<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Asciidoctor 2.0.12">
<title>mzIdentML extension for crosslinking data</title>
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400,700">
<style>
/* Asciidoctor default stylesheet | MIT License | https://asciidoctor.org */
/* Uncomment @import statement to use as custom stylesheet */
/*@import "https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400,700";*/
article,aside,details,figcaption,figure,footer,header,hgroup,main,nav,section{display:block}
audio,video{display:inline-block}
audio:not([controls]){display:none;height:0}
html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}
a{background:none}
a:focus{outline:thin dotted}
a:active,a:hover{outline:0}
h1{font-size:2em;margin:.67em 0}
abbr[title]{border-bottom:1px dotted}
b,strong{font-weight:bold}
dfn{font-style:italic}
hr{-moz-box-sizing:content-box;box-sizing:content-box;height:0}
mark{background:#ff0;color:#000}
code,kbd,pre,samp{font-family:monospace;font-size:1em}
pre{white-space:pre-wrap}
q{quotes:"\201C" "\201D" "\2018" "\2019"}
small{font-size:80%}
sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}
sup{top:-.5em}
sub{bottom:-.25em}
img{border:0}
svg:not(:root){overflow:hidden}
figure{margin:0}
fieldset{border:1px solid silver;margin:0 2px;padding:.35em .625em .75em}
legend{border:0;padding:0}
button,input,select,textarea{font-family:inherit;font-size:100%;margin:0}
button,input{line-height:normal}
button,select{text-transform:none}
button,html input[type="button"],input[type="reset"],input[type="submit"]{-webkit-appearance:button;cursor:pointer}
button[disabled],html input[disabled]{cursor:default}
input[type="checkbox"],input[type="radio"]{box-sizing:border-box;padding:0}
button::-moz-focus-inner,input::-moz-focus-inner{border:0;padding:0}
textarea{overflow:auto;vertical-align:top}
table{border-collapse:collapse;border-spacing:0}
*,*::before,*::after{-moz-box-sizing:border-box;-webkit-box-sizing:border-box;box-sizing:border-box}
html,body{font-size:100%}
body{background:#fff;color:rgba(0,0,0,.8);padding:0;margin:0;font-family:"Noto Serif","DejaVu Serif",serif;font-weight:400;font-style:normal;line-height:1;position:relative;cursor:auto;tab-size:4;word-wrap:anywhere;-moz-osx-font-smoothing:grayscale;-webkit-font-smoothing:antialiased}
a:hover{cursor:pointer}
img,object,embed{max-width:100%;height:auto}
object,embed{height:100%}
img{-ms-interpolation-mode:bicubic}
.left{float:left!important}
.right{float:right!important}
.text-left{text-align:left!important}
.text-right{text-align:right!important}
.text-center{text-align:center!important}
.text-justify{text-align:justify!important}
.hide{display:none}
img,object,svg{display:inline-block;vertical-align:middle}
textarea{height:auto;min-height:50px}
select{width:100%}
.subheader,.admonitionblock td.content>.title,.audioblock>.title,.exampleblock>.title,.imageblock>.title,.listingblock>.title,.literalblock>.title,.stemblock>.title,.openblock>.title,.paragraph>.title,.quoteblock>.title,table.tableblock>.title,.verseblock>.title,.videoblock>.title,.dlist>.title,.olist>.title,.ulist>.title,.qlist>.title,.hdlist>.title{line-height:1.45;color:#7a2518;font-weight:400;margin-top:0;margin-bottom:.25em}
div,dl,dt,dd,ul,ol,li,h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6,pre,form,p,blockquote,th,td{margin:0;padding:0}
a{color:#2156a5;text-decoration:underline;line-height:inherit}
a:hover,a:focus{color:#1d4b8f}
a img{border:0}
p{font-family:inherit;font-weight:400;font-size:1em;line-height:1.6;margin-bottom:1.25em;text-rendering:optimizeLegibility}
p aside{font-size:.875em;line-height:1.35;font-style:italic}
h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6{font-family:"Open Sans","DejaVu Sans",sans-serif;font-weight:300;font-style:normal;color:#ba3925;text-rendering:optimizeLegibility;margin-top:1em;margin-bottom:.5em;line-height:1.0125em}
h1 small,h2 small,h3 small,#toctitle small,.sidebarblock>.content>.title small,h4 small,h5 small,h6 small{font-size:60%;color:#e99b8f;line-height:0}
h1{font-size:2.125em}
h2{font-size:1.6875em}
h3,#toctitle,.sidebarblock>.content>.title{font-size:1.375em}
h4,h5{font-size:1.125em}
h6{font-size:1em}
hr{border:solid #dddddf;border-width:1px 0 0;clear:both;margin:1.25em 0 1.1875em;height:0}
em,i{font-style:italic;line-height:inherit}
strong,b{font-weight:bold;line-height:inherit}
small{font-size:60%;line-height:inherit}
code{font-family:"Droid Sans Mono","DejaVu Sans Mono",monospace;font-weight:400;color:rgba(0,0,0,.9)}
ul,ol,dl{font-size:1em;line-height:1.6;margin-bottom:1.25em;list-style-position:outside;font-family:inherit}
ul,ol{margin-left:1.5em}
ul li ul,ul li ol{margin-left:1.25em;margin-bottom:0;font-size:1em}
ul.square li ul,ul.circle li ul,ul.disc li ul{list-style:inherit}
ul.square{list-style-type:square}
ul.circle{list-style-type:circle}
ul.disc{list-style-type:disc}
ol li ul,ol li ol{margin-left:1.25em;margin-bottom:0}
dl dt{margin-bottom:.3125em;font-weight:bold}
dl dd{margin-bottom:1.25em}
abbr,acronym{text-transform:uppercase;font-size:90%;color:rgba(0,0,0,.8);border-bottom:1px dotted #ddd;cursor:help}
abbr{text-transform:none}
blockquote{margin:0 0 1.25em;padding:.5625em 1.25em 0 1.1875em;border-left:1px solid #ddd}
blockquote cite{display:block;font-size:.9375em;color:rgba(0,0,0,.6)}
blockquote cite::before{content:"\2014 \0020"}
blockquote cite a,blockquote cite a:visited{color:rgba(0,0,0,.6)}
blockquote,blockquote p{line-height:1.6;color:rgba(0,0,0,.85)}
@media screen and (min-width:768px){h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6{line-height:1.2}
h1{font-size:2.75em}
h2{font-size:2.3125em}
h3,#toctitle,.sidebarblock>.content>.title{font-size:1.6875em}
h4{font-size:1.4375em}}
table{background:#fff;margin-bottom:1.25em;border:solid 1px #dedede;word-wrap:normal}
table thead,table tfoot{background:#f7f8f7}
table thead tr th,table thead tr td,table tfoot tr th,table tfoot tr td{padding:.5em .625em .625em;font-size:inherit;color:rgba(0,0,0,.8);text-align:left}
table tr th,table tr td{padding:.5625em .625em;font-size:inherit;color:rgba(0,0,0,.8)}
table tr.even,table tr.alt{background:#f8f8f7}
table thead tr th,table tfoot tr th,table tbody tr td,table tr td,table tfoot tr td{line-height:1.6}
h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6{line-height:1.2;word-spacing:-.05em}
h1 strong,h2 strong,h3 strong,#toctitle strong,.sidebarblock>.content>.title strong,h4 strong,h5 strong,h6 strong{font-weight:400}
.center{margin-left:auto;margin-right:auto}
.stretch{width:100%}
.clearfix::before,.clearfix::after,.float-group::before,.float-group::after{content:" ";display:table}
.clearfix::after,.float-group::after{clear:both}
:not(pre).nobreak{word-wrap:normal}
:not(pre).nowrap{white-space:nowrap}
:not(pre).pre-wrap{white-space:pre-wrap}
:not(pre):not([class^=L])>code{font-size:.9375em;font-style:normal!important;letter-spacing:0;padding:.1em .5ex;word-spacing:-.15em;background:#f7f7f8;-webkit-border-radius:4px;border-radius:4px;line-height:1.45;text-rendering:optimizeSpeed}
pre{color:rgba(0,0,0,.9);font-family:"Droid Sans Mono","DejaVu Sans Mono",monospace;line-height:1.45;text-rendering:optimizeSpeed}
pre code,pre pre{color:inherit;font-size:inherit;line-height:inherit}
pre>code{display:block}
pre.nowrap,pre.nowrap pre{white-space:pre;word-wrap:normal}
em em{font-style:normal}
strong strong{font-weight:400}
.keyseq{color:rgba(51,51,51,.8)}
kbd{font-family:"Droid Sans Mono","DejaVu Sans Mono",monospace;display:inline-block;color:rgba(0,0,0,.8);font-size:.65em;line-height:1.45;background:#f7f7f7;border:1px solid #ccc;-webkit-border-radius:3px;border-radius:3px;-webkit-box-shadow:0 1px 0 rgba(0,0,0,.2),0 0 0 .1em white inset;box-shadow:0 1px 0 rgba(0,0,0,.2),0 0 0 .1em #fff inset;margin:0 .15em;padding:.2em .5em;vertical-align:middle;position:relative;top:-.1em;white-space:nowrap}
.keyseq kbd:first-child{margin-left:0}
.keyseq kbd:last-child{margin-right:0}
.menuseq,.menuref{color:#000}
.menuseq b:not(.caret),.menuref{font-weight:inherit}
.menuseq{word-spacing:-.02em}
.menuseq b.caret{font-size:1.25em;line-height:.8}
.menuseq i.caret{font-weight:bold;text-align:center;width:.45em}
b.button::before,b.button::after{position:relative;top:-1px;font-weight:400}
b.button::before{content:"[";padding:0 3px 0 2px}
b.button::after{content:"]";padding:0 2px 0 3px}
p a>code:hover{color:rgba(0,0,0,.9)}
#header,#content,#footnotes,#footer{width:100%;margin-left:auto;margin-right:auto;margin-top:0;margin-bottom:0;max-width:62.5em;*zoom:1;position:relative;padding-left:.9375em;padding-right:.9375em}
#header::before,#header::after,#content::before,#content::after,#footnotes::before,#footnotes::after,#footer::before,#footer::after{content:" ";display:table}
#header::after,#content::after,#footnotes::after,#footer::after{clear:both}
#content{margin-top:1.25em}
#content::before{content:none}
#header>h1:first-child{color:rgba(0,0,0,.85);margin-top:2.25rem;margin-bottom:0}
#header>h1:first-child+#toc{margin-top:8px;border-top:1px solid #dddddf}
#header>h1:only-child,body.toc2 #header>h1:nth-last-child(2){border-bottom:1px solid #dddddf;padding-bottom:8px}
#header .details{border-bottom:1px solid #dddddf;line-height:1.45;padding-top:.25em;padding-bottom:.25em;padding-left:.25em;color:rgba(0,0,0,.6);display:-ms-flexbox;display:-webkit-flex;display:flex;-ms-flex-flow:row wrap;-webkit-flex-flow:row wrap;flex-flow:row wrap}
#header .details span:first-child{margin-left:-.125em}
#header .details span.email a{color:rgba(0,0,0,.85)}
#header .details br{display:none}
#header .details br+span::before{content:"\00a0\2013\00a0"}
#header .details br+span.author::before{content:"\00a0\22c5\00a0";color:rgba(0,0,0,.85)}
#header .details br+span#revremark::before{content:"\00a0|\00a0"}
#header #revnumber{text-transform:capitalize}
#header #revnumber::after{content:"\00a0"}
#content>h1:first-child:not([class]){color:rgba(0,0,0,.85);border-bottom:1px solid #dddddf;padding-bottom:8px;margin-top:0;padding-top:1rem;margin-bottom:1.25rem}
#toc{border-bottom:1px solid #e7e7e9;padding-bottom:.5em}
#toc>ul{margin-left:.125em}
#toc ul.sectlevel0>li>a{font-style:italic}
#toc ul.sectlevel0 ul.sectlevel1{margin:.5em 0}
#toc ul{font-family:"Open Sans","DejaVu Sans",sans-serif;list-style-type:none}
#toc li{line-height:1.3334;margin-top:.3334em}
#toc a{text-decoration:none}
#toc a:active{text-decoration:underline}
#toctitle{color:#7a2518;font-size:1.2em}
@media screen and (min-width:768px){#toctitle{font-size:1.375em}
body.toc2{padding-left:15em;padding-right:0}
#toc.toc2{margin-top:0!important;background:#f8f8f7;position:fixed;width:15em;left:0;top:0;border-right:1px solid #e7e7e9;border-top-width:0!important;border-bottom-width:0!important;z-index:1000;padding:1.25em 1em;height:100%;overflow:auto}
#toc.toc2 #toctitle{margin-top:0;margin-bottom:.8rem;font-size:1.2em}
#toc.toc2>ul{font-size:.9em;margin-bottom:0}
#toc.toc2 ul ul{margin-left:0;padding-left:1em}
#toc.toc2 ul.sectlevel0 ul.sectlevel1{padding-left:0;margin-top:.5em;margin-bottom:.5em}
body.toc2.toc-right{padding-left:0;padding-right:15em}
body.toc2.toc-right #toc.toc2{border-right-width:0;border-left:1px solid #e7e7e9;left:auto;right:0}}
@media screen and (min-width:1280px){body.toc2{padding-left:20em;padding-right:0}
#toc.toc2{width:20em}
#toc.toc2 #toctitle{font-size:1.375em}
#toc.toc2>ul{font-size:.95em}
#toc.toc2 ul ul{padding-left:1.25em}
body.toc2.toc-right{padding-left:0;padding-right:20em}}
#content #toc{border-style:solid;border-width:1px;border-color:#e0e0dc;margin-bottom:1.25em;padding:1.25em;background:#f8f8f7;-webkit-border-radius:4px;border-radius:4px}
#content #toc>:first-child{margin-top:0}
#content #toc>:last-child{margin-bottom:0}
#footer{max-width:none;background:rgba(0,0,0,.8);padding:1.25em}
#footer-text{color:rgba(255,255,255,.8);line-height:1.44}
#content{margin-bottom:.625em}
.sect1{padding-bottom:.625em}
@media screen and (min-width:768px){#content{margin-bottom:1.25em}
.sect1{padding-bottom:1.25em}}
.sect1:last-child{padding-bottom:0}
.sect1+.sect1{border-top:1px solid #e7e7e9}
#content h1>a.anchor,h2>a.anchor,h3>a.anchor,#toctitle>a.anchor,.sidebarblock>.content>.title>a.anchor,h4>a.anchor,h5>a.anchor,h6>a.anchor{position:absolute;z-index:1001;width:1.5ex;margin-left:-1.5ex;display:block;text-decoration:none!important;visibility:hidden;text-align:center;font-weight:400}
#content h1>a.anchor::before,h2>a.anchor::before,h3>a.anchor::before,#toctitle>a.anchor::before,.sidebarblock>.content>.title>a.anchor::before,h4>a.anchor::before,h5>a.anchor::before,h6>a.anchor::before{content:"\00A7";font-size:.85em;display:block;padding-top:.1em}
#content h1:hover>a.anchor,#content h1>a.anchor:hover,h2:hover>a.anchor,h2>a.anchor:hover,h3:hover>a.anchor,#toctitle:hover>a.anchor,.sidebarblock>.content>.title:hover>a.anchor,h3>a.anchor:hover,#toctitle>a.anchor:hover,.sidebarblock>.content>.title>a.anchor:hover,h4:hover>a.anchor,h4>a.anchor:hover,h5:hover>a.anchor,h5>a.anchor:hover,h6:hover>a.anchor,h6>a.anchor:hover{visibility:visible}
#content h1>a.link,h2>a.link,h3>a.link,#toctitle>a.link,.sidebarblock>.content>.title>a.link,h4>a.link,h5>a.link,h6>a.link{color:#ba3925;text-decoration:none}
#content h1>a.link:hover,h2>a.link:hover,h3>a.link:hover,#toctitle>a.link:hover,.sidebarblock>.content>.title>a.link:hover,h4>a.link:hover,h5>a.link:hover,h6>a.link:hover{color:#a53221}
details,.audioblock,.imageblock,.literalblock,.listingblock,.stemblock,.videoblock{margin-bottom:1.25em}
details>summary:first-of-type{cursor:pointer;display:list-item;outline:none;margin-bottom:.75em}
.admonitionblock td.content>.title,.audioblock>.title,.exampleblock>.title,.imageblock>.title,.listingblock>.title,.literalblock>.title,.stemblock>.title,.openblock>.title,.paragraph>.title,.quoteblock>.title,table.tableblock>.title,.verseblock>.title,.videoblock>.title,.dlist>.title,.olist>.title,.ulist>.title,.qlist>.title,.hdlist>.title{text-rendering:optimizeLegibility;text-align:left;font-family:"Noto Serif","DejaVu Serif",serif;font-size:1rem;font-style:italic}
table.tableblock.fit-content>caption.title{white-space:nowrap;width:0}
.paragraph.lead>p,#preamble>.sectionbody>[class="paragraph"]:first-of-type p{font-size:1.21875em;line-height:1.6;color:rgba(0,0,0,.85)}
table.tableblock #preamble>.sectionbody>[class="paragraph"]:first-of-type p{font-size:inherit}
.admonitionblock>table{border-collapse:separate;border:0;background:none;width:100%}
.admonitionblock>table td.icon{text-align:center;width:80px}
.admonitionblock>table td.icon img{max-width:none}
.admonitionblock>table td.icon .title{font-weight:bold;font-family:"Open Sans","DejaVu Sans",sans-serif;text-transform:uppercase}
.admonitionblock>table td.content{padding-left:1.125em;padding-right:1.25em;border-left:1px solid #dddddf;color:rgba(0,0,0,.6);word-wrap:anywhere}
.admonitionblock>table td.content>:last-child>:last-child{margin-bottom:0}
.exampleblock>.content{border-style:solid;border-width:1px;border-color:#e6e6e6;margin-bottom:1.25em;padding:1.25em;background:#fff;-webkit-border-radius:4px;border-radius:4px}
.exampleblock>.content>:first-child{margin-top:0}
.exampleblock>.content>:last-child{margin-bottom:0}
.sidebarblock{border-style:solid;border-width:1px;border-color:#dbdbd6;margin-bottom:1.25em;padding:1.25em;background:#f3f3f2;-webkit-border-radius:4px;border-radius:4px}
.sidebarblock>:first-child{margin-top:0}
.sidebarblock>:last-child{margin-bottom:0}
.sidebarblock>.content>.title{color:#7a2518;margin-top:0;text-align:center}
.exampleblock>.content>:last-child>:last-child,.exampleblock>.content .olist>ol>li:last-child>:last-child,.exampleblock>.content .ulist>ul>li:last-child>:last-child,.exampleblock>.content .qlist>ol>li:last-child>:last-child,.sidebarblock>.content>:last-child>:last-child,.sidebarblock>.content .olist>ol>li:last-child>:last-child,.sidebarblock>.content .ulist>ul>li:last-child>:last-child,.sidebarblock>.content .qlist>ol>li:last-child>:last-child{margin-bottom:0}
.literalblock pre,.listingblock>.content>pre{-webkit-border-radius:4px;border-radius:4px;overflow-x:auto;padding:1em;font-size:.8125em}
@media screen and (min-width:768px){.literalblock pre,.listingblock>.content>pre{font-size:.90625em}}
@media screen and (min-width:1280px){.literalblock pre,.listingblock>.content>pre{font-size:1em}}
.literalblock pre,.listingblock>.content>pre:not(.highlight),.listingblock>.content>pre[class="highlight"],.listingblock>.content>pre[class^="highlight "]{background:#f7f7f8}
.literalblock.output pre{color:#f7f7f8;background:rgba(0,0,0,.9)}
.listingblock>.content{position:relative}
.listingblock code[data-lang]::before{display:none;content:attr(data-lang);position:absolute;font-size:.75em;top:.425rem;right:.5rem;line-height:1;text-transform:uppercase;color:inherit;opacity:.5}
.listingblock:hover code[data-lang]::before{display:block}
.listingblock.terminal pre .command::before{content:attr(data-prompt);padding-right:.5em;color:inherit;opacity:.5}
.listingblock.terminal pre .command:not([data-prompt])::before{content:"$"}
.listingblock pre.highlightjs{padding:0}
.listingblock pre.highlightjs>code{padding:1em;-webkit-border-radius:4px;border-radius:4px}
.listingblock pre.prettyprint{border-width:0}
.prettyprint{background:#f7f7f8}
pre.prettyprint .linenums{line-height:1.45;margin-left:2em}
pre.prettyprint li{background:none;list-style-type:inherit;padding-left:0}
pre.prettyprint li code[data-lang]::before{opacity:1}
pre.prettyprint li:not(:first-child) code[data-lang]::before{display:none}
table.linenotable{border-collapse:separate;border:0;margin-bottom:0;background:none}
table.linenotable td[class]{color:inherit;vertical-align:top;padding:0;line-height:inherit;white-space:normal}
table.linenotable td.code{padding-left:.75em}
table.linenotable td.linenos{border-right:1px solid currentColor;opacity:.35;padding-right:.5em}
pre.pygments .lineno{border-right:1px solid currentColor;opacity:.35;display:inline-block;margin-right:.75em}
pre.pygments .lineno::before{content:"";margin-right:-.125em}
.quoteblock{margin:0 1em 1.25em 1.5em;display:table}
.quoteblock:not(.excerpt)>.title{margin-left:-1.5em;margin-bottom:.75em}
.quoteblock blockquote,.quoteblock p{color:rgba(0,0,0,.85);font-size:1.15rem;line-height:1.75;word-spacing:.1em;letter-spacing:0;font-style:italic;text-align:justify}
.quoteblock blockquote{margin:0;padding:0;border:0}
.quoteblock blockquote::before{content:"\201c";float:left;font-size:2.75em;font-weight:bold;line-height:.6em;margin-left:-.6em;color:#7a2518;text-shadow:0 1px 2px rgba(0,0,0,.1)}
.quoteblock blockquote>.paragraph:last-child p{margin-bottom:0}
.quoteblock .attribution{margin-top:.75em;margin-right:.5ex;text-align:right}
.verseblock{margin:0 1em 1.25em}
.verseblock pre{font-family:"Open Sans","DejaVu Sans",sans;font-size:1.15rem;color:rgba(0,0,0,.85);font-weight:300;text-rendering:optimizeLegibility}
.verseblock pre strong{font-weight:400}
.verseblock .attribution{margin-top:1.25rem;margin-left:.5ex}
.quoteblock .attribution,.verseblock .attribution{font-size:.9375em;line-height:1.45;font-style:italic}
.quoteblock .attribution br,.verseblock .attribution br{display:none}
.quoteblock .attribution cite,.verseblock .attribution cite{display:block;letter-spacing:-.025em;color:rgba(0,0,0,.6)}
.quoteblock.abstract blockquote::before,.quoteblock.excerpt blockquote::before,.quoteblock .quoteblock blockquote::before{display:none}
.quoteblock.abstract blockquote,.quoteblock.abstract p,.quoteblock.excerpt blockquote,.quoteblock.excerpt p,.quoteblock .quoteblock blockquote,.quoteblock .quoteblock p{line-height:1.6;word-spacing:0}
.quoteblock.abstract{margin:0 1em 1.25em;display:block}
.quoteblock.abstract>.title{margin:0 0 .375em;font-size:1.15em;text-align:center}
.quoteblock.excerpt>blockquote,.quoteblock .quoteblock{padding:0 0 .25em 1em;border-left:.25em solid #dddddf}
.quoteblock.excerpt,.quoteblock .quoteblock{margin-left:0}
.quoteblock.excerpt blockquote,.quoteblock.excerpt p,.quoteblock .quoteblock blockquote,.quoteblock .quoteblock p{color:inherit;font-size:1.0625rem}
.quoteblock.excerpt .attribution,.quoteblock .quoteblock .attribution{color:inherit;text-align:left;margin-right:0}
p.tableblock:last-child{margin-bottom:0}
td.tableblock>.content{margin-bottom:1.25em;word-wrap:anywhere}
td.tableblock>.content>:last-child{margin-bottom:-1.25em}
table.tableblock,th.tableblock,td.tableblock{border:0 solid #dedede}
table.grid-all>*>tr>*{border-width:1px}
table.grid-cols>*>tr>*{border-width:0 1px}
table.grid-rows>*>tr>*{border-width:1px 0}
table.frame-all{border-width:1px}
table.frame-ends{border-width:1px 0}
table.frame-sides{border-width:0 1px}
table.frame-none>colgroup+*>:first-child>*,table.frame-sides>colgroup+*>:first-child>*{border-top-width:0}
table.frame-none>:last-child>:last-child>*,table.frame-sides>:last-child>:last-child>*{border-bottom-width:0}
table.frame-none>*>tr>:first-child,table.frame-ends>*>tr>:first-child{border-left-width:0}
table.frame-none>*>tr>:last-child,table.frame-ends>*>tr>:last-child{border-right-width:0}
table.stripes-all tr,table.stripes-odd tr:nth-of-type(odd),table.stripes-even tr:nth-of-type(even),table.stripes-hover tr:hover{background:#f8f8f7}
th.halign-left,td.halign-left{text-align:left}
th.halign-right,td.halign-right{text-align:right}
th.halign-center,td.halign-center{text-align:center}
th.valign-top,td.valign-top{vertical-align:top}
th.valign-bottom,td.valign-bottom{vertical-align:bottom}
th.valign-middle,td.valign-middle{vertical-align:middle}
table thead th,table tfoot th{font-weight:bold}
tbody tr th{background:#f7f8f7}
tbody tr th,tbody tr th p,tfoot tr th,tfoot tr th p{color:rgba(0,0,0,.8);font-weight:bold}
p.tableblock>code:only-child{background:none;padding:0}
p.tableblock{font-size:1em}
ol{margin-left:1.75em}
ul li ol{margin-left:1.5em}
dl dd{margin-left:1.125em}
dl dd:last-child,dl dd:last-child>:last-child{margin-bottom:0}
ol>li p,ul>li p,ul dd,ol dd,.olist .olist,.ulist .ulist,.ulist .olist,.olist .ulist{margin-bottom:.625em}
ul.checklist,ul.none,ol.none,ul.no-bullet,ol.no-bullet,ol.unnumbered,ul.unstyled,ol.unstyled{list-style-type:none}
ul.no-bullet,ol.no-bullet,ol.unnumbered{margin-left:.625em}
ul.unstyled,ol.unstyled{margin-left:0}
ul.checklist{margin-left:.625em}
ul.checklist li>p:first-child>.fa-square-o:first-child,ul.checklist li>p:first-child>.fa-check-square-o:first-child{width:1.25em;font-size:.8em;position:relative;bottom:.125em}
ul.checklist li>p:first-child>input[type="checkbox"]:first-child{margin-right:.25em}
ul.inline{display:-ms-flexbox;display:-webkit-box;display:flex;-ms-flex-flow:row wrap;-webkit-flex-flow:row wrap;flex-flow:row wrap;list-style:none;margin:0 0 .625em -1.25em}
ul.inline>li{margin-left:1.25em}
.unstyled dl dt{font-weight:400;font-style:normal}
ol.arabic{list-style-type:decimal}
ol.decimal{list-style-type:decimal-leading-zero}
ol.loweralpha{list-style-type:lower-alpha}
ol.upperalpha{list-style-type:upper-alpha}
ol.lowerroman{list-style-type:lower-roman}
ol.upperroman{list-style-type:upper-roman}
ol.lowergreek{list-style-type:lower-greek}
.hdlist>table,.colist>table{border:0;background:none}
.hdlist>table>tbody>tr,.colist>table>tbody>tr{background:none}
td.hdlist1,td.hdlist2{vertical-align:top;padding:0 .625em}
td.hdlist1{font-weight:bold;padding-bottom:1.25em}
td.hdlist2{word-wrap:anywhere}
.literalblock+.colist,.listingblock+.colist{margin-top:-.5em}
.colist td:not([class]):first-child{padding:.4em .75em 0;line-height:1;vertical-align:top}
.colist td:not([class]):first-child img{max-width:none}
.colist td:not([class]):last-child{padding:.25em 0}
.thumb,.th{line-height:0;display:inline-block;border:solid 4px #fff;-webkit-box-shadow:0 0 0 1px #ddd;box-shadow:0 0 0 1px #ddd}
.imageblock.left{margin:.25em .625em 1.25em 0}
.imageblock.right{margin:.25em 0 1.25em .625em}
.imageblock>.title{margin-bottom:0}
.imageblock.thumb,.imageblock.th{border-width:6px}
.imageblock.thumb>.title,.imageblock.th>.title{padding:0 .125em}
.image.left,.image.right{margin-top:.25em;margin-bottom:.25em;display:inline-block;line-height:0}
.image.left{margin-right:.625em}
.image.right{margin-left:.625em}
a.image{text-decoration:none;display:inline-block}
a.image object{pointer-events:none}
sup.footnote,sup.footnoteref{font-size:.875em;position:static;vertical-align:super}
sup.footnote a,sup.footnoteref a{text-decoration:none}
sup.footnote a:active,sup.footnoteref a:active{text-decoration:underline}
#footnotes{padding-top:.75em;padding-bottom:.75em;margin-bottom:.625em}
#footnotes hr{width:20%;min-width:6.25em;margin:-.25em 0 .75em;border-width:1px 0 0}
#footnotes .footnote{padding:0 .375em 0 .225em;line-height:1.3334;font-size:.875em;margin-left:1.2em;margin-bottom:.2em}
#footnotes .footnote a:first-of-type{font-weight:bold;text-decoration:none;margin-left:-1.05em}
#footnotes .footnote:last-of-type{margin-bottom:0}
#content #footnotes{margin-top:-.625em;margin-bottom:0;padding:.75em 0}
.gist .file-data>table{border:0;background:#fff;width:100%;margin-bottom:0}
.gist .file-data>table td.line-data{width:99%}
div.unbreakable{page-break-inside:avoid}
.big{font-size:larger}
.small{font-size:smaller}
.underline{text-decoration:underline}
.overline{text-decoration:overline}
.line-through{text-decoration:line-through}
.aqua{color:#00bfbf}
.aqua-background{background:#00fafa}
.black{color:#000}
.black-background{background:#000}
.blue{color:#0000bf}
.blue-background{background:#0000fa}
.fuchsia{color:#bf00bf}
.fuchsia-background{background:#fa00fa}
.gray{color:#606060}
.gray-background{background:#7d7d7d}
.green{color:#006000}
.green-background{background:#007d00}
.lime{color:#00bf00}
.lime-background{background:#00fa00}
.maroon{color:#600000}
.maroon-background{background:#7d0000}
.navy{color:#000060}
.navy-background{background:#00007d}
.olive{color:#606000}
.olive-background{background:#7d7d00}
.purple{color:#600060}
.purple-background{background:#7d007d}
.red{color:#bf0000}
.red-background{background:#fa0000}
.silver{color:#909090}
.silver-background{background:#bcbcbc}
.teal{color:#006060}
.teal-background{background:#007d7d}
.white{color:#bfbfbf}
.white-background{background:#fafafa}
.yellow{color:#bfbf00}
.yellow-background{background:#fafa00}
span.icon>.fa{cursor:default}
a span.icon>.fa{cursor:inherit}
.admonitionblock td.icon [class^="fa icon-"]{font-size:2.5em;text-shadow:1px 1px 2px rgba(0,0,0,.5);cursor:default}
.admonitionblock td.icon .icon-note::before{content:"\f05a";color:#19407c}
.admonitionblock td.icon .icon-tip::before{content:"\f0eb";text-shadow:1px 1px 2px rgba(155,155,0,.8);color:#111}
.admonitionblock td.icon .icon-warning::before{content:"\f071";color:#bf6900}
.admonitionblock td.icon .icon-caution::before{content:"\f06d";color:#bf3400}
.admonitionblock td.icon .icon-important::before{content:"\f06a";color:#bf0000}
.conum[data-value]{display:inline-block;color:#fff!important;background:rgba(0,0,0,.8);-webkit-border-radius:50%;border-radius:50%;text-align:center;font-size:.75em;width:1.67em;height:1.67em;line-height:1.67em;font-family:"Open Sans","DejaVu Sans",sans-serif;font-style:normal;font-weight:bold}
.conum[data-value] *{color:#fff!important}
.conum[data-value]+b{display:none}
.conum[data-value]::after{content:attr(data-value)}
pre .conum[data-value]{position:relative;top:-.125em}
b.conum *{color:inherit!important}
.conum:not([data-value]):empty{display:none}
dt,th.tableblock,td.content,div.footnote{text-rendering:optimizeLegibility}
h1,h2,p,td.content,span.alt{letter-spacing:-.01em}
p strong,td.content strong,div.footnote strong{letter-spacing:-.005em}
p,blockquote,dt,td.content,span.alt{font-size:1.0625rem}
p{margin-bottom:1.25rem}
.sidebarblock p,.sidebarblock dt,.sidebarblock td.content,p.tableblock{font-size:1em}
.exampleblock>.content{background:#fffef7;border-color:#e0e0dc;-webkit-box-shadow:0 1px 4px #e0e0dc;box-shadow:0 1px 4px #e0e0dc}
.print-only{display:none!important}
@page{margin:1.25cm .75cm}
@media print{*{-webkit-box-shadow:none!important;box-shadow:none!important;text-shadow:none!important}
html{font-size:80%}
a{color:inherit!important;text-decoration:underline!important}
a.bare,a[href^="#"],a[href^="mailto:"]{text-decoration:none!important}
a[href^="http:"]:not(.bare)::after,a[href^="https:"]:not(.bare)::after{content:"(" attr(href) ")";display:inline-block;font-size:.875em;padding-left:.25em}
abbr[title]::after{content:" (" attr(title) ")"}
pre,blockquote,tr,img,object,svg{page-break-inside:avoid}
thead{display:table-header-group}
svg{max-width:100%}
p,blockquote,dt,td.content{font-size:1em;orphans:3;widows:3}
h2,h3,#toctitle,.sidebarblock>.content>.title{page-break-after:avoid}
#header,#content,#footnotes,#footer{max-width:none}
#toc,.sidebarblock,.exampleblock>.content{background:none!important}
#toc{border-bottom:1px solid #dddddf!important;padding-bottom:0!important}
body.book #header{text-align:center}
body.book #header>h1:first-child{border:0!important;margin:2.5em 0 1em}
body.book #header .details{border:0!important;display:block;padding:0!important}
body.book #header .details span:first-child{margin-left:0!important}
body.book #header .details br{display:block}
body.book #header .details br+span::before{content:none!important}
body.book #toc{border:0!important;text-align:left!important;padding:0!important;margin:0!important}
body.book #toc,body.book #preamble,body.book h1.sect0,body.book .sect1>h2{page-break-before:always}
.listingblock code[data-lang]::before{display:block}
#footer{padding:0 .9375em}
.hide-on-print{display:none!important}
.print-only{display:block!important}
.hide-for-print{display:none!important}
.show-for-print{display:inherit!important}}
@media print,amzn-kf8{#header>h1:first-child{margin-top:1.25rem}
.sect1{padding:0!important}
.sect1+.sect1{border:0}
#footer{background:none}
#footer-text{color:rgba(0,0,0,.6);font-size:.9em}}
@media amzn-kf8{#header,#content,#footnotes,#footer{padding:0}}
</style>
</head>
<body class="book toc2 toc-left">
<div id="header">
<h1>mzIdentML extension for crosslinking data</h1>
<div id="toc" class="toc2">
<div id="toctitle">Table of Contents</div>
<ul class="sectlevel1">
<li><a href="#_preface">Preface</a></li>
<li><a href="#abstract">Abstract</a></li>
<li><a href="#introduction">1. Introduction</a>
<ul class="sectlevel2">
<li><a href="#background">1.1. Background</a></li>
<li><a href="#supported-use-cases">1.2. Supported Crosslinking Use Cases</a></li>
<li><a href="#document-structure">1.3. Document Structure and Changes from mzIdentML Version 1.2.0 to 1.3.0 to Support Crosslinking Results</a></li>
<li><a href="#_availability_of_documentation_and_example_files">1.4. Availability of Documentation and Example Files</a></li>
</ul>
</li>
<li><a href="#_controlled_vocabularies_for_encoding_crosslinks">2. Controlled Vocabularies for Encoding Crosslinks</a></li>
<li><a href="#_encoding_the_results_of_crosslinking_searches">3. Encoding the Results of Crosslinking Searches</a>
<ul class="sectlevel2">
<li><a href="#_introduction">3.1. Introduction</a></li>
<li><a href="#_spectrumidentificationprotocol_elements">3.2. &lt;SpectrumIdentificationProtocol&gt; Elements</a></li>
<li><a href="#_encoding_crosslinked_peptides_in_the_element_sequencecollection">3.3. Encoding Crosslinked Peptides in the Element &lt;SequenceCollection&gt;</a></li>
<li><a href="#_encoding_identified_crosslinks_in_spectrumidentificationitem_elements">3.4. Encoding Identified Crosslinks in &lt;SpectrumIdentificationItem&gt; Elements</a></li>
</ul>
</li>
<li><a href="#_scores_and_thresholds">4. Scores and Thresholds</a>
<ul class="sectlevel2">
<li><a href="#_introduction_2">4.1. Introduction</a></li>
<li><a href="#_thresholds">4.2. Thresholds</a></li>
<li><a href="#_scores">4.3. Scores</a></li>
<li><a href="#_fdr_specific_comments">4.4. FDR Specific Comments</a></li>
</ul>
</li>
<li><a href="#_unsupported_use_cases_and_future_directions">5. Unsupported Use Cases and Future Directions</a></li>
<li><a href="#_appendix_i_comparison_of_rules_for_crosslink_donor_and_crosslink_acceptor_depending_on_context">6. Appendix I. Comparison of rules for “crosslink donor” and “crosslink acceptor” depending on context</a></li>
<li><a href="#_appendix_ii_example_encodings_of_crosslinker_reagents_as_searchmodification_elements">7. Appendix II. Example encodings of crosslinker reagents as &lt;SearchModification&gt; elements</a>
<ul class="sectlevel2">
<li><a href="#_bissulfosuccinimidyl_suberate_bs3">7.1. Bis(sulfosuccinimidyl) suberate (BS3)</a></li>
<li><a href="#_1_ethyl_3_3_dimethylaminopropylcarbodiimide_hydrochloride_edc">7.2. 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride (EDC)</a></li>
<li><a href="#_nhs_diazirine_succinimidyl_44_azipentanoate_sda">7.3. (NHS-Diazirine) succinimidyl 4,4'-azipentanoate (SDA)</a></li>
</ul>
</li>
<li><a href="#authors">8. Authors Information</a></li>
<li><a href="#_glossary">9. Glossary</a></li>
<li><a href="#_references">10. References</a></li>
<li><a href="#_intellectual_property_statement">11. Intellectual Property Statement</a></li>
<li><a href="#_copyright_notice">12. Copyright Notice</a></li>
</ul>
</div>
</div>
<div id="content">
<div class="sect1">
<h2 id="_preface">Preface</h2>
<div class="sectionbody">
<div class="paragraph">
<p><strong>mzIdentML: exchange format for peptides and proteins identified from mass spectra</strong></p>
</div>
<div class="paragraph">
<p><strong>Extension for crosslinking approaches</strong></p>
</div>
<div class="paragraph">
<p>(This extension is an <em>addendum</em> to mzIdentML version 1.3.0)</p>
</div>
<div class="paragraph">
<p><em>Status of This Document</em></p>
</div>
<div class="paragraph">
<p>This document presents a final specification for the mzIdentML data format
developed by the HUPO Proteomics Standards Initiative. Distribution is
unlimited.</p>
</div>
<div class="paragraph">
<p><em>Version of This Document</em></p>
</div>
<div class="paragraph">
<p>Date created: June 24, 2024</p>
</div>
<div class="paragraph">
<p>Last updated: *BUILDDATE*</p>
</div>
<div class="paragraph">
<p>Based on commit: <a href="https://github.com/HUPO-PSI/mzIdentML/commit/*COMMITHASH*">*COMMITHASH*</a> - <a href="https://github.com/HUPO-PSI/mzIdentML/commits/master/specification_document/specdoc1_3/asciidoc/crosslinking_ext.adoc">Commit History</a></p>
</div>
<div class="paragraph">
<p>The current version of this document is: version 1.0.0 final June 24, 2024</p>
</div>
<div class="paragraph">
<p>The latest (draft) version of this document may be found at <a href="https://github.com/HUPO-PSI/mzIdentML" class="bare">https://github.com/HUPO-PSI/mzIdentML</a>.</p>
</div>
<div class="paragraph">
<p><em>Type of This Document</em></p>
</div>
<div class="paragraph">
<p>This document is a <em>recommendation</em> for a common, community-driven standard data exchange format in proteomics.</p>
</div>
<div class="paragraph">
<p><em>Authors</em></p>
</div>
<div class="paragraph">
<p>Please see <a href="#authors">Chapter 8</a> for details on the authors and editors of this document.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="abstract">Abstract</h2>
<div class="sectionbody">
<div class="paragraph">
<p>The Human Proteome Organisation (HUPO) Proteomics Standards Initiative (PSI) defines community standards for data representation in proteomics to facilitate data comparison, exchange and verification. This document defines the extension of the mzIdentML data standard to describe the outputs of proteomics search engines or similar software for the identification of crosslinked peptides.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="introduction">1. Introduction</h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="background">1.1. Background</h3>
<div class="paragraph">
<p>A large number of proteomics search engines are available, each outputting results in a variety of file formats. mzIdentML <a href="https://paperpile.com/c/YrD2gH/vjf16">[1</a>] is a HUPO-PSI endorsed community standard that provides a common file format for identification results. This extension document has been released in parallel with mzIdentML version 1.3. mzIdentML version 1.3 supports extensions for additional features or use cases which can be described in additional documents, rather than editing the original specification document.</p>
</div>
<div class="paragraph">
<p>This mzIdentML 1.3.0 extension document provides further information on the encoding of crosslinking Mass Spectrometry (MS) results in mzIdentML. It has two aims:</p>
</div>
<div class="paragraph">
<p>1- Extend the crosslinking use cases supported by mzIdentML version 1.2.0 (2017) to cover what is currently required by the state-of-the-art in the field. Particular attention is paid to workflows using cleavable crosslinkers.</p>
</div>
<div class="paragraph">
<p>2- Provide further clarification and documentation on how to encode crosslinking data in an mzIdentML file.</p>
</div>
</div>
<div class="sect2">
<h3 id="supported-use-cases">1.2. Supported Crosslinking Use Cases</h3>
<div class="paragraph">
<p>Already supported use case in mzIdentML 1.2.0:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Two crosslinked peptides (the crosslinking product that is typically of most interest).</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>New use cases supported in this extension (part of mzIdentML 1.3.0):</p>
</div>
<div class="paragraph">
<p>1- Reporting cleavable crosslinkers. MS-cleavable crosslinkers can cleave upon activation in the mass spectrometer, releasing the individual peptides and thus enabling their individual analysis. Section 7.11 of the main mzIdentML 1.3.0 specification gives a new mechanism for encoding identifications based on multiple spectra, using this is essential for some cleavable crosslinker workflows. Such a workflow is used as an example in Section 7.11 of the main specification document.</p>
</div>
<div class="paragraph">
<p>2- Internally linked peptides (commonly known as “looplinks”). Internally linked peptides are cases where both ends of the crosslinker are within a single peptide, not between two copies of the same peptide. This type of crosslinking product is therefore necessarily intramolecular.</p>
</div>
<div class="paragraph">
<p>3- Noncovalently associated peptides. Some spectra may show the fragmentation of two different peptides which were not crosslinked but stayed associated due to noncovalent interactions <a href="https://paperpile.com/c/YrD2gH/IanGW">[2</a>]. Both peptides together appear as a single precursor species in the instrument, as opposed to ‘chimeric’ spectra where a single peptide is selected as precursor but additional peptide(s) fall within the same selection window. Identifying these noncovalently associated peptides may improve the accuracy of the results as it can prevent them from being misidentified as crosslinked peptides.</p>
</div>
<div class="paragraph">
<p>4- Additionally, the encoding of scores applicable to crosslinking MS results, and their corresponding thresholds, has been clarified and improved.</p>
</div>
<div class="paragraph">
<p>An overview of the different crosslinking product types and their support in mzIdentML is given in Figure 1. For discussion of the product types that are not supported in this version of mzIdentML (crosslinkers with more than two reactive groups, higher order crosslinked peptides) see §5.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 10.7142%;">
<col style="width: 10.7142%;">
<col style="width: 10.7142%;">
<col style="width: 10.7142%;">
<col style="width: 14.2857%;">
<col style="width: 10.7142%;">
<col style="width: 10.7142%;">
<col style="width: 10.7142%;">
<col style="width: 10.7149%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-center valign-middle"><p class="tableblock"><strong class="small">no crosslinker reaction</strong></p></td>
<td class="tableblock halign-center valign-middle"><div class="content"><div class="imageblock">
<div class="content">
<img src="img/crosslinking_ext/image9.jpg" alt="image" width="42" height="20">
</div>
</div>
<div class="paragraph">
<p><span class="small">linear peptide / free peptide</span></p>
</div></div></td>
<td class="tableblock halign-center valign-middle"></td>
<td class="tableblock halign-center valign-middle"></td>
<td class="tableblock halign-center valign-middle"></td>
<td class="tableblock halign-center valign-middle"></td>
<td class="tableblock halign-center valign-middle"><div class="content"><div class="imageblock">
<div class="content">
<img src="img/crosslinking_ext/image9.jpg" alt="image" width="43" height="15">
</div>
</div>
<div class="imageblock">
<div class="content">
<img src="img/crosslinking_ext/image9.jpg" alt="image" width="43" height="14">
</div>
</div>
<div class="paragraph">
<p><span class="small">non-covalently associated peptides</span></p>
</div></div></td>
<td class="tableblock halign-center valign-middle"></td>
<td class="tableblock halign-center valign-middle"></td>
</tr>
<tr>
<td class="tableblock halign-center valign-middle"><div class="content"><div class="paragraph">
<p><strong class="small">crosslinker reaction</strong></p>
</div></div></td>
<td class="tableblock halign-center valign-middle"></td>
<td class="tableblock halign-center valign-middle"><div class="content"><div class="imageblock">
<div class="content">
<img src="img/crosslinking_ext/image5.jpg" alt="image" width="56" height="33">
</div>
</div>
<div class="paragraph">
<p><span class="small">crosslinker modified peptide (monolink or dead-end link)</span></p>
</div></div></td>
<td class="tableblock halign-center valign-middle"><div class="content"><div class="imageblock">
<div class="content">
<img src="img/crosslinking_ext/image8.jpg" alt="image" width="63" height="41">
</div>
</div>
<div class="paragraph">
<p><span class="small">crosslinked peptides</span></p>
</div></div></td>
<td class="tableblock halign-center valign-middle"><div class="content"><div class="imageblock">
<div class="content">
<img src="img/crosslinking_ext/image4.png" alt="image" width="46" height="34">
</div>
</div>
<div class="paragraph">
<p><span class="small">cleavable crosslinker</span></p>
</div></div></td>
<td class="tableblock halign-center valign-middle"><div class="content"><div class="imageblock">
<div class="content">
<img src="img/crosslinking_ext/image6.jpg" alt="image" width="66" height="26">
</div>
</div>
<div class="paragraph">
<p><span class="small">internally linked peptide (looplink)</span></p>
</div></div></td>
<td class="tableblock halign-center valign-middle"></td>
<td class="tableblock halign-center valign-middle"><div class="content"><div class="imageblock">
<div class="content">
<img src="img/crosslinking_ext/image7.jpg" alt="image" width="63" height="45">
</div>
</div>
<div class="paragraph">
<p><span class="small">crosslinked peptides from crosslinkers with more than two reactive groups</span></p>
</div></div></td>
<td class="tableblock halign-center valign-middle"><div class="content"><div class="imageblock">
<div class="content">
<img src="img/crosslinking_ext/image10.jpg" alt="image" width="63" height="41">
</div>
</div>
<div class="paragraph">
<p><span class="small">higher order crosslinked peptides</span></p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-center valign-middle"><div class="content"><div class="paragraph">
<p><strong class="small">mzIdentML version supporting</strong></p>
</div></div></td>
<td class="tableblock halign-center valign-middle" colspan="2"><p class="tableblock"><strong class="small">1.1.0</strong></p></td>
<td class="tableblock halign-center valign-middle"><p class="tableblock"><strong class="small">1.2.0</strong></p></td>
<td class="tableblock halign-center valign-middle" colspan="3"><p class="tableblock"><strong class="small">1.3.0</strong></p></td>
<td class="tableblock halign-center valign-middle" colspan="2"><p class="tableblock"><strong class="small">Unsupported</strong></p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Figure 1. Summary of mzIdentML support for crosslinking product types.</strong></p>
</div>
</div>
<div class="sect2">
<h3 id="document-structure">1.3. Document Structure and Changes from mzIdentML Version 1.2.0 to 1.3.0 to Support Crosslinking Results</h3>
<div class="paragraph">
<p>mzIdentML version 1.3.0 makes two significant changes: (i) a new mechanism for encoding identifications based on multiple spectra, including the retiral of the old method for doing this; (ii) the ability to supplement the specification with extension documents covering specific use cases. The general guidance on the mzIdentML file format given in the main specification document all applies here, with this extension document giving guidance on the use of the controlled vocabulary (CV) terms specific to crosslinking.</p>
</div>
<div class="paragraph">
<p>The previously supported crosslinking use case was described in the main mzIdentML 1.2.0 specification document. In mzIdentML 1.3.0 this information has been moved to this extension document but it remains unchanged. The only change to the previous version 1.2.0 support for crosslinking is regarding identifications based on multiple spectra, this change is covered in Section 7.11 of the main mzIdentML 1.3.0 specification document.</p>
</div>
<div class="paragraph">
<p>The new use cases supported in this extension (version 1.0.0, compatible with mzIdentML version 1.3.0) are explained in detail in the following Sections of this document (denoted with an §). All of them have new controlled vocabulary terms associated with them.</p>
</div>
<div class="paragraph">
<p>Section 3 of this extension document is organised on the basis of elements in the mzIdentML schema, see Figure 2. For each of the new use cases, the list below states the relevant sections of this document and the new CV terms.</p>
</div>
<div class="paragraph">
<p>1- Reporting cleavable crosslinkers. See §3.2.2. Three new CV terms have been created related to encoding the derivatives of cleavable crosslinkers:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>“cleavable crosslinker stub” (MS:1003346),</p>
</li>
<li>
<p>“Unimod derivative code” (MS:1003347),</p>
</li>
<li>
<p>“crosslinker cleavage characteristics” (MS:1003390).</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>2- Internally linked peptides (a.k.a. “looplinks”). See Sections §3.3 &amp; §3.4.3. One new CV term has been created to allow the encoding:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>“looplink spectrum identification item” (MS:1003329).</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>3- Noncovalently associated peptides. See §3.2.1 and §3.4.2. Two new CV terms have been created related to noncovalently associated peptides:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>“noncovalently associated peptides search” (MS:1003330),</p>
</li>
<li>
<p>“noncovalently associated peptides spectrum identification item” (MS:1003331).</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>4- Improvements in the encoding of scores and thresholds related to crosslinking results. See Section §4. Seven CV terms have been created:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>"crosslinked PSM-level global FDR" (MS:1003337),</p>
</li>
<li>
<p>“peptide-pair sequence-level global FDR” (MS:1003338),</p>
</li>
<li>
<p>“peptide-pair passes threshold” (MS:1003339),</p>
</li>
<li>
<p>“residue-pair passes threshold” (MS:1003340),</p>
</li>
<li>
<p>“protein-protein interaction passes threshold” (MS:1003341),</p>
</li>
<li>
<p>“regular expression for whether interaction score derived from crosslinking passes threshold” (MS:1003342),</p>
</li>
<li>
<p>“FDR applied separately to self crosslinks and protein heteromeric crosslinks” (MS:1003343),</p>
</li>
<li>
<p>“residue pair ref” (MS:1003344).</p>
</li>
<li>
<p>“regular expression for residue-pair ref” (MS:1003345)</p>
</li>
</ul>
</div>
</div>
<div class="sect2">
<h3 id="_availability_of_documentation_and_example_files">1.4. Availability of Documentation and Example Files</h3>
<div class="paragraph">
<p>All documents in their most recent form are available on the PSI website (<a href="http://www.psidev.info/mzidentml"><span class="underline">http://www.psidev.info/mzidentml</span></a>) and at the mzIdentML GitHub project (<a href="https://github.com/HUPO-PSI/mzIdentML/tree/master/specification_document"><span class="underline">https://github.com/HUPO-PSI/mzIdentML/tree/master/specification_document</span></a>).</p>
</div>
<div class="paragraph">
<p>The example files supporting this extension document are available at <a href="https://github.com/HUPO-PSI/mzIdentML/blob/master/examples/1_3examples/crosslinking/" class="bare">https://github.com/HUPO-PSI/mzIdentML/blob/master/examples/1_3examples/crosslinking/</a><span class="[.underline"><a href="https://github.com/HUPO-PSI/mzIdentML/blob/master/examples/1_3examples/crosslinking/" class="bare">https://github.com/HUPO-PSI/mzIdentML/blob/master/examples/1_3examples/crosslinking/</a></span>].</p>
</div>
<div class="paragraph">
<p>The example files are:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Xlink_EDC_mzIdentML_1_3_0_draft.mzid (internally linked peptides),</p>
</li>
<li>
<p>multiple_spectra_per_id_1_3_0_draft.mzid (identification based on multiple spectra),</p>
</li>
<li>
<p>noncovalently_assoc_1_3_0_draft.mzid (noncovalently associated peptides),</p>
</li>
<li>
<p>scores_and_thresholds_1_3_0_draft.mzid (scores and thresholds).</p>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_controlled_vocabularies_for_encoding_crosslinks">2. Controlled Vocabularies for Encoding Crosslinks</h2>
<div class="sectionbody">
<div class="paragraph">
<p>A collection of terms for describing a certain domain is called a controlled vocabulary (CV) <a href="https://paperpile.com/c/YrD2gH/9dZZN">[3</a>]. Section 4.1 of the main mzIdentML 1.3.0 document describes the use of CVs in mzIdentML. The PSI-MS CV (<a href="https://github.com/HUPO-PSI/psi-ms-CV"><span class="underline">https://github.com/HUPO-PSI/psi-ms-CV</span></a>) can be used to encode many types of technical information in mzIdentML (e.g. statistical scores, mass spectrometers, etc). There are two other CVs that are relevant to encoding crosslinking data in mzIdentML: Unimod and XLMOD.
XLMOD (<a href="https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/master/cv/XLMOD.obo">https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/master/cv/XLMOD.obo</a>) represents the crosslinker reagents.
Unimod (<a href="http://www.unimod.org/obo/unimod.obo">http://www.unimod.org/obo/unimod.obo</a>) represents the resulting modifications in the crosslinked peptides/proteins.</p>
</div>
<div class="paragraph">
<p>At the time of writing (Unimod v2.1, XLMOD v1.1.12) both CVs have advantages and disadvantages when used for encoding crosslinking results in mzIdentML. For example, the representation of heterobifunctional crosslinkers (crosslinkers with different reactive groups) is better in XLMOD. However, the representation of the derivatives from a cleavable crosslinker is more complete in Unimod. Which CV (XLMOD or Unimod) to use for encoding crosslinker modifications is left as the implementers’ choice.</p>
</div>
<div class="paragraph">
<p>There is also some overlap between the information stored in these CVs and the contents of the &lt;SearchModification&gt; elements in mzIdentML. The &lt;SearchModification&gt; elements can encode: the derivatives of cleavable crosslinkers, namely the crosslinker stub as a peptide modification on the MS3 level and crosslinker cleavability as stub fragments on the MS2 level; and crosslinker specificity (including heterobifunctional crosslinkers). Implementers SHOULD describe the crosslinker modifications searched for as &lt;SearchModification&gt; elements; this provides a consistent way of retrieving crosslinker modification information regardless of which CV has been used, see §3.2.2.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_encoding_the_results_of_crosslinking_searches">3. Encoding the Results of Crosslinking Searches</h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_introduction">3.1. Introduction</h3>
<div class="paragraph">
<p>mzIdentML documents MUST indicate that they are implementing the guidance in this extension document by including the following CV term inside the top-level &lt;MzIdentML&gt; element, immediately after the &lt;cvList&gt; element:</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;cvParam cvRef="PSI-MS" accession="MS:1003385" name="mzIdentML crosslinking extension document version" value="1.0.0"/&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p>Crosslinked peptides presented a challenge for mzIdentML 1.2.0, since more than one peptide can be identified from the same spectrum.</p>
</div>
<div class="paragraph">
<p>mzIdentML 1.2.0 solved this by:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>introducing the “crosslink donor” (MS:1002509) and “crosslink acceptor” (MS:1002510) CV terms – the values of these terms associate <strong>either</strong> &lt;SearchModification&gt; elements (see §3.2.2) or &lt;Modification&gt; elements (see §3.3);</p>
</li>
<li>
<p>introducing the “crosslink spectrum identification item” (MS:1002511) CV term – the values of these terms group &lt;SpectrumIdentificationItem&gt; elements within a &lt;SpectrumIdentificationResult&gt; (see §3.4).</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Note that “crosslink donor” (MS:1002509) and “crosslink acceptor” (MS:1002510) are used in two different contexts:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>/MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/ ModificationParams/SearchModification – encoding the modifications searched for (including the specificity, see §3.2.2);</p>
</li>
<li>
<p>/MzIdentML/SequenceCollection/Peptide/Modification - encoding the actual modifications present in the crosslinked peptides (§3.3).</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>The rules that govern their use differ in each context, the details of these rules are given in §3.2.2 &amp; §3.3. To emphasise that they differ, Appendix I compares them. Appendix I presents no new information on how to encode crosslinking results in mzIdentML.</p>
</div>
<div class="paragraph">
<p>Figure 2 gives an overview of how the subsections here (§3) relate to the elements in an mzIdentML file.</p>
</div>
<div class="imageblock">
<div class="content">
<img src="img/crosslinking_ext/image2.png" alt="image" width="864" height="562">
</div>
</div>
<div class="paragraph">
<p><strong>Figure 2. Overview of the mzIdentML 1.3.0. Format (crosslinking extension).</strong> Elements are labelled with the section from this document that contains guidance on how to encode them.</p>
</div>
</div>
<div class="sect2">
<h3 id="_spectrumidentificationprotocol_elements">3.2. &lt;SpectrumIdentificationProtocol&gt; Elements</h3>
<div class="paragraph">
<p>A &lt;SpectrumIdentificationProtocol&gt; element describes the parameters and settings of a spectrum identification analysis. There may be several of these protocols included in one mzIdentML file. In the case of analysis workflows in which an identification is based on multiple spectra (see Section 7.11 of the main mzIdentML 1.3.0 specification document), these spectra identifications may be included in different &lt;SpectrumIdentificationList&gt; elements, each associated with a different &lt;SpectrumIdentificationProtocol&gt;.</p>
</div>
<div class="paragraph">
<p>Section 2 of the main mzIdentML 1.3.0 specification document states that “all search parameters should be described in sufficient detail to enable a user to run the same or a similar search on the same or another search engine”. As far as possible, the information that would be needed to reannotate the mass spectra SHOULD be included. The &lt;FragmentTolerance&gt; and &lt;ParentTolerance&gt; subelements of &lt;SpectrumIdentificationProtocol&gt; SHOULD be completed.</p>
</div>
<div class="paragraph">
<p>Two child elements of &lt;SpectrumIdentificationProtocol&gt; are covered in more detail here:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>&lt;AdditionalSearchParams&gt; (§3.2.1),</p>
</li>
<li>
<p>&lt;ModificationParams&gt; (§3.2.2).</p>
</li>
</ul>
</div>
<div class="sect3">
<h4 id="_additional_search_parameters">3.2.1. Additional Search Parameters</h4>
<div class="paragraph">
<p><strong>Path:</strong> <span class="underline">/MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/AdditionalSearchParams</span></p>
</div>
<div class="paragraph">
<p>If a crosslinking search has been performed then the CV term “crosslinking search” (MS:1002494) MUST be present within the &lt;AdditionalSearchParams&gt; subelement of every &lt;SpectrumIdentificationProtocol&gt; associated with that search (see Figure 3).</p>
</div>
<div class="paragraph">
<p>The ion series that were searched for SHOULD also be included here.</p>
</div>
<div class="paragraph">
<p><em><span class="underline">New supported use case in this extension - noncovalently associated peptides:</span></em> mzIdentML 1.2.1 introduces a new CV term – “noncovalently associated peptides search” (MS:1003330). If pairs of noncovalently associated peptides were also searched for, then the &lt;SpectrumIdentificationProtocol&gt; elements MUST also contain this new CV term within their &lt;AdditionalSearchParams&gt; subelement, see Figure 3.</p>
</div>
<div class="paragraph">
<p>The new CV term "FDR applied separately to self crosslinks and protein heteromeric crosslinks" (MS:1003343), see §4.4, which SHOULD be present is also shown in Figure 3.</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;AnalysisProtocolCollection&gt;

....
&lt;SpectrumIdentificationProtocol analysisSoftware_ref="ID_software" id="SearchProtocol_1"&gt;

&lt;SearchType&gt;

&lt;cvParam accession="MS:1001083" cvRef="PSI-MS" name="ms-ms search"/&gt;

&lt;/SearchType&gt;

&lt;AdditionalSearchParams&gt;

&lt;cvParam accession="MS:1001211" cvRef="PSI-MS" name="parent mass type mono"/&gt;

&lt;cvParam accession="MS:1001256" cvRef="PSI-MS" name="fragment mass type mono"/&gt;

&lt;cvParam accession="MS:1002494" cvRef="PSI-MS" name="crosslinking search"/&gt;

&lt;cvParam accession="MS:1003330" cvRef="PSI-MS"

name="noncovalently associated peptides search"/&gt;
....

&lt;cvParam accession="MS:1003343" cvRef="PSI-MS"

....
name="FDR applied separately to self crosslinks and protein heteromeric crosslinks" value="true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1001118" name="param: b ion"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1001262" name="param: y ion"/&gt;

&lt;/AdditionalSearchParams&gt;

....

&lt;/SpectrumIdentificationProtocol&gt;
....

&lt;/AnalysisProtocolCollection&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>Figure 3. XML snippet showing crosslinking related CV terms.</strong> If a crosslinking search has been performed, MS:1002494 <strong>MUST</strong> be present. If noncovalently associated peptides have also been searched for, then MS:1003330 <strong>MUST</strong> also be present. The new CV term "FDR applied separately to self crosslinks and protein heteromeric crosslinks" (MS:1003343) which <strong>SHOULD</strong> be present is also shown, see §4.4.</p>
</div>
</div>
<div class="sect3">
<h4 id="_modification_parameters">3.2.2. Modification Parameters</h4>
<div class="paragraph">
<p><strong>Path:</strong> <span class="underline">/MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/ModificationParams/SearchModification</span></p>
</div>
<div class="paragraph">
<p>The &lt;SpectrumIdentificationProtocol&gt; element encodes the modifications that were searched for within its &lt;ModificationParams&gt; subelement. These are encoded in &lt;SearchModification&gt; elements within &lt;ModificationParams&gt;.</p>
</div>
<div class="paragraph">
<p>mzIdentML version 1.3.0 introduces two new CV terms to link &lt;SearchModification&gt; elements and &lt;Modification&gt; elements - “search modification id” (MS:1003392) which goes inside &lt;SearchModification&gt; elements, and “search modification id ref” (MS:1003393) which goes inside &lt;Modification&gt; elements. Making this link is optional but recommended where possible. In the case of open modification searches, such a link cannot be made. See Section 7.12 of the main mzIdentML specification document.</p>
</div>
<div class="paragraph">
<p>Each crosslinker reagent is defined by multiple &lt;SearchModification&gt; elements that contain either the “crosslink donor” (MS:1002509) or “crosslink acceptor” (MS:1002510) CV term. An example is given in Figure 4(i). The residue specificities of the crosslinkers used SHOULD be encoded here, examples are given in Appendix II.</p>
</div>
<div class="paragraph">
<p>The value slot of the crosslink donor and acceptor CV terms is interpreted as a local identifier for the &lt;SearchModification&gt; elements describing a single reagent. The rules governing the use of the crosslink donor and acceptor CV terms in &lt;SearchModification&gt; elements are given in Figure 4(ii).</p>
</div>
<div class="paragraph">
<p>There may be more than two &lt;SearchModification&gt; elements required. For example, if the crosslinker reacts with the sidechains and also with the protein termini, see Appendix II for examples.</p>
</div>
<div class="paragraph">
<p>&lt;SearchModification&gt; elements can contain one or more children of the CV term “peptide modification details” (MS:1001471). These CV terms can encode information on neutral losses, see Figure 4(i).</p>
</div>
<div class="paragraph">
<p><em><span class="underline">New supported use case in this extension - cleavable crosslinkers:</span></em> mzIdentML 1.3.0 adds three new CV terms relating to modifications from cleavable crosslinkers – “cleavable crosslinker stub” (MS:1003346), “Unimod derivative code” (MS:1003347) and “crosslinker cleavage characteristics” (MS:1003390).</p>
</div>
<div class="paragraph">
<p>At the MS3 level, where single peptides and part of the cleaved crosslinker are identified, the crosslinker modifications SHOULD include the CV term “cleavable crosslinker stub” (MS:1003346).</p>
</div>
<div class="paragraph">
<p>The crosslink stub modification MUST also have a suitably sourced CV term for the reagent (see Appendix II). Additionally, if Unimod is being used as the CV, then the CV term “Unimod derivative code” (MS:1003347) MAY be used to state which derivative of the cleaved crosslinker is identified. The single-letter derivative codes in Unimod are chosen arbitrarily when a linker definition is added to Unimod. For instance, in <a href="https://unimod.org/xlink.html"><span class="underline">https://unimod.org/xlink.html</span></a> one can find the examples "A for alkene, S for sulfenic acid, and T for thiol", and e.g. Xlink:DSS uses W for loss of water. There is no formal vocabulary for the single-letter codes. "UNIMOD derivative code" must be equal to one of the derivative codes defined in the corresponding Unimod entry (not a random character unrelated to the definition). An example Unimod entry is at <a href="https://www.unimod.org/modifications_view.php?editid1=1842"><span class="underline">https://www.unimod.org/modifications_view.php?editid1=1842 .</span></a></p>
</div>
<div class="paragraph">
<p>At the MS2 level, the new CV term “crosslinker cleavage characteristics” (MS:1003390) signifies that the crosslinker is cleavable and on cleavage can leave a given stub. This can lead to additional stub fragments in the MS2 spectra that contain the crosslinker stub instead of the whole crosslinker plus the second peptide. Each “crosslinker cleavage characteristics” CV term represents one possible crosslinker stub. It has a structured value -</p>
</div>
<div class="paragraph">
<p><em>name</em>:_mass_:_pairs with_</p>
</div>
<div class="paragraph">
<p><em>Name</em> must be a single character to identify this stub. The scope of <em>name</em> is restricted to that crosslinker definition, i.e. they need only be unique within that crosslinker definition not the whole file or the &lt;SpectrumIdentification&gt; element. <em>Mass</em> gives the monoisotopic mass delta of the resulting stub in Daltons. <em>Pairs with</em> MUST be a sequence of one or more characters, giving the <em>name(s)</em> of the partner stub(s). See Appendix II for examples.</p>
</div>
<div class="paragraph">
<p>Note that the choice of which &lt;SearchModification&gt; is the donor and which one is the acceptor is arbitrary.</p>
</div>
<div class="paragraph">
<p><strong>(i)</strong></p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;SpectrumIdentificationProtocol&gt;

...

&lt;ModificationParams&gt;

____
&lt;SearchModification fixedMod="false" massDelta="138.06808" residues="S T Y K"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="BS3_donor"/&gt;

&lt;cvParam cvRef="XLMOD" accession="XLMOD:02000" name="BS3"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002509" name="crosslink donor"

value="0"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="138.06808" residues="."&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002057"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="BS3_donor_n_term"/&gt;

&lt;cvParam cvRef="XLMOD" accession="XLMOD:02000" name="BS3"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink donor"

value="0" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="0.0" residues="S T Y K"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="BS3_acceptor"/&gt;

&lt;cvParam cvRef="XLMOD" accession="XLMOD:02000" name="BS3"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor"

value="0"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="0.0" residues="."&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002058"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="BS3_acceptor_n_term"&gt;

&lt;cvParam cvRef="XLMOD" accession="XLMOD:02000" name="BS3"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor"

value="0" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="15.994919" residues="M"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="Mox"/&gt;

&lt;cvParam accession="UNIMOD:35" name="Oxidation" cvRef="UNIMOD" /&gt;

&lt;cvParam accession="MS:1001524" name="fragment neutral loss"

cvRef="PSI-MS" value="63.998291" unitAccession="UO:0000221" unitName="dalton" unitCvRef="UO"/&gt;

&lt;/SearchModification&gt;
____

&lt;/ModificationParams&gt;

...

&lt;/SpectrumIdentificationProtocol&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>(ii)</strong></p>
</div>
<div class="ulist">
<ul>
<li>
<p><strong>At least two</strong> &lt;SearchModification&gt; elements SHOULD be used to encode each crosslink reagent, to encode the site specificity of both the donor and acceptor termini of the reagent.</p>
</li>
<li>
<p>The value slot of the crosslink donor and acceptor CV terms is interpreted as a local identifier for the &lt;SearchModification&gt; elements describing a single reagent.</p>
</li>
<li>
<p>The choice of which reactive group is the donor and which is the acceptor is arbitrary.</p>
</li>
<li>
<p>The crosslink donor &lt;SearchModification&gt; element <strong>MUST</strong> have the attribute massDelta = the mass gain from the crosslink reagent.</p>
</li>
<li>
<p>The crosslink acceptor peptide’s &lt;SearchModification&gt; element <strong>MUST</strong> have massDelta = 0.</p>
</li>
<li>
<p><strong>Both</strong> acceptor and donor <strong>MUST</strong> have a suitably sourced &lt;cvParam&gt;.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p><strong>Figure 4.</strong> <strong>The use of the “crosslink donor” (MS:1002509) and “crosslink acceptor” (MS:1002510) CV terms in &lt;SearchModification&gt; elements.</strong></p>
</div>
<div class="paragraph">
<p><strong>(i)</strong> XML snippet showing the “crosslink donor” (MS:1002509) and “crosslink acceptor” (MS:1002510) CV terms used in &lt;SearchModification&gt;, shows encoding for the BS3 crosslinking reagent. It also shows a modification with a neutral loss.</p>
</div>
<div class="paragraph">
<p><strong>(ii)</strong> The rules applying to the use of the “crosslink donor” (MS:1002509) and “crosslink acceptor” (MS:1002510) CV terms within &lt;SearchModification&gt;.</p>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_encoding_crosslinked_peptides_in_the_element_sequencecollection">3.3. Encoding Crosslinked Peptides in the Element &lt;SequenceCollection&gt;</h3>
<div class="paragraph">
<p><span class="underline"><strong>Path:</strong> /MzIdentML/SequenceCollection</span></p>
</div>
<div class="paragraph">
<p>The peptides that have been identified are encoded in the &lt;SequenceCollection&gt; element. This will include both crosslinked and uncrosslinked peptides.</p>
</div>
<div class="paragraph">
<p>A word of warning about redundancy, it is not the intention of mzIdentML that every &lt;SpectrumIdentificationItem&gt; (§3.4) references a new &lt;Peptide&gt; in &lt;SequenceCollection&gt; – “the combination of &lt;Peptide&gt; sequence and modifications MUST be unique in the file” (main mzIdentML specification document, Section 6.68). However, each distinct combination of crosslinked peptides will require a new pair of &lt;Peptide&gt; elements in &lt;SequenceCollection&gt;.</p>
</div>
<div class="paragraph">
<p>To represent the crosslinked peptides, mzIdentML 1.2.0 added a mechanism for linking two different &lt;Peptide&gt; elements together, using the CV terms “crosslink donor” (MS:1002509) and “crosslink acceptor” (MS:1002510). An identical value for these terms indicates that they are grouped together, see Figure 5(i).</p>
</div>
<div class="paragraph">
<p>The rules governing the use of the crosslink donor and acceptor CV terms in &lt;Modification&gt; elements are given in Figure 5(ii).</p>
</div>
<div class="paragraph">
<p>As of mzIdentML 1.3.0, &lt;Modification&gt; elements MAY contain the CV term "search modification id ref" (MS:1003393) to link a &lt;Modification&gt; to a &lt;SearchModification&gt; element. The value of this term is the unique id of the &lt;SearchModification&gt; as defined by its "search modification id" (MS:1003392) CV term. It is recommended to use this approach for the encoding of modifications from crosslinkers, see Appendix II.</p>
</div>
<div class="paragraph">
<p><em><span class="underline">New supported use case in this extension - internally linked peptide:</span></em> An internally linked peptide has both ends of the crosslinker within it. To encode an internally crosslinked peptide the &lt;Peptide&gt; can contain one &lt;Modification&gt; element with the “crosslink donor” CV term and one &lt;Modification&gt; element with the “crosslink acceptor” CV term. The same rules apply to these CV terms when encoding internally linked peptides as when encoding crosslinked peptides (Figure 5 (ii)). For an example of how to encode an internally linked peptide, see Figure 5(iii).</p>
</div>
<div class="paragraph">
<p>The accompanying example file <a href="https://github.com/HUPO-PSI/mzIdentML/blob/master/examples/1_3_0examples/crosslinking/multiple_spectra_per_id_1.3.0_draft.mzid"><span class="underline">multiple_spectra_per_id_1.3.0_draft.mzid</span></a> illustrates a common cleavable crosslinker workflow <a href="https://paperpile.com/c/YrD2gH/osIjk">[4</a>].</p>
</div>
<div class="paragraph">
<p>Child CV terms of “peptide modification details” (MS:1001471) can be included in &lt;Modification&gt; elements to provide additional information about the modification, including the new cleavable crosslinker related CV terms, see §3.2.2. This is not recommended if the &lt;Modification&gt; elements have "search modification id ref" (MS:1003393) CV terms to link them to a &lt;SearchModification&gt; element, as it would add unnecessary duplication to the file.</p>
</div>
<div class="paragraph">
<p>The encoding for crosslinked peptides MAY be combined with the encoding for modification localisation scoring, using the same mechanism (main mzIdentML 1.3.0 document, Section 5.2.8).</p>
</div>
<div class="paragraph">
<p><strong>(i)</strong></p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;SequenceCollection&gt;

____
&lt;Peptide id="30491856_30492180_2_4_p1"&gt;

&lt;PeptideSequence&gt;AAFTKQAADK&lt;/PeptideSequence&gt;

&lt;Modification monoisotopicMassDelta="138.0680796" location="5"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003393"

name="search modification id ref" value="DSS_donor"/&gt;

&lt;cvParam accession="XL:00002" cvRef="PSI-MS" name="Xlink:DSS"/&gt;

&lt;cvParam accession="MS:1002509" cvRef="PSI-MS" name="crosslink donor" value="*5448*"/&gt;

&lt;/Modification&gt;

&lt;/Peptide&gt;

&lt;Peptide id="30491856_30492180_2_4_p2"&gt;

&lt;PeptideSequence&gt;AMYPPKEDR&lt;/PeptideSequence&gt;

&lt;Modification monoisotopicMassDelta="0.0" location="6"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003393"

name="search modification id ref" value="DSS_acceptor"/&gt;

&lt;cvParam accession="MS:1002510" cvRef="PSI-MS" name="crosslink acceptor" value="*5448*"/&gt;

&lt;/Modification&gt;

&lt;/Peptide&gt;

...
____


&lt;/SequenceCollection&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>(ii)</strong> If a pair of crosslinked peptides has been identified:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>One peptide’s &lt;Modification&gt; element MUST be flagged as “crosslink donor” and one MUST be flagged as “crosslink acceptor”.</p>
</li>
<li>
<p>A unique identifier linking exactly <strong>two</strong> &lt;Modification&gt; elements together <strong>MUST</strong> be in the value slot. (Thereby excluding the representation of trimeric crosslinkers, see §6.)</p>
</li>
<li>
<p>If the CV term “search modification id ref” (MS:1003393) is being used then the crosslink donor MUST be chosen to match the end marked as the donor in the corresponding &lt;SearchModification&gt; elements, see §3.2.2. If that CV term is not used, or if the preceding rule does not unambiguously define which end to mark as donor (e.g. because the crosslinker is symmetrical) then the export software SHOULD use the following rules to choose the crosslink donor as the: longer peptide, then higher peptide neutral mass, then alphabetical order.</p>
</li>
<li>
<p>The crosslink donor &lt;Modification&gt; element <strong>MUST</strong> have the attribute monoisotopicMassDelta = the mass gain from the crosslink reagent.</p>
</li>
<li>
<p>The crosslink acceptor peptide’s &lt;Modification&gt; element <strong>MUST</strong> have monoisotopicMassDelta = 0.</p>
</li>
<li>
<p>The crosslink donor peptide’s &lt;Modification&gt; element <strong>MUST</strong> have a suitably sourced cvParam for the crosslink. The crosslink acceptor peptide’s &lt;Modification&gt; element <strong>MUST</strong> <strong>NOT</strong> have a cvParam for the reagent.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p><strong>(iii)</strong></p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;SequenceCollection&gt;

____
&lt;Peptide id="peptide_7_1"&gt;

&lt;PeptideSequence&gt;DVIQSLVDDDLVAK&lt;/PeptideSequence&gt;

&lt;Modification location="10" residues="D" monoisotopicMassDelta="-18.010565"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003393"

name="search modification id ref" value="EDC_donor"/&gt;

&lt;cvParam accession="UNIMOD:2018" name="Xlink:EDC" cvRef="UNIMOD"/&gt;

&lt;cvParam accession="MS:1002509" cvRef="PSI-MS" name="crosslink donor"

value="*100*"/&gt;

&lt;/Modification&gt;

&lt;Modification location="14" residues="K" monoisotopicMassDelta="0.0"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003393"

name="search modification id ref" value="EDC_acceptor"/&gt;

&lt;cvParam accession="MS:1002510" cvRef="PSI-MS" name="crosslink acceptor"

value="*100*"/&gt;

&lt;/Modification&gt;

&lt;/Peptide&gt;

...
____

&lt;/SequenceCollection&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>Figure 5. Encoding Crosslinked Peptides in the Element &lt;SequenceCollection&gt;</strong></p>
</div>
<div class="paragraph">
<p>(i) XML snippet showing the encoding of crosslinked peptides.</p>
</div>
<div class="paragraph">
<p>(ii) The rules applying to the use of the “crosslink donor” (MS:1002509) and “crosslink acceptor” (MS:1002510) CV terms within &lt;Modification&gt; elements.</p>
</div>
<div class="paragraph">
<p>(iii) XML snippet showing the encoding of an internally linked peptide.</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;SequenceCollection&gt;

&lt;!-- linear peptides--&gt;

&lt;Peptide id="p1_linear"&gt;

&lt;PeptideSequence&gt;PEPKR&lt;/PeptideSequence&gt;

&lt;Modification location="4" monoisotopicMassDelta="176.01433"&gt;

____
&lt;cvParam cvRef="PSI-MS" accession="MS:1003393"
____

name="search modification id ref" value="DSSO_monolink_W"/&gt;

&lt;cvParam accession="UNIMOD:1842" cvRef="UNIMOD" name="Xlink:DSSO"/&gt;

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code" value="W" cvRef="PSI-MS" /&gt;

&lt;/Modification&gt;

&lt;/Peptide&gt;

&lt;!-- crosslinked peptides --&gt;

&lt;Peptide id="p1"&gt;

&lt;PeptideSequence&gt;PEPKR&lt;/PeptideSequence&gt;

&lt;Modification location="4" monoisotopicMassDelta="158.003765"&gt;

____
&lt;cvParam cvRef="PSI-MS" accession="MS:1003393"
____

name="search modification id ref" value="DSSO_donor"/&gt;

&lt;cvParam accession="UNIMOD:1842" cvRef="UNIMOD" name="Xlink:DSSO"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002509" name="crosslink donor" value="1"/&gt;

&lt;/Modification&gt;

&lt;/Peptide&gt;

&lt;Peptide id="p2"&gt;

&lt;PeptideSequence&gt;TIDYK&lt;/PeptideSequence&gt;

&lt;Modification location="4" monoisotopicMassDelta="0"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003393"

name="search modification id ref" value="DSSO_acceptor"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor" value="1"/&gt;

&lt;/Modification&gt;

&lt;/Peptide&gt;

&lt;!-- MS3 peptides are separately listed, as they are linear stub modified peptides --&gt;

&lt;Peptide id="p1_a"&gt;

&lt;PeptideSequence&gt;PEPKR&lt;/PeptideSequence&gt;

&lt;Modification location="4" monoisotopicMassDelta="54.010565"&gt;

____
&lt;cvParam cvRef="PSI-MS" accession="MS:1003393"
____

name="search modification id ref" value="DSSO_crosslink_stub_a"/&gt;

&lt;cvParam accession="UNIMOD:1842" cvRef="UNIMOD" name="Xlink:DSSO"/&gt;

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code" value="A" cvRef="PSI-MS" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003346" name="cleavable crosslinker stub"/&gt;

&lt;/Modification&gt;

&lt;/Peptide&gt;

&lt;Peptide id="p1_t"&gt;

&lt;PeptideSequence&gt;PEPKR&lt;/PeptideSequence&gt;

&lt;Modification location="4" monoisotopicMassDelta="85.982635"&gt;

____
&lt;cvParam cvRef="PSI-MS" accession="MS:1003393"
____

name="search modification id ref" value="DSSO_crosslink_stub_t"/&gt;

&lt;cvParam accession="UNIMOD:1842" cvRef="UNIMOD" name="Xlink:DSSO"/&gt;

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code" value="T" cvRef="PSI-MS" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003346" name="cleavable crosslinker stub"/&gt;

&lt;/Modification&gt;

&lt;/Peptide&gt;

&lt;Peptide id="p2_a"&gt;

&lt;PeptideSequence&gt;TIDYK&lt;/PeptideSequence&gt;

&lt;Modification location="4" monoisotopicMassDelta="54.010565"&gt;

____
&lt;cvParam cvRef="PSI-MS" accession="MS:1003393"
____

name="search modification id ref" value="DSSO_crosslink_stub_a"/&gt;

&lt;cvParam accession="UNIMOD:1842" cvRef="UNIMOD" name="Xlink:DSSO"/&gt;

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code" value="A" cvRef="PSI-MS" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003346" name="cleavable crosslinker stub"/&gt;

&lt;/Modification&gt;

&lt;/Peptide&gt;

&lt;Peptide id="p2_t"&gt;

&lt;PeptideSequence&gt;TIDYK&lt;/PeptideSequence&gt;

&lt;Modification location="4" monoisotopicMassDelta="85.982635"&gt;

____
&lt;cvParam cvRef="PSI-MS" accession="MS:1003393"
____

name="search modification id ref" value="DSSO_crosslink_stub_t"/&gt;

&lt;cvParam accession="UNIMOD:1842" cvRef="UNIMOD" name="Xlink:DSSO"/&gt;

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code" value="T" cvRef="PSI-MS" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003346" name="cleavable crosslinker stub"/&gt;

&lt;/Modification&gt;

&lt;/Peptide&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>Figure 6. XML snippet showing the encoding of modifications from cleavable crosslinkers.</strong> The new CV terms are shown: “crosslinker stub” (MS:1003346) and “Unimod derivative code” (MS:1003347). This example also uses the new CV term "search modification id ref" (MS:1003393) to reference the corresponding &lt;SearchModification&gt; elements.</p>
</div>
</div>
<div class="sect2">
<h3 id="_encoding_identified_crosslinks_in_spectrumidentificationitem_elements">3.4. Encoding Identified Crosslinks in &lt;SpectrumIdentificationItem&gt; Elements</h3>
<div class="sect3">
<h4 id="_identifications_of_crosslinked_peptides">3.4.1. Identifications of Crosslinked Peptides</h4>
<div class="paragraph">
<p><span class="underline"><strong>Path:</strong> /MzIdentML/DataCollection/AnalysisData/SpectrumIdentificationList/SpectrumIdentificationResult</span></p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult&gt; elements report the evidence associated with the identification of particular peptides.</p>
</div>
<div class="paragraph">
<p>A pair of crosslinked peptides within a given &lt;SpectrumIdentificationResult&gt; MUST be reported as two instances of &lt;SpectrumIdentificationItem&gt; having a shared local unique identifier as the value for the CV term “crosslink spectrum identification item” (MS:1002511). Locally unique means unique within the containing &lt;SpectrumIdentificationResult&gt;. See Figure 7(i). The rules governing the use of the “crosslink spectrum identification item” CV term are given in Figure 7(ii).</p>
</div>
<div class="paragraph">
<p><strong>(i)</strong></p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-img" data-lang="img">&lt;SpectrumIdentificationResult spectraData_ref="SID_1" spectrumID="index=2776" id="SIR_1"&gt;

____
&lt;SpectrumIdentificationItem passThreshold="true" rank="*1*"

peptide_ref="30491856_30492180_2_4_p1" experimentalMassToCharge= "569.7912"

calculatedMassToCharge="569.79054" chargeState="4" id="SII_1_1"&gt;

&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_psm121558473_pep30491845_protP02768-A_target_535"/&gt;

&lt;cvParam accession="MS:1002511" cvRef="PSI-MS" value="*1*"

name="crosslink spectrum identification item"/&gt;

&lt;cvParam accession="MS:1002545" cvRef="PSI-MS"

value="1.3111826921077734" name="xi:score"/&gt;

&lt;cvParam accession="MS:1003344" cvRef="PSI-MS" value="54321.a" name="Residue pair ref"/&gt;

&lt;/SpectrumIdentificationItem&gt;

&lt;SpectrumIdentificationItem passThreshold="true" rank="*1*"

peptide_ref="30491715_30491845_3_7_p0" experimentalMassToCharge= "569.7912"

calculatedMassToCharge="569.79054" chargeState="4" id="SII_1_2"&gt;

&lt;PeptideEvidenceRef

peptideEvidence_ref="pepevid_psm121558473_pep30491715_protP02768-A_target_411"/&gt;

&lt;cvParam accession="MS:1002511" cvRef="PSI-MS" value="*1*"

name="crosslink spectrum identification item"/&gt;

&lt;cvParam accession="MS:1002545" cvRef="PSI-MS"

value="1.3111826921077734" name="xi:score"/&gt;

&lt;cvParam accession="MS:1003344" cvRef="PSI-MS" value="54321.b" name="Residue pair ref"/&gt;

&lt;/SpectrumIdentificationItem&gt;
____

&lt;/SpectrumIdentificationResult&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>(ii)</strong> If a crosslinked pair of peptides has been identified:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>There MUST be <strong>two</strong> &lt;SpectrumIdentificationItem&gt; elements with the same rank value.</p>
</li>
<li>
<p>Both MUST have the “crosslink spectrum identification item” cvParam, and the value acts as a <strong>local</strong> identifier within the &lt;SpectrumIdentificationResult&gt; to group these two elements together.</p>
</li>
<li>
<p>The experimentalMassToCharge, calculatedMassToCharge and chargeState MUST be identical over both SII elements, indicating the overall values for the pair.</p>
</li>
<li>
<p>If the search engine applies a score to the paired identification, both &lt;SpectrumIdentificationItem&gt; elements MUST have the same cvParam capturing the value.</p>
</li>
<li>
<p>The two &lt;SpectrumIdentificationItem&gt; elements MAY also have independent scores for the two chains (not shown).</p>
</li>
</ul>
</div>
<div class="paragraph">
<p><strong>Figure 7.</strong> <strong>Encoding the identification of a pair of crosslinked peptides.</strong> (i) Example XML snippet. (ii) The rules governing the use of “crosslink spectrum identification item”.</p>
</div>
</div>
<div class="sect3">
<h4 id="_identifications_of_noncovalently_associated_peptides">3.4.2. Identifications of Noncovalently Associated Peptides</h4>
<div class="paragraph">
<p><span class="underline"><strong>Path:</strong> /MzIdentML/DataCollection/AnalysisData/SpectrumIdentificationList/SpectrumIdentificationResult</span></p>
</div>
<div class="paragraph">
<p><em><span class="underline">New supported use case in this extension - noncovalently associated peptides:</span></em> mzIdentML 1.2.1 introduces a new CV term “noncovalently associated peptides spectrum identification item” (MS:1003331) to encode such identifications (see §1.2). It operates in the same way as “crosslink spectrum identification item”, by using the value of the CV term to group the identifications together, see Figure 8(i).</p>
</div>
<div class="paragraph">
<p>As indicated above, to use the “noncovalently associated peptides spectrum identification item” (MS:1003331), the element &lt;AdditionalSearchParams&gt; MUST contain the CV term “noncovalently associated peptides search” (MS:1003330), see Figure 3.</p>
</div>
<div class="paragraph">
<p>The rules governing the use of the “noncovalently associated peptides spectrum identification item” CV term are given in Figure 8 (ii) and are analogous to those governing the use of “crosslink spectrum identification item”. The peptides referred to will be linear, uncrosslinked peptides.</p>
</div>
<div class="paragraph">
<p><strong>(i)</strong></p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;SpectrumIdentificationResult spectraData_ref="SID_1" spectrumID="index=2776" id="SIR_1"&gt;

____
&lt;SpectrumIdentificationItem passThreshold="true" rank="*1*"

peptide_ref=p1" experimentalMassToCharge= "569.7912"

calculatedMassToCharge="569.79054" chargeState="4" id="SII_1_1"&gt;

&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_pep_1"/&gt;

&lt;cvParam accession="MS:1003331" cvRef="PSI-MS" value="*1*"

name="noncovalently associated peptides spectrum identification item"/&gt;

&lt;cvParam accession="MS:1002545" cvRef="PSI-MS"

value="1.3111826921077734" name="xi:score"/&gt;

&lt;/SpectrumIdentificationItem&gt;

&lt;SpectrumIdentificationItem passThreshold="true" rank="*1*"

peptide_ref="p2" experimentalMassToCharge= "569.7912"

calculatedMassToCharge="569.79054" chargeState="4" id="SII_1_2"&gt;

&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_pep_2"/&gt;

&lt;cvParam accession="MS:1003331" cvRef="PSI-MS" value="*1*"

name="noncovalently associated peptides spectrum identification item"/&gt;

&lt;cvParam accession="MS:1002545" cvRef="PSI-MS"

value="1.3111826921077734" name="xi:score"/&gt;

&lt;/SpectrumIdentificationItem&gt;
____

&lt;/SpectrumIdentificationResult&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>(ii)</strong> If a pair of <strong>noncovalently associated peptides</strong> has been identified:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>There MUST be <strong>two</strong> &lt;SpectrumIdentificationItem&gt; elements with the same rank value.</p>
</li>
<li>
<p>Both MUST have the “noncovalently associated peptides spectrum identification item” cvParam, and the value acts as a <strong>local</strong> identifier within the &lt;SpectrumIdentificationResult&gt; to group these two elements together.</p>
</li>
<li>
<p>The experimentalMassToCharge, calculatedMassToCharge and chargeState MUST be identical over both SII elements, indicating the overall values for the pair.</p>
</li>
<li>
<p>If the search engine applies a score to the paired identification, both &lt;SpectrumIdentificationItem&gt; elements MUST have the same cvParam capturing the value.</p>
</li>
<li>
<p>The two &lt;SpectrumIdentificationItem&gt; elements MAY also have independent scores for the two chains (not shown).</p>
</li>
</ul>
</div>
<div class="paragraph">
<p><strong>Figure 8.</strong> <strong>Encoding the identification of a pair of noncovalently associated peptides.</strong> (i) Example XML snippet. (ii) The rules governing the use of “noncovalently associated peptides spectrum identification item”.</p>
</div>
</div>
<div class="sect3">
<h4 id="_identifications_of_an_internally_linked_peptide">3.4.3. Identifications of an Internally Linked Peptide</h4>
<div class="paragraph">
<p><span class="underline"><strong>Path:</strong> /MzIdentML/DataCollection/AnalysisData/SpectrumIdentificationList/SpectrumIdentificationResult</span></p>
</div>
<div class="paragraph">
<p><em><span class="underline">New supported use case in this extension - internally linked peptide:</span></em> mzIdentML 1.3.0 introduces a new CV term – “looplink spectrum identification item” (MS:1003329) – to allow the encoding of internally linked peptides (a.k.a. “looplinks”), see Figure 9. The &lt;SpectrumIdentificationItem&gt; element will refer to a &lt;Peptide&gt; containing both crosslink donor and crosslink acceptor modifications (as shown in Figure 5(iii)).</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;SpectrumIdentificationResult spectraData_ref="SID_1" spectrumID="index=2776" id="SIR_1"&gt;

____
&lt;SpectrumIdentificationItem passThreshold="true" rank="*1*"

peptide_ref="*looplink_p1*" experimentalMassToCharge= "569.7912"

calculatedMassToCharge="569.79054" chargeState="4" id="SII_1_1"&gt;

&lt;PeptideEvidenceRef peptideEvidence_ref="*looplink_p1_pep_evid*"/&gt;

&lt;cvParam accession="MS:1003329" cvRef="PSI-MS"

name="looplink spectrum identification item"/&gt;

&lt;cvParam accession="MS:1002545" cvRef="PSI-MS"

value="1.3111826921077734" name="xi:score"/&gt;

&lt;/SpectrumIdentificationItem&gt;
____

&lt;/SpectrumIdentificationResult&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>Figure 9.</strong> <strong>XML snippet including the encoding of an identification of an internally linked peptide.</strong> Within a &lt;SpectrumIdentificationResult&gt;, a &lt;SpectrumIdentificationItem&gt; element may be marked as referring to a looplink containing peptide by including the CV term “looplink spectrum identification item” (MS:1003329) CV term. This &lt;SpectrumIdentificationItem&gt; will refer to a &lt;Peptide&gt; containing both crosslink donor and crosslink acceptor modifications (as shown in Figure 5(iii)).</p>
</div>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_scores_and_thresholds">4. Scores and Thresholds</h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_introduction_2">4.1. Introduction</h3>
<div class="paragraph">
<p>This section addresses the encoding of error control procedures. This consists of encoding scores (§4.3) and the corresponding thresholds (§4.2) applied to those scores. The contents of this section are all optional; at the PSM level, providing threshold information and identifications that fall below the given significance threshold is encouraged.</p>
</div>
<div class="paragraph">
<p><em>“Depending on the intended purpose of the file, the file producer MAY wish to report a number of identifications that fall below the given significance threshold, for example to allow global statistical analyses to be performed which are not possible if only identifications passing the threshold are reported.”</em> (Section 7.4 of the main mzIdentML 1.3.0 specification document)</p>
</div>
<div class="paragraph">
<p>mzIdentML also provides the option not to encode the peptide spectrum matches that fell below the threshold applied. (§4.2)</p>
</div>
<div class="paragraph">
<p>The correspondence between scores and the applied thresholds is indicated by using the same CV term for both. That is, the same CV term will be used within the &lt;Threshold&gt; element and within either the related &lt;SpectrumIdentificationItem&gt; element or the related &lt;ProteinDetectionHypothesis&gt; element.</p>
</div>
<div class="paragraph">
<p>One specific type of score is an FDR (False Discovery Rate) score. Comments specific to FDR are in §4.4.</p>
</div>
<div class="paragraph">
<p>There are different points in the analysis at which thresholds may be applied <a href="https://paperpile.com/c/YrD2gH/kAbLF">[5</a>] <a href="https://paperpile.com/c/YrD2gH/ghaA0">[6</a>]. These correspond to different levels of consolidation at which analyses may be performed. Scores and thresholds are encoded differently in mzIdentML depending on the level of consolidation at which they were applied. For crosslinking studies encoded in mzIdentML, the possible levels are:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>crosslink containing PSM (also known as Crosslink Spectrum Match, CSM), see §4.3.1,</p>
</li>
<li>
<p>unique peptide-pair, see §4.3.2,</p>
</li>
<li>
<p>unique residue-pair, see §4.3.3,</p>
</li>
<li>
<p>protein-protein interaction (PPI) see §4.3.3.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Unique residue-pair and protein-protein interaction level scores are described in the same section as they are encoded using the same mechanism.</p>
</div>
<div class="paragraph">
<p>The example file <a href="https://github.com/HUPO-PSI/mzIdentML/blob/master/examples/1_3_0examples/crosslinking/scores_and_thresholds_1_3_0_draft.mzid"><span class="underline">scores_and_thresholds_1_3_0_draft.mzid</span></a> gives a simplified example containing two crosslinks and shows scores and thresholds applied at all four levels. Figures 10, 11, 14 and 15 are XML-snippets from that example file.</p>
</div>
<div class="paragraph">
<p>mzIdentML allows peptide-level scores to be associated with “unique peptides” (not arbitrary groups of peptides). There are three mutually exclusive definitions of “unique peptide”:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>“group PSMs by sequence” (MS:1002496);</p>
</li>
<li>
<p>“group PSMs by sequence with modifications” (MS:1002497);</p>
</li>
<li>
<p>“group PSMs by sequence with modifications and charge” (MS:1002498).</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>If peptide level (re)scoring is used, exactly one of these CV terms must be placed in the &lt;AdditionalSearchParams&gt; element to state the definition of “unique peptide” in use (see Section 5.2.7 of the main specification document). As these are mutually exclusive, an error control procedure which uses more than one definition of “unique peptide” cannot be fully captured by mzIdentML.</p>
</div>
</div>
<div class="sect2">
<h3 id="_thresholds">4.2. Thresholds</h3>
<div class="paragraph">
<p>Section 7.4 of the main mzIdentML specification document gives general guidance on the encoding of thresholds and what has passed them. Note that thresholds are encoded in two different places: in the &lt;SpectrumIdentificationProtocol&gt; element and in the &lt;ProteinDetectionProtocol&gt; element. In both cases, they are encoded using CV terms inside a &lt;Threshold&gt; element, see Figure 10.</p>
</div>
<div class="paragraph">
<p>The &lt;Threshold&gt; element inside &lt;SpectrumIdentificationProtocol&gt; gives the thresholds associated with &lt;SpectrumIdentificationItem&gt; elements. These thresholds apply at the crosslinked PSM level and at a unique peptide level.</p>
</div>
<div class="paragraph">
<p>Analogously, the &lt;Threshold&gt; element inside &lt;ProteinDetectionProtocol&gt; includes the thresholds associated with &lt;ProteinDetectionHypothesis&gt; elements. These thresholds apply at the unique residue-pair level and PPI level.</p>
</div>
<div class="paragraph">
<p>The elements &lt;SpectrumIdentificationItem&gt; and &lt;ProteinDetectionHypothesis&gt; have a mandatory Boolean attribute <em>passThreshold</em> that allows a file producer to indicate that an identification has passed the given thresholds or that it has been manually validated.</p>
</div>
<div class="paragraph">
<p>The <em>passThreshold</em> attribute of &lt;SpectrumIdentificationItem&gt; relates only to the passing of PSM-level thresholds (see Section 5.2.7 of the main specification document, final paragraph therein).</p>
</div>
<div class="paragraph">
<p>To enable additional thresholding at the peptide-pair level in the context of crosslinking, a new CV term is required for all PSMs (“peptide-pair passes threshold”, MS:1003339) as shown in Figure 11. This is similar to the general guidance on peptide level thresholds given in Section 5.2.7 of the main specification document.</p>
</div>
<div class="paragraph">
<p>The <em>passThreshold</em> attribute of &lt;ProteinDetectionHypothesis&gt; only relates to the presence or absence of proteins, it is not directly related to the identification of crosslinks. Whether or not residue-pairs or PPIs have passed significance thresholds is encoded by including the new CV terms “residue-pair passes threshold” (MS:1003340) or “protein-protein interaction passes threshold” (MS:1003341) in the &lt;ProteinDetectionHypothesis&gt; element. The values of these CV terms include an identifier that associates them with a specific residue pair or PPI, see Figure 14.</p>
</div>
<div class="paragraph">
<p>At each level of consolidation there may be multiple scores. Therefore, for each level there is a mechanism for encoding whether the identification passed when all scores are considered:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>for PSM-level identifications this is the <em>passThreshold</em> attribute of &lt;SpectrumIdentificiationItem&gt;;</p>
</li>
<li>
<p>at peptide-pair level it is the “peptide-pair passes threshold” (MS:1003339) CV term;</p>
</li>
<li>
<p>at residue-pair level it is the “residue-pair passes threshold” (MS:1003340) CV term;</p>
</li>
<li>
<p>and for PPIs it is the “protein-pair passes threshold” (MS:1003341) CV term.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>If the file producer does not want to indicate that thresholds have been set, all identification elements (&lt;SpectrumIdentificationItem&gt; and &lt;ProteinDetectionHypothesis&gt;) MUST have the attribute passThreshold = “true" and the “no threshold" CV term should be provided within the &lt;SpectrumIdentificationProtocol&gt; and &lt;ProteinDetectionProtocol&gt; (Section 7.4 of the main mzIdentML 1.3.0 specification document). In this case, the new “residue-pair passes threshold" (MS:1003340) and “protein-protein interaction passes threshold" (MS:1003341) CV terms can be omitted.</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;AnalysisProtocolCollection &gt;

&lt;SpectrumIdentificationProtocol analysisSoftware_ref="xiFDR_id" id="SearchProtocol_1_17022"&gt;

&lt;SearchType&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1001083" name="ms-ms search"/&gt;

&lt;/SearchType&gt;

&lt;AdditionalSearchParams&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1001211" name="parent mass type mono"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002494" name="crosslinking search"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1001256" name="fragment mass type mono"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002490" name="peptide-level scoring"/&gt;

&lt;cvParam cvRef="PSI-MS" accession = "MS:1002496" name="group PSMs by sequence"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003343"

____
name="FDR applied separately to self crosslinks

and protein heteromeric crosslinks" /&gt;
____

&lt;cvParam accession="MS:1001118" name="param: b ion" cvRef="PSI-MS" /&gt;

&lt;cvParam accession="MS:1001262" name="param: y ion" cvRef="PSI-MS" /&gt;

&lt;/AdditionalSearchParams&gt;

&lt;ModificationParams .../&gt;

&lt;Enzymes .../&gt;

&lt;FragmentTolerance .../&gt;

&lt;ParentTolerance .../&gt;

&lt;Threshold&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003337"

____
name="crosslinked PSM-level global FDR" value="0.05"/&gt;
____

&lt;cvParam cvRef="PSI-MS" accession="MS:1003338"

____
name="peptide-pair sequence-level global FDR" value="0.05"/&gt;
____

&lt;/Threshold&gt;

&lt;/SpectrumIdentificationProtocol&gt;

&lt;ProteinDetectionProtocol analysisSoftware_ref="xiFDR_id" id="pdp1"&gt;

&lt;Threshold&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677"

____
name="residue-pair-level global FDR" value="0.05"/&gt;
____

&lt;cvParam cvRef="PSI-MS" accession="MS:1002676"

____
name="protein-pair-level global FDR" value="0.05"/&gt;
____

&lt;/Threshold&gt;

&lt;/ProteinDetectionProtocol&gt;

&lt;/AnalysisProtocolCollection&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>Figure 10. XML snippet showing the thresholds applied at all four levels of consolidation.</strong> These are - PSM, peptide-pair, residue pair and PPI. The CV terms MS:1002490 and MS:1002496 are required to enable peptide level rescoring (mzIdentML main specification Section 5.2.7) and to state the definition of ‘unique peptide’ being used.</p>
</div>
</div>
<div class="sect2">
<h3 id="_scores">4.3. Scores</h3>
<div class="sect3">
<h4 id="_match_level_scores">4.3.1. Match Level Scores</h4>
<div class="paragraph">
<p>Match level scores are stored in &lt;SpectrumIdentificationItem&gt; elements.</p>
</div>
<div class="paragraph">
<p>The CV mapping rules for &lt;SpectrumIdentificationItem&gt; are straightforward – there is only one, which states ‘MAY supply a child term of <a href="https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1001405"><span class="underline">MS:1001405 (spectrum identification result details)</span></a> one or more times’.</p>
</div>
<div class="paragraph">
<p>CV terms to encode match level scores must therefore be children of <a href="https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1001405"><span class="underline">MS:1001405</span></a> in the CV’s “is a” hierarchy.</p>
</div>
<div class="paragraph">
<p>Those which also meet the CV mapping rules for the &lt;Threshold&gt; element can also be used to encode the Threshold applied.</p>
</div>
<div class="paragraph">
<p>See Section 7.11 of the main mzIdentML document for guidance specific to PSM-level scores for identifications based on multiple spectra.</p>
</div>
</div>
<div class="sect3">
<h4 id="_peptide_level_scores">4.3.2. Peptide Level Scores</h4>
<div class="paragraph">
<p>Peptide level scores are also stored in &lt;SpectrumIdentificationItem&gt; elements and everything in §4.3.1 also applies here.</p>
</div>
<div class="paragraph">
<p>Section 5.2.7 of the main mzIdentML specification document describes the encoding of peptide-level scores and statistical measures. The encoding of crosslinking results MAY also be combined with the peptide-level re-scoring mechanism described there, but with specific CV terms for scores associated with crosslinked peptides rather than PSM-level terms (as stated in Section 5.2.7 of main specification document).</p>
</div>
<div class="paragraph">
<p>Where needed, new CV terms for search specific scores of crosslinked peptides should be added as a child of (i.e. with an “is a” relationship to) the CV term “interaction score derived from crosslinking” (MS:1002664).</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;SpectrumIdentificationList id="SII_LIST_1_1" &gt;

&lt;SpectrumIdentificationResult spectrumID="index=26630" spectraData_ref="SD_17022_recal_B210619_02_Lumos_ZC_CO_190_D2I_SDA-WT1.mgf" id="SIR_1"&gt;

&lt;SpectrumIdentificationItem chargeState="5" experimentalMassToCharge="1135.3259479607323"

____
calculatedMassToCharge="1135.3254335427703"

peptide_ref="16734061838_ISDKRAPSQGGLENEGVFEELLR_16734063165_GAEDEEEEEDVGFEQNFEEMLESVTR_4_9_p1" rank="1"

passThreshold="false" id="SII_1_1"&gt;
____

&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_pep_16734063165"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002511" name="crosslink spectrum identification item" value="1"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002545" name="xi:score" value="25.929927957127177"/&gt;

_&lt;!-- crosslinked PSM level global FDR --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1003337 name="crosslinked PSM-level global FDR" value="0.06"/&gt;

_&lt;!-- peptide pair global FDR --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1002520" value="GAEDEEEEEDVGFEQNFEEMLESVTR-ISDKRAPSQGGLENEGVFEELLR"

____
name="peptide group ID"/&gt;
____

&lt;cvParam cvRef="PSI-MS" accession="MS:1003338" name="peptide-pair sequence-level global FDR" value="0.06"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003339" name="peptide-pair passes threshold" value="false"/&gt;

_&lt;!-- residue pair ref value="1.b" --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1003344" value="11.b" name="Residue-pair ref"/&gt;

&lt;/SpectrumIdentificationItem&gt;

&lt;SpectrumIdentificationItem chargeState="5" experimentalMassToCharge="1135.3259479607323"

calculatedMassToCharge="1135.3254335427703"

____
peptide_ref="16734061838_ISDKRAPSQGGLENEGVFEELLR_16734063165_GAEDEEEEEDVGFEQNFEEMLESVTR_4_9_p0" rank="1"

passThreshold="false" id="SII_1_2"&gt;
____

&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_pep_16734061838"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002511" name="crosslink spectrum identification item" value="1"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002545" name="xi:score" value="25.929927957127177"/&gt;

_&lt;!-- crosslinked PSM level global FDR --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1003337" name="crosslinked PSM-level global FDR" value="0.06"/&gt;

_&lt;!-- peptide pair global FDR --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1002520" value="GAEDEEEEEDVGFEQNFEEMLESVTR-ISDKRAPSQGGLENEGVFEELLR"

____
name="peptide group ID"/&gt;
____

&lt;cvParam cvRef="PSI-MS" accession="MS:1003338" name="peptide-pair sequence-level global FDR" value="0.06"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003339" name="peptide-pair passes threshold" value="false"/&gt;

_&lt;!-- residue pair ref value="11.a" --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1003344" value="11.a" name="Residue-pair ref"/&gt;

&lt;/SpectrumIdentificationItem&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1000797" name="peak list scans" value="40560"/&gt;

&lt;/SpectrumIdentificationResult&gt;

&lt;SpectrumIdentificationResult spectrumID="index=23414" spectraData_ref="SD_17022_recal_B210619_04_Lumos_ZC_CO_190_D2I_SDA-WT3.mgf" id="SIR_2"&gt;

&lt;SpectrumIdentificationItem chargeState="6" experimentalMassToCharge="752.7466713415814"

____
calculatedMassToCharge="752.41371619677"

peptide_ref="16734068348_TAAPTVCcmLLVLGQADKVLEEVDWLIKR_16734057553_SCcmKDLQILQASK_18_1_p1" rank="1"

passThreshold="true" id="SII_2_1"&gt;
____

&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_pep_16734057553"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002511" name="crosslink spectrum identification item" value="2"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002545" name="xi:score" value="21.55734182309742"/&gt;

_&lt;!-- crosslinked PSM level global FDR --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1003337" name="crosslinked PSM-level global FDR" value="0.03"/&gt;

_&lt;!-- peptide pair global FDR --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1002520" value="SCKDLQILQASK-TAAPTVCLLVLGQADKVLEEVDWLIKR" name="peptide group ID"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003338" name="peptide-pair sequence-level global FDR" value="0.03"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003339" name="peptide-pair passes threshold" value="true"/&gt;

_&lt;!-- residue pair ref value="22.b" --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1003344" value="22.b" name="Residue-pair ref"/&gt;

&lt;/SpectrumIdentificationItem&gt;

&lt;SpectrumIdentificationItem chargeState="6" experimentalMassToCharge="752.7466713415814"

____
calculatedMassToCharge="752.41371619677"

peptide_ref="16734068348_TAAPTVCcmLLVLGQADKVLEEVDWLIKR_16734057553_SCcmKDLQILQASK_18_1_p0" rank="1"

passThreshold="true" id="SII_2_2"&gt;
____

&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_pep_16734068348"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002511" name="crosslink spectrum identification item" value="2"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002545" name="xi:score" value="21.55734182309742"/&gt;

_&lt;!-- crosslinked PSM level global FDR --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1003337" name="crosslinked PSM-level global FDR" value="0.03"/&gt;

_&lt;!-- peptide pair global FDR --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1002520" value="SCKDLQILQASK-TAAPTVCLLVLGQADKVLEEVDWLIKR" name="peptide group ID"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003338" name="peptide-pair sequence-level global FDR" value="0.03"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003339" name="peptide-pair passes threshold" value="true"/&gt;

_&lt;!-- residue pair ref value="22.a" --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1003344" value="22.a" name="Residue-pair ref"/&gt;

&lt;/SpectrumIdentificationItem&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1000797" name="peak list scans" value="38065"/&gt;

&lt;/SpectrumIdentificationResult&gt;

&lt;/SpectrumIdentificationList&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>Figure 11.</strong> <strong>XML snippet including the encoding of scores for PSM-level matches and peptide pairs.</strong> These are encoded inside &lt;SpectrumIdentificationItem&gt; elements. “peptide-pair passes threshold” (MS:1003339) would become relevant if there was more than one score for that peptide pair (sharing the same “peptide group ID”), it states whether the peptide pair passed when all scores and thresholds are considered. This is analogous to the <em>passThreshold</em> attribute of &lt;SpectrumIdentificationItem&gt; elements for PSM-level scores.</p>
</div>
</div>
<div class="sect3">
<h4 id="_interaction_level_scores_unique_residue_pairs_and_ppi">4.3.3. Interaction Level Scores (Unique Residue-Pairs and PPI)</h4>
<div class="paragraph">
<p>mzIdentML uses the same mechanism to encode scores for interactions at both the unique residue-pair level and protein-protein interaction level. This encoding was put forward in mzIdentML 1.2.0 and remains unchanged. Where a residue-pair level score gives the position of the crosslinked residue, a protein-protein interaction (PPI) score will instead have the value ‘null’.</p>
</div>
<div class="paragraph">
<p>mzIdentML encodes these with the same mechanism it uses to address the protein inference problem, that is, within &lt;ProteinAmbiguityGroup&gt; elements. More specifically, these scores go inside &lt;ProteinDetectionHypothesis&gt; elements. All such scores must therefore meet the CV mapping rules of &lt;ProteinDetectionHypothesis&gt; elements.</p>
</div>
<div class="paragraph">
<p>As the encoding of interaction scores uses &lt;ProteinAmbiguityGroup&gt; elements, the guidance in Section 5.2.1 (Protein grouping encoding) of the main specification also applies here and MUST be followed. This means that ambiguity about which protein a crosslinked peptide came from must be reflected in how the &lt;ProteinDetectionHypothesis&gt; elements containing the score are assigned to &lt;ProteinAmbiguityGroup&gt; elements, see Figure 12.</p>
</div>
<div class="imageblock">
<div class="content">
<img src="img/crosslinking_ext/image1.png" alt="image" width="624" height="396">
</div>
</div>
<div class="paragraph">
<p><strong>Figure 12. Ambiguity at PPI level.</strong> Ambiguity regarding which protein is crosslinked (protein inference problem) MUST be reflected in how the &lt;ProteinDetectionHypothesis&gt; elements containing interaction scores are assigned to &lt;ProteinAmbiguityGroup&gt; elements, see Section 5.2.1 (Protein grouping encoding) of the main specification. Shown here with PPI level scores.</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;ProteinAmbiguityGroup id="PAG_0"&gt;

&lt;ProteinDetectionHypothesis dBSequence_ref="dbseq_P02771" passThreshold="true" id="PAG_0_PDH_0"&gt;

&lt;PeptideHypothesis peptideEvidence_ref="pepevid_psm252637369_pep54601081"&gt;

&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_1_1"/&gt;

&lt;/PeptideHypothesis&gt;

...

&lt;cvParam cvRef="PSI-MS" accession="MS:1002676" name="protein-pair-level global FDR" value="100.b:null:0.001:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="106.b:146:0.0294:true"/&gt;

&lt;/ProteinDetectionHypothesis&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002415" name="protein group passes threshold" value="true"/&gt;

&lt;/ProteinAmbiguityGroup&gt;

&lt;ProteinAmbiguityGroup id="PAG_1"&gt;

&lt;ProteinDetectionHypothesis dBSequence_ref="dbseq_P02768" passThreshold="true" id="PAG_1_PDH_0"&gt;

&lt;PeptideHypothesis peptideEvidence_ref="pepevid_psm252637369_pep54600650"&gt;

&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_1_2"/&gt;

&lt;/PeptideHypothesis&gt;

&lt;PeptideHypothesis peptideEvidence_ref="pepevid_psm252633422_pep54604445_protP02768-A_target_52"&gt;

&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_2_1"/&gt;

&lt;/PeptideHypothesis&gt;

....

&lt;cvParam cvRef="PSI-MS" accession="MS:1002676" name="protein-pair-level global FDR" value="100.a:null:0.001:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="106.a:436:0.0294:true"/&gt;

&lt;/ProteinDetectionHypothesis&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002415" name="protein group passes threshold" value="true"/&gt;

&lt;/ProteinAmbiguityGroup&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>Figure 13. XML snippet showing the CV terms "protein-pair-level global FDR" (MS:1002676) and "residue-pair-level global FDR" (MS:1002677).</strong></p>
</div>
<div class="paragraph">
<p>The XML snippet in Figure 13 shows the "protein-pair-level global FDR" (MS:1002676) and "residue-pair-level global FDR" (MS:1002677) CV terms, these CV terms have the parent CV term “interaction score derived from crosslinking” (MS:1002664). Where needed, new CV terms for search specific interaction scores should be added as children of the CV term “interaction score derived from crosslinking” (MS:1002664).</p>
</div>
<div class="paragraph">
<p>These CV terms must have a paired structure of int_ID.a|b:POS|null:SCORE_OR_VALUE:PASS_THRESHOLD</p>
</div>
<div class="paragraph">
<p><strong>1 2 3 4</strong></p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p>The two partners in the interaction share the same integer value for ID followed by a or b. If there is ambiguity in protein identification, two different ProteinDetectionHypothesis (PDH) elements, within the same ProteinAmbiguityGroup (PAG), MAY share the same ID and suffix (a or b). A given identifier (integer and suffix) value MUST NOT be used in more than one PAG.</p>
</li>
<li>
<p>The export software MAY indicate the general position of the interaction (potentially taking on board multiple pairs of crosslinked peptides), with respect to the protein sequence – using a 1-based counting system. A “null” MAY be used if the export software does not wish to include a value.</p>
</li>
<li>
<p>The score or statistical value for the interaction.</p>
</li>
<li>
<p>“true” or “false” to indicate whether the score or value has passed a reported threshold in the file. If no threshold is defined, then PASS_THRESHOLD is always true.</p>
</li>
</ol>
</div>
<div class="paragraph">
<p>The first “int_ID” part of the value MUST be identical/shared between interaction level scores if they refer to the same residue pair or PPI.</p>
</div>
<div class="paragraph">
<p>The new CV term “Residue pair ref” (MS:1003344) SHOULD be included within &lt;SpectrumIdentificationItem&gt; elements to indicate that these are the spectra which supported the linking of a specific residue pair. The value of the new “Residue pair ref” CV term is the “<em>int_ID.a|b</em>” part of the values, see Figure 11. More than one “Residue pair ref” (MS:1003344) CV term (with different values) can be included in a single &lt;SpectrumIdentificationItem&gt; element if it has been taken as evidence for more than one linked residue pair.</p>
</div>
<div class="paragraph">
<p>It is not a requirement that the &lt;SpectrumIdentificationItem&gt; elements containing “Residue pair ref” (MS:1003344) place the linkage sites at the same position in the peptide as the residue-pair they are claiming to support. Hence, analyses which utilise link site reassignment can be encoded in mzIdentML. (Some analyses may look at a collection of spectra to reach a conclusion about where the linkage site was, therefore some identifications may end up supporting a residue-pair that places the linkage site at a different position from where they themselves did).</p>
</div>
<div class="paragraph">
<p>References to supporting &lt;SpectrumIdentificationItem&gt; elements for PPIs are given by the &lt;SpectrumIdentificationItemRef&gt; elements inside &lt;PeptideHypothesis&gt; elements in &lt;ProteinDectectionHypothesis&gt;. This performs the equivalent role as the “Residue pair ref” (MS:1003344) CV term does for residue-pair interactions.</p>
</div>
<div class="paragraph">
<p>See Figure 14 for an example of encoding residue-pair and PPI level scores.</p>
</div>
<div class="paragraph">
<p>Positional ambiguity of the residues linked can be encoded by repeating the score CV terms, keeping the same identifier (integer and suffix) , for each of the positional alternatives, see Figure 15. This may be due to ambiguity regarding the position of the peptide in the protein sequences (protein inference problem) or ambiguity regarding the linkage site in the peptide.</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;ProteinDetectionList id="PDL_1" &gt;

&lt;ProteinAmbiguityGroup id="PAG_0"&gt;

&lt;ProteinDetectionHypothesis dBSequence_ref="dbseq_ggFANCI_target" passThreshold="true" id="PAG_0_PDH_0"&gt;

&lt;PeptideHypothesis peptideEvidence_ref="pepevid_pep_16734063165"&gt;

&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_1_1"/&gt;

&lt;/PeptideHypothesis&gt;

&lt;PeptideHypothesis peptideEvidence_ref="pepevid_pep_16734057553"&gt;

&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_2_1"/&gt;

&lt;/PeptideHypothesis&gt;

&lt;PeptideHypothesis peptideEvidence_ref="pepevid_pep_16734068348"&gt;

&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_2_2"/&gt;

&lt;/PeptideHypothesis&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002403" name="group representative"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1001593" name="group member with undefined relationship OR ortholog protein"/&gt;

_&lt;!-- forms a protein heteromeric PPI with its partner 10.a in PAG_1_PDH_0 --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1002676" name="protein-pair-level global FDR" value="10.b:null:0.059:false"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003341" name="protein-protein interaction passes threshold" value="10:false"/&gt;

_&lt;!-- forms a self PPI with its partner 20.b in PAG_0_PDH_0 --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1002676" name="protein-pair-level global FDR" value="20.a:null:0.030:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002676" name="protein-pair-level global FDR" value="20.b:null:0.030:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003341" name="protein-protein interaction passes threshold" value="20:true"/&gt;

_&lt;!-- forms a protein heteromeric crosslink with its partner 11.a in PAG_1_PDH_0 --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="11.b:697:0.06:false"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003340" name="residue-pair passes threshold" value="11:false"/&gt;

_&lt;!-- forms a self crosslink with its partner 22.b in PAG_0_PDH_0 --&gt;_

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="22.a:1095:0.01:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="22.b:339:0.01:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003340" name="residue-pair passes threshold" value="22:true"/&gt;

&lt;/ProteinDetectionHypothesis&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002415" name="protein group passes threshold" value="true"/&gt;

&lt;/ProteinAmbiguityGroup&gt;

&lt;ProteinAmbiguityGroup id="PAG_1"&gt;

&lt;ProteinDetectionHypothesis dBSequence_ref="dbseq_ggFANCD2_target" passThreshold="true" id="PAG_1_PDH_0"&gt;

&lt;PeptideHypothesis peptideEvidence_ref="pepevid_pep_16734061838"&gt;

&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_1_2"/&gt;

&lt;/PeptideHypothesis&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002403" name="group representative"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1001593" name="group member with undefined relationship OR ortholog protein"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002676" name="protein-pair-level global FDR" value="10.a:null:059:false"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003341" name="protein-protein interaction passes threshold" value="10:false"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="11.a:36:0.06:false"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003340" name="residue-pair passes threshold" value="11:false"/&gt;

&lt;/ProteinDetectionHypothesis&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002415" name="protein group passes threshold" value="true"/&gt;

&lt;/ProteinAmbiguityGroup&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002404" name="count of identified proteins" value="2"/&gt;

&lt;/ProteinDetectionList&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>Figure 14. XML snippet including the encoding of scores for residue-pairs and PPIs.</strong> These are encoded inside &lt;ProteinDetectionHypothesis&gt; elements. The CV terms "residue-pair passes threshold" (MS:1003340) and “protein-pair passes threshold” (MS:1003341) would become relevant if there was more than one score for those residue or protein pairs (sharing the same integer id part of their value). These are analogous to the <em>passThreshold</em> attribute of &lt;SpectrumIdentificationItem&gt; elements.</p>
</div>
<div class="paragraph">
<p>(i)</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;ProteinAmbiguityGroup id="PAG_0"&gt;

&lt;!-- example of both peptide ambiguity (classical protein inference) and site ambiguity with in a peptide --&gt;

&lt;ProteinDetectionHypothesis dBSequence_ref="dbseq_A_target" passThreshold="true" id="PAG_0_PDH_0"&gt;

...

&lt;!-- each possible linksite in the originating peptide is referenced here as a possible residue pair--&gt;

&lt;!-- the first two have the same score as there is no fragmentation distinguishing the two neighbouring residues--&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="22.a:1095:0.01:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="22.a:1096:0.01:true"/&gt;

&lt;!-- the third residue would be a possible linksite, but there is some fragments speaking in favour of the first two, therefore this one has a lower score and hence a worse FDR--&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="22.a:1091:0.09:false"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003341" name="residue-pair passes threshold" value="22:true"/&gt;

&lt;/ProteinDetectionHypothesis&gt;

&lt;ProteinDetectionHypothesis dBSequence_ref="dbseq_B_target" passThreshold="true" id="PAG_0_PDH_1"&gt;

...

&lt;!-- (all) peptide(s) for site a could also come from a different protein--&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="22.a:295:0.01:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="22.a:296:0.01:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="22.a:291:0.09:false"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003341" name="residue-pair passes threshold" value="22:true"/&gt;

&lt;/ProteinDetectionHypothesis&gt;

&lt;/ProteinAmbiguityGroup&gt;

&lt;ProteinAmbiguityGroup id="PAG_1"&gt;

&lt;ProteinDetectionHypothesis dBSequence_ref="dbseq_C_target" passThreshold="true" id="PAG_1_PDH_0"&gt;

...

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="22.b:339:0.01:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003341" name="residue-pair passes threshold" value="22:true"/&gt;

&lt;/ProteinDetectionHypothesis&gt;

&lt;/ProteinAmbiguityGroup&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p>(ii)</p>
</div>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;ProteinAmbiguityGroup id="PAG_0"&gt;

&lt;ProteinDetectionHypothesis dBSequence_ref="dbseq_B_target" passThreshold="true" id="PAG_0_PDH_1"&gt;

...

&lt;!-- peptide has two possible link sites and is present in two places in protein B--&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="23.a:1095:0.01:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="23.a:1091:0.09:false"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="23.a:295:0.01:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="23.a:291:0.09:false"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="23.b:339:0.01:true"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003341" name="residue-pair passes threshold" value="23:true"/&gt;

&lt;/ProteinDetectionHypothesis&gt;

&lt;/ProteinAmbiguityGroup&gt;</code></pre>
</div>
</div>
<div class="paragraph">
<p><strong>Figure 15. XML snippet including the encoding of positional ambiguity of residue pairs.</strong> In panel (i), residue-pair 22 is a protein heteromeric crosslink where the “a” end of the crosslink is ambiguous between two proteins and there are three possible positions of the crosslink in peptide “a”. In panel (ii), residue pair 23 is a self link but there is ambiguity about where peptide “a” came from within that protein (two possible positions) and two possible link sites in peptide “a”, giving a total of four possible residues.</p>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_fdr_specific_comments">4.4. FDR Specific Comments</h3>
<div class="paragraph">
<p>Section 7.5 of the main mzIdentML 1.3.0 specification document (‘Using decoy databases to set different thresholds of false discovery rate’) states that:</p>
</div>
<div class="ulist">
<ul>
<li>
<p><em>A &lt;SpectrumIdentificationItem&gt; can be marked as matching a decoy peptide using the isDecoy attribute of the referenced &lt;PeptideEvidence&gt; element, thus allowing the false discovery rate to be calculated across an entire file.</em></p>
</li>
<li>
<p><em>Implementers of the format SHOULD report the peptide identifications</em> [including those of decoy peptides] <em>that pass the threshold they wish to communicate to a consumer of the data.</em></p>
</li>
<li>
<p><em>It is not guaranteed that a consumer of an mzIdentML file will be able to calculate other results, or global false discovery rates, using different thresholds from the reported information, although in some circumstances they may be able to, for example, if a user reports the complete output of a search against a target and decoy search.</em></p>
</li>
</ul>
</div>
<div class="paragraph">
<p>CV terms exist for FDR scores at each level of consolidation:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>"crosslinked PSM-level global FDR" (MS:1003337)</p>
</li>
<li>
<p>“peptide-pair sequence-level global FDR” (MS:1003339)</p>
</li>
<li>
<p>“residue-pair-level global FDR” (MS:1002677)</p>
</li>
<li>
<p>“protein-pair-level global FDR” (MS:1002676)</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>A new CV term “FDR applied separately to self crosslinks and protein heteromeric crosslinks” (MS:1003343) has been introduced to encode whether self crosslinks (crosslinks between peptides within one protein sequence) and protein heteromeric crosslinks (crosslinks between distinct protein sequences) were grouped separately for FDR analysis <a href="https://paperpile.com/c/YrD2gH/kAbLF">[5</a>]. This CV term goes within the &lt;AdditionalSearchParameters&gt; element (see Figure 3).</p>
</div>
<div class="paragraph">
<p>The value of “FDR applied separately to self crosslinks and protein heteromeric crosslinks” (MS:1003343) is a boolean, stating whether or not this happened. This CV term SHOULD be supplied. If it is omitted then it is unspecified whether self and heteromeric links were grouped separately for analysis (there is no default value).</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_unsupported_use_cases_and_future_directions">5. Unsupported Use Cases and Future Directions</h2>
<div class="sectionbody">
<div class="paragraph">
<p>The two unsupported crosslinking product types shown in Figure 1 are: crosslinkers with more than two reactive groups and higher order crosslinks (arbitrarily many peptides identified with many crosslinks between them).</p>
</div>
<div class="paragraph">
<p>Crosslinkers with more than two reactive groups <a href="https://paperpile.com/c/YrD2gH/8J3I">[7</a>] cannot be represented using the current model for two reasons. First, the donor/acceptor mechanism for crosslinked Peptides in &lt;SequenceCollection&gt; elements (§3.2) restricts the number of reactive groups to two. Second, there can be at most two crosslinked &lt;SpectrumidentificationItem&gt; elements, each of which references an identified peptide within a &lt;SpectrumIdentificationResult&gt; (§3.3).</p>
</div>
<div class="paragraph">
<p>In the case of higher order crosslinks, the specification already allows the encoding of this in the &lt;Peptide&gt; elements within &lt;SequenceCollection&gt; (or rather nothing forbids it), see §3.2. It is only the restriction of there being at most two crosslinked &lt;SpectrumIdentificationItem&gt; elements that share the same value within a &lt;SpectrumIdentificationResult&gt; that prevents the encoding of higher order crosslinks.</p>
</div>
<div class="paragraph">
<p>It would be possible to support higher order crosslinks by allowing <em>n</em> crosslinked &lt;SpectrumidentificationItem&gt; elements within a &lt;SpectrumIdentificationResult&gt;. This would pose some problems for the validation of the documents. These would not be insurmountable because the number of peptides that are crosslinked could be derived from &lt;Peptide&gt; elements in &lt;SequenceCollection&gt;. However, this would make validation significantly more complex, to alleviate this an additional CV term in peptide that links peptides as part of a "crosslink-group" independent of the crosslinker could be introduced.</p>
</div>
<div class="paragraph">
<p>In the case of crosslinkers with more than two reactive groups and the identification of higher order crosslinks, there was no demand for supporting these use cases at this point in time and so, for the sake of simplicity and minimal changes, they are still not supported.</p>
</div>
<div class="paragraph">
<p>This remains an open question for future versions of the specification. There are other use cases in which <em>n</em> &lt;SpectrumIdentificationItem&gt; elements need to be associated. Characterisation of antibodies or other multi-chain proteins that contain complex patterns of disulfide bonds (representing endogenous crosslinks) by top-down mass spectrometry would be an example of this.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_appendix_i_comparison_of_rules_for_crosslink_donor_and_crosslink_acceptor_depending_on_context">6. Appendix I. Comparison of rules for “crosslink donor” and “crosslink acceptor” depending on context</h2>
<div class="sectionbody">
<div class="paragraph">
<p>The CV terms “crosslink donor” (MS:1002509) and “crosslink acceptor” (MS:1002510) are used in two different contexts:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>/MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/ ModificationParams/SearchModification – encoding the modifications searched for (including specificity, see §3.1.2);</p>
</li>
<li>
<p>/MzIdentML/SequenceCollection/Peptide/Modification - encoding the modifications of crosslinked peptides (§3.2).</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Table 1 summarises the commonalities and differences between the rules governing their use in these two contexts.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Element &lt;SearchModification&gt; (see §3.1.2)</strong></th>
<th class="tableblock halign-left valign-top"><strong>Element &lt;Peptide&gt;&lt;Modification&gt; (see §3.2)</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Two or more</strong> &lt;SearchModification&gt; elements are needed to describe the specificity of a single crosslinker with two reactive groups. All of the donor and acceptor CV terms contained in these <strong>MUST</strong> share a unique identifier in their value slot.</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A unique identifier linking these <strong>two</strong> Modification elements together <strong>MUST</strong> be in the value slot. (Thereby excluding the representation of trimeric crosslinkers.)</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">The choice of which end is the ‘donor’ and which end is the ‘acceptor’ is arbitrary.</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">If the CV term “search modification id ref” (MS:1003393) is being used then the crosslink donor <strong>MUST</strong> be chosen to match the end marked as the donor in the corresponding &lt;SearchModification&gt; elements, see §3.2.2. If that CV term is not used, or if the preceding rule does not unambiguously define which end to mark as donor (e.g. because the crosslinker is symmetrical) then the export software <strong>SHOULD</strong> use the following rules to choose the crosslink donor as the: longer peptide, then higher peptide neutral mass, then alphabetical order.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">The element(s) containing the crosslink donor CV term <strong>MUST</strong> have their mass delta attribute = the mass gained from the crosslink reagent.</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">The element(s) containing the crosslink acceptor CV term <strong>MUST</strong> have their mass delta attribute = 0.</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Both</strong> crosslink donor and crosslink acceptor <strong>MUST</strong> have a suitably sourced cvParam for the crosslink.</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">The crosslink donor peptide’s Modification element <strong>MUST</strong> have a suitably sourced cvParam for the crosslink.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">The crosslink acceptor peptide’s Modification element <strong>MUST NOT</strong> have a cvParam for the reagent.</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Table 1.</strong> The rules governing the use of “crosslink donor” (MS:1002509) and “crosslink acceptor” (MS:1002510) differ depending on the context.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_appendix_ii_example_encodings_of_crosslinker_reagents_as_searchmodification_elements">7. Appendix II. Example encodings of crosslinker reagents as &lt;SearchModification&gt; elements</h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_bissulfosuccinimidyl_suberate_bs3">7.1. Bis(sulfosuccinimidyl) suberate (BS3)</h3>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;SpectrumIdentificationProtocol&gt;

...

&lt;ModificationParams&gt;

____
&lt;SearchModification fixedMod="false" massDelta="138.06808" residues="S T Y K"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="BS3_donor"/&gt;

&lt;cvParam cvRef="XLMOD" accession="XLMOD:02000" name="BS3"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002509" name="crosslink donor"

value="0"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="138.06808" residues="."&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002057"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="BS3_donor_n_term"/&gt;

&lt;cvParam cvRef="XLMOD" accession="XLMOD:02000" name="BS3"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink donor"

value="0" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="0.0" residues="S T Y K"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="BS3_acceptor"/&gt;

&lt;cvParam cvRef="XLMOD" accession="XLMOD:02000" name="BS3"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor"

value="0"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="0.0" residues="."&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002058"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="BS3_acceptor_n_term"/&gt;

&lt;cvParam cvRef="XLMOD" accession="XLMOD:02000" name="BS3"/&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor"

value="0" /&gt;

&lt;/SearchModification&gt;
____

&lt;/ModificationParams&gt;

...

&lt;/SpectrumIdentificationProtocol&gt;</code></pre>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_1_ethyl_3_3_dimethylaminopropylcarbodiimide_hydrochloride_edc">7.2. 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride (EDC)</h3>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;SpectrumIdentificationProtocol&gt;

...

&lt;ModificationParams&gt;

____
&lt;SearchModification fixedMod="false" massDelta="-18.010565" residues="K"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="EDC_donor"/&gt;

&lt;cvParam accession="UNIMOD:2018" name="Xlink:EDC" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002509" name="crosslink donor"

value="1" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="-18.010565" residues="*.*"&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002057"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="EDC_donor_n_term"/&gt;

&lt;cvParam accession="UNIMOD:2018" name="Xlink:EDC" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink donor"

value="1" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="0.0" residues="D E"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="EDC_acceptor"/&gt;

&lt;cvParam accession="UNIMOD:2018" name="Xlink:EDC" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor"

value="1" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="0.0" residues="*.*"&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002058"

name="modification specificity protein C-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="EDC_acceptor_c_term"/&gt;

&lt;cvParam accession="UNIMOD:2018" name="Xlink:EDC" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor"

value="1" /&gt;

&lt;/SearchModification&gt;
____

&lt;/ModificationParams&gt;

...

&lt;/SpectrumIdentificationProtocol&gt;</code></pre>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_nhs_diazirine_succinimidyl_44_azipentanoate_sda">7.3. (NHS-Diazirine) succinimidyl 4,4'-azipentanoate (SDA)</h3>
<div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;SpectrumIdentificationProtocol&gt;

...

&lt;ModificationParams&gt;

____
&lt;SearchModification fixedMod="false" massDelta="100.05243" residues="K S T Y"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="SDA_monolink_W"/&gt;

&lt;cvParam accession="UNIMOD:2000" cvRef="UNIMOD" name="Xlink:SDA"/&gt;
____

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code"

____
value="W" cvRef="PSI-MS" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="100.05243" residues="."&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002057"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="SDA_monolink_W_n_term"/&gt;

&lt;cvParam accession="UNIMOD:2000" cvRef="UNIMOD" name="Xlink:SDA"/&gt;
____

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code"

____
value="W" cvRef="PSI-MS" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="82.041865" residues="K S T Y"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="SDA_crosslink_donor"/&gt;

&lt;cvParam accession="UNIMOD:2000" name="Xlink:DSSO" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002509" name="crosslink donor"

value="*2*" /&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics" value="S:82.041865:O"

cvRef="PSI-MS"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="82.041865" residues="."&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002057"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="SDA_crosslink_donor_n_term"/&gt;

&lt;cvParam accession="UNIMOD:1842" name="Xlink:DSSO" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink donor"

value="*2*" /&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics" value="S:82.041865:O"

cvRef="PSI-MS"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="0.0" residues="."&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="SDA_crosslink_acceptor"/&gt;

&lt;cvParam accession="UNIMOD:2000" name="Xlink:SDA" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor"

value="*2*" /&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="O:0:S" cvRef="PSI-MS"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="0.0" residues="."&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002057"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id"

value="SDA_crosslink_acceptor_n_term"/&gt;

&lt;cvParam accession="UNIMOD:2000" name="Xlink:SDA" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor"

value="*2*" /&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="O:0:S" cvRef="PSI-MS"/&gt;

&lt;/SearchModification&gt;
____

&lt;/ModificationParams&gt;

...

&lt;/SpectrumIdentificationProtocol&gt;

=== *7.4 Disuccinimidyl sulfoxide (DSSO)*

&lt;SpectrumIdentificationProtocol&gt;

...

&lt;ModificationParams&gt;

____
&lt;SearchModification fixedMod="false" massDelta="175.030314" residues="K S T Y"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="DSSO_monolink_M"/&gt;

&lt;cvParam accession="UNIMOD:1842" cvRef="UNIMOD" name="Xlink:DSSO"/&gt;
____

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code"

____
value="M" cvRef="PSI-MS" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="176.01433" residues="K S T Y"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="DSSO_monolink_W"/&gt;

&lt;cvParam accession="UNIMOD:1842" cvRef="UNIMOD" name="Xlink:DSSO"/&gt;
____

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code"

____
value="W" cvRef="PSI-MS" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="175.030314" residues="."&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002057"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="DSSO_monolink_M_n_term"/&gt;

&lt;cvParam accession="UNIMOD:1842" cvRef="UNIMOD" name="Xlink:DSSO"/&gt;
____

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code"

____
value="M" cvRef="PSI-MS" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="176.01433" residues="."&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002057"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="DSSO_monolink_W_n_term"/&gt;

&lt;cvParam accession="UNIMOD:1842" cvRef="UNIMOD" name="Xlink:DSSO"/&gt;
____

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code"

____
value="W" cvRef="PSI-MS" /&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="158.003765" residues="K S T Y"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="DSSO_crosslink_donor"/&gt;

&lt;cvParam accession="UNIMOD:1842" name="Xlink:DSSO" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002509" name="crosslink donor"

value="3" /&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="A:54.0105647:ST" cvRef="PSI-MS"/&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="S:103.9932001:A" cvRef="PSI-MS"/&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="T:85.9826354:A" cvRef="PSI-MS"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="158.003765" residues="."&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002057"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="DSSO_crosslink_donor_n_term"/&gt;

&lt;cvParam accession="UNIMOD:1842" name="Xlink:DSSO" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink donor"

value="3" /&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="A:54.0105647:ST" cvRef="PSI-MS"/&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="S:103.9932001:A" cvRef="PSI-MS"/&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="T:85.9826354:A" cvRef="PSI-MS"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="0.0" residues="K S T Y"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="DSSO_crosslink_acceptor"/&gt;

&lt;cvParam accession="UNIMOD:1842" name="Xlink:DSSO" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor"

value="3" /&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="A:54.0105647:ST" cvRef="PSI-MS"/&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="S:103.9932001:A" cvRef="PSI-MS"/&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="T:85.9826354:A" cvRef="PSI-MS"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="0.0" residues="."&gt;

&lt;SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002057"

name="modification specificity protein N-term"/&gt;

&lt;/SpecificityRules&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id"

value="DSSO_crosslink_acceptor_n_term"/&gt;

&lt;cvParam accession="UNIMOD:1842" name="Xlink:DSSO" cvRef="UNIMOD" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor"

value="3" /&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="A:54.0105647:ST" cvRef="PSI-MS"/&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="S:103.9932001:A" cvRef="PSI-MS"/&gt;

&lt;cvParam accession="MS:1003390" name="crosslinker cleavage characteristics"

value="T:85.9826354:A" cvRef="PSI-MS"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="54.010565" residues="K S T Y"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="DSSO_crosslink_stub_a"/&gt;

&lt;cvParam accession="UNIMOD:1842" name="Xlink:DSSO" cvRef="UNIMOD" /&gt;

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code"

value="A" cvRef="PSI-MS" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003346" name="cleavable crosslinker stub"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="85.982636" residues="K S T Y"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="DSSO_crosslink_stub_t"/&gt;

&lt;cvParam accession="UNIMOD:1842" name="Xlink:DSSO" cvRef="UNIMOD" /&gt;

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code"

value="T" cvRef="PSI-MS" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003346" name="cleavable crosslinker stub"/&gt;

&lt;/SearchModification&gt;

&lt;SearchModification fixedMod="false" massDelta="103.9932" residues="K S T Y"&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003392"

name="search modification id" value="DSSO_crosslink_stub_s"/&gt;

&lt;cvParam accession="UNIMOD:1842" name="Xlink:DSSO" cvRef="UNIMOD" /&gt;

&lt;cvParam accession="MS:1003347" name="UNIMOD derivative code"

value="S" cvRef="PSI-MS" /&gt;

&lt;cvParam cvRef="PSI-MS" accession="MS:1003346" name="cleavable crosslinker stub"/&gt;

&lt;/SearchModification&gt;
____

&lt;/ModificationParams&gt;

...

&lt;/SpectrumIdentificationProtocol&gt;</code></pre>
</div>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="authors">8. Authors Information</h2>
<div class="sectionbody">
<div class="paragraph">
<p>Authors of this extension:</p>
</div>
<div class="paragraph">
<p>Colin W. Combe, University of Edinburgh, <a href="mailto:colin.combe@ed.ac.uk">colin.combe@ed.ac.uk</a></p>
</div>
<div class="paragraph">
<p>Lars Kolbowski, Technische Universität Berlin, <a href="mailto:lars.kolbowski@tu-berlin.de">lars.kolbowski@tu-berlin.de</a></p>
</div>
<div class="paragraph">
<p>Lutz Fischer, Technische Universität Berlin, <a href="mailto:lutz.fischer@tu-berlin.de">lutz.fischer@tu-berlin.de</a></p>
</div>
<div class="paragraph">
<p>Ville Koskinen, Matrix Science Ltd, <a href="mailto:villek@matrixscience.com">villek@matrixscience.com</a></p>
</div>
<div class="paragraph">
<p>Joshua Klein, University of Boston, <a href="mailto:mobiusklein@gmail.com">mobiusklein@gmail.com</a></p>
</div>
<div class="paragraph">
<p>Alexander Leitner, ETH Zurich, <a href="mailto:leitner@imsb.biol.ethz.ch">leitner@imsb.biol.ethz.ch</a></p>
</div>
<div class="paragraph">
<p>Juan Antonio Vizcaíno, European Molecular Biology Laboratory, EMBL-EBI,</p>
</div>
<div class="paragraph">
<p><a href="mailto:juan@ebi.ac.uk">juan@ebi.ac.uk</a></p>
</div>
<div class="paragraph">
<p>Andy Jones, University of Liverpool, <a href="mailto:Andrew.Jones@liverpool.ac.uk">Andrew.Jones@liverpool.ac.uk</a></p>
</div>
<div class="paragraph">
<p>Juri Rappsilber, Technische Universität Berlin, <a href="mailto:Juri.Rappsilber@tu-Berlin.de">Juri.Rappsilber@tu-Berlin.de</a></p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_glossary">9. Glossary</h2>
<div class="sectionbody">
<div class="paragraph">
<p>Cleavable Crosslinker - a crosslinker that can be broken in two to release the individual peptides (in a modified form), thus enabling their individual analysis. In crosslinking studies this typically refers to <em>MS-cleavable crosslinkers</em>.</p>
</div>
<div class="paragraph">
<p>Controlled Vocabulary (CV) - A structured collection of terms describing a certain Crosslink - The covalent bond formed by a <em>crosslinker</em>.</p>
</div>
<div class="paragraph">
<p>Crosslink Acceptor - one end of a crosslinking reaction, the other is the <em>crosslink donor</em>. (Assumes the crosslinker has only two reactive groups.)</p>
</div>
<div class="paragraph">
<p>Crosslink Donor - one end of a crosslinking reaction, the other is the <em>crosslink acceptor</em>. (Assumes the crosslinker has only two reactive groups.)</p>
</div>
<div class="paragraph">
<p>Crosslinker - A chemical reagent. A molecule which creates a covalent bond either between proteins or within the same protein chain. This bond preserves proximity information which would otherwise be destroyed by the enzymatic digestion of the proteins. The proximity information is then recovered by identifying the peptides via mass spectrometry.</p>
</div>
<div class="paragraph">
<p>Crosslinker Modified Peptide - a peptide where one reactive group of a <em>crosslinker</em> has reacted with one of its amino acids, but the other reactive group has not reacted with any animo acid and no covalent bond is formed.</p>
</div>
<div class="paragraph">
<p>Crosslinker Specificity - The specificity of a crosslinker gives the amino acids it will react with. A <em>crosslinker</em> has reactive groups that react with amino acids or their side-chains, the reactive groups determine the specificity of the crosslinker.</p>
</div>
<div class="paragraph">
<p>Crosslinking Product - the result of a crosslinking reaction.</p>
</div>
<div class="paragraph">
<p>Crosslink Spectrum Match (CSM) - the subset of PSM level matches that contain a crosslink.</p>
</div>
<div class="paragraph">
<p>Decoy (decoy databases, decoy peptide) - decoys are a set of artificially generated sequences used to assess the performance of an identification algorithm. Decoys are typically created by randomising or reversing the sequences of the target proteins. These new sequences are then added to the database used for analysis. <em>False Discovery Rates</em> may use the identifications of decoys to estimate the error rate in the data.</p>
</div>
<div class="paragraph">
<p>False Discovery Rate (FDR) - the fraction of identifications that are predicted to be incorrect (false positives) among the total number of identifications made.</p>
</div>
<div class="paragraph">
<p>Global FDR - the False Discovery Rate across the whole dataset, as opposed to ‘local FDR’ which calculates the error rate within a given score window.</p>
</div>
<div class="paragraph">
<p>Heterobifunctional Crosslinker - a <em>crosslinker</em> with two reactive groups in which the reactive groups, and hence the specificity of each end, are different.</p>
</div>
<div class="paragraph">
<p>Higher Order Crosslink - A <em>crosslinking product</em> in which there are arbitrarily many peptides with many crosslinks between them.</p>
</div>
<div class="paragraph">
<p>Internally Linked Peptide - A crosslinking product in which both ends of the crosslinker have reacted within a single peptide (that is, within the same identical peptide, not between two copies of the same peptide). This type of crosslinking product is known to be intramolecular.</p>
</div>
<div class="paragraph">
<p>Looplink - colloquial name for an <em>internally linked peptide</em>.</p>
</div>
<div class="paragraph">
<p>Modification - a chemical change to an amino acid.</p>
</div>
<div class="paragraph">
<p>MS-Cleavable Crosslinker - a type of <em>cleavable crosslinker</em>, MS-cleavable crosslinkers can cleave upon activation in the mass spectrometer, releasing the individual peptides and thus enabling their individual analysis.</p>
</div>
<div class="paragraph">
<p>Noncovalently Associated Peptides - two different peptides which were not crosslinked but stayed associated with each other throughout the workflow, due to noncovalent interactions.</p>
</div>
<div class="paragraph">
<p>Monolink - colloquial name for a <em>crosslinker modified peptide</em>. Unimod uses this term to describe some derivatives of crosslinkers (e.g. see <a href="http://www.unimod.org/modifications_view.php?editid1=1842" class="bare">http://www.unimod.org/modifications_view.php?editid1=1842</a>).</p>
</div>
<div class="paragraph">
<p>Open Modification Search - a search strategy which allows for any type of mass shift to occur at any residue within a peptide sequence. In contrast to a "closed" search, which is limited to a defined set of modifications, an open modification search allows for the identification of novel or unexpected modifications</p>
</div>
<div class="paragraph">
<p>Posterior Error Probability - a statistical measure of the probability that an identification is incorrect.</p>
</div>
<div class="paragraph">
<p>Protein Heteromeric Crosslinks - crosslinks between distinct protein sequences.</p>
</div>
<div class="paragraph">
<p>Protein-Protein Interaction (PPI) - an interaction between proteins. In the context of crosslinking, it is a level of consolidation at which crosslinks may be analysed.</p>
</div>
<div class="paragraph">
<p>PSI-MS - the Human Proteome Organization (HUPO) Proteomics Standards Initiative’s (PSI) controlled vocabulary for Mass Spectrometry (MS)</p>
</div>
<div class="paragraph">
<p>Peptide Spectrum Match (PSM) - a match to one or more peptides in a mass spectrum. The lowest, unconsolidated level at which analysis of identifications can occur. Because a single “match” may identify more than one peptide, the concept ‘PSM’ does not correspond directly with “spectrum identification item” in mzIdentML, no element in mzIdentML does correspond directly to “a match”.</p>
</div>
<div class="paragraph">
<p>PSM-level - the level of analysis that looks at individual <em>peptide spectrum matches</em>.</p>
</div>
<div class="paragraph">
<p>Residue-pair - a unique pair of crosslinked residues, irrespective of the peptides identified. A level of consolidation higher than unique peptide but lower than PPI.</p>
</div>
<div class="paragraph">
<p>Self Crosslinks - crosslinks between peptides within one protein sequence.</p>
</div>
<div class="paragraph">
<p>Trimeric Crosslinker - a crosslinker with three reactive groups.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_references">10. References</h2>
<div class="sectionbody">
<div class="paragraph">
<p>1. <a href="http://paperpile.com/b/YrD2gH/vjf16">Vizcaíno JA, Mayer G, Perkins S, Barsnes H, Vaudel M, Perez-Riverol Y, et al. The mzIdentML Data Standard Version 1.2, Supporting Advances in Proteome Informatics. Mol Cell Proteomics. 2017;16: 1275–1285.</a></p>
</div>
<div class="paragraph">
<p>2. <a href="http://paperpile.com/b/YrD2gH/IanGW">Giese SH, Belsom A, Sinn L, Fischer L, Rappsilber J. Noncovalently Associated Peptides Observed during Liquid Chromatography-Mass Spectrometry and Their Effect on Cross-Link Analyses. Anal Chem. 2019;91: 2678–2685.</a></p>
</div>
<div class="paragraph">
<p>3. <a href="http://paperpile.com/b/YrD2gH/9dZZN">Mayer G, Jones AR, Binz P-A, Deutsch EW, Orchard S, Montecchi-Palazzi L, et al. Controlled vocabularies and ontologies in proteomics: overview, principles and practice. Biochim Biophys Acta. 2014;1844: 98–107.</a></p>
</div>
<div class="paragraph">
<p>4. <a href="http://paperpile.com/b/YrD2gH/osIjk">Liu F, Lössl P, Scheltema R, Viner R, Heck AJR. Optimized fragmentation schemes and data analysis strategies for proteome-wide cross-link identification. Nat Commun. 2017;8: 15473.</a></p>
</div>
<div class="paragraph">
<p>5. <a href="http://paperpile.com/b/YrD2gH/kAbLF">Lenz S, Sinn LR, O’Reilly FJ, Fischer L, Wegner F, Rappsilber J. Reliable identification of protein-protein interactions by crosslinking mass spectrometry. Nat Commun. 2021;12: 1–11.</a></p>
</div>
<div class="paragraph">
<p>6. <a href="http://paperpile.com/b/YrD2gH/ghaA0">Fischer L, Rappsilber J. Quirks of Error Estimation in Cross-Linking/Mass Spectrometry. Anal Chem. 2017;89: 3829.</a></p>
</div>
<div class="paragraph">
<p>7. <a href="http://paperpile.com/b/YrD2gH/8J3I">Mohr JP, Caudal A, Tian R, Bruce JE. Multidimensional Cross-Linking and Real-Time Informatics for Multiprotein Interaction Studies. J Proteome Res. 2024;23. doi:</a>http://dx.doi.org/10.1021/acs.jproteome.3c00455[10.1021/acs.jproteome.3c00455]</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_intellectual_property_statement">11. Intellectual Property Statement</h2>
<div class="sectionbody">
<div class="paragraph">
<p>The PSI takes no position regarding the validity or scope of any intellectual property or other rights that might be claimed to pertain to the implementation or use of the technology described in this document or the extent to which any license under such rights might or might not be available; neither does it represent that it has made any effort to identify any such rights. Copies of claims of rights made available for publication and any assurances of licenses to be made available, or the result of an attempt made to obtain a general license or permission for the use of such proprietary rights by implementers or users of this specification can be obtained from the PSI Chair.</p>
</div>
<div class="paragraph">
<p>The PSI invites any interested party to bring to its attention any copyrights, patents or patent applications, or other proprietary rights which may cover technology that may be required to practice this recommendation. Please address the information to the PSI Chair (see contacts information at PSI website).</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_copyright_notice">12. Copyright Notice</h2>
<div class="sectionbody">
<div class="paragraph">
<p>Copyright &#169; Proteomics Standards Initiative (2023). All Rights Reserved.</p>
</div>
<div class="paragraph">
<p>This document and translations of it may be copied and furnished to others, and derivative works that comment on or otherwise explain it or assist in its implementation may be prepared, copied, published and distributed, in whole or in part, without restriction of any kind, provided that the above copyright notice and this paragraph are included on all such copies and derivative works. However, this document itself may not be modified in any way, such as by removing the copyright notice or references to the PSI or other organizations, except as needed for the purpose of developing Proteomics Recommendations in which case the procedures for copyrights defined in the PSI Document process must be followed, or as required to translate it into languages other than English.</p>
</div>
<div class="paragraph">
<p>The limited permissions granted above are perpetual and will not be revoked by the PSI or its successors or assigns.</p>
</div>
<div class="paragraph">
<p>This document and the information contained herein is provided on an "AS IS" basis and THE PROTEOMICS STANDARDS INITIATIVE DISCLAIMS ALL WARRANTIES, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTY THAT THE USE OF THE INFORMATION HEREIN WILL NOT INFRINGE ANY RIGHTS OR ANY IMPLIED WARRANTIES OF MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE."</p>
</div>
</div>
</div>
</div>
<div id="footer">
<div id="footer-text">
Last updated 2025-02-22 21:34:18 +0100
</div>
</div>
</body>
</html>