<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Asciidoctor 2.0.12">
<title>mzIdentML: exchange format for peptides and proteins identified from mass spectra</title>
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400,700">
<style>
/* Asciidoctor default stylesheet | MIT License | https://asciidoctor.org */
/* Uncomment @import statement to use as custom stylesheet */
/*@import "https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400,700";*/
article,aside,details,figcaption,figure,footer,header,hgroup,main,nav,section{display:block}
audio,video{display:inline-block}
audio:not([controls]){display:none;height:0}
html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}
a{background:none}
a:focus{outline:thin dotted}
a:active,a:hover{outline:0}
h1{font-size:2em;margin:.67em 0}
abbr[title]{border-bottom:1px dotted}
b,strong{font-weight:bold}
dfn{font-style:italic}
hr{-moz-box-sizing:content-box;box-sizing:content-box;height:0}
mark{background:#ff0;color:#000}
code,kbd,pre,samp{font-family:monospace;font-size:1em}
pre{white-space:pre-wrap}
q{quotes:"\201C" "\201D" "\2018" "\2019"}
small{font-size:80%}
sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}
sup{top:-.5em}
sub{bottom:-.25em}
img{border:0}
svg:not(:root){overflow:hidden}
figure{margin:0}
fieldset{border:1px solid silver;margin:0 2px;padding:.35em .625em .75em}
legend{border:0;padding:0}
button,input,select,textarea{font-family:inherit;font-size:100%;margin:0}
button,input{line-height:normal}
button,select{text-transform:none}
button,html input[type="button"],input[type="reset"],input[type="submit"]{-webkit-appearance:button;cursor:pointer}
button[disabled],html input[disabled]{cursor:default}
input[type="checkbox"],input[type="radio"]{box-sizing:border-box;padding:0}
button::-moz-focus-inner,input::-moz-focus-inner{border:0;padding:0}
textarea{overflow:auto;vertical-align:top}
table{border-collapse:collapse;border-spacing:0}
*,*::before,*::after{-moz-box-sizing:border-box;-webkit-box-sizing:border-box;box-sizing:border-box}
html,body{font-size:100%}
body{background:#fff;color:rgba(0,0,0,.8);padding:0;margin:0;font-family:"Noto Serif","DejaVu Serif",serif;font-weight:400;font-style:normal;line-height:1;position:relative;cursor:auto;tab-size:4;word-wrap:anywhere;-moz-osx-font-smoothing:grayscale;-webkit-font-smoothing:antialiased}
a:hover{cursor:pointer}
img,object,embed{max-width:100%;height:auto}
object,embed{height:100%}
img{-ms-interpolation-mode:bicubic}
.left{float:left!important}
.right{float:right!important}
.text-left{text-align:left!important}
.text-right{text-align:right!important}
.text-center{text-align:center!important}
.text-justify{text-align:justify!important}
.hide{display:none}
img,object,svg{display:inline-block;vertical-align:middle}
textarea{height:auto;min-height:50px}
select{width:100%}
.subheader,.admonitionblock td.content>.title,.audioblock>.title,.exampleblock>.title,.imageblock>.title,.listingblock>.title,.literalblock>.title,.stemblock>.title,.openblock>.title,.paragraph>.title,.quoteblock>.title,table.tableblock>.title,.verseblock>.title,.videoblock>.title,.dlist>.title,.olist>.title,.ulist>.title,.qlist>.title,.hdlist>.title{line-height:1.45;color:#7a2518;font-weight:400;margin-top:0;margin-bottom:.25em}
div,dl,dt,dd,ul,ol,li,h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6,pre,form,p,blockquote,th,td{margin:0;padding:0}
a{color:#2156a5;text-decoration:underline;line-height:inherit}
a:hover,a:focus{color:#1d4b8f}
a img{border:0}
p{font-family:inherit;font-weight:400;font-size:1em;line-height:1.6;margin-bottom:1.25em;text-rendering:optimizeLegibility}
p aside{font-size:.875em;line-height:1.35;font-style:italic}
h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6{font-family:"Open Sans","DejaVu Sans",sans-serif;font-weight:300;font-style:normal;color:#ba3925;text-rendering:optimizeLegibility;margin-top:1em;margin-bottom:.5em;line-height:1.0125em}
h1 small,h2 small,h3 small,#toctitle small,.sidebarblock>.content>.title small,h4 small,h5 small,h6 small{font-size:60%;color:#e99b8f;line-height:0}
h1{font-size:2.125em}
h2{font-size:1.6875em}
h3,#toctitle,.sidebarblock>.content>.title{font-size:1.375em}
h4,h5{font-size:1.125em}
h6{font-size:1em}
hr{border:solid #dddddf;border-width:1px 0 0;clear:both;margin:1.25em 0 1.1875em;height:0}
em,i{font-style:italic;line-height:inherit}
strong,b{font-weight:bold;line-height:inherit}
small{font-size:60%;line-height:inherit}
code{font-family:"Droid Sans Mono","DejaVu Sans Mono",monospace;font-weight:400;color:rgba(0,0,0,.9)}
ul,ol,dl{font-size:1em;line-height:1.6;margin-bottom:1.25em;list-style-position:outside;font-family:inherit}
ul,ol{margin-left:1.5em}
ul li ul,ul li ol{margin-left:1.25em;margin-bottom:0;font-size:1em}
ul.square li ul,ul.circle li ul,ul.disc li ul{list-style:inherit}
ul.square{list-style-type:square}
ul.circle{list-style-type:circle}
ul.disc{list-style-type:disc}
ol li ul,ol li ol{margin-left:1.25em;margin-bottom:0}
dl dt{margin-bottom:.3125em;font-weight:bold}
dl dd{margin-bottom:1.25em}
abbr,acronym{text-transform:uppercase;font-size:90%;color:rgba(0,0,0,.8);border-bottom:1px dotted #ddd;cursor:help}
abbr{text-transform:none}
blockquote{margin:0 0 1.25em;padding:.5625em 1.25em 0 1.1875em;border-left:1px solid #ddd}
blockquote cite{display:block;font-size:.9375em;color:rgba(0,0,0,.6)}
blockquote cite::before{content:"\2014 \0020"}
blockquote cite a,blockquote cite a:visited{color:rgba(0,0,0,.6)}
blockquote,blockquote p{line-height:1.6;color:rgba(0,0,0,.85)}
@media screen and (min-width:768px){h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6{line-height:1.2}
h1{font-size:2.75em}
h2{font-size:2.3125em}
h3,#toctitle,.sidebarblock>.content>.title{font-size:1.6875em}
h4{font-size:1.4375em}}
table{background:#fff;margin-bottom:1.25em;border:solid 1px #dedede;word-wrap:normal}
table thead,table tfoot{background:#f7f8f7}
table thead tr th,table thead tr td,table tfoot tr th,table tfoot tr td{padding:.5em .625em .625em;font-size:inherit;color:rgba(0,0,0,.8);text-align:left}
table tr th,table tr td{padding:.5625em .625em;font-size:inherit;color:rgba(0,0,0,.8)}
table tr.even,table tr.alt{background:#f8f8f7}
table thead tr th,table tfoot tr th,table tbody tr td,table tr td,table tfoot tr td{line-height:1.6}
h1,h2,h3,#toctitle,.sidebarblock>.content>.title,h4,h5,h6{line-height:1.2;word-spacing:-.05em}
h1 strong,h2 strong,h3 strong,#toctitle strong,.sidebarblock>.content>.title strong,h4 strong,h5 strong,h6 strong{font-weight:400}
.center{margin-left:auto;margin-right:auto}
.stretch{width:100%}
.clearfix::before,.clearfix::after,.float-group::before,.float-group::after{content:" ";display:table}
.clearfix::after,.float-group::after{clear:both}
:not(pre).nobreak{word-wrap:normal}
:not(pre).nowrap{white-space:nowrap}
:not(pre).pre-wrap{white-space:pre-wrap}
:not(pre):not([class^=L])>code{font-size:.9375em;font-style:normal!important;letter-spacing:0;padding:.1em .5ex;word-spacing:-.15em;background:#f7f7f8;-webkit-border-radius:4px;border-radius:4px;line-height:1.45;text-rendering:optimizeSpeed}
pre{color:rgba(0,0,0,.9);font-family:"Droid Sans Mono","DejaVu Sans Mono",monospace;line-height:1.45;text-rendering:optimizeSpeed}
pre code,pre pre{color:inherit;font-size:inherit;line-height:inherit}
pre>code{display:block}
pre.nowrap,pre.nowrap pre{white-space:pre;word-wrap:normal}
em em{font-style:normal}
strong strong{font-weight:400}
.keyseq{color:rgba(51,51,51,.8)}
kbd{font-family:"Droid Sans Mono","DejaVu Sans Mono",monospace;display:inline-block;color:rgba(0,0,0,.8);font-size:.65em;line-height:1.45;background:#f7f7f7;border:1px solid #ccc;-webkit-border-radius:3px;border-radius:3px;-webkit-box-shadow:0 1px 0 rgba(0,0,0,.2),0 0 0 .1em white inset;box-shadow:0 1px 0 rgba(0,0,0,.2),0 0 0 .1em #fff inset;margin:0 .15em;padding:.2em .5em;vertical-align:middle;position:relative;top:-.1em;white-space:nowrap}
.keyseq kbd:first-child{margin-left:0}
.keyseq kbd:last-child{margin-right:0}
.menuseq,.menuref{color:#000}
.menuseq b:not(.caret),.menuref{font-weight:inherit}
.menuseq{word-spacing:-.02em}
.menuseq b.caret{font-size:1.25em;line-height:.8}
.menuseq i.caret{font-weight:bold;text-align:center;width:.45em}
b.button::before,b.button::after{position:relative;top:-1px;font-weight:400}
b.button::before{content:"[";padding:0 3px 0 2px}
b.button::after{content:"]";padding:0 2px 0 3px}
p a>code:hover{color:rgba(0,0,0,.9)}
#header,#content,#footnotes,#footer{width:100%;margin-left:auto;margin-right:auto;margin-top:0;margin-bottom:0;max-width:62.5em;*zoom:1;position:relative;padding-left:.9375em;padding-right:.9375em}
#header::before,#header::after,#content::before,#content::after,#footnotes::before,#footnotes::after,#footer::before,#footer::after{content:" ";display:table}
#header::after,#content::after,#footnotes::after,#footer::after{clear:both}
#content{margin-top:1.25em}
#content::before{content:none}
#header>h1:first-child{color:rgba(0,0,0,.85);margin-top:2.25rem;margin-bottom:0}
#header>h1:first-child+#toc{margin-top:8px;border-top:1px solid #dddddf}
#header>h1:only-child,body.toc2 #header>h1:nth-last-child(2){border-bottom:1px solid #dddddf;padding-bottom:8px}
#header .details{border-bottom:1px solid #dddddf;line-height:1.45;padding-top:.25em;padding-bottom:.25em;padding-left:.25em;color:rgba(0,0,0,.6);display:-ms-flexbox;display:-webkit-flex;display:flex;-ms-flex-flow:row wrap;-webkit-flex-flow:row wrap;flex-flow:row wrap}
#header .details span:first-child{margin-left:-.125em}
#header .details span.email a{color:rgba(0,0,0,.85)}
#header .details br{display:none}
#header .details br+span::before{content:"\00a0\2013\00a0"}
#header .details br+span.author::before{content:"\00a0\22c5\00a0";color:rgba(0,0,0,.85)}
#header .details br+span#revremark::before{content:"\00a0|\00a0"}
#header #revnumber{text-transform:capitalize}
#header #revnumber::after{content:"\00a0"}
#content>h1:first-child:not([class]){color:rgba(0,0,0,.85);border-bottom:1px solid #dddddf;padding-bottom:8px;margin-top:0;padding-top:1rem;margin-bottom:1.25rem}
#toc{border-bottom:1px solid #e7e7e9;padding-bottom:.5em}
#toc>ul{margin-left:.125em}
#toc ul.sectlevel0>li>a{font-style:italic}
#toc ul.sectlevel0 ul.sectlevel1{margin:.5em 0}
#toc ul{font-family:"Open Sans","DejaVu Sans",sans-serif;list-style-type:none}
#toc li{line-height:1.3334;margin-top:.3334em}
#toc a{text-decoration:none}
#toc a:active{text-decoration:underline}
#toctitle{color:#7a2518;font-size:1.2em}
@media screen and (min-width:768px){#toctitle{font-size:1.375em}
body.toc2{padding-left:15em;padding-right:0}
#toc.toc2{margin-top:0!important;background:#f8f8f7;position:fixed;width:15em;left:0;top:0;border-right:1px solid #e7e7e9;border-top-width:0!important;border-bottom-width:0!important;z-index:1000;padding:1.25em 1em;height:100%;overflow:auto}
#toc.toc2 #toctitle{margin-top:0;margin-bottom:.8rem;font-size:1.2em}
#toc.toc2>ul{font-size:.9em;margin-bottom:0}
#toc.toc2 ul ul{margin-left:0;padding-left:1em}
#toc.toc2 ul.sectlevel0 ul.sectlevel1{padding-left:0;margin-top:.5em;margin-bottom:.5em}
body.toc2.toc-right{padding-left:0;padding-right:15em}
body.toc2.toc-right #toc.toc2{border-right-width:0;border-left:1px solid #e7e7e9;left:auto;right:0}}
@media screen and (min-width:1280px){body.toc2{padding-left:20em;padding-right:0}
#toc.toc2{width:20em}
#toc.toc2 #toctitle{font-size:1.375em}
#toc.toc2>ul{font-size:.95em}
#toc.toc2 ul ul{padding-left:1.25em}
body.toc2.toc-right{padding-left:0;padding-right:20em}}
#content #toc{border-style:solid;border-width:1px;border-color:#e0e0dc;margin-bottom:1.25em;padding:1.25em;background:#f8f8f7;-webkit-border-radius:4px;border-radius:4px}
#content #toc>:first-child{margin-top:0}
#content #toc>:last-child{margin-bottom:0}
#footer{max-width:none;background:rgba(0,0,0,.8);padding:1.25em}
#footer-text{color:rgba(255,255,255,.8);line-height:1.44}
#content{margin-bottom:.625em}
.sect1{padding-bottom:.625em}
@media screen and (min-width:768px){#content{margin-bottom:1.25em}
.sect1{padding-bottom:1.25em}}
.sect1:last-child{padding-bottom:0}
.sect1+.sect1{border-top:1px solid #e7e7e9}
#content h1>a.anchor,h2>a.anchor,h3>a.anchor,#toctitle>a.anchor,.sidebarblock>.content>.title>a.anchor,h4>a.anchor,h5>a.anchor,h6>a.anchor{position:absolute;z-index:1001;width:1.5ex;margin-left:-1.5ex;display:block;text-decoration:none!important;visibility:hidden;text-align:center;font-weight:400}
#content h1>a.anchor::before,h2>a.anchor::before,h3>a.anchor::before,#toctitle>a.anchor::before,.sidebarblock>.content>.title>a.anchor::before,h4>a.anchor::before,h5>a.anchor::before,h6>a.anchor::before{content:"\00A7";font-size:.85em;display:block;padding-top:.1em}
#content h1:hover>a.anchor,#content h1>a.anchor:hover,h2:hover>a.anchor,h2>a.anchor:hover,h3:hover>a.anchor,#toctitle:hover>a.anchor,.sidebarblock>.content>.title:hover>a.anchor,h3>a.anchor:hover,#toctitle>a.anchor:hover,.sidebarblock>.content>.title>a.anchor:hover,h4:hover>a.anchor,h4>a.anchor:hover,h5:hover>a.anchor,h5>a.anchor:hover,h6:hover>a.anchor,h6>a.anchor:hover{visibility:visible}
#content h1>a.link,h2>a.link,h3>a.link,#toctitle>a.link,.sidebarblock>.content>.title>a.link,h4>a.link,h5>a.link,h6>a.link{color:#ba3925;text-decoration:none}
#content h1>a.link:hover,h2>a.link:hover,h3>a.link:hover,#toctitle>a.link:hover,.sidebarblock>.content>.title>a.link:hover,h4>a.link:hover,h5>a.link:hover,h6>a.link:hover{color:#a53221}
details,.audioblock,.imageblock,.literalblock,.listingblock,.stemblock,.videoblock{margin-bottom:1.25em}
details>summary:first-of-type{cursor:pointer;display:list-item;outline:none;margin-bottom:.75em}
.admonitionblock td.content>.title,.audioblock>.title,.exampleblock>.title,.imageblock>.title,.listingblock>.title,.literalblock>.title,.stemblock>.title,.openblock>.title,.paragraph>.title,.quoteblock>.title,table.tableblock>.title,.verseblock>.title,.videoblock>.title,.dlist>.title,.olist>.title,.ulist>.title,.qlist>.title,.hdlist>.title{text-rendering:optimizeLegibility;text-align:left;font-family:"Noto Serif","DejaVu Serif",serif;font-size:1rem;font-style:italic}
table.tableblock.fit-content>caption.title{white-space:nowrap;width:0}
.paragraph.lead>p,#preamble>.sectionbody>[class="paragraph"]:first-of-type p{font-size:1.21875em;line-height:1.6;color:rgba(0,0,0,.85)}
table.tableblock #preamble>.sectionbody>[class="paragraph"]:first-of-type p{font-size:inherit}
.admonitionblock>table{border-collapse:separate;border:0;background:none;width:100%}
.admonitionblock>table td.icon{text-align:center;width:80px}
.admonitionblock>table td.icon img{max-width:none}
.admonitionblock>table td.icon .title{font-weight:bold;font-family:"Open Sans","DejaVu Sans",sans-serif;text-transform:uppercase}
.admonitionblock>table td.content{padding-left:1.125em;padding-right:1.25em;border-left:1px solid #dddddf;color:rgba(0,0,0,.6);word-wrap:anywhere}
.admonitionblock>table td.content>:last-child>:last-child{margin-bottom:0}
.exampleblock>.content{border-style:solid;border-width:1px;border-color:#e6e6e6;margin-bottom:1.25em;padding:1.25em;background:#fff;-webkit-border-radius:4px;border-radius:4px}
.exampleblock>.content>:first-child{margin-top:0}
.exampleblock>.content>:last-child{margin-bottom:0}
.sidebarblock{border-style:solid;border-width:1px;border-color:#dbdbd6;margin-bottom:1.25em;padding:1.25em;background:#f3f3f2;-webkit-border-radius:4px;border-radius:4px}
.sidebarblock>:first-child{margin-top:0}
.sidebarblock>:last-child{margin-bottom:0}
.sidebarblock>.content>.title{color:#7a2518;margin-top:0;text-align:center}
.exampleblock>.content>:last-child>:last-child,.exampleblock>.content .olist>ol>li:last-child>:last-child,.exampleblock>.content .ulist>ul>li:last-child>:last-child,.exampleblock>.content .qlist>ol>li:last-child>:last-child,.sidebarblock>.content>:last-child>:last-child,.sidebarblock>.content .olist>ol>li:last-child>:last-child,.sidebarblock>.content .ulist>ul>li:last-child>:last-child,.sidebarblock>.content .qlist>ol>li:last-child>:last-child{margin-bottom:0}
.literalblock pre,.listingblock>.content>pre{-webkit-border-radius:4px;border-radius:4px;overflow-x:auto;padding:1em;font-size:.8125em}
@media screen and (min-width:768px){.literalblock pre,.listingblock>.content>pre{font-size:.90625em}}
@media screen and (min-width:1280px){.literalblock pre,.listingblock>.content>pre{font-size:1em}}
.literalblock pre,.listingblock>.content>pre:not(.highlight),.listingblock>.content>pre[class="highlight"],.listingblock>.content>pre[class^="highlight "]{background:#f7f7f8}
.literalblock.output pre{color:#f7f7f8;background:rgba(0,0,0,.9)}
.listingblock>.content{position:relative}
.listingblock code[data-lang]::before{display:none;content:attr(data-lang);position:absolute;font-size:.75em;top:.425rem;right:.5rem;line-height:1;text-transform:uppercase;color:inherit;opacity:.5}
.listingblock:hover code[data-lang]::before{display:block}
.listingblock.terminal pre .command::before{content:attr(data-prompt);padding-right:.5em;color:inherit;opacity:.5}
.listingblock.terminal pre .command:not([data-prompt])::before{content:"$"}
.listingblock pre.highlightjs{padding:0}
.listingblock pre.highlightjs>code{padding:1em;-webkit-border-radius:4px;border-radius:4px}
.listingblock pre.prettyprint{border-width:0}
.prettyprint{background:#f7f7f8}
pre.prettyprint .linenums{line-height:1.45;margin-left:2em}
pre.prettyprint li{background:none;list-style-type:inherit;padding-left:0}
pre.prettyprint li code[data-lang]::before{opacity:1}
pre.prettyprint li:not(:first-child) code[data-lang]::before{display:none}
table.linenotable{border-collapse:separate;border:0;margin-bottom:0;background:none}
table.linenotable td[class]{color:inherit;vertical-align:top;padding:0;line-height:inherit;white-space:normal}
table.linenotable td.code{padding-left:.75em}
table.linenotable td.linenos{border-right:1px solid currentColor;opacity:.35;padding-right:.5em}
pre.pygments .lineno{border-right:1px solid currentColor;opacity:.35;display:inline-block;margin-right:.75em}
pre.pygments .lineno::before{content:"";margin-right:-.125em}
.quoteblock{margin:0 1em 1.25em 1.5em;display:table}
.quoteblock:not(.excerpt)>.title{margin-left:-1.5em;margin-bottom:.75em}
.quoteblock blockquote,.quoteblock p{color:rgba(0,0,0,.85);font-size:1.15rem;line-height:1.75;word-spacing:.1em;letter-spacing:0;font-style:italic;text-align:justify}
.quoteblock blockquote{margin:0;padding:0;border:0}
.quoteblock blockquote::before{content:"\201c";float:left;font-size:2.75em;font-weight:bold;line-height:.6em;margin-left:-.6em;color:#7a2518;text-shadow:0 1px 2px rgba(0,0,0,.1)}
.quoteblock blockquote>.paragraph:last-child p{margin-bottom:0}
.quoteblock .attribution{margin-top:.75em;margin-right:.5ex;text-align:right}
.verseblock{margin:0 1em 1.25em}
.verseblock pre{font-family:"Open Sans","DejaVu Sans",sans;font-size:1.15rem;color:rgba(0,0,0,.85);font-weight:300;text-rendering:optimizeLegibility}
.verseblock pre strong{font-weight:400}
.verseblock .attribution{margin-top:1.25rem;margin-left:.5ex}
.quoteblock .attribution,.verseblock .attribution{font-size:.9375em;line-height:1.45;font-style:italic}
.quoteblock .attribution br,.verseblock .attribution br{display:none}
.quoteblock .attribution cite,.verseblock .attribution cite{display:block;letter-spacing:-.025em;color:rgba(0,0,0,.6)}
.quoteblock.abstract blockquote::before,.quoteblock.excerpt blockquote::before,.quoteblock .quoteblock blockquote::before{display:none}
.quoteblock.abstract blockquote,.quoteblock.abstract p,.quoteblock.excerpt blockquote,.quoteblock.excerpt p,.quoteblock .quoteblock blockquote,.quoteblock .quoteblock p{line-height:1.6;word-spacing:0}
.quoteblock.abstract{margin:0 1em 1.25em;display:block}
.quoteblock.abstract>.title{margin:0 0 .375em;font-size:1.15em;text-align:center}
.quoteblock.excerpt>blockquote,.quoteblock .quoteblock{padding:0 0 .25em 1em;border-left:.25em solid #dddddf}
.quoteblock.excerpt,.quoteblock .quoteblock{margin-left:0}
.quoteblock.excerpt blockquote,.quoteblock.excerpt p,.quoteblock .quoteblock blockquote,.quoteblock .quoteblock p{color:inherit;font-size:1.0625rem}
.quoteblock.excerpt .attribution,.quoteblock .quoteblock .attribution{color:inherit;text-align:left;margin-right:0}
p.tableblock:last-child{margin-bottom:0}
td.tableblock>.content{margin-bottom:1.25em;word-wrap:anywhere}
td.tableblock>.content>:last-child{margin-bottom:-1.25em}
table.tableblock,th.tableblock,td.tableblock{border:0 solid #dedede}
table.grid-all>*>tr>*{border-width:1px}
table.grid-cols>*>tr>*{border-width:0 1px}
table.grid-rows>*>tr>*{border-width:1px 0}
table.frame-all{border-width:1px}
table.frame-ends{border-width:1px 0}
table.frame-sides{border-width:0 1px}
table.frame-none>colgroup+*>:first-child>*,table.frame-sides>colgroup+*>:first-child>*{border-top-width:0}
table.frame-none>:last-child>:last-child>*,table.frame-sides>:last-child>:last-child>*{border-bottom-width:0}
table.frame-none>*>tr>:first-child,table.frame-ends>*>tr>:first-child{border-left-width:0}
table.frame-none>*>tr>:last-child,table.frame-ends>*>tr>:last-child{border-right-width:0}
table.stripes-all tr,table.stripes-odd tr:nth-of-type(odd),table.stripes-even tr:nth-of-type(even),table.stripes-hover tr:hover{background:#f8f8f7}
th.halign-left,td.halign-left{text-align:left}
th.halign-right,td.halign-right{text-align:right}
th.halign-center,td.halign-center{text-align:center}
th.valign-top,td.valign-top{vertical-align:top}
th.valign-bottom,td.valign-bottom{vertical-align:bottom}
th.valign-middle,td.valign-middle{vertical-align:middle}
table thead th,table tfoot th{font-weight:bold}
tbody tr th{background:#f7f8f7}
tbody tr th,tbody tr th p,tfoot tr th,tfoot tr th p{color:rgba(0,0,0,.8);font-weight:bold}
p.tableblock>code:only-child{background:none;padding:0}
p.tableblock{font-size:1em}
ol{margin-left:1.75em}
ul li ol{margin-left:1.5em}
dl dd{margin-left:1.125em}
dl dd:last-child,dl dd:last-child>:last-child{margin-bottom:0}
ol>li p,ul>li p,ul dd,ol dd,.olist .olist,.ulist .ulist,.ulist .olist,.olist .ulist{margin-bottom:.625em}
ul.checklist,ul.none,ol.none,ul.no-bullet,ol.no-bullet,ol.unnumbered,ul.unstyled,ol.unstyled{list-style-type:none}
ul.no-bullet,ol.no-bullet,ol.unnumbered{margin-left:.625em}
ul.unstyled,ol.unstyled{margin-left:0}
ul.checklist{margin-left:.625em}
ul.checklist li>p:first-child>.fa-square-o:first-child,ul.checklist li>p:first-child>.fa-check-square-o:first-child{width:1.25em;font-size:.8em;position:relative;bottom:.125em}
ul.checklist li>p:first-child>input[type="checkbox"]:first-child{margin-right:.25em}
ul.inline{display:-ms-flexbox;display:-webkit-box;display:flex;-ms-flex-flow:row wrap;-webkit-flex-flow:row wrap;flex-flow:row wrap;list-style:none;margin:0 0 .625em -1.25em}
ul.inline>li{margin-left:1.25em}
.unstyled dl dt{font-weight:400;font-style:normal}
ol.arabic{list-style-type:decimal}
ol.decimal{list-style-type:decimal-leading-zero}
ol.loweralpha{list-style-type:lower-alpha}
ol.upperalpha{list-style-type:upper-alpha}
ol.lowerroman{list-style-type:lower-roman}
ol.upperroman{list-style-type:upper-roman}
ol.lowergreek{list-style-type:lower-greek}
.hdlist>table,.colist>table{border:0;background:none}
.hdlist>table>tbody>tr,.colist>table>tbody>tr{background:none}
td.hdlist1,td.hdlist2{vertical-align:top;padding:0 .625em}
td.hdlist1{font-weight:bold;padding-bottom:1.25em}
td.hdlist2{word-wrap:anywhere}
.literalblock+.colist,.listingblock+.colist{margin-top:-.5em}
.colist td:not([class]):first-child{padding:.4em .75em 0;line-height:1;vertical-align:top}
.colist td:not([class]):first-child img{max-width:none}
.colist td:not([class]):last-child{padding:.25em 0}
.thumb,.th{line-height:0;display:inline-block;border:solid 4px #fff;-webkit-box-shadow:0 0 0 1px #ddd;box-shadow:0 0 0 1px #ddd}
.imageblock.left{margin:.25em .625em 1.25em 0}
.imageblock.right{margin:.25em 0 1.25em .625em}
.imageblock>.title{margin-bottom:0}
.imageblock.thumb,.imageblock.th{border-width:6px}
.imageblock.thumb>.title,.imageblock.th>.title{padding:0 .125em}
.image.left,.image.right{margin-top:.25em;margin-bottom:.25em;display:inline-block;line-height:0}
.image.left{margin-right:.625em}
.image.right{margin-left:.625em}
a.image{text-decoration:none;display:inline-block}
a.image object{pointer-events:none}
sup.footnote,sup.footnoteref{font-size:.875em;position:static;vertical-align:super}
sup.footnote a,sup.footnoteref a{text-decoration:none}
sup.footnote a:active,sup.footnoteref a:active{text-decoration:underline}
#footnotes{padding-top:.75em;padding-bottom:.75em;margin-bottom:.625em}
#footnotes hr{width:20%;min-width:6.25em;margin:-.25em 0 .75em;border-width:1px 0 0}
#footnotes .footnote{padding:0 .375em 0 .225em;line-height:1.3334;font-size:.875em;margin-left:1.2em;margin-bottom:.2em}
#footnotes .footnote a:first-of-type{font-weight:bold;text-decoration:none;margin-left:-1.05em}
#footnotes .footnote:last-of-type{margin-bottom:0}
#content #footnotes{margin-top:-.625em;margin-bottom:0;padding:.75em 0}
.gist .file-data>table{border:0;background:#fff;width:100%;margin-bottom:0}
.gist .file-data>table td.line-data{width:99%}
div.unbreakable{page-break-inside:avoid}
.big{font-size:larger}
.small{font-size:smaller}
.underline{text-decoration:underline}
.overline{text-decoration:overline}
.line-through{text-decoration:line-through}
.aqua{color:#00bfbf}
.aqua-background{background:#00fafa}
.black{color:#000}
.black-background{background:#000}
.blue{color:#0000bf}
.blue-background{background:#0000fa}
.fuchsia{color:#bf00bf}
.fuchsia-background{background:#fa00fa}
.gray{color:#606060}
.gray-background{background:#7d7d7d}
.green{color:#006000}
.green-background{background:#007d00}
.lime{color:#00bf00}
.lime-background{background:#00fa00}
.maroon{color:#600000}
.maroon-background{background:#7d0000}
.navy{color:#000060}
.navy-background{background:#00007d}
.olive{color:#606000}
.olive-background{background:#7d7d00}
.purple{color:#600060}
.purple-background{background:#7d007d}
.red{color:#bf0000}
.red-background{background:#fa0000}
.silver{color:#909090}
.silver-background{background:#bcbcbc}
.teal{color:#006060}
.teal-background{background:#007d7d}
.white{color:#bfbfbf}
.white-background{background:#fafafa}
.yellow{color:#bfbf00}
.yellow-background{background:#fafa00}
span.icon>.fa{cursor:default}
a span.icon>.fa{cursor:inherit}
.admonitionblock td.icon [class^="fa icon-"]{font-size:2.5em;text-shadow:1px 1px 2px rgba(0,0,0,.5);cursor:default}
.admonitionblock td.icon .icon-note::before{content:"\f05a";color:#19407c}
.admonitionblock td.icon .icon-tip::before{content:"\f0eb";text-shadow:1px 1px 2px rgba(155,155,0,.8);color:#111}
.admonitionblock td.icon .icon-warning::before{content:"\f071";color:#bf6900}
.admonitionblock td.icon .icon-caution::before{content:"\f06d";color:#bf3400}
.admonitionblock td.icon .icon-important::before{content:"\f06a";color:#bf0000}
.conum[data-value]{display:inline-block;color:#fff!important;background:rgba(0,0,0,.8);-webkit-border-radius:50%;border-radius:50%;text-align:center;font-size:.75em;width:1.67em;height:1.67em;line-height:1.67em;font-family:"Open Sans","DejaVu Sans",sans-serif;font-style:normal;font-weight:bold}
.conum[data-value] *{color:#fff!important}
.conum[data-value]+b{display:none}
.conum[data-value]::after{content:attr(data-value)}
pre .conum[data-value]{position:relative;top:-.125em}
b.conum *{color:inherit!important}
.conum:not([data-value]):empty{display:none}
dt,th.tableblock,td.content,div.footnote{text-rendering:optimizeLegibility}
h1,h2,p,td.content,span.alt{letter-spacing:-.01em}
p strong,td.content strong,div.footnote strong{letter-spacing:-.005em}
p,blockquote,dt,td.content,span.alt{font-size:1.0625rem}
p{margin-bottom:1.25rem}
.sidebarblock p,.sidebarblock dt,.sidebarblock td.content,p.tableblock{font-size:1em}
.exampleblock>.content{background:#fffef7;border-color:#e0e0dc;-webkit-box-shadow:0 1px 4px #e0e0dc;box-shadow:0 1px 4px #e0e0dc}
.print-only{display:none!important}
@page{margin:1.25cm .75cm}
@media print{*{-webkit-box-shadow:none!important;box-shadow:none!important;text-shadow:none!important}
html{font-size:80%}
a{color:inherit!important;text-decoration:underline!important}
a.bare,a[href^="#"],a[href^="mailto:"]{text-decoration:none!important}
a[href^="http:"]:not(.bare)::after,a[href^="https:"]:not(.bare)::after{content:"(" attr(href) ")";display:inline-block;font-size:.875em;padding-left:.25em}
abbr[title]::after{content:" (" attr(title) ")"}
pre,blockquote,tr,img,object,svg{page-break-inside:avoid}
thead{display:table-header-group}
svg{max-width:100%}
p,blockquote,dt,td.content{font-size:1em;orphans:3;widows:3}
h2,h3,#toctitle,.sidebarblock>.content>.title{page-break-after:avoid}
#header,#content,#footnotes,#footer{max-width:none}
#toc,.sidebarblock,.exampleblock>.content{background:none!important}
#toc{border-bottom:1px solid #dddddf!important;padding-bottom:0!important}
body.book #header{text-align:center}
body.book #header>h1:first-child{border:0!important;margin:2.5em 0 1em}
body.book #header .details{border:0!important;display:block;padding:0!important}
body.book #header .details span:first-child{margin-left:0!important}
body.book #header .details br{display:block}
body.book #header .details br+span::before{content:none!important}
body.book #toc{border:0!important;text-align:left!important;padding:0!important;margin:0!important}
body.book #toc,body.book #preamble,body.book h1.sect0,body.book .sect1>h2{page-break-before:always}
.listingblock code[data-lang]::before{display:block}
#footer{padding:0 .9375em}
.hide-on-print{display:none!important}
.print-only{display:block!important}
.hide-for-print{display:none!important}
.show-for-print{display:inherit!important}}
@media print,amzn-kf8{#header>h1:first-child{margin-top:1.25rem}
.sect1{padding:0!important}
.sect1+.sect1{border:0}
#footer{background:none}
#footer-text{color:rgba(0,0,0,.6);font-size:.9em}}
@media amzn-kf8{#header,#content,#footnotes,#footer{padding:0}}
</style>
</head>
<body class="book toc2 toc-left">
<div id="header">
<h1>mzIdentML: exchange format for peptides and proteins identified from mass spectra</h1>
<div id="toc" class="toc2">
<div id="toctitle">Table of Contents</div>
<ul class="sectlevel1">
<li><a href="#_preface">Preface</a></li>
<li><a href="#abstract">Abstract</a></li>
<li><a href="#_introduction">Introduction</a>
<ul class="sectlevel2">
<li><a href="#_background">Background</a></li>
<li><a href="#_document_structure">Document Structure</a></li>
</ul>
</li>
<li><a href="#_use_cases_for_mzidentml">Use Cases for mzIdentML</a></li>
<li><a href="#_concepts_and_terminology">Concepts and Terminology</a></li>
<li><a href="#_relationship_to_other_specifications">Relationship to Other Specifications</a>
<ul class="sectlevel2">
<li><a href="#_the_psi_mass_spectrometry_controlled_vocabulary_cv">The PSI Mass Spectrometry Controlled Vocabulary (CV)</a></li>
<li><a href="#_validation_of_controlled_vocabulary_terms">Validation of controlled vocabulary terms</a></li>
<li><a href="#_changes_from_version_1_1_0_to_version_1_2_0">Changes from version 1.1.0 to version 1.2.0</a></li>
<li><a href="#_changes_from_version_1_2_0_to_1_3_0">Changes from version 1.2.0 to 1.3.0</a></li>
</ul>
</li>
<li><a href="#_format_scope_and_specific_use_cases">Format scope and specific use cases</a>
<ul class="sectlevel2">
<li><a href="#_handling_updates_to_the_controlled_vocabulary">Handling updates to the controlled vocabulary</a></li>
<li><a href="#_identifying_the_input_spectrum_for_each_result">Identifying the input spectrum for each result</a></li>
<li><a href="#_comments_on_specific_use_cases">Comments on Specific use cases</a></li>
<li><a href="#_other_supporting_materials">Other supporting materials</a></li>
</ul>
</li>
<li><a href="#_model_in_xml_schema">Model in XML Schema</a>
<ul class="sectlevel2">
<li><a href="#_element_mzidentml">Element &lt;MzIdentML&gt;</a></li>
<li><a href="#_element_additionalsearchparams">Element &lt;AdditionalSearchParams&gt;</a></li>
<li><a href="#_element_affiliation">Element &lt;Affiliation&gt;</a></li>
<li><a href="#_element_ambiguousresidue">Element &lt;AmbiguousResidue&gt;</a></li>
<li><a href="#_element_analysiscollection">Element &lt;AnalysisCollection&gt;</a></li>
<li><a href="#_element_analysisdata">Element &lt;AnalysisData&gt;</a></li>
<li><a href="#_element_analysisparams">Element &lt;AnalysisParams&gt;</a></li>
<li><a href="#_element_analysisprotocolcollection">Element &lt;AnalysisProtocolCollection&gt;</a></li>
<li><a href="#_element_analysissamplecollection">Element &lt;AnalysisSampleCollection&gt;</a></li>
<li><a href="#_element_analysissoftware">Element &lt;AnalysisSoftware&gt;</a></li>
<li><a href="#_element_analysissoftwarelist">Element &lt;AnalysisSoftwareList&gt;</a></li>
<li><a href="#_element_auditcollection">Element &lt;AuditCollection&gt;</a></li>
<li><a href="#_element_bibliographicreference">Element &lt;BibliographicReference&gt;</a></li>
<li><a href="#_element_contactrole">Element &lt;ContactRole&gt;</a></li>
<li><a href="#_element_customizations">Element &lt;Customizations&gt;</a></li>
<li><a href="#_element_cv">Element &lt;cv&gt;</a></li>
<li><a href="#_element_cvlist">Element &lt;cvList&gt;</a></li>
<li><a href="#_element_cvparam">Element &lt;cvParam&gt;</a></li>
<li><a href="#_element_databasefilters">Element &lt;DatabaseFilters&gt;</a></li>
<li><a href="#_element_databasename">Element &lt;DatabaseName&gt;</a></li>
<li><a href="#_element_databasetranslation">Element &lt;DatabaseTranslation&gt;</a></li>
<li><a href="#_element_datacollection">Element &lt;DataCollection&gt;</a></li>
<li><a href="#_element_dbsequence">Element &lt;DBSequence&gt;</a></li>
<li><a href="#_element_enzyme">Element &lt;Enzyme&gt;</a></li>
<li><a href="#_element_enzymename">Element &lt;EnzymeName&gt;</a></li>
<li><a href="#_element_enzymes">Element &lt;Enzymes&gt;</a></li>
<li><a href="#_element_exclude">Element &lt;Exclude&gt;</a></li>
<li><a href="#_element_externalformatdocumentation">Element &lt;ExternalFormatDocumentation&gt;</a></li>
<li><a href="#_element_fileformat">Element &lt;FileFormat&gt;</a></li>
<li><a href="#_element_filter">Element &lt;Filter&gt;</a></li>
<li><a href="#_element_filtertype">Element &lt;FilterType&gt;</a></li>
<li><a href="#_element_fragmentarray">Element &lt;FragmentArray&gt;</a></li>
<li><a href="#_element_fragmentation">Element &lt;Fragmentation&gt;</a></li>
<li><a href="#_element_fragmentationtable">Element &lt;FragmentationTable&gt;</a></li>
<li><a href="#_element_fragmenttolerance">Element &lt;FragmentTolerance&gt;</a></li>
<li><a href="#_element_include">Element &lt;Include&gt;</a></li>
<li><a href="#_element_inputs">Element &lt;Inputs&gt;</a></li>
<li><a href="#_element_inputspectra">Element &lt;InputSpectra&gt;</a></li>
<li><a href="#_element_inputspectrumidentifications">Element &lt;InputSpectrumIdentifications&gt;</a></li>
<li><a href="#_element_iontype">Element &lt;IonType&gt;</a></li>
<li><a href="#_element_masstable">Element &lt;MassTable&gt;</a></li>
<li><a href="#_element_measure">Element &lt;Measure&gt;</a></li>
<li><a href="#_element_modification">Element &lt;Modification&gt;</a></li>
<li><a href="#_element_modificationparams">Element &lt;ModificationParams&gt;</a></li>
<li><a href="#_element_organization">Element &lt;Organization&gt;</a></li>
<li><a href="#_element_parent">Element &lt;Parent&gt;</a></li>
<li><a href="#_element_parenttolerance">Element &lt;ParentTolerance&gt;</a></li>
<li><a href="#_element_peptide">Element &lt;Peptide&gt;</a></li>
<li><a href="#_element_peptideevidence">Element &lt;PeptideEvidence&gt;</a></li>
<li><a href="#_element_peptideevidenceref">Element &lt;PeptideEvidenceRef&gt;</a></li>
<li><a href="#_element_peptidehypothesis">Element &lt;PeptideHypothesis&gt;</a></li>
<li><a href="#_element_peptidesequence">Element &lt;PeptideSequence&gt;</a></li>
<li><a href="#_element_person">Element &lt;Person&gt;</a></li>
<li><a href="#_element_proteinambiguitygroup">Element &lt;ProteinAmbiguityGroup&gt;</a></li>
<li><a href="#_element_proteindetection">Element &lt;ProteinDetection&gt;</a></li>
<li><a href="#_element_proteindetectionhypothesis">Element &lt;ProteinDetectionHypothesis&gt;</a></li>
<li><a href="#_element_proteindetectionlist">Element &lt;ProteinDetectionList&gt;</a></li>
<li><a href="#_element_proteindetectionprotocol">Element &lt;ProteinDetectionProtocol&gt;</a></li>
<li><a href="#_element_provider">Element &lt;Provider&gt;</a></li>
<li><a href="#_element_residue">Element &lt;Residue&gt;</a></li>
<li><a href="#_element_role">Element &lt;Role&gt;</a></li>
<li><a href="#_element_sample">Element &lt;Sample&gt;</a></li>
<li><a href="#_element_searchdatabase">Element &lt;SearchDatabase&gt;</a></li>
<li><a href="#_element_searchdatabaseref">Element &lt;SearchDatabaseRef&gt;</a></li>
<li><a href="#_element_searchmodification">Element &lt;SearchModification&gt;</a></li>
<li><a href="#_element_searchtype">Element &lt;SearchType&gt;</a></li>
<li><a href="#_element_seq">Element &lt;Seq&gt;</a></li>
<li><a href="#_element_sequencecollection">Element &lt;SequenceCollection&gt;</a></li>
<li><a href="#_element_siteregexp">Element &lt;SiteRegexp&gt;</a></li>
<li><a href="#_element_softwarename">Element &lt;SoftwareName&gt;</a></li>
<li><a href="#_element_sourcefile">Element &lt;SourceFile&gt;</a></li>
<li><a href="#_element_specificityrules">Element &lt;SpecificityRules&gt;</a></li>
<li><a href="#_element_spectradata">Element &lt;SpectraData&gt;</a></li>
<li><a href="#_element_spectrumidentification">Element &lt;SpectrumIdentification&gt;</a></li>
<li><a href="#_element_spectrumidentificationitem">Element &lt;SpectrumIdentificationItem&gt;</a></li>
<li><a href="#_element_spectrumidentificationitemref">Element &lt;SpectrumIdentificationItemRef&gt;</a></li>
<li><a href="#_element_spectrumidentificationlist">Element &lt;SpectrumIdentificationList&gt;</a></li>
<li><a href="#_element_spectrumidentificationprotocol">Element &lt;SpectrumIdentificationProtocol&gt;</a></li>
<li><a href="#_element_spectrumidentificationresult">Element &lt;SpectrumIdentificationResult&gt;</a></li>
<li><a href="#_element_spectrumidformat">Element &lt;SpectrumIDFormat&gt;</a></li>
<li><a href="#_element_subsample">Element &lt;SubSample&gt;</a></li>
<li><a href="#_element_substitutionmodification">Element &lt;SubstitutionModification&gt;</a></li>
<li><a href="#_element_threshold">Element &lt;Threshold&gt;</a></li>
<li><a href="#_element_translationtable">Element &lt;TranslationTable&gt;</a></li>
<li><a href="#_element_userparam">Element &lt;userParam&gt;</a></li>
</ul>
</li>
<li><a href="#_specific_comments_on_schema">Specific Comments on schema</a>
<ul class="sectlevel2">
<li><a href="#_file_extension_and_compression">File extension and compression</a></li>
<li><a href="#_referencing_elements_within_the_document">Referencing elements within the document</a></li>
<li><a href="#_searches_against_nucleotide_sequences">Searches against nucleotide sequences</a></li>
<li><a href="#_reporting_peptide_and_protein_identifications_passing_a_significance_threshold">Reporting peptide and protein identifications passing a significance threshold</a></li>
<li><a href="#_using_decoy_databases_to_set_different_thresholds_of_false_discovery_rate">Using decoy databases to set different thresholds of false discovery rate</a></li>
<li><a href="#_database_filter">Database Filter</a></li>
<li><a href="#_types_of_parameters_and_values">Types of parameters and values</a></li>
<li><a href="#_reporting_fragmentation_ions">Reporting fragmentation ions</a></li>
<li><a href="#_enzyme_definition">Enzyme definition</a></li>
<li><a href="#_unknown_modifications">Unknown modifications</a></li>
<li><a href="#_identifications_based_on_multiple_mass_spectra">Identifications based on multiple mass spectra</a></li>
<li><a href="#_linking_searchmodification_elements_to_modification_elements">Linking SearchModification elements to Modification elements</a></li>
</ul>
</li>
<li><a href="#_conclusions">Conclusions</a></li>
<li><a href="#_authors_and_contributors">Authors and Contributors</a></li>
<li><a href="#_references">References</a></li>
<li><a href="#_intellectual_property_statement">Intellectual Property Statement</a></li>
<li><a href="#_copyright_notice">Copyright Notice</a></li>
</ul>
</div>
</div>
<div id="content">
<div class="sect1">
<h2 id="_preface">Preface</h2>
<div class="sectionbody">
<div class="paragraph">
<p><em>Status of This Document</em></p>
</div>
<div class="paragraph">
<p>This document presents a final specification for the mzIdentML data format developed by the HUPO Proteomics Standards Initiative.
Distribution is unlimited.</p>
</div>
<div class="paragraph">
<p><em>Version of This Document</em></p>
</div>
<div class="paragraph">
<p>The current version of this document is: version 1.3.0 final, June 2024.</p>
</div>
<div class="paragraph">
<p>Date created: June 24, 2024</p>
</div>
<div class="paragraph">
<p>Last updated: BUILDDATE</p>
</div>
<div class="paragraph">
<p>Based on commit: <a href="https://github.com/HUPO-PSI/mzTab/commit/COMMITHASH">COMMITHASH</a> - <a href="https://github.com/HUPO-PSI/mzTab/commits/master/specification_document-releases/2_0-Metabolomics-Release/mzTab_format_specification_2_0-M_release.adoc">Commit History</a></p>
</div>
<div class="paragraph">
<p>The current version of this document is: version version 1.3.0 final, June 2024</p>
</div>
<div class="paragraph">
<p>The latest (draft) version of this document may be found at <a href="https://github.com/HUPO-PSI/mzTab" class="bare">https://github.com/HUPO-PSI/mzTab</a>.</p>
</div>
<div class="paragraph">
<p><em>Type of This Document</em></p>
</div>
<div class="paragraph">
<p>This document is a <em>recommendation</em> for a common, community-driven standard data exchange format in proteomics.</p>
</div>
<div class="paragraph">
<p><em>Authors</em></p>
</div>
<div class="paragraph">
<p>Please see <a href="#authors">[authors]</a> for details on the authors and editors of this document.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="abstract">Abstract</h2>
<div class="sectionbody">
<div class="paragraph">
<p>The Human Proteome Organisation (HUPO) Proteomics Standards Initiative (PSI) defines community standards for data representation in proteomics to facilitate data comparison, exchange and verification.
The Proteomics Informatics Working Group is developing standards for describing the results of identification and quantitation processes for proteins, peptides and protein modifications from mass spectrometry.
This document defines an XML schema that can be used to describe the outputs of proteomics search engines or similar software for peptide/protein identification from mass spectrometry (MS) data.</p>
</div>
<div class="paragraph">
<p><strong>Contents</strong></p>
</div>
<div class="paragraph">
<p><a href="#abstract">Abstract 1</a></p>
</div>
<div class="paragraph">
<p><a href="#introduction">1. Introduction 4</a></p>
</div>
<div class="paragraph">
<p><a href="#background">1.1 Background 4</a></p>
</div>
<div class="paragraph">
<p><a href="#document-structure">1.2 Document Structure 4</a></p>
</div>
<div class="paragraph">
<p><a href="#use-cases-for-mzidentml">2. Use Cases for mzIdentML 4</a></p>
</div>
<div class="paragraph">
<p><a href="#concepts-and-terminology">3. Concepts and Terminology 6</a></p>
</div>
<div class="paragraph">
<p><a href="#relationship-to-other-specifications">4. Relationship to Other Specifications 6</a></p>
</div>
<div class="paragraph">
<p><a href="#the-psi-mass-spectrometry-controlled-vocabulary-cv">4.1 The PSI Mass Spectrometry Controlled Vocabulary (CV) 7</a></p>
</div>
<div class="paragraph">
<p><a href="#validation-of-controlled-vocabulary-terms">4.2 Validation of controlled vocabulary terms 7</a></p>
</div>
<div class="paragraph">
<p><a href="#changes-from-version-1.1.0-to-version-1.2.0">4.3 Changes from version 1.1.0 to version 1.2.0 8</a></p>
</div>
<div class="paragraph">
<p><a href="#changes-from-version-1.2.0-to-1.3.0">4.4 Changes from version 1.2.0 to 1.3.0 8</a></p>
</div>
<div class="paragraph">
<p><a href="#format-scope-and-specific-use-cases">5. Format scope and specific use cases 9</a></p>
</div>
<div class="paragraph">
<p><a href="#handling-updates-to-the-controlled-vocabulary">5.1.1 Handling updates to the controlled vocabulary 9</a></p>
</div>
<div class="paragraph">
<p><a href="#identifying-the-input-spectrum-for-each-result">5.1.2 Identifying the input spectrum for each result 9</a></p>
</div>
<div class="paragraph">
<p><a href="#comments-on-specific-use-cases">5.2 Comments on Specific use cases 11</a></p>
</div>
<div class="paragraph">
<p><a href="#protein-grouping-encoding">5.2.1 Protein grouping encoding 11</a></p>
</div>
<div class="paragraph">
<p><a href="#support-for-de-novo-sequencing-approaches">5.2.2 Support for <em>de novo</em> sequencing approaches 15</a></p>
</div>
<div class="paragraph">
<p><a href="#spectral-library-searches">5.2.3 Spectral library searches 15</a></p>
</div>
<div class="paragraph">
<p><a href="#multiple-database-search-engines">5.2.4 Multiple database search engines 16</a></p>
</div>
<div class="paragraph">
<p><a href="#pre-fractionation-of-samples-prior-to-ms-and-splitting-of-searches">5.2.5 Pre-fractionation of samples prior to MS and splitting of searches 16</a></p>
</div>
<div class="paragraph">
<p><a href="#encoding-replicate-samples">5.2.6 Encoding replicate samples 17</a></p>
</div>
<div class="paragraph">
<p><a href="#peptide-level-scores-and-statistical-measures">5.2.7 Peptide-level scores and statistical measures 17</a></p>
</div>
<div class="paragraph">
<p><a href="#encoding-modification-localisation-scores">5.2.8 Encoding modification localisation scores 20</a></p>
</div>
<div class="paragraph">
<p><a href="#encoding-results-of-crosslinking-searches">5.2.9 Encoding results of crosslinking searches 22</a></p>
</div>
<div class="paragraph">
<p><a href="#encoding-proteogenomics-annotation-data">5.2.10 Encoding proteogenomics annotation data 22</a></p>
</div>
<div class="paragraph">
<p><a href="#other-supporting-materials">5.3 Other supporting materials 24</a></p>
</div>
<div class="paragraph">
<p><a href="#model-in-xml-schema">6. Model in XML Schema 24</a></p>
</div>
<div class="paragraph">
<p><a href="#element-mzidentml">6.1 Element &lt;MzIdentML&gt; 24</a></p>
</div>
<div class="paragraph">
<p><a href="#element-additionalsearchparams">6.2 Element &lt;AdditionalSearchParams&gt; 26</a></p>
</div>
<div class="paragraph">
<p><a href="#element-affiliation">6.3 Element &lt;Affiliation&gt; 28</a></p>
</div>
<div class="paragraph">
<p><a href="#element-ambiguousresidue">6.4 Element &lt;AmbiguousResidue&gt; 28</a></p>
</div>
<div class="paragraph">
<p><a href="#element-analysiscollection">6.5 Element &lt;AnalysisCollection&gt; 29</a></p>
</div>
<div class="paragraph">
<p><a href="#element-analysisdata">6.6 Element &lt;AnalysisData&gt; 29</a></p>
</div>
<div class="paragraph">
<p><a href="#element-analysisparams">6.7 Element &lt;AnalysisParams&gt; 30</a></p>
</div>
<div class="paragraph">
<p><a href="#element-analysisprotocolcollection">6.8 Element &lt;AnalysisProtocolCollection&gt; 30</a></p>
</div>
<div class="paragraph">
<p><a href="#element-analysissamplecollection">6.9 Element &lt;AnalysisSampleCollection&gt; 31</a></p>
</div>
<div class="paragraph">
<p><a href="#element-analysissoftware">6.10 Element &lt;AnalysisSoftware&gt; 31</a></p>
</div>
<div class="paragraph">
<p><a href="#element-analysissoftwarelist">6.11 Element &lt;AnalysisSoftwareList&gt; 32</a></p>
</div>
<div class="paragraph">
<p><a href="#element-auditcollection">6.12 Element &lt;AuditCollection&gt; 32</a></p>
</div>
<div class="paragraph">
<p><a href="#element-bibliographicreference">6.13 Element &lt;BibliographicReference&gt; 33</a></p>
</div>
<div class="paragraph">
<p><a href="#element-contactrole">6.14 Element &lt;ContactRole&gt; 33</a></p>
</div>
<div class="paragraph">
<p><a href="#element-customizations">6.15 Element &lt;Customizations&gt; 34</a></p>
</div>
<div class="paragraph">
<p><a href="#element-cv">6.16 Element &lt;cv&gt; 34</a></p>
</div>
<div class="paragraph">
<p><a href="#element-cvlist">6.17 Element &lt;cvList&gt; 34</a></p>
</div>
<div class="paragraph">
<p><a href="#element-cvparam">6.18 Element &lt;cvParam&gt; 35</a></p>
</div>
<div class="paragraph">
<p><a href="#element-databasefilters">6.19 Element &lt;DatabaseFilters&gt; 35</a></p>
</div>
<div class="paragraph">
<p><a href="#element-databasename">6.20 Element &lt;DatabaseName&gt; 36</a></p>
</div>
<div class="paragraph">
<p><a href="#element-databasetranslation">6.21 Element &lt;DatabaseTranslation&gt; 36</a></p>
</div>
<div class="paragraph">
<p><a href="#element-datacollection">6.22 Element &lt;DataCollection&gt; 37</a></p>
</div>
<div class="paragraph">
<p><a href="#element-dbsequence">6.23 Element &lt;DBSequence&gt; 38</a></p>
</div>
<div class="paragraph">
<p><a href="#element-enzyme">6.24 Element &lt;Enzyme&gt; 40</a></p>
</div>
<div class="paragraph">
<p><a href="#element-enzymename">6.25 Element &lt;EnzymeName&gt; 40</a></p>
</div>
<div class="paragraph">
<p><a href="#element-enzymes">6.26 Element &lt;Enzymes&gt; 41</a></p>
</div>
<div class="paragraph">
<p><a href="#element-exclude">6.27 Element &lt;Exclude&gt; 41</a></p>
</div>
<div class="paragraph">
<p><a href="#element-externalformatdocumentation">6.28 Element &lt;ExternalFormatDocumentation&gt; 42</a></p>
</div>
<div class="paragraph">
<p><a href="#element-fileformat">6.29 Element &lt;FileFormat&gt; 42</a></p>
</div>
<div class="paragraph">
<p><a href="#element-filter">6.30 Element &lt;Filter&gt; 43</a></p>
</div>
<div class="paragraph">
<p><a href="#element-filtertype">6.31 Element &lt;FilterType&gt; 43</a></p>
</div>
<div class="paragraph">
<p><a href="#element-fragmentarray">6.32 Element &lt;FragmentArray&gt; 44</a></p>
</div>
<div class="paragraph">
<p><a href="#element-fragmentation">6.33 Element &lt;Fragmentation&gt; 44</a></p>
</div>
<div class="paragraph">
<p><a href="#element-fragmentationtable">6.34 Element &lt;FragmentationTable&gt; 44</a></p>
</div>
<div class="paragraph">
<p><a href="#element-fragmenttolerance">6.35 Element &lt;FragmentTolerance&gt; 45</a></p>
</div>
<div class="paragraph">
<p><a href="#element-include">6.36 Element &lt;Include&gt; 45</a></p>
</div>
<div class="paragraph">
<p><a href="#element-inputs">6.37 Element &lt;Inputs&gt; 46</a></p>
</div>
<div class="paragraph">
<p><a href="#element-inputspectra">6.38 Element &lt;InputSpectra&gt; 46</a></p>
</div>
<div class="paragraph">
<p><a href="#element-inputspectrumidentifications">6.39 Element &lt;InputSpectrumIdentifications&gt; 46</a></p>
</div>
<div class="paragraph">
<p><a href="#element-iontype">6.40 Element &lt;IonType&gt; 47</a></p>
</div>
<div class="paragraph">
<p><a href="#element-masstable">6.41 Element &lt;MassTable&gt; 49</a></p>
</div>
<div class="paragraph">
<p><a href="#element-measure">6.42 Element &lt;Measure&gt; 49</a></p>
</div>
<div class="paragraph">
<p><a href="#element-modification">6.43 Element &lt;Modification&gt; 50</a></p>
</div>
<div class="paragraph">
<p><a href="#element-modificationparams">6.44 Element &lt;ModificationParams&gt; 51</a></p>
</div>
<div class="paragraph">
<p><a href="#element-organization">6.45 Element &lt;Organization&gt; 52</a></p>
</div>
<div class="paragraph">
<p><a href="#element-parent">6.46 Element &lt;Parent&gt; 53</a></p>
</div>
<div class="paragraph">
<p><a href="#element-parenttolerance">6.47 Element &lt;ParentTolerance&gt; 53</a></p>
</div>
<div class="paragraph">
<p><a href="#element-peptide">6.48 Element &lt;Peptide&gt; 53</a></p>
</div>
<div class="paragraph">
<p><a href="#element-peptideevidence">6.49 Element &lt;PeptideEvidence&gt; 55</a></p>
</div>
<div class="paragraph">
<p><a href="#element-peptideevidenceref">6.50 Element &lt;PeptideEvidenceRef&gt; 56</a></p>
</div>
<div class="paragraph">
<p><a href="#element-peptidehypothesis">6.51 Element &lt;PeptideHypothesis&gt; 57</a></p>
</div>
<div class="paragraph">
<p><a href="#element-peptidesequence">6.52 Element &lt;PeptideSequence&gt; 58</a></p>
</div>
<div class="paragraph">
<p><a href="#element-person">6.53 Element &lt;Person&gt; 58</a></p>
</div>
<div class="paragraph">
<p><a href="#element-proteinambiguitygroup">6.54 Element &lt;ProteinAmbiguityGroup&gt; 59</a></p>
</div>
<div class="paragraph">
<p><a href="#element-proteindetection">6.55 Element &lt;ProteinDetection&gt; 60</a></p>
</div>
<div class="paragraph">
<p><a href="#element-proteindetectionhypothesis">6.56 Element &lt;ProteinDetectionHypothesis&gt; 60</a></p>
</div>
<div class="paragraph">
<p><a href="#element-proteindetectionlist">6.57 Element &lt;ProteinDetectionList&gt; 62</a></p>
</div>
<div class="paragraph">
<p><a href="#element-proteindetectionprotocol">6.58 Element &lt;ProteinDetectionProtocol&gt; 63</a></p>
</div>
<div class="paragraph">
<p><a href="#element-provider">6.59 Element &lt;Provider&gt; 64</a></p>
</div>
<div class="paragraph">
<p><a href="#element-residue">6.60 Element &lt;Residue&gt; 65</a></p>
</div>
<div class="paragraph">
<p><a href="#element-role">6.61 Element &lt;Role&gt; 65</a></p>
</div>
<div class="paragraph">
<p><a href="#element-sample">6.62 Element &lt;Sample&gt; 65</a></p>
</div>
<div class="paragraph">
<p><a href="#element-searchdatabase">6.63 Element &lt;SearchDatabase&gt; 66</a></p>
</div>
<div class="paragraph">
<p><a href="#element-searchdatabaseref">6.64 Element &lt;SearchDatabaseRef&gt; 67</a></p>
</div>
<div class="paragraph">
<p><a href="#element-searchmodification">6.65 Element &lt;SearchModification&gt; 68</a></p>
</div>
<div class="paragraph">
<p><a href="#element-searchtype">6.66 Element &lt;SearchType&gt; 69</a></p>
</div>
<div class="paragraph">
<p><a href="#element-seq">6.67 Element &lt;Seq&gt; 69</a></p>
</div>
<div class="paragraph">
<p><a href="#element-sequencecollection">6.68 Element &lt;SequenceCollection&gt; 70</a></p>
</div>
<div class="paragraph">
<p><a href="#element-siteregexp">6.69 Element &lt;SiteRegexp&gt; 72</a></p>
</div>
<div class="paragraph">
<p><a href="#element-softwarename">6.70 Element &lt;SoftwareName&gt; 72</a></p>
</div>
<div class="paragraph">
<p><a href="#element-sourcefile">6.71 Element &lt;SourceFile&gt; 72</a></p>
</div>
<div class="paragraph">
<p><a href="#element-specificityrules">6.72 Element &lt;SpecificityRules&gt; 73</a></p>
</div>
<div class="paragraph">
<p><a href="#element-spectradata">6.73 Element &lt;SpectraData&gt; 74</a></p>
</div>
<div class="paragraph">
<p><a href="#element-spectrumidentification">6.74 Element &lt;SpectrumIdentification&gt; 74</a></p>
</div>
<div class="paragraph">
<p><a href="#element-spectrumidentificationitem">6.75 Element &lt;SpectrumIdentificationItem&gt; 75</a></p>
</div>
<div class="paragraph">
<p><a href="#element-spectrumidentificationitemref">6.76 Element &lt;SpectrumIdentificationItemRef&gt; 78</a></p>
</div>
<div class="paragraph">
<p><a href="#element-spectrumidentificationlist">6.77 Element &lt;SpectrumIdentificationList&gt; 79</a></p>
</div>
<div class="paragraph">
<p><a href="#element-spectrumidentificationprotocol">6.78 Element &lt;SpectrumIdentificationProtocol&gt; 80</a></p>
</div>
<div class="paragraph">
<p><a href="#element-spectrumidentificationresult">6.79 Element &lt;SpectrumIdentificationResult&gt; 83</a></p>
</div>
<div class="paragraph">
<p><a href="#element-spectrumidformat">6.80 Element &lt;SpectrumIDFormat&gt; 84</a></p>
</div>
<div class="paragraph">
<p><a href="#element-subsample">6.81 Element &lt;SubSample&gt; 84</a></p>
</div>
<div class="paragraph">
<p><a href="#element-substitutionmodification">6.82 Element &lt;SubstitutionModification&gt; 85</a></p>
</div>
<div class="paragraph">
<p><a href="#element-threshold">6.83 Element &lt;Threshold&gt; 85</a></p>
</div>
<div class="paragraph">
<p><a href="#element-translationtable">6.84 Element &lt;TranslationTable&gt; 87</a></p>
</div>
<div class="paragraph">
<p><a href="#element-userparam">6.85 Element &lt;userParam&gt; 87</a></p>
</div>
<div class="paragraph">
<p><a href="#specific-comments-on-schema">7. Specific Comments on schema 88</a></p>
</div>
<div class="paragraph">
<p><a href="#file-extension-and-compression">7.1 File extension and compression 88</a></p>
</div>
<div class="paragraph">
<p><a href="#referencing-elements-within-the-document">7.2 Referencing elements within the document 88</a></p>
</div>
<div class="paragraph">
<p><a href="#searches-against-nucleotide-sequences">7.3 Searches against nucleotide sequences 88</a></p>
</div>
<div class="paragraph">
<p><a href="#reporting-peptide-and-protein-identifications-passing-a-significance-threshold">7.4 Reporting peptide and protein identifications passing a significance threshold 89</a></p>
</div>
<div class="paragraph">
<p><a href="#using-decoy-databases-to-set-different-thresholds-of-false-discovery-rate">7.5 Using decoy databases to set different thresholds of false discovery rate 90</a></p>
</div>
<div class="paragraph">
<p><a href="#database-filter">7.6 Database Filter 91</a></p>
</div>
<div class="paragraph">
<p><a href="#types-of-parameters-and-values">7.7 Types of parameters and values 91</a></p>
</div>
<div class="paragraph">
<p><a href="#reporting-fragmentation-ions">7.8 Reporting fragmentation ions 91</a></p>
</div>
<div class="paragraph">
<p><a href="#internal-fragments-and-immonium-ions">7.8.1 Internal fragments and immonium ions 92</a></p>
</div>
<div class="paragraph">
<p><a href="#encoding-neutral-loss-fragment-ions">7.8.2 Encoding Neutral loss fragment ions 92</a></p>
</div>
<div class="paragraph">
<p><a href="#enzyme-definition">7.9 Enzyme definition 93</a></p>
</div>
<div class="paragraph">
<p><a href="#unknown-modifications">7.10 Unknown modifications 94</a></p>
</div>
<div class="paragraph">
<p><a href="#identifications-based-on-multiple-mass-spectra">7.11 Identifications based on multiple mass spectra 94</a></p>
</div>
<div class="paragraph">
<p><a href="#linking-searchmodification-elements-to-modification-elements">7.12 Linking SearchModification elements to Modification elements 98</a></p>
</div>
<div class="paragraph">
<p><a href="#conclusions">8. Conclusions 99</a></p>
</div>
<div class="paragraph">
<p><a href="#authors-and-contributors">9. Authors and Contributors 99</a></p>
</div>
<div class="paragraph">
<p><a href="#references">10. References 101</a></p>
</div>
<div class="paragraph">
<p><a href="#intellectual-property-statement">11. Intellectual Property Statement 101</a></p>
</div>
<div class="paragraph">
<p><a href="#copyright-notice">Copyright Notice 102</a></p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_introduction">Introduction</h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_background">Background</h3>
<div class="paragraph">
<p>This document addresses the systematic description of (poly)peptide identification and characterisation based upon mass spectrometry (MS).
A large number of different proteomics search engines are available that produce output in a variety of different formats.
It is intended that mzIdentML will provide a common standard format for identification results to support a range of scenarios encountered in proteome informatics. mzIdentML has been developed with a view to supporting the following general tasks (more specific use cases are provided in Section 2.):</p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p><em>The discovery of relevant results,</em> so that, for example, data sets in a database that use a particular technique or combination of techniques can be identified and studied by experimentalists during experiment design or data analysis.</p>
</li>
<li>
<p><em>The sharing of best practice</em>, so that, for example, analyses that have been particularly successful at identifying a certain group of peptides/proteins can be interpreted by consumers of the data.</p>
</li>
<li>
<p><em>The evaluation of results</em>, so that, for example, sufficient information is provided about how a particular analysis was performed to allow the results to be critically evaluated.</p>
</li>
<li>
<p><em>The sharing of data sets,</em> so that, for example, public repositories can import or export data, or multi-site projects can share results to support integrated analysis.</p>
</li>
<li>
<p><em>The creation of a format for input to analysis software</em>, for example, allowing software to be designed that provides a meta-score over the output from several search engines.</p>
</li>
<li>
<p><em>An internal format for pipeline analysis software, for example, allowing analysis software to store intermediate results from different stages of an identification pipeline, prior to the final results being assembled in a single mzIdentML file.</em></p>
</li>
</ol>
</div>
<div class="paragraph">
<p>This document presents a specification, not a tutorial.
As such, the presentation of technical details is deliberately direct.
The role of the text is to describe the model and justify design decisions made.
The document does not discuss how the models should be used in practice, consider tool support for data capture or storage, or provide comprehensive examples of the models in use.
Tutorial material can be located on the PSI website (see <a href="http://www.psidev.info/mzidentml" class="bare">http://www.psidev.info/mzidentml</a>).</p>
</div>
</div>
<div class="sect2">
<h3 id="_document_structure">Document Structure</h3>
<div class="paragraph">
<p>The remainder of this document is structured as follows.
Section 2. lists use cases mzIdentML is designed to support.
Section 3. describes the terminology used.
Section 4. describes how the specification presented in Section 4. relates to other specifications, both those that it extends and those that it is intended to complement.
Section 6. contains the documentation for the XML schema which is generated automatically and several parts of the schema are documented in more detail in Section 7. Conclusions are presented in Section 8..</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_use_cases_for_mzidentml">Use Cases for mzIdentML</h2>
<div class="sectionbody">
<div class="paragraph">
<p>The following use cases have driven the development of the mzIdentML data model and XML schema, and are used to define the scope of the format in the current version.</p>
</div>
<div class="paragraph">
<p>It should be possible to create a tool that loads an mzIdentML document and enables users to examine results from an MS or MSn analysis.
As of mzIdentML version 1.3.0, there is support for aggregating evidence from multiple MS levels by using the encoding for identifications based on multiple spectra given in Section 7.11. There should be sufficient information for the tool to generate output reports that conform to the requirements made by journals for publication and that conform to the relevant MIAPE guidelines.
For example:<br>
· For a Peptide Mass Fingerprint (PMF) search, it should be possible to display the spectrum and show the matches of the peaks to the relevant peptides.<br>
· For an MS/MS search, it should be possible to locate which spectrum matched to which peptide in the original file.</p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p>There should be sufficient information stored in the instance document to enable a user to run the same or a similar search on the same or another search engine.
This means that all search parameters should be described in sufficient detail and that sufficient information is available to determine which database (if any) the data were searched against.
The peak lists data do not need to be included in the instance document, but do need to be suitably referenced.</p>
</li>
<li>
<p>It should be possible to save the results of searching a decoy database in the same instance document as the results from the target database.
It should then be possible to write a viewer application that enables a user to investigate the effect of changing, for example, a threshold value on the false discovery rate.
This would only be possible if results that are generally considered lower quality from the search are also saved in the mzIdentML document (rather than just top matches) and if the results from the decoy search are also saved.
It would only be possible to do this at the peptide level for an MS/MS search, because changing thresholds would normally have some effect on the protein grouping algorithm.</p>
</li>
<li>
<p>It should be possible to save manual or automated annotation of proteins/peptides in an instance document.
A third-party tool could be used to save annotations and validations of identified proteins/peptides to an existing instance document.</p>
</li>
<li>
<p>It should be possible to save the results from a search of a metabolically labelled sample.
For example, with a 14N/15N experiment, two separate sets of amino acid masses are used, and it must be possible to tell which masses were used for each peptide result.</p>
</li>
<li>
<p>For a search of multiple peak lists, it should be possible to identify the spectrum that matched a particular peptide or protein reported by the search engine.
For example, in an LC-MS/MS run, it should be possible to refer back to the spectrum in the peak list file that was searched and from there, if the information is available, to be able to determine the retention time of the spectrum.</p>
</li>
<li>
<p>It should be possible to search a file to retrieve all molecules that have a specified modification.</p>
</li>
<li>
<p>It should be possible to store the results of a search of spectra against other spectra – <em>i.e.</em>, a spectral library search.</p>
</li>
<li>
<p>It should be possible to store the results of a top-down search, <em>i.e.</em>, analysis of complete proteins.</p>
</li>
<li>
<p>Support should be provided for storing fragmentation data so that for example viewers could display which ions in the input data match predicted ion fragment masses.</p>
</li>
<li>
<p>There should be support for storing the results of searches of peptides against nucleic acid databases, including the information about which translation frame the matches were found in.</p>
</li>
<li>
<p>It should be possible to combine the results from multiple search engines into one mzIdentML document.
For example, the peptide spectrum matches (PSMs) from two or more different search engines could be combined using a third tool to give one set of protein results.</p>
</li>
<li>
<p>It should be possible to store <em>de novo</em> peptide sequencing results, to the extent that it will be possible to enumerate and record all possible matches found by a <em>de novo</em> technique.
However, we anticipate that this can produce large files.</p>
</li>
<li>
<p>It should be possible to store the results of MS/MS crosslinking approaches, whereby two peptides ed using chemical reagents or biologically occurring modifications have been identified (newly added in mzIdentML 1.2).
New use cases for crosslinking data have been added in mzIdentML 1.3. From this version, this information is available in the mzIdentML crosslinking extension document, and not in this main specification document.</p>
</li>
<li>
<p>It should be possible to store at a basic level of detail the molecular interaction data that can be inferred from crosslinking approaches (newly added in mzIdentML 1.2).</p>
</li>
<li>
<p>It should be possible to represent statistical values or scores associated with the positions of modifications on a peptide chain (newly added in mzIdentML 1.2).</p>
</li>
<li>
<p>It should be possible to represent statistical values or scores associated with peptide identifications, formed from groups of redundant peptide-spectrum matches (PSMs) reporting on the same peptide (newly added in mzIdentML 1.2).</p>
</li>
<li>
<p>It should be possible to capture the output of proteogenomics analyses such as the mapping of peptides to gene models and chromosomes (newly added in mzIdentML 1.2).</p>
</li>
</ol>
</div>
<div class="paragraph">
<p>There should be limited support for:</p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p>Sequence tagging, in which short sequences defined by a <em>de novo</em> process are used to characterize spectra.
The final results from a sequence-tag-filtered search can be stored in mzIdentML, but the details of tag generation and filtering cannot.</p>
</li>
</ol>
</div>
<div class="paragraph">
<p>The following use cases <span class="underline">will not be</span> supported in version 1.2 of mzIdentML:</p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p>It should be possible to store relative and absolute quantitation information at the peptide and protein level using all the popular techniques – this is captured in mzQuantML and also in mzTab.</p>
</li>
<li>
<p>Support for complex workflows where multiple data processing algorithms are combined in a pipeline, <em>i.e.</em>, only “final” results are represented in mzIdentML v1.2 in one protein list.</p>
</li>
</ol>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_concepts_and_terminology">Concepts and Terminology</h2>
<div class="sectionbody">
<div class="paragraph">
<p>This document assumes familiarity with XML Schema notation (<a href="http://www.w3.org/XML/Schema">www.w3.org/XML/Schema</a>).
The key words “MUST,” “MUST NOT,” “REQUIRED,” “SHALL,” “SHALL NOT,” “SHOULD,” “SHOULD NOT,” “RECOMMENDED,” “MAY,” and “OPTIONAL” are to be interpreted as described in RFC-2119 (<a href="http://www.ietf.org/rfc/rfc2119.txt" class="bare">http://www.ietf.org/rfc/rfc2119.txt</a>).</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_relationship_to_other_specifications">Relationship to Other Specifications</h2>
<div class="sectionbody">
<div class="paragraph">
<p>The specification described in this document is not being developed in isolation; indeed, it is designed to be complementary to, and thus used in conjunction with, several existing and emerging models.
Related specifications include the following:</p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p><em>MIAPE MSI</em> (<a href="http://www.psidev.info/groups/miape" class="bare">http://www.psidev.info/groups/miape</a>) The Minimum Information About a Proteomics Experiment: Mass spectrometry Informatics document defines a checklist of information that should be reported about such a study.</p>
</li>
<li>
<p><em>mzML</em> (<a href="http://www.psidev.info/mzml" class="bare">http://www.psidev.info/mzml</a>). mzML is the PSI standard for capturing mass spectra / peak lists resulting from MS in proteomics.
It is RECOMMENDED that mzIdentML should be used in conjunction with mzML, although it is possible to use mzIdentML with other formats of mass spectra.
This document does not assume familiarity with mzML (1).</p>
</li>
<li>
<p><em>mzQuantML</em> (<a href="http://www.psidev.info/mzquantml" class="bare">http://www.psidev.info/mzquantml</a>). mzQuantML is the PSI standard for capturing quantitative proteomics data from MS (2) mzQuantML files that report quantitative data MAY reference mzIdentML files containing the detailed identification data.</p>
</li>
<li>
<p><em>mzTab</em> (<a href="http://www.psidev.info/mztab" class="bare">http://www.psidev.info/mztab</a>). mzTab is the PSI standard that can represent identification and quantification results at different levels of detail, in a tab-delimited format (3). mzTab results can represent a subset of all the information included in a mzidentML file.
However, in mzTab 1.0 while there is support for some of the new features represented in mzidentML 1.2 (e.g. ambiguity in the modification position), other features are not explicitly supported (e.g. crosslinking). mzTab files MAY reference mzIdentML files.</p>
</li>
<li>
<p><em>PSI-MI XML / MITAB</em> (<a href="http://www.psidev.info/groups/molecular-interactions" class="bare">http://www.psidev.info/groups/molecular-interactions</a>).
The PSI has developed specifications for molecular interaction evidence in XML format (PSI-MI) and tab-separated (MITAB).
When MS crosslinking data is stored in mzIdentML, including evidence for protein interactions, the file can act in a complementary manner (e.g. providing source data) to a PSI-MI (4) or MITAB file.</p>
</li>
</ol>
</div>
<div class="sect2">
<h3 id="_the_psi_mass_spectrometry_controlled_vocabulary_cv">The PSI Mass Spectrometry Controlled Vocabulary (CV)</h3>
<div class="paragraph">
<p>The PSI-MS controlled vocabulary (CV) (5) is intended to provide terms for annotation of mzML, mzIdentML, mzQuantML and other PSI standard file formats.
The CV has been generated by collection of terms from software vendors and academic groups working in the area of MS and proteome informatics.
Some terms describe attributes that must be coupled with a numerical value attribute in the &lt;cvParam&gt; element (e.g. MS:1001191 “p-value”) and optionally a unit for that value (e.g. MS:1001117, “theoretical mass”, units = dalton).
The terms that require a value are denoted by having a “datatype” key-value pair in the CV itself: MS:1001172 "mascot:expectation value" value-type:xsd:double.
Terms that need to be qualified with units are denoted by having a “has_units” key in the CV itself (relationship: has_units: UO:0000221 ! dalton).
The details of which terms are allowed or required in a given schema section is reported in the mapping file (Section 4.2).</p>
</div>
<div class="paragraph">
<p>As recommended by the PSI CV guidelines, psi-ms.obo should be dynamically maintained <em>via</em> the <a href="mailto:psidev-ms-vocab@lists.sourceforge.net">psidev-ms-vocab@lists.sourceforge.net</a> mailing list that allows any user to request new terms, in agreement with the community involved.
Once a consensus is reached among the community the new terms are added within a few business days.</p>
</div>
<div class="paragraph">
<p>In general, modifications SHOULD be sourced from Unimod (<a href="http://www.unimod.org/obo/unimod.obo" class="bare">http://www.unimod.org/obo/unimod.obo</a>) where possible.
For encoding crosslinking results, the XLMOD-CV SHOULD be used (<a href="https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/master/cv/XLMOD.obo" class="bare">https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/master/cv/XLMOD.obo</a>), unless suitable terms can be obtained from Unimod at a later date.</p>
</div>
<div class="paragraph">
<p>The following ontologies or controlled vocabularies specified below may also be suitable or required in certain instances:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Unit Ontology (<a href="http://bioportal.bioontology.org/ontologies/UO?p=classes&amp;conceptid=root" class="bare">http://bioportal.bioontology.org/ontologies/UO?p=classes&amp;conceptid=root</a>).</p>
</li>
<li>
<p>ChEBI (<a href="http://www.ebi.ac.uk/chebi/" class="bare">http://www.ebi.ac.uk/chebi/</a>).</p>
</li>
<li>
<p>For describing sample types, any suitable and stable ontologies MAY be used.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>The PSI Protein modifications CV (<a href="http://psidev.cvs.sourceforge.net/viewvc/psidev/psi/mod/data/PSI-MOD.obo" class="bare">http://psidev.cvs.sourceforge.net/viewvc/psidev/psi/mod/data/PSI-MOD.obo</a>) is now DEPRECATED for use in mzIdentML.
It is RECOMMENDED to use Unimod wherever possible.</p>
</div>
</div>
<div class="sect2">
<h3 id="_validation_of_controlled_vocabulary_terms">Validation of controlled vocabulary terms</h3>
<div class="paragraph">
<p>The correct usage of controlled vocabulary terms within mzIdentML is governed by the use of a mapping file that defines each XML location (XPath) where a &lt;cvParam&gt; instance can be used, and the allowed terms from the PSI-MS, or other CVs.
The mapping file is read and interpreted by validation software, checking that the data annotation is consistent.
The mapping file needs to be checked and updated when the structure of the CV is changed, and in some instances when new terms are added to the CV.
XML paths are associated with CV terms along with a requirement level (MAY, SHOULD or MUST) defining what should be reported by validation software if one of the mapped terms is not provided in an instance document.
Syntactic and semantic validation SHOULD be checked using the official Java-based validator available from <a href="http://www.psidev.info/mzidentml" class="bare">http://www.psidev.info/mzidentml</a>.</p>
</div>
</div>
<div class="sect2">
<h3 id="_changes_from_version_1_1_0_to_version_1_2_0">Changes from version 1.1.0 to version 1.2.0</h3>
<div class="paragraph">
<p>The primary update requiring the change from version 1.1.0 to version 1.2.0 is in the inclusion of guidelines for encoding protein group results (Section 5.2.1).
Several examples referenced throughout the document are annotated with version 1.1.0. In these cases, it can be assumed that these files are also valid 1.2.0 files, since they do not include protein inference results.
Other minor changes have been made to the specification since version 1.1.0, with regards to the encoding of specific workflows – notably searches where pre-fractionation has been performed (Section 5.2.5), searches employing multiple search engines (Section 5.2.4), <em>de novo</em> sequencing (Section 5.2.2) and spectral library searches (5.2.3).</p>
</div>
<div class="paragraph">
<p>Several new CV terms are now mandatory within the &lt;SpectrumIdentificationProtocol&gt; element - enabling the new features to be differentiated and recognised automatically by processing software, as follows. <strong>1…<em>n</em> of the following terms MUST be present:</strong></p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 33.3333%;">
<col style="width: 33.3333%;">
<col style="width: 33.3334%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>CV term name</strong></th>
<th class="tableblock halign-left valign-top"><strong>Accession</strong></th>
<th class="tableblock halign-left valign-top"><strong>Reference to section within this document</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>peptide-level scoring</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1002490</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">5.2.7</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>modification localization scoring</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1002491</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">5.2.8</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>consensus scoring</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1002492</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">5.2.4</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>sample pre-fractionation</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1002493</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">5.2.5</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>crosslinking search</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1002494</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">5.2.9</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>de novo search</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1001010</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">5.2.2</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>spectral library search</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1001031</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">5.2.3</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>proteogenomics search</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1002635</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">5.2.10</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>no special processing</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1002495</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Used to indicate that none of the above (new) features have been included in the file.</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>Table 1 New CV terms now mandatory (1&#8230;&#8203;<em>n</em> terms MUST be present) within the &lt;SpectrumIdentificationProtocol&gt; element in mzIdentML 1.2. Terms “de novo search” and “spectral library search” MUST appear under the &lt;SearchType&gt; element.
All other terms MUST appear under the &lt;AdditionalSearchParams&gt; element.</p>
</div>
</div>
<div class="sect2">
<h3 id="_changes_from_version_1_2_0_to_1_3_0">Changes from version 1.2.0 to 1.3.0</h3>
<div class="paragraph">
<p>The core of this specification document remains unchanged from 1.2.0, but version 1.3.0 now supports extensions for additional features or use cases, described in additional extension documents located in the same folder.
At the time of writing, an extension for crosslinking data exists and an extension for glycopeptides is in progress, but others may take place in the future.</p>
</div>
<div class="paragraph">
<p>It is important to highlight that extension documents cannot introduce any changes to the mzIdentML schema.
As such they are primarily about introducing and describing the use of CV terms.</p>
</div>
<div class="paragraph">
<p>Implementers only need to incorporate the extensions if supporting the specific extra features or use cases described there.
A document signals which extensions it requires by including &lt;cvParam&gt; elements that are children of the term MS:1003373, "mzIdentML extension version", immediately after the &lt;cvList&gt; element inside the &lt;MzIdentML&gt; element.</p>
</div>
<div class="paragraph">
<p>This is the only change to the XML schema definition for mzIdentML 1.3.0: &lt;cvParam&gt; elements can be included immediately after the &lt;cvList&gt; element inside the top level &lt;MzIdetnML&gt; element, to permit declaring the version of extension documents.</p>
</div>
<div class="paragraph">
<p>A new section has also been added to explain how to encode identifications coming from multiple spectra (Section 7.11).
The “combined spectra” type of input file format from version 1.2.0 has been retired and is not part of the 1.3.0 specification.</p>
</div>
<div class="paragraph">
<p>Additionally, two new CV terms have been introduced to provide an optional mechanism for linking the &lt;Modification&gt; elements inside &lt;Peptide&gt; elements to &lt;SearchModification&gt; elements (Section 7.12).
These are:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>“search modification id” (MS:1003392),</p>
</li>
<li>
<p>“search modification id ref” (MS:1003393).</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>Furthermore, different typos and small details have been refined throughout the text.
As an example, the hyphenated term “cross-linking” (used in version 1.2.0) has now been de-hyphenated throughout this 1.3.0 updated specification document.</p>
</div>
<div class="paragraph">
<p>Finally, in the section 5.2.2 (“Support for de novo sequencing approaches”), a clarification has been added providing more details about how to represent <em>de novo</em> searches, including the use of the CV param “<em>de novo</em> search or no database used” (MS:1000394).</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_format_scope_and_specific_use_cases">Format scope and specific use cases</h2>
<div class="sectionbody">
<div class="sect2">
<h3 id="_handling_updates_to_the_controlled_vocabulary">Handling updates to the controlled vocabulary</h3>
<div class="paragraph">
<p>In brief, when a new term is required, the file producers must contact the CV working group <em>via</em> e-mail (<a href="http://psidev-ms-vocab@lists.sourceforge.net">psidev-ms-vocab@lists.sourceforge.net</a>) and request the new term.
It is anticipated that problems may arise if a consumer of the file encounters a new CV term and they are not working from the latest version of the CV file.
It has been decided that rather than aim for a workaround to this issue, it can be expected that data file consumers must ensure that the OBO file is up-to-date.</p>
</div>
</div>
<div class="sect2">
<h3 id="_identifying_the_input_spectrum_for_each_result">Identifying the input spectrum for each result</h3>
<div class="paragraph">
<p>A &lt;SpectrumIdentificationResult&gt; is linked to the source spectrum (in an external file) from which the identifications are made by way of a reference in the spectrumID attribute and <em>via</em> the &lt;SpectraData&gt; element which stores the URL of the file in the location attribute.
It is advantageous if there is a consistent system for identifying spectra in different file formats.
The following table is implemented in the PSI-MS CV for providing consistent identifiers for different spectrum file formats.
A CV term MUST be imported into the &lt;SpectraData&gt; element to demonstrate which system for identifying input spectra is being used in the spectrumID attribute of &lt;SpectrumIdentificationResult&gt;.</p>
</div>
<div class="paragraph">
<p><em>It is encouraged but not mandatory that a valid mzIdentML file is accompanied by the set of spectra that were searched.</em> It is acknowledged that in many cases it will be useful to have an mzIdentML file and the input spectra together, there are practical problems processing such data depending on the spectrum file format (e.g. in case of proprietary formats), and cases where an mzIdentML is useful even if the spectra data is not available.</p>
</div>
<div class="paragraph">
<p><strong>Update from version 1.2.0:</strong></p>
</div>
<div class="paragraph">
<p>Version 1.1.0 of the specification document states “the CV holds the definite specification for legal encodings of spectrumID values”.
In version 1.2, the only legal ways of referencing a spectrum identification format are provided below in Table 1. Any new spectral formats that cannot fit into this schema require an update to this document.</p>
</div>
<div class="paragraph">
<p><strong>Update included in version 1.3.0:</strong></p>
</div>
<div class="paragraph">
<p><em>All CV terms containing combined spectra input types have been deprecated.</em></p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>ID</strong></th>
<th class="tableblock halign-left valign-top"><strong>Term</strong></th>
<th class="tableblock halign-left valign-top"><strong>Data type</strong></th>
<th class="tableblock halign-left valign-top"><strong>Comment</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1000768</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Thermo nativeID format</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">controllerType=xsd:nonNegativeInteger controllerNumber=xsd:positiveInteger scan=xsd:positiveInteger</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">controller=0 is usually the mass spectrometer. Space-separated values.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1000769</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Waters nativeID format</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">function=xsd:positiveInteger process=xsd:nonNegativeInteger scan=xsd:nonNegativeInteger</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Space-separated values.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1000770</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">WIFF nativeID format</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">sample=xsd:nonNegativeInteger period=xsd:nonNegativeInteger cycle=xsd:nonNegativeInteger experiment=xsd:nonNegativeInteger</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Space-separated values.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1000771</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bruker/Agilent YEP nativeID format</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">scan=xsd:nonNegativeInteger</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1000772</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bruker BAF nativeID format</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">scan=xsd:nonNegativeInteger</p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1000773</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Bruker FID nativeID format</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">file=xsd:IDREF</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">The nativeID must be the same as the source file ID.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1000774</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">multiple peak list nativeID format</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">index=xsd:nonNegativeInteger</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Used for referencing peak list files with multiple spectra, i.e. MGF, PKL, merged DTA files. Index is the spectrum number in the file, starting from 0.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1000775</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">single peak list nativeID format</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">file=xsd:IDREF</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">The nativeID must be the same as the source file ID. Used for referencing peak list files with one spectrum per file, typically in a folder of PKL or DTAs, where each sourceFileRef is different.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1000776</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">scan number only nativeID format</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">scan=xsd:nonNegativeInteger</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Used for referencing mzXML, or a DTA folder where native scan numbers can be derived.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1000777</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">spectrum identifier nativeID format</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">spectrum=xsd:nonNegativeInteger</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Used for referencing mzData. The spectrum ID attribute is referenced.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">MS:1001530</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">mzML unique identifier</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd:string</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Used for referencing mzML. The value of the spectrum ID attribute is referenced directly.</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>Table 2. CV terms and rules implemented in the PSI-MS CV for formulating the “nativeID” to identify spectra in different file formats.</p>
</div>
<div class="paragraph">
<p>In mzIdentML, the spectrum ID attribute should be constructed following the data type specification in Table 2. As an example, to reference the third spectrum (index=2) in an MGF (Mascot Generic Format) file:</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult id="Res1" spectrumID="index=2" SpectraData_ref="InputSpectra1"&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;SpectraData location="local/mgf/merge.mgf" id="SD_1" &gt;</p>
</div>
<div class="paragraph">
<p>&lt;FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001062" name="Mascot MGF file" cvRef="PSI-MS" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIDFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1000774" name="multiple peak list nativeID format" cvRef="PSI-MS" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIDFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectraData&gt;</p>
</div>
</div>
<div class="sect2">
<h3 id="_comments_on_specific_use_cases">Comments on Specific use cases</h3>
<div class="paragraph">
<p>Many special use cases for mzIdentML were considered during its development.
Each of these use cases has a corresponding example file that exercises the relevant part of the schema and provides a reference implementation example (see supporting documentation).
Authors of software that create mzIdentML are encouraged to examine the examples that accompany this format release before implementing the writer.
Further, such authors are encouraged to use the validator before releasing any new writer code and working with the PSI PI Working Group to resolve any issues.
In the subsections below, we comment on a few of the notable use cases that were considered – in particular those newly added in mzIdentML 1.2.</p>
</div>
<div class="sect3">
<h4 id="_protein_grouping_encoding">Protein grouping encoding</h4>
<div class="paragraph">
<p>This section is newly inserted in the mzIdentML version 1.2.0 specifications.
In version 1.1.0, CV terms had been proposed for representing set relationships between different proteins within groups, but there was not a requirement that particular terms were used.
A given data structure from software could be mapped onto the hierarchy &lt;ProteinAmbiguityGroup&gt; and &lt;ProteinDetectionHypothesis&gt; in mzIdentML in different ways, leading to difficulties for data consumers.
As such, a working group has now agreed a more rigid encoding detailed as follows and in (6).</p>
</div>
<div class="olist arabic">
<ol class="arabic">
<li>
<p>As in mzIdentML version 1.1, a single protein accession that has been cited by software is captured in mzIdentML in &lt;ProteinDetectionHypothesis&gt; (PDH).</p>
<div class="olist loweralpha">
<ol class="loweralpha" type="a">
<li>
<p>A PDH MAY contain scores or statistical values produced by the export software, encoded as CV terms.</p>
</li>
</ol>
</div>
</li>
<li>
<p>A “protein group”, representing a “biological entity” for which the software claims independent evidence is present, MUST be mapped onto &lt;ProteinAmbiguityGroup&gt; (PAG).</p>
<div class="olist loweralpha">
<ol class="loweralpha" type="a">
<li>
<p>A PAG MAY have additional scores produced by the export software, encoded as CV terms.</p>
</li>
</ol>
</div>
</li>
<li>
<p>The reporting of protein identification thresholds is now mapped onto PAGs.
There is no desire to change the core XML Schema Document (XSD) for mzIdentML and as such, a new CV term “protein group passes threshold” value= “xsd:boolean” MUST be present on every PAG (MS:1002415).
If no thresholding has been done by the software, all protein groups MUST be annotated as “protein group passes threshold” value= “true”.</p>
<div class="olist loweralpha">
<ol class="loweralpha" type="a">
<li>
<p>The attribute <em>passThreshold</em> = “true|false” remains present on PDH and MAY be used if software packages wish to report a two-level hierarchy of thresholds applied.
However, it is not expected that consuming software will use this attribute to determine which proteins have been reported as identified.</p>
</li>
</ol>
</div>
</li>
<li>
<p>The &lt;ProteinDetectionList&gt; MUST contain the CV term “count of identified proteins” value= “xsd:integer” (MS:1002404).
The value MUST be derived from the count of PAGs passing the threshold reported in the file and will be checked by validation software.</p>
</li>
<li>
<p>Few software packages report “protein clusters” at present, but for those packages that wish to report clusters, a CV term “cluster identifier” value = “xsd:integer” SHOULD be used (MS:1002407).
The integer identifier MUST be shared by all PAGs belonging to the same cluster.
An optional term “count of identified clusters” value = “xsd:integer” (MS:1002406) MAY be annotated on the &lt;ProteinDetectionList&gt;.</p>
</li>
<li>
<p>Every PDH MUST be annotated as either a “leading protein” (MS:1002401) or a “non-leading protein” (MS:1002402), as defined in Table 2, within a PAG.
This recommendation thus makes it explicit for consuming software whether one or more proteins have stronger evidence than others in the group (see Table 2 for examples).</p>
<div class="olist loweralpha">
<ol class="loweralpha" type="a">
<li>
<p>An additional term, “group representative” (MS:1002403) MAY be used to annotate one PDH, which is also flagged as a “leading protein”, if the export software wishes to enforce that only one of potential several “leading proteins” will be interpreted by the consuming software as the representative of the group, for example acting as a tiebreaker.</p>
</li>
<li>
<p>If the export software does not explicitly flag one protein as the “group representative”, it is assumed that if consuming software requires a single accession to represent the group, an arbitrary choice will be made (among “leading proteins” only if these exist).</p>
</li>
</ol>
</div>
</li>
<li>
<p>Any PDHs MAY be annotated with terms present in the CV for spectrum/sequence same-set, spectrum/sequence subset, spectrum/sequence subsumable, marginally distinguished and so on (Table 2).</p>
<div class="olist loweralpha">
<ol class="loweralpha" type="a">
<li>
<p>A PDH MAY be annotated with more than one of these terms if appropriate to describe the complex set relationships that exist within a group.</p>
</li>
<li>
<p>Developers of software packages MAY propose additional terms for describing group membership of PDHs, which will be incorporated into the CV.</p>
</li>
<li>
<p>The associated value for these CV terms MAY be used to annotate which PDH(s) are the super/same-set of the annotated PDH.</p>
</li>
<li>
<p>There is no expectation that consuming software should be aware of these terms, but they may be useful in internal pipeline or visualization software packages that are specifically designed to work with this terminology set.</p>
</li>
</ol>
</div>
</li>
<li>
<p>Some PDHs could be mapped to more than one PAG, for example where proteins are multiply subsumed.
To capture these cases, multiple PDHs in different PAGs MAY reference the same &lt;DBSequence&gt;.</p>
</li>
</ol>
</div>
<div class="paragraph">
<p>The semantic validation software has been updated to encode these rules and report errors (“MUST” rule), warnings (“SHOULD” rule) or informational messages (“MAY” rule).</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
<col style="width: 20%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>mzIdentML context</strong></th>
<th class="tableblock halign-left valign-top"><strong>CV term</strong></th>
<th class="tableblock halign-left valign-top"><strong>Values</strong></th>
<th class="tableblock halign-left valign-top"><strong>Require-ment level</strong></th>
<th class="tableblock halign-left valign-top"><strong>Description</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinDetection-List</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">count of identified proteins</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd:integer</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MUST</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">The value reported MUST equal the number of PAGs with “protein group passes threshold” value = “true”</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinDetection-List</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">count of identified clusters</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd:integer</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MAY</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">If protein clusters have been reported in the file, the exporter may choose to annotate the ProteinDetectionList with the number identified above threshold.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinAmbiguity-Group</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">number of distinct protein sequences</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd:integer</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MAY</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">The number of distinct protein sequences among the PDHs in the group. For example, if there are two PDHs with different identifiers that have identical full length sequences, the value would be 1.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinAmbiguity-Group</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">cluster identifier</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd:integer</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MAY</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">An identifier applied to protein groups to indicate that they are linked by shared peptides.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinDetection-Hypothesis</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>leading protein</p>
</div>
<div class="paragraph">
<p>OR</p>
</div>
<div class="paragraph">
<p>non-leading protein</p>
</div></div></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">-</p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>MUST</p>
</div>
<div class="paragraph">
<p>OR</p>
</div>
<div class="paragraph">
<p>MUST</p>
</div></div></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Every PDH in each PAG MUST be flagged as a leading protein or a non-leading protein and each PAG MUST contain at least one leading protein, but MAY contain more than one. A “leading protein” is defined as a protein that has the strongest or near strongest (further explained in Table 3) set of evidence for being present in the sample studied, amongst the grouped protein accessions. A “non-leading protein” is defined as a protein that has (substantially) less evidence than other proteins within the same group, and is thus less likely to have been present in the sample studied.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinDetection-Hypothesis</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">group representative</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">-</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MAY</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Each PAG MAY contain zero or one PDH flagged as the group representative, if the software wishes to flag a preference (often arbitrary or for example based on alphabetical ordering) amongst the leading proteins. The group representative term can thus be viewed a “tiebreaker” if the export software wishes to make this distinction.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinAmbiguity-Group</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">protein group passes threshold</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd:Boolean</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MUST</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Each PAG MUST be annotated with a Boolean CV term indicating whether the PAG has passed the threshold reported in the ProteinDetectionProtocol.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinDetection-Hypothesis</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">sequence same-set protein</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd: “list_of_ strings” space separated list of PDH IDs that are same-set.</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MAY</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A protein that is indistinguishable or equivalent to another protein in the group, having matches to an identical set of peptide sequences.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinDetection-Hypothesis</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">spectrum same-set protein</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd: “list_of_ strings” space separated list of PDH IDs that are same-set.</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MAY</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A protein that is indistinguishable or equivalent to another protein in the group, having PSMs derived from the same set of spectra.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinDetection-Hypothesis</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">sequence sub-set protein</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd: “list_of_ strings” space separated list of PDH IDs that are super-set.</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MAY</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A protein for which the matched peptide sequences are a subset of the matched peptide sequences for another protein in the group.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinDetection-Hypothesis</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">spectrum sub-set protein</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd: “list_of_ strings” space separated list of PDH IDs that are super-set.</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MAY</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A protein for which the matched spectra are a subset of the matched spectra for another protein in the group.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinDetection-Hypothesis</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">sequence multiply subsumable protein</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd: “list_of_ strings” space separated list of PDH IDs that subsume this PDH.</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MAY</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A protein for which the matched peptide sequences are the same, or a subset of, the matched peptide sequences for two or more other proteins combined. These other proteins need not all be in the same group.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinDetection-Hypothesis</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">spectrum multiply subsumable protein</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd: “list_of_ strings” space separated list of PDH IDs that subsume this PDH.</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MAY</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A protein for which the matched spectra are the same, or a subset of, the matched spectra for two or more other proteins combined. These other proteins need not all be in the same group.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>ProteinDetection-Hypothesis</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">marginally distinguished protein</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">-</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MAY</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Assigned to a non-leading PDH that has some independent evidence to support its presence relative to the leading protein(s) e.g. the PDH may have a unique peptide but not sufficient to be promoted as, for example, a leading protein of another a PAG.</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>Table 3. New CV terms for reporting protein set (group) relationships and global statistics about the protein identification results.
The semantic validation software for mzIdentML version 1.2.0 reports an error (MUST), a warning (SHOULD) or an informational message (MAY) if these terms are not reported within the file.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 33.3333%;">
<col style="width: 33.3333%;">
<col style="width: 33.3334%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Scenario</strong></th>
<th class="tableblock halign-left valign-top"><strong>Software preference</strong></th>
<th class="tableblock halign-left valign-top"><strong>Encoding</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Software scores A and B as same-set, C and D as subset.</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Software wishes to make A the group representative (arbitrary)</p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>A = leading protein &amp; group representative</p>
</div>
<div class="paragraph">
<p>B = leading protein</p>
</div>
<div class="paragraph">
<p>C = non-leading protein</p>
</div>
<div class="paragraph">
<p>D = non-leading protein</p>
</div>
<div class="paragraph">
<p>(Use of formal same-set and subset notation is also allowed but optional)</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>As above</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Software does not wish to choose which is the group representative</p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>A = leading protein</p>
</div>
<div class="paragraph">
<p>B = leading protein</p>
</div>
<div class="paragraph">
<p>C = non-leading protein</p>
</div>
<div class="paragraph">
<p>D = non-leading protein</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Software scores A as best protein, B, C and D are all subset or subsumed</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">N/A</p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>A = leading protein</p>
</div>
<div class="paragraph">
<p>B = non-leading protein</p>
</div>
<div class="paragraph">
<p>C = non-leading protein</p>
</div>
<div class="paragraph">
<p>D = non-leading protein</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Software scores all four proteins as same-set or more generally as having equal evidence</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Software wishes to make A the group representative (arbitrary)</p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>A = leading protein &amp; group representative</p>
</div>
<div class="paragraph">
<p>B = leading protein</p>
</div>
<div class="paragraph">
<p>C = leading protein</p>
</div>
<div class="paragraph">
<p>D = leading protein</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>As above</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Software does not wish to choose which is the group representative</p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>A = leading protein</p>
</div>
<div class="paragraph">
<p>B = leading protein</p>
</div>
<div class="paragraph">
<p>C = leading protein</p>
</div>
<div class="paragraph">
<p>D = leading protein</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Software scores A as having slightly more evidence than B. B has additional weak independent evidence relative to A. C and D have less evidence than either A or B.</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Software wishes to assign A as the leading protein and the independent evidence for B is not sufficient for it to form a new PAG.</p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>A = leading protein</p>
</div>
<div class="paragraph">
<p>B = non-leading protein &amp; marginally distinguished (optional)</p>
</div>
<div class="paragraph">
<p>C = non-leading protein</p>
</div>
<div class="paragraph">
<p>D = non-leading protein</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>As above</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Software does not wish to choose which is the leading protein out of A and B or group representative</p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>A = leading protein</p>
</div>
<div class="paragraph">
<p>B = leading protein</p>
</div>
<div class="paragraph">
<p>C = non-leading protein</p>
</div>
<div class="paragraph">
<p>D = non-leading protein</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>As above</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Software does not wish to choose which is the leading protein but does select a group representative</p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>A = leading protein &amp; group representative</p>
</div>
<div class="paragraph">
<p>B = leading protein</p>
</div>
<div class="paragraph">
<p>C = non-leading protein</p>
</div>
<div class="paragraph">
<p>D = non-leading protein</p>
</div></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Table 4.</strong> A summary of grouping options and recommendation for CV term annotations, assuming a group of four related proteins A-D.</p>
</div>
</div>
<div class="sect3">
<h4 id="_support_for_de_novo_sequencing_approaches">Support for <em>de novo</em> sequencing approaches</h4>
<div class="paragraph">
<p>In mzIdentML version 1.1, &lt;SpectrumIdentificationItem&gt; had a mandatory sub-element &lt;PeptideEvidenceRef&gt; to link each PSM to all the proteins from which it could have originated.
The inclusion of these mandatory requirements makes it difficult to represent results from <em>de novo</em> sequencing and spectral library search approaches where PSMs may not necessarily have originated from a protein database search.
As such, in mzIdentML 1.2.0 &lt;PeptideEvidenceRef&gt; has a cardinality of 0&#8230;&#8203;many.
In all cases of sequence database search, export software MUST include all &lt;PeptideEvidenceRef&gt; elements for every PSM.
In version mzIdentML 1.3.0, further clarifications are added: &lt;SearchDatabase&gt; will include the CV param MS:1000394 (“de novo search or no database used”) and “No database” will be included as an userParam in its subelement &lt;DatabaseName&gt;.</p>
</div>
<div class="paragraph">
<p><em>De novo</em> sequencing approaches are therefore supported, but only in a relatively straightforward manner, where complete peptide sequences are identified.
Proposals for representing partial peptide sequences or sequence tags are encouraged for future iterations of the standard.</p>
</div>
</div>
<div class="sect3">
<h4 id="_spectral_library_searches">Spectral library searches</h4>
<div class="paragraph">
<p>An alternative to sequence database searches for identifying peptides from MS data is to search a pre-compiled library (spectral library) of PSMs.
These spectral library searches are supported in mzIdentML.
The recommended encoding is similar to sequence database search results.
The main difference is that a &lt;Peptide&gt; entity SHOULD record each library entry that has been matched against.
Additional scores or metadata about the library entity SHOULD be included as &lt;cvParam&gt; elements on &lt;Peptide&gt;.
For searches against spectral archives i.e. where the identity of the library entry is unknown (there is no a peptide sequence assignment to the spectrum in the library), the encoding SHOULD include an empty string in &lt;PeptideSequence&gt;.</p>
</div>
<div class="paragraph">
<p>Note – there has been no formal change to the schema or CV requirements from mzIdentML 1.1 to 1.2.0 around spectral library encoding, but the intended encoding has changed.
The mzIdentML 1.1. specifications stated that spectral library entries should be encoded within &lt;DBSequence&gt;, which does not well model the data produced.</p>
</div>
</div>
<div class="sect3">
<h4 id="_multiple_database_search_engines">Multiple database search engines</h4>
<div class="paragraph">
<p>Proteomics research groups now commonly analyze MS data using multiple search engines and combine results to improve the number of peptide and protein identifications that can be made.
The output of such approaches can be represented in mzIdentML as follows (see Section 6 for documentation of the model elements).
Note that the RECOMMENDED encoding has changed since the version 1.1.0 specification as a result of community feedback.
It has been decided that throughout mzIdentML, the spectrum referenced from a &lt;SpectrumIdentificationResult&gt; MUST be unique within a file <em>i.e.</em> only one set of ranked results can be provided per spectrum.
This has implications for encoding the results of multiple search engines, as only consensus results (after they have been combined) can be represented in a valid mzIdentML 1.2.0 file.
If exporters wish to maintain the original search engine results, these MAY be encoded using &lt;cvParam&gt; elements within &lt;SpectrumIdentificationItem&gt; containing additional scores, statistics and indicating the original rankings from the source search engine.</p>
</div>
<div class="paragraph">
<p>The &lt;SpectrumIdentification&gt; element MUST reference a &lt;SpectrumIdentificationProtocol&gt; holding representative parameters used across all search engines (i.e. search tolerances, enzyme and modifications), since these are MANDATORY elements.
If the same search parameters were not employed in all source searches, the parameters should be set with superset or widest values i.e. all modifications that have been searched, widest tolerances and so on.
All search engines that have been employed SHOULD be represented within the &lt;AnalysisSoftwareList&gt;.
The &lt;AnalysisSoftwareList&gt; SHOULD also record the software used to combine results.
It must also be highlighted that mzIdentML cannot be used to model the order in which the software was used (it does not support workflows).</p>
</div>
<div class="paragraph">
<p>The same encoding MAY also be used to describe other approaches where different search protocols are applied to the same spectra (for example using different parameter sets with the same search engine), and subsequently combined.
In this case, it is RECOMMENDED that only a single (assumed best) score of any given type is represented once per &lt;SpectrumIdentificationItem&gt;.</p>
</div>
</div>
<div class="sect3">
<h4 id="_pre_fractionation_of_samples_prior_to_ms_and_splitting_of_searches">Pre-fractionation of samples prior to MS and splitting of searches</h4>
<div class="paragraph">
<p>It is common in many workflows for pre-fractionation of a sample to be performed prior to MS, for example <em>via</em> 1D or 2D gel electrophoresis or 2D LC.
In some scenarios results of database searches are combined prior to protein inference and in other instances there is no combination of results prior to protein inference.
We have identified the following scenarios and describe the RECOMMENDED encoding in each case in Table 5 below.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Scenario</strong></th>
<th class="tableblock halign-left valign-top"><strong>Encoding</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Scenario 1. i) A sample is fractionated into <em>n</em> sub-samples, prior to <em>n</em> runs on the MS; ii) the search engine performs <em>n</em> searches, producing <em>n</em> protein-lists.</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><em>n</em> mzIdentML files SHOULD be produced, each containing 1 &lt;SpectrumIdentificationList&gt;, 1 &lt;SpectrumIdentificationProtocol&gt;, 1&lt;SpectrumIdentification&gt;, 1 &lt;ProteinDetection&gt;, 1 &lt;ProteinDetectionList&gt;.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Scenario 2. i) A sample is fractionated into <em>n</em> sub-samples, prior to <em>n</em> runs on the MS; ii) the search engine imports <em>n</em> peak lists and performs <em>n</em> searches but internally integrates results to produce one protein list.</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>One single mzIdentML file SHOULD contain <em>n</em> &lt;SpectrumIdentificationList&gt;s, <em>n</em> &lt;SpectrumIdentificationProtocol&gt;s, <em>n</em> &lt;SpectrumIdentification&gt;s, 1 &lt;ProteinDetection&gt;, 1 &lt;ProteinDetectionList&gt;.</p>
</div>
<div class="paragraph">
<p>The &lt;SpectrumIdentificationProtocol&gt;s MUST indicate that pre-fractionation has taken place, using the CV term indicated in Table 1.</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Scenario 3. i) A sample is fractionated into <em>n</em> sub-samples, prior to <em>n</em> runs on the MS; ii) the search engine performs <em>n</em> searches, producing <em>n</em> lists of spectrum identifications; iii) post-processing software integrates results to produce one protein list.</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">As Scenario 2.</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Scenario 4. i) There is no sample pre-fractionation and one run on the MS. ii) The spectra are split into <em>n</em> peak list files for searching (for example for parallelisation on a cluster), producing <em>n</em> lists of PSMs iii) post-processing software re-combines results into one mzIdentML file producing 1 protein list.</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">One single mzIdentML file SHOULD contain 1 &lt;SpectrumIdentificationList&gt;, 1 &lt;SpectrumIdentificationProtocol&gt;, 1 &lt;SpectrumIdentification&gt; referencing <em>n</em> &lt;InputSpectra&gt; sub-elements, 1 &lt;ProteinDetection&gt;, 1 &lt;ProteinDetectionList&gt;.</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>Table 5. A description of RECOMMENDED encodings in mzIdentML, where sample pre-fractionation has taken place.</p>
</div>
</div>
<div class="sect3">
<h4 id="_encoding_replicate_samples">Encoding replicate samples</h4>
<div class="paragraph">
<p>One mzIdentML file is intended to capture the analysis of one sample, including rules for pre-fractionation as discussed in Section 5.2.5. For encoding replicate samples (biological or technical), separate mzIdentML files SHOULD be used.
A naming convention using suffixes MAY be adopted but the specifications of such are beyond the scope of this document.</p>
</div>
</div>
<div class="sect3">
<h4 id="_peptide_level_scores_and_statistical_measures">Peptide-level scores and statistical measures</h4>
<div class="paragraph">
<p>The format was designed with explicit support for encoding scores or statistical measures for PSMs, for individual proteins and for protein groups.
However, the original design contained no explicit (schema level) support for peptide-level scores i.e. after redundant PSMs reporting on the same peptide have been removed.
One of the challenges in this space is defining the mechanism of grouping multiple PSMs for the same <em>distinct peptide –</em> since in different contexts a distinct peptide could encompass one of the following concepts:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>A peptide sequence with a given set of modifications – located in specified positions, identified from a single charge state.</p>
</li>
<li>
<p>A peptide sequence with a given set of modifications – located in specified positions, identified from different charge state ions.</p>
</li>
<li>
<p>A peptide sequence with a given set of modifications – regardless of the positions of modifications.</p>
</li>
<li>
<p>A peptide sequence regardless of the presence/absence of different modifications.</p>
</li>
</ul>
</div>
<div class="paragraph">
<p>A mechanism for encoding these different types of distinct peptide grouping in the mzIdentML 1.2.0 specifications has been defined, using CV terms as described in Figure 1. Three CV terms have currently been added to the PSI-MS CV: “group PSMs by sequence” (MS:1002496), “group PSMs by sequence with modifications” (MS:1002497) and “group PSMs by sequence with modifications and charge” (MS:1002498).</p>
</div>
<div class="paragraph">
<p>The following additional features have also been added to mzIdentML 1.2.0 to support peptide-level scores (Figure 1).
First, an additional CV term “peptide-level scoring” (<strong>MS:1002490) MUST be included (when this process is being reported)</strong> in &lt;SpectrumIdentificationProtocol&gt;, as shown in Figure 1A and Table 1. In addition, the &lt;SpectrumIdentificationProtocol&gt; contains a &lt;Threshold&gt; element, used in previous versions, for representing the threshold applied at the PSM level.
In mzIdentML 1.2, the element can now be used to demonstrate the threshold applied at the PSM and/or peptide-level, through the use of appropriate CV terms.</p>
</div>
<div class="paragraph">
<p>Additionally, a mechanism is needed for capturing how different PSMs are grouped into a single entity.
This is achieved by adding a CV term to every PSM in the file “peptide group ID” (MS:1002520), whereby the associated value is a unique identifier shared between all PSMs in the same peptide group.
In the example in Figure 1D, the unique identifier used is the peptide sequence itself (since when grouping by sequence irrespective of modification status this value must be unique), although this could be an arbitrary value such as an integer code.</p>
</div>
<div class="paragraph">
<p>The mzIdentML file must be able to record scores or statistical values at the peptide level.
This is achieved <em>via</em> adding CV terms with identical values to all PSMs within the same peptide-group, with an indication that it is a peptide-level value, <em>via</em> the convention of the prefix “peptide:” in the CV term name (Figure 1E).
Finally, a mechanism has been added for recording peptides both above and below the threshold, to allow complete statistical re-evaluation by downstream software.
PSM-level threshold is covered <em>via</em> the <em>passThreshold</em> attribute on the &lt;SpectrumIdentificationItem&gt; element.
To enable additional thresholding at the peptide-level, a new CV term is required for all PSMs (“peptide passes threshold”, MS:1002500) as shown in Figure 1F.</p>
</div>
<div class="paragraph">
<p><span class="image"><img src="img//media/image1.wmf" alt="image" width="643" height="857"></span></p>
</div>
<div class="paragraph">
<p>Figure 1. The mechanism for encoding peptide-level statistics in mzIdentML 1.2.0.</p>
</div>
</div>
<div class="sect3">
<h4 id="_encoding_modification_localisation_scores">Encoding modification localisation scores</h4>
<div class="paragraph">
<p>A new addition to mzIdentML 1.2.0 is the ability to attach scores or statistical values to the position of a modification, with regards to the peptide sequence.
A variety of software packages now export such values, since it is common for there to be more than one possible site of modification.
Evidence from the presence or absence of fragment ions can enable a calculation of the likelihood of different possibilities.
Such evidence trail is particularly important for some downstream uses of the data, such as profiling motifs for positions of modifications or populating databases with “experimentally observed” modification sites.</p>
</div>
<div class="paragraph">
<p>The encoding of such scores is achieved in mzIdentML 1.2.0 by making use of a regular expression attached within a &lt;cvParam&gt; at the level of &lt;SpectrumIdentificationItem&gt;.
The following additional features to be present in mzIdentML 1.2.0 (Figure 2).</p>
</div>
<div class="paragraph">
<p>To ensure that downstream software is aware that a file contains modification scores, a CV term is added to the &lt;SpectrumIdenticationProtocol&gt; - “Modification localization scoring” (MS:1002491), as shown in Figure 2A and Table 1. Some approaches apply a statistical threshold for accepting or rejecting that a modification position has been confidently identified.
The (re-usable) &lt;Peptide&gt; element has an attribute <em>via</em> which the residue and location of a modification can be recorded.
To remain backwards compatible, we recommend that the software implementing mod scoring (and export) in mzIdentML should continue to use these attributes, populating with the most likely modification position (Figure 2C).
A new CV term (REQUIRED when MS:1002491 is present in the protocol) must be added to every &lt;Modification&gt; element – called “modification index” (MS:1002504), where the value serves as a unique identifier (local only to the containing &lt;Peptide&gt;) to be referenced from the PSM (Figure 2D).</p>
</div>
<div class="paragraph">
<p>The modification scores themselves are added as CV terms to the &lt;SpectrumIdentificationItem&gt; element referencing the peptide (e.g. “phosphoRS score”, Figure 2E), with a value provided as a regular expression of four values in a defined order - <strong>MOD_INDEX, SCORE</strong>, <strong>POSITION</strong>, <strong>PASS_THRESHOLD.
MOD_INDEX is a reference to the</strong> “modification index” <strong>identifier provided in the referenced &lt;Peptide&gt; - &lt;Modification&gt; element.
SCORE represents the score or statistical value (double data type) for the given position.
POSITION is the scored modification position with respect to the peptide sequence (where position = 0 is the N-terminus, and the peptide length + 1 is used to indicate the C-terminus).
The POSITION can include the bar symbol ‘|’, as a logical OR, if the score relates to multiple positions that can be distinguished.
PASS_THRESHOLD holds a Boolean (true, false) value to indicate whether the modification position passes the threshold described above.</strong></p>
</div>
<div class="paragraph">
<p><strong>&lt;cvParam accession="MS:1002380" cvRef="PSI-MS" value="1:0.03:2|3:true" name="modification rescored by false localization rate"/&gt;<br>
&lt;cvParam accession="MS:1002380" cvRef="PSI-MS" value="1:0.97:8|9:false" name="modification rescored by false localization rate"/&gt;</strong></p>
</div>
<div class="paragraph">
<p><strong>The mechanism described MAY be used in conjunction with peptide-level scoring, using specific CV terms for peptide-level modification re-scoring.</strong></p>
</div>
<div class="paragraph">
<p><span class="image"><img src="img//media/image2.wmf" alt="image" width="456" height="875"></span></p>
</div>
<div class="paragraph">
<p>Figure 2. The specification in mzIdentML 1.2.0 for encoding modification localization scores, using CV terms.</p>
</div>
</div>
<div class="sect3">
<h4 id="_encoding_results_of_crosslinking_searches">Encoding results of crosslinking searches</h4>
<div class="paragraph">
<p>See the crosslinking extension document in the same folder as this document for all the details.</p>
</div>
</div>
<div class="sect3">
<h4 id="_encoding_proteogenomics_annotation_data">Encoding proteogenomics annotation data</h4>
<div class="paragraph">
<p>It is now common to use tandem MS data to improve current gene model annotations, in so-called proteogenomics approaches, for example based on making peptide identifications against the official gene models or against alternative databases generated by gene finders, mapping mRNA transcripts or six frame genome translations.
Where identifications do not match the official genes, they give evidence in support of updates to the gene models.
One of the key concepts required is the mapping of peptides back to chromosomes, including, for example, where they map across splice junctions.
File format specifications are under development that can be used directly for genome visualisation, such as adaptations of the BED and BAM (7) formats commonly used in genomics.
To ensure a consistent export is possible from mzIdentML to formats designed for genome visualisation or annotation, in mzIdentML 1.2, a consistent encoding of the chromosomal mappings has been developed, as exemplified in Figure 3.</p>
</div>
<div class="paragraph">
<p><span class="image"><img src="img//media/image3.wmf" alt="image" width="664" height="768"></span></p>
</div>
<div class="paragraph">
<p>Figure 3. The encoding for chromosomal coordinates in mzIdentML in support of proteogenomics approaches.</p>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_other_supporting_materials">Other supporting materials</h3>
<div class="paragraph">
<p>Example files demonstrating the different uses cases have been developed and are available from the following location: <a href="https://github.com/HUPO-PSI/mzIdentML/tree/master/examples/1_2examples">https://github.com/HUPO-PSI/mzIdentML/tree/master/examples/</a>.
The sub-folder names indicate the features of the format being used in each example.</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_model_in_xml_schema">Model in XML Schema</h2>
<div class="sectionbody">
<div class="paragraph">
<p>An overview of the schema is presented in Figure 4. The following documentation is automatically generated from the XML Schema.</p>
</div>
<div class="paragraph">
<p><span class="image"><img src="img//media/image4.png" alt="image" width="664" height="373"></span></p>
</div>
<div class="paragraph">
<p>Figure 4. A diagrammatic overview of the mzIdentML schema (generated by <a href="https://github.com/dgis/xsddiagram">xsddiagram</a>).</p>
</div>
<div class="sect2">
<h3 id="_element_mzidentml">Element &lt;MzIdentML&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The upper-most hierarchy level of mzIdentML with sub-containers for example describing software, protocols and search results (spectrum identifications or protein detection results).</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MzIdentMLType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|creationDate |xsd:dateTime |optional |The date on which the file was produced.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|version |versionRegex |required |The version of the schema this instance document refers to, in the format x.y.z.
Changes to z should not affect prevent instance documents from validating.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvlist">cvList</a> |1 |1 |The list of controlled vocabularies used in the file.
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
Here, it is used to declare which extension documents are used.
|<a href="#element-analysissoftwarelist">AnalysisSoftwareList</a> |0 |1 |The software packages used to perform the analyses.
|<a href="#element-provider">Provider</a> |0 |1 |The Provider of the mzIdentML record in terms of the contact and software.
|<a href="#element-auditcollection">AuditCollection</a> |0 |1 |The complete set of Contacts (people and organisations) for this file.
|<a href="#element-analysissamplecollection">AnalysisSampleCollection</a> |0 |1 |The samples analysed can optionally be recorded using CV terms for descriptions.
If a composite sample has been analysed, the subsample association can be used to build a hierarchical description.
|<a href="#element-sequencecollection">SequenceCollection</a> |0 |1 |The collection of sequences (DBSequence or Peptide) identified and their relationship between each other (PeptideEvidence) to be referenced elsewhere in the results.
|<a href="#element-analysiscollection">AnalysisCollection</a> |1 |1 |The analyses performed to get the results, which map the input and output data sets.
Analyses are for example: SpectrumIdentification (resulting in peptides) or ProteinDetection (assemble proteins from peptides).
|<a href="#element-analysisprotocolcollection">AnalysisProtocolCollection</a> |1 |1 |The collection of protocols which include the parameters and settings of the performed analyses.
|<a href="#element-datacollection">DataCollection</a> |1 |1 |The collection of input and output data sets of the analyses.
|<a href="#element-bibliographicreference">BibliographicReference</a> |0 |unbounded |Any bibliographic references associated with the file</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Graphical Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="image"><img src="img//media/image5.png" alt="image" width="282" height="496"></span></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;MzIdentML xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" id="" xsi:schemaLocation="http://psidev.info/psi/pi/mzIdentML/1.2 mzIdentML1.2.0.xsd" creationDate="2014-06-03T14:52:58.665376Z" version="1.2.0" xmlns="http://psidev.info/psi/pi/mzIdentML/1.2"&gt;

&lt;cvList&gt;

&lt;cv fullName="Proteomics Standards Initiative Mass Spectrometry Vocabularies"
version="2.32.0"
uri="https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master/psi-ms.obo"
id="PSI-MS" /&gt;

&lt;cv fullName="UNIMOD" uri="http://www.unimod.org/obo/unimod.obo" id="UNIMOD" /&gt;

&lt;cv fullName="UNIT-ONTOLOGY" uri="http://obo.cvs.sourceforge.net/*checkout*/obo/obo/ontology/phenotype/unit.obo" id="UO" /&gt;

&lt;/cvList&gt;

&lt;AnalysisSoftwareList&gt;

...

&lt;/MzIdentML&gt;</code></pre>
</div>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_additionalsearchparams">Element &lt;AdditionalSearchParams&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The search parameters other than the modifications searched.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ParamListType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |1 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;AdditionalSearchParams&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001211" name="parent mass type mono"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001256" name="fragment mass type mono"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002492" name="consensus scoring"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002490" name="peptide-level scoring"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002497" name="group PSMs by sequence with modifications"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002491" name="modification localization scoring"/&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/AdditionalSearchParams&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/AdditionalSearchParams</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001302">MS:1001302</a> (search engine specific input parameter) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001005">MS:1001005</a> (SEQUEST:CleavesAt)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001007">MS:1001007</a> (SEQUEST:OutputLines)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001009">MS:1001009</a> (SEQUEST:DescriptionLines)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001026">MS:1001026</a> (SEQUEST:NormalizeXCorrValues)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001028">MS:1001028</a> (SEQUEST:SequenceHeaderFilter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001032">MS:1001032</a> (SEQUEST:SequencePartialFilter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001037">MS:1001037</a> (SEQUEST:ShowFragmentIons)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001038">MS:1001038</a> (SEQUEST:Consensus)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001042">MS:1001042</a> (SEQUEST:LimitTo)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001046">MS:1001046</a> (SEQUEST:sort by dCn)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001302">et al.</a></p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001066">MS:1001066</a> (ions series considered in search) one or more times</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001210">MS:1001210</a> (mass type settings) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001211">MS:1001211</a> (parent mass type mono)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001212">MS:1001212</a> (parent mass type average)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001255">MS:1001255</a> (fragment mass type average)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001256">MS:1001256</a> (fragment mass type mono)</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of MS:1002489 (special processing) one or more times</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001211" name="parent mass type mono"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001256" name="fragment mass type mono"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002490" name="peptide-level scoring"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002496" name="group PSMs by sequence"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001118" cvRef="PSI-MS" name="param: b ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001149" cvRef="PSI-MS" name="param: b ion-NH3 DEPRECATED"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001150" cvRef="PSI-MS" name="param: b ion-H2O DEPRECATED"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001262" cvRef="PSI-MS" name="param: y ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001151" cvRef="PSI-MS" name="param: y ion-NH3 DEPRECATED"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001152" cvRef="PSI-MS" name="param: y ion-H2O DEPRECATED"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002494" cvRef="PSI-MS" name="crosslinking search"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="no special processing" cvRef="PSI-MS" accession="MS:1002495" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002492" name="consensus scoring"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002497" name="group PSMs by sequence with modifications"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002491" name="modification localization scoring"/&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example userParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;userParam value="true" name="TargetDecoyApproach"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="-1" name="MinIsotopeError"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="2" name="MaxIsotopeError"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="HCD" name="FragmentMethod"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="QExactive" name="Instrument"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="iTRAQ" name="Protocol"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="2" name="NumTolerableTermini"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="1" name="NumMatchesPerSpec"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="2" name="MaxNumModifications"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="6" name="MinPepLength"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="40" name="MaxPepLength"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="2" name="MinCharge"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="3" name="MaxCharge"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="2a_uniprot" name="Mascot User Comment"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="ESI-QUAD" name="Mascot Instrument Name"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="input_consensusXML" unitName="xsd:string" value="leitner004.consensusXML"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="input_decoys" unitName="xsd:string" value=""/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="decoy_prefix" unitName="xsd:integer" value="1"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="decoy_string" unitName="xsd:string" value="decoy"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="precursor:min_charge" unitName="xsd:integer" value="3"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="precursor:max_charge" unitName="xsd:integer" value="7"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="fragment:mass_tolerance_xlinks" unitName="xsd:double" value="0.3"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="peptide:min_size" unitName="xsd:integer" value="5"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="cross_link:residue1" unitName="xsd:string" value="[K]"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="cross_link:residue2" unitName="xsd:string" value="[K]"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="cross_link:mass" unitName="xsd:double" value="138.0680796"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="cross_link:mass_isoshift" unitName="xsd:double" value="12.075321"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="cross_link:mass_monolink" unitName="xsd:string" value="[156.07864431, 155.094628715]"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="modifications:variable_max_per_peptide" unitName="xsd:integer" value="2"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="algorithm:candidate_search" unitName="xsd:string" value="enumeration"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="charges" unitName="xsd:string" value="2,3,4,5,6"/&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Example for peptide-level statistics:</strong></p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001211" name="parent mass type mono"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001256" name="fragment mass type mono"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002490" name="peptide-level scoring"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002496" name="group PSMs by sequence"&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p><strong>Example for sample pre-fractionation:</strong></p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001256" name="fragment mass type mono"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001211" name="parent mass type mono"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002493" name="sample pre-fractionation" value="MudPIT"&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p><strong>Example for proteogenomics:</strong></p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002635" name="proteogenomics search" value=""&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p><strong>Example for crosslinking:</strong></p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001211" name="parent mass type mono"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002494" name="crosslinking search"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001256" name="fragment mass type mono"&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p><strong>Example for modification position scoring:</strong></p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002491" name="modification localization scoring"&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p><strong>Example for de novo sequencing:</strong></p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001010" name=" de novo search "&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p><strong>Example for consensus scoring:</strong></p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002492" name="consensus scoring"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002392" name="PIA:PSM sets created" value="true"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002391" name="PIA:Combined FDRScore calculated" value="true"&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
</div>
<div class="sect2">
<h3 id="_element_affiliation">Element &lt;Affiliation&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The organization a person belongs to.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">AffiliationType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|organization_ref |xsd:string |required |A reference to the organization this contact belongs to.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;Affiliation organization_ref="ORG_DOC_OWNER"&gt;&lt;/Affiliation&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_ambiguousresidue">Element &lt;AmbiguousResidue&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Ambiguous residues e.g. X can be specified by the Code attribute and a set of parameters for example giving the different masses that will be used in the search.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">AmbiguousResidueType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|code |chars |required |The single letter code of the ambiguous residue e.g. X.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |1 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;AmbiguousResidue code="X"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001360" cvRef="PSI-MS" value="A C D E F G H I K L M N O P Q R S T U V W Y" name="alternate single letter codes"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/AmbiguousResidue&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/MassTable/AmbiguousResidue</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001359">MS:1001359</a> (ambiguous residues) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001360">MS:1001360</a> (alternate single letter codes)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001361">MS:1001361</a> (alternate mass)</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;cvParam accession="MS:1001360" cvRef="PSI-MS" value="D N" name="alternate single letter codes"/&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_analysiscollection">Element &lt;AnalysisCollection&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The analyses performed to get the results, which map the input and output data sets. Analyses are for example: SpectrumIdentification (resulting in peptides) or ProteinDetection (assemble proteins from peptides).</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">AnalysisCollectionType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-spectrumidentification">SpectrumIdentification</a> |1 |unbounded |An Analysis which tries to identify peptides in input spectra, referencing the database searched, the input spectra, the output results and the protocol that is run.
|<a href="#element-proteindetection">ProteinDetection</a> |0 |1 |An Analysis which assembles a set of peptides (e.g. from a spectra search analysis) to proteins.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;AnalysisCollection &gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentification spectrumIdentificationProtocol_ref="SearchProtocol_1_4299" spectrumIdentificationList_ref="SII_LIST_1_1_4299_120114_20_Orbi2_ZC_QC_220_HSAd0-d4-1to1-3_Din.raw" id="SpecIdent__4299_120114_20_Orbi2_ZC_QC_220_HSAd0-d4-1to1-3_Din.raw"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;InputSpectra spectraData_ref="SD_4299_120114_20_Orbi2_ZC_QC_220_HSAd0-d4-1to1-3_Din.raw"&gt;&lt;/InputSpectra&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SearchDatabaseRef searchDatabase_ref="SDB_4299_203"&gt;&lt;/SearchDatabaseRef&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentification&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentification spectrumIdentificationProtocol_ref="SearchProtocol_1_4299" spectrumIdentificationList_ref="SII_LIST_1_1_4299_120114_09_Orbi2_ZC_QC_220_HSAd0-d4-1to4-2_Din.raw" id="SpecIdent__4299_120114_09_Orbi2_ZC_QC_220_HSAd0-d4-1to4-2_Din.raw"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;InputSpectra spectraData_ref="SD_4299_120114_09_Orbi2_ZC_QC_220_HSAd0-d4-1to4-2_Din.raw"&gt;&lt;/InputSpectra&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/AnalysisCollection&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_analysisdata">Element &lt;AnalysisData&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Data sets generated by the analyses, including peptide and protein lists.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">AnalysisDataType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-spectrumidentificationlist">SpectrumIdentificationList</a> |1 |unbounded |Represents the set of all search results from SpectrumIdentification.
|<a href="#element-proteindetectionlist">ProteinDetectionList</a> |0 |1 |The protein list resulting from a protein detection process.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;AnalysisData&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationList xmlns="http://psidev.info/psi/pi/mzIdentML/1.2" id="SII_LIST_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult spectrumID="index=6451" spectraData_ref="SD_COMBINED_SE_0" id="SIR_8947"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="2" experimentalMassToCharge="679.817322" calculatedMassToCharge="679.818488" peptide_ref="AVMDDFAAFVEK_##Oxidation(M):3" rank="1" passThreshold="false" id="SIR_8947_SII_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="AVMDDFAAFVEK_generic</p>
</div></div></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP00000401820</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:known_378_389"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&lt;PeptideEvidenceRef peptideEvidence_ref="AVMDDFAAFVEK_generic</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP00000421027</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:putative_420_431"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&lt;PeptideEvidenceRef peptideEvidence_ref="AVMDDFAAFVEK_generic</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP00000483421</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:known_357_368"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&#8230;&#8203;
&lt;/AnalysisData&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_analysisparams">Element &lt;AnalysisParams&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The parameters and settings for the protein detection given as CV terms.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ParamListType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |1 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;AnalysisParams&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:SigThreshold" value="0.05" cvRef="PSI-MS" accession="MS:1001316" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:MaxProteinHits" value="Auto" cvRef="PSI-MS" accession="MS:1001317" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:ProteinScoringMethod" value="MudPIT" cvRef="PSI-MS" accession="MS:1001318" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:MinMSMSThreshold" value="0" cvRef="PSI-MS" accession="MS:1001319" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:ShowHomologousProteinsWithSamePeptides" value="1" cvRef="PSI-MS" accession="MS:1001320" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:ShowHomologousProteinsWithSubsetOfPeptides" value="10" cvRef="PSI-MS" accession="MS:1001321" /&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/AnalysisParams&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/ProteinDetectionProtocol/AnalysisParams</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001302">MS:1001302</a> (search engine specific input parameter) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001005">MS:1001005</a> (SEQUEST:CleavesAt</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001007">MS:1001007</a> (SEQUEST:OutputLines)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001009">MS:1001009</a> (SEQUEST:DescriptionLines)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001026">MS:1001026</a> (SEQUEST:NormalizeXCorrValues)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001028">MS:1001028</a> (SEQUEST:SequenceHeaderFilter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001032">MS:1001032</a> (SEQUEST:SequencePartialFilter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001037">MS:1001037</a> (SEQUEST:ShowFragmentIons)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001038">MS:1001038</a> (SEQUEST:Consensus)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001042">MS:1001042</a> (SEQUEST:LimitTo)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001046">MS:1001046</a> (SEQUEST:sort by dCn)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001302">et al.</a></p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001194">MS:1001194</a> (quality estimation with decoy database) one or more times</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam name="mascot:SigThreshold" value="0.05" cvRef="PSI-MS" accession="MS:1001316" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:MaxProteinHits" value="Auto" cvRef="PSI-MS" accession="MS:1001317" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:ProteinScoringMethod" value="MudPIT" cvRef="PSI-MS" accession="MS:1001318" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:MinMSMSThreshold" value="0" cvRef="PSI-MS" accession="MS:1001319" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:ShowHomologousProteinsWithSamePeptides" value="1" cvRef="PSI-MS" accession="MS:1001320" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:ShowHomologousProteinsWithSubsetOfPeptides" value="10" cvRef="PSI-MS" accession="MS:1001321" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:RequireBoldRed" value="0" cvRef="PSI-MS" accession="MS:1001322" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:UseUnigeneClustering" value="false" cvRef="PSI-MS" accession="MS:1001323" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:IncludeErrorTolerantMatches" value="1" cvRef="PSI-MS" accession="MS:1001324" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:ShowDecoyMatches" value="0" cvRef="PSI-MS" accession="MS:1001325" /&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_analysisprotocolcollection">Element &lt;AnalysisProtocolCollection&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The collection of protocols which include the parameters and settings of the performed analyses.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">AnalysisProtocolCollectionType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-spectrumidentificationprotocol">SpectrumIdentificationProtocol</a> |1 |unbounded |The parameters and settings of a SpectrumIdentification analysis.
|<a href="#element-proteindetectionprotocol">ProteinDetectionProtocol</a> |0 |1 |The parameters and settings of a ProteinDetection process.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;AnalysisProtocolCollection xmlns="http://psidev.info/psi/pi/mzIdentML/1.2"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationProtocol analysisSoftware_ref="AS_mascot_server" id="SIP"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SearchType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001083" cvRef="PSI-MS" value="" name="ms-ms search"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SearchType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;AdditionalSearchParams&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001211" cvRef="PSI-MS" name="parent mass type mono"/&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/AnalysisProtocolCollection&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_analysissamplecollection">Element &lt;AnalysisSampleCollection&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The samples analysed can optionally be recorded using CV terms for descriptions. If a composite sample has been analysed, the subsample association can be used to build a hierarchical description.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">AnalysisSampleCollectionType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-sample">Sample</a> |1 |unbounded |A description of the sample analysed by mass spectrometry using CVParams or UserParams.
If a composite sample has been analysed, a parent sample should be defined, which references subsamples.
This represents any kind of substance used in an experimental workflow, such as whole organisms, cells, DNA, solutions, compounds and experimental substances (gels, arrays etc.).</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_analysissoftware">Element &lt;AnalysisSoftware&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The software used for performing the analyses.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">AnalysisSoftwareType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|uri |xsd:anyURI |optional |URI of the analysis software e.g. manufacturer&#8217;s website |version |xsd:string |optional |The version of Software used.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-contactrole">ContactRole</a> |0 |1 |The Contact that provided the document instance.
|<a href="#element-softwarename">SoftwareName</a> |1 |1 |The name of the analysis software package, sourced from a CV if available.
|<a href="#element-customizations">Customizations</a> |0 |1 |Any customizations to the software, such as alternative scoring mechanisms implemented, should be documented here as free text.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;AnalysisSoftware uri="http://code.google.com/p/ehu-bio/downloads/list" version="1.1-beta4" name="PAnalyzer (v1.1-beta4)" id="PAnalyzer"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;ContactRole contact_ref="PAnalyzer_Author"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Role&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001271" cvRef="PSI-MS" name="researcher"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Role&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/ContactRole&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SoftwareName&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/AnalysisSoftware&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_analysissoftwarelist">Element &lt;AnalysisSoftwareList&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The software packages used to perform the analyses.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">AnalysisSoftwareListType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-analysissoftware">AnalysisSoftware</a> |1 |unbounded |The software used for performing the analyses.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;AnalysisSoftwareList xmlns="http://psidev.info/psi/pi/mzIdentML/1.2"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;AnalysisSoftware version="Beta (v9979)" name="MS-GF+" id="ID_software"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SoftwareName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002048" cvRef="PSI-MS" name="MS-GF+"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SoftwareName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/AnalysisSoftware&gt;</p>
</div>
<div class="paragraph">
<p>&lt;AnalysisSoftware name="FalseDiscoveryRate_2014-07-02 12-04-18" id="FalseDiscoveryRate_2014-07-02 12-04-18"&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/AnalysisSoftwareList&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_auditcollection">Element &lt;AuditCollection&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The complete set of Contacts (people and organisations) for this file.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">AuditCollectionType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-person">Person</a> |1 |unbounded |A person&#8217;s name and contact details.
Any additional information such as the address, contact email etc. should be supplied using CV parameters or user parameters.
|<a href="#element-organization">Organization</a> |1 |unbounded |Organizations are entities like companies, universities, government agencies.
Any additional information such as the address, email etc. should be supplied either as CV parameters or as user parameters.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;AuditCollection xmlns="http://psidev.info/psi/pi/mzIdentML/1.2"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Person lastName="secondName" firstName="firstname" id="PERSON_DOC_OWNER"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000587" name="contact address" value="address"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Affiliation organization_ref="ORG_DOC_OWNER"&gt;&lt;/Affiliation&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Person&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Organization id="ORG_DOC_OWNER" name="myworkplace"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000586" name="contact name" value="address"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/AuditCollection&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_bibliographicreference">Element &lt;BibliographicReference&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Any bibliographic references associated with the file</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">BibliographicReferenceType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|authors |xsd:string |optional |The names of the authors of the reference.
|doi |xsd:string |optional |The DOI of the referenced publication.
|editor |xsd:string |optional |The editor(s) of the reference.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|issue |xsd:string |optional |The issue name or number.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|pages |xsd:string |optional |The page numbers.
|publication |xsd:string |optional |The name of the journal, book etc.
|publisher |xsd:string |optional |The publisher of the publication.
|title |xsd:string |optional |The title of the BibliographicReference.
|volume |xsd:string |optional |The volume name or number.
|year |xsd:int |optional |The year of publication.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;BibliographicReference id="10.1002/(SICI)1522-2683(19991201)20:18&lt;3551::AID-ELPS3551&gt;3.0.CO;2-2" name="Probability-based protein identification by searching sequence databases using mass spectrometry data" authors="David N. Perkins, Darryl J. C. Pappin, David M. Creasy, John S. Cottrell" publication="Electrophoresis" publisher="Wiley VCH" editor="" year="1999" volume="20" issue="18" pages="3551-3567" title="Probability-based protein identification by searching sequence databases using mass spectrometry data" /&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_contactrole">Element &lt;ContactRole&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top"><strong>Depending on context:</strong><br>
<strong>1</strong>: The contact details of the organisation or person that produced the software<br>
<strong>2</strong>: Contact details for the Material. The association to ContactRole could specify, for example, the creator or provider of the Material.<br>
<strong>3</strong>: The Contact that provided the document instance.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ContactRoleType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|contact_ref |xsd:string |required |When a ContactRole is used, it specifies which Contact the role is associated with.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-role">Role</a> |1 |1 |The roles (lab equipment sales, contractor, etc.) the Contact fills.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;ContactRole contact_ref="PERSON_DOC_OWNER"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Role&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001271" name="researcher"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Role&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/ContactRole&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_customizations">Element &lt;Customizations&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Any customizations to the software, such as alternative scoring mechanisms implemented, should be documented here as free text.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd:string</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Customizations&gt;</p>
</div>
<div class="paragraph">
<p>No customisations</p>
</div>
<div class="paragraph">
<p>&lt;/Customizations&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_cv">Element &lt;cv&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A source controlled vocabulary from which cvParams will be obtained.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">cvType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|fullName |xsd:string |required |The full name of the CV.
|id |xsd:string |required |The unique identifier of this cv within the document to be referenced by cvParam elements.
|uri |xsd:anyURI |required |The URI of the source CV.
|version |xsd:string |optional |The version of the CV.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="listingblock">
<div class="content">
<pre class="highlight"><code class="language-xml" data-lang="xml">&lt;cv fullName="Proteomics Standards Initiative Mass Spectrometry Vocabularies"
version="2.32.0"
uri=" https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master/psi-ms.obo"
id="PSI-MS" /&gt;</code></pre>
</div>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_cvlist">Element &lt;cvList&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The list of controlled vocabularies used in the file.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">CVListType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cv">cv</a> |1 |unbounded |A source-controlled vocabulary from which cvParams will be obtained.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvList&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cv id="PSI-MS" fullName="Proteomics Standards Initiative Mass Spectrometry Vocabularies" uri="https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master/psi-ms.obo" version="3.15.0"&gt;&lt;/cv&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cv id="UNIMOD" fullName="UNIMOD" uri="http://www.unimod.org/obo/unimod.obo"&gt;&lt;/cv&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cv id="UO" fullName="UNIT-ONTOLOGY" uri="https://raw.githubusercontent.com/bio-ontology-research-group/unit-ontology/master/unit.obo"&gt;&lt;/cv&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cv id="XLMOD" fullName="PSI cross-link modifications" uri="https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/master/cv/XLMOD-1.0.0.obo"&gt;&lt;/cv&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/cvList&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_cvparam">Element &lt;cvParam&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A single entry from an ontology or a controlled vocabulary.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">CVParamType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|accession |xsd:string |required |The accession or ID number of this CV term in the source CV.
|cvRef |xsd:string |required |A reference to the cv element from which this term originates.
|name |xsd:string |required |The name of the parameter.
|unitAccession |xsd:string |optional |An accession number identifying the unit within the OBO foundry Unit CV.
|unitCvRef |xsd:string |optional |If a unit term is referenced, this attribute MUST refer to the CV 'id' attribute defined in the cvList in this file.
|unitName |xsd:string |optional |The name of the unit.
|value |xsd:string |optional |The user-entered value of the parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002520" name="peptide group ID" value="CCPQCCSSGCSQNLCGPLCVTTPYYCTR_<mark>Carbamidomethyl&#169;:1</mark>Carbamidomethyl&#169;:2##Carbamidomethyl(C</p>
</div>
<div class="paragraph">
<p>:5<mark>Carbamidomethyl&#169;:6</mark>Carbamidomethyl&#169;:10<mark>Carbamidomethyl&#169;:15</mark>Carbamidomethyl(C</p>
</div>
<div class="paragraph">
<p>:19##Carbamidomethyl&#169;:26"&gt;&lt;/cvParam&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_databasefilters">Element &lt;DatabaseFilters&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The specification of filters applied to the database searched.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">DatabaseFiltersType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-filter">Filter</a> |1 |unbounded |Filters applied to the search database.
The filter MUST include at least one of Include and Exclude.
If both are used, it is assumed that inclusion is performed first.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;DatabaseFilters&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Filter&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FilterType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001020" cvRef="PSI-MS" name="DB filter taxonomy"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FilterType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Filter&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/DatabaseFilters&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_databasename">Element &lt;DatabaseName&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The database name may be given as a cvParam if it maps exactly to one of the release databases listed in the CV, otherwise a userParam should be used.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ParamType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |1 |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |1 |1 |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;DatabaseName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="uniprot-human-reviewed-trypsin-april-2016_concatenated_target_decoy.fasta"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/DatabaseName&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/DataCollection/Inputs/SearchDatabase/DatabaseName</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001013">MS:1001013</a> (database name) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001084">MS:1001084</a> (database nr)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001104">MS:1001104</a> (database UniProtKB/Swiss-Prot)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001142">MS:1001142</a> (database IPI_human)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001285">MS:1001285</a> (database IPI_mouse)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001286">MS:1001286</a> (database IPI_rat)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001287">MS:1001287</a> (database IPI_zebrafish)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001288">MS:1001288</a> (database IPI_chicken)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001289">MS:1001289</a> (database IPI_cow)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001290">MS:1001290</a> (database IPI_arabidopsis)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002060">MS:1002060</a> (database UniProtKB/TrEMBL)</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;cvParam accession="<a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001104">MS:1001104</a>" cvRef="PSI-MS" name="database UniProtKB/Swiss-Prot"/&gt;</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example userParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;userParam name="fawaz_PXD000652_combined_concatenated_target_decoy.fasta"&gt;&lt;/userParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="no description"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="Rosetta_uniprot_20130402_mouse_SWISS_can_iso_ECOLI.fasta"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="Ros_Uniprot_20130402.fasta"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="26Syeast_test.fasta"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="Ros_Uniprot_Ecoli_20130402.fasta" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="uniprot-human-reviewed-trypsin-april-2016_concatenated_target_decoy.fasta"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="HSA-Active.FASTA"&gt;&lt;/userParam&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_databasetranslation">Element &lt;DatabaseTranslation&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A specification of how a nucleic acid sequence database was translated for searching.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">DatabaseTranslationType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|frames |listOfAllowedFrames |optional |The frames in which the nucleic acid sequence has been translated as a space separated list</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-translationtable">TranslationTable</a> |1 |unbounded |The table used to translate codons into nucleic acids e.g. by reference to the NCBI translation table.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;DatabaseTranslation frames="1 2 3 -1 -2 -3"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;TranslationTable id="TT_1" name="Standard"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001025" name="translation table" cvRef="PSI-MS" value="FFLLSSSSYY**CC*WLLLLPPPPHHQQRRRRIIIMTTTTNNKKSSRRVVVVAAAADDEEGGGG" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001410" name="translation start codons" cvRef="PSI-MS" value="---M---------------M---------------M----------------------------" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001423" name="translation table description" cvRef="PSI-MS" value="http://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/index.cgi?chapter=cgencodes#SG1" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/TranslationTable&gt;</p>
</div>
<div class="paragraph">
<p>&lt;TranslationTable id="TT_2" name="Vertebrate Mitochondrial"&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/DatabaseTranslation&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_datacollection">Element &lt;DataCollection&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The collection of input and output data sets of the analyses.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">DataCollectionType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-inputs">Inputs</a> |1 |1 |The inputs to the analyses including the databases searched, the spectral data and the source file converted to mzIdentML.
|<a href="#element-analysisdata">AnalysisData</a> |1 |1 |Data sets generated by the analyses, including peptide and protein lists.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Graphical Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="image"><img src="img//media/image6.png" alt="http://www.peptideatlas.org/PSI/schemas/mzIdentML/1.2/figures/DataCollection.png" width="566" height="642"></span></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;DataCollection&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Inputs xmlns="http://psidev.info/psi/pi/mzIdentML/1.2"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SearchDatabase numDatabaseSequences="57566" location="E:\Work\PSI\mzIdentML\ProteinInference\Rosetta2\FASTAswithecoli\Rosetta_uniprot_20130402_</p>
</div>
<div class="paragraph">
<p>mouse_SWISS_can_iso_ECOLI.fasta" id="SearchDB_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001348" cvRef="PSI-MS" name="FASTA format"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;DatabaseName&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/DataCollection&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_dbsequence">Element &lt;DBSequence&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A database sequence from the specified SearchDatabase (nucleic acid or amino acid). If the sequence is nucleic acid, the source nucleic acid sequence should be given in the seq attribute rather than a translated sequence.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">DBSequenceType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|accession |xsd:string |required |The unique accession of this sequence.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|length |xsd:int |optional |The length of the sequence as a number of bases or residues.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|searchDatabase_ref |xsd:string |required |The source database of this sequence.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-seq">Seq</a> |0 |1 |The actual sequence of amino acids or nucleic acid.
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
<col style="width: 12.5%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;DBSequence accession="sp</p>
</div></div></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">P20029</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">GRP78_MOUSE 78 kDa glucose-regulated protein OS=Mus musculus GN=Hspa5</p>
<p class="tableblock">PE=1&#8230;&#8203;" searchDatabase_ref="SearchDB_1" length="655" name="sp</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">P20029</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">GRP78_MOUSE 78 kDa glucose-regula</p>
<p class="tableblock">ted protein OS=Mus musculus GN=Hspa5 PE=1 SV=3" id="dbseq_sp</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">P20029</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">GRP78_MOUSE 78 kDa glucose-regulated</p>
<p class="tableblock">protein OS=Mus musculus GN=Hspa5 PE=1&#8230;&#8203;"&gt;</p>
<p class="tableblock">&lt;Seq&gt;MMKFTVVAAALLLLGAVRAEEEDKKEDVGTVVGIDLGTTYSCVGVFKNGRVEIIANDQGNRITPSYVAFTPEGERLIGDAAKNQLTSNPENTVFDA</p>
<p class="tableblock">KRLIGRTWNDPSVQQDIKFLPFKVVEKKTKPYIQVDIGGGQTKTFAPEEISAMVLTKMKETAEAYLGKKVTHAVVTVPAYFNDAQRQATKDAGTIAGLNVM</p>
<p class="tableblock">RIINEPTAAAIAYGLDKREGEKNILVFDLGGGTFDVSLLTIDNGVFEVVATNGDTHLGGEDFDQRVMEHFIKLYKKKTGKDVRKDNRAVQKLRREVEKAKR</p>
<p class="tableblock">ALSSQHQARIEIESFFEGEDFSETLTRAKFEELNMDLFRSTMKPVQKVLEDSDLKKSDIDEIVLVGGSTRIPKIQQLVKEFFNGKEPSRGINPDEAVAYGA</p>
<p class="tableblock">AVQAGVLSGDQDTGDLVLLDVCPLTLGIETVGGVMTKLIPRNTVVPTKKSQIFSTASDNQPTVTIKVYEGERPLTKDNHLLGTFDLTGIPPAPRGVPQIEV</p>
<p class="tableblock">TFEIDVNGILRVTAEDKGTGNKNKITITNDQNRLTPEEIERMVNDAEKFAEEDKKLKERIDTRNELESYAYSLKNQIGDKEKLGGKLSSEDKETMEKAVEE</p>
<p class="tableblock">KIEWLESHQDADIEDFKAKKKELEEIVQPIISKLYGSGGPPPTGEEDTSEKDEL&lt;/Seq&gt;</p>
<p class="tableblock">&lt;/DBSequence&gt;</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Example for proteogenomics search:</strong></p>
</div>
<div class="paragraph">
<p>&lt;DBSequence searchDatabase_ref="SearchDB_1" accession="generic|A_ENSP00000284981|" id="dbseq_generic|A_ENSP00000284981|"&gt;</p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002637" name="chromosome name" value="21"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002638" name="chromosome strand" value="-"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002644" name="genome reference version" value="Homo_sapiens.GRCh38.77.gff3"&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p>&lt;/DBSequence&gt;</p>
</div>
</div>
<div class="sect2">
<h3 id="_element_enzyme">Element &lt;Enzyme&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The details of an individual cleavage enzyme should be provided by giving a regular expression or a CV term if a "standard" enzyme cleavage has been performed.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">EnzymeType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|cTermGain a| xsd:string with restriction</p>
</div>
<div class="paragraph">
<p>[A-Za-z0-9 ]+</p>
</div>
<div class="paragraph">
<p>|optional |Element formula gained at CTerm.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|minDistance |xsd:int |optional |Minimal distance for another cleavage (minimum: 1).
|missedCleavages |xsd:int |optional |The number of missed cleavage sites allowed by the search.
The attribute MUST be provided if an enzyme has been used.
|nTermGain a| xsd:string with restriction</p>
</div>
<div class="paragraph">
<p>[A-Za-z0-9 ]+</p>
</div>
<div class="paragraph">
<p>|optional |Element formula gained at NTerm.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|semiSpecific |xsd:boolean |optional |Set to true if the enzyme cleaves semi-specifically (i.e. one terminus MUST cleave according to the rules, the other can cleave at any residue), false if the enzyme cleavage is assumed to be specific to both termini (accepting for any missed cleavages).</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-siteregexp">SiteRegexp</a> |0 |1 |Regular expression for specifying the enzyme cleavage site.
|<a href="#element-enzymename">EnzymeName</a> |0 |1 |The name of the enzyme from a CV.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Enzyme missedCleavages="1" semiSpecific="false" cTermGain="OH" nTermGain="H" id="ENZ_0"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SiteRegexp&gt;(?&#8656;[KR])&lt;/SiteRegexp&gt;</p>
</div>
<div class="paragraph">
<p>&lt;EnzymeName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001313" cvRef="PSI-MS" name="Trypsin/P"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/EnzymeName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Enzyme&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_enzymename">Element &lt;EnzymeName&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The name of the enzyme from a CV.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ParamListType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |1 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;EnzymeName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001251" name="Trypsin"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/EnzymeName&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/Enzymes/Enzyme/EnzymeName</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001045">MS:1001045</a> (cleavage agent name) only once</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001091">MS:1001091</a> (NoEnzyme)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001251">MS:1001251</a> (Trypsin)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001303">MS:1001303</a> (Arg-C)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001304">MS:1001304</a> (Asp-N)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001305">MS:1001305</a> (Asp-N_ambic)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001306">MS:1001306</a> (Chymotrypsin)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001307">MS:1001307</a> (CNBr)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001308">MS:1001308</a> (Formic_acid)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001309">MS:1001309</a> (Lys-C)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001310">MS:1001310</a> (Lys-C/P)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001045">et al.</a></p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001251" name="Trypsin"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001313" cvRef="PSI-MS" name="Trypsin/P"/&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_enzymes">Element &lt;Enzymes&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The list of enzymes used in experiment</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">EnzymesType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|independent |xsd:boolean |optional |If there are multiple enzymes specified, this attribute is set to true if cleavage with different enzymes is performed independently.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-enzyme">Enzyme</a> |1 |unbounded |The details of an individual cleavage enzyme should be provided by giving a regular expression or a CV term if a "standard" enzyme cleavage has been performed.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Enzymes&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Enzyme missedCleavages="1" semiSpecific="false" cTermGain="OH" nTermGain="H" id="ENZ_0"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SiteRegexp&gt;(?&#8656;[KR])&lt;/SiteRegexp&gt;</p>
</div>
<div class="paragraph">
<p>&lt;EnzymeName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001313" cvRef="PSI-MS" name="Trypsin/P"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/EnzymeName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Enzyme&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/Enzymes&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_exclude">Element &lt;Exclude&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">All sequences fulfilling the specifed criteria are excluded.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ParamListType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |1 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/DatabaseFilters/Filter/Exclude</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001512">MS:1001512</a> (Sequence database filters) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001090">MS:1001090</a> (taxonomy nomenclature)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001201">MS:1001201</a> (DB MW filter maximum)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001202">MS:1001202</a> (DB MW filter minimum)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001203">MS:1001203</a> (DB PI filter maximum)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001204">MS:1001204</a> (DB PI filter minimum)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001467">MS:1001467</a> (taxonomy: NCBI TaxID)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001468">MS:1001468</a> (taxonomy: common name)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001469">MS:1001469</a> (taxonomy: scientific name)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001470">MS:1001470</a> (taxonomy: Swiss-Prot ID)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001513">MS:1001513</a> (DB sequence filter pattern)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001512">et al.</a></p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_externalformatdocumentation">Element &lt;ExternalFormatDocumentation&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A URI to access documentation and tools to interpret the external format of the ExternalData instance. For example, XML Schema or static libraries (APIs) to access binary formats.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd:anyURI</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_fileformat">Element &lt;FileFormat&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The format of the ExternalData file, for example "tiff" for image files.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">FileFormatType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |1 |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001401" name="X!Tandem xml format"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FileFormat&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/DataCollection/Inputs/SearchDatabase/FileFormat</p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001347">MS:1001347</a> (database file formats) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001348">MS:1001348</a> (FASTA format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001349">MS:1001349</a> (ASN.1)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001350">MS:1001350</a> (NCBI <strong>.p</strong>)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001351">MS:1001351</a> (clustal aln)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001352">MS:1001352</a> (embl em)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001353">MS:1001353</a> (NBRF PIR)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001462">MS:1001462</a> (PEFF format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002659">MS:1002659</a> (UniProtKB text sequence format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002660">MS:1002660</a> (UniProtKB XML sequence format)</p>
</div>
<div class="paragraph">
<p>Path /MzIdentML/DataCollection/Inputs/SourceFile/FileFormat</p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001040">MS:1001040</a> (intermediate analysis format) only once</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000742">MS:1000742</a> (Bioworks SRF format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001107">MS:1001107</a> (data stored in database)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001199">MS:1001199</a> (Mascot DAT format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001200">MS:1001200</a> (SEQUEST out file format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001242">MS:1001242</a> (SEQUEST out folder)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001243">MS:1001243</a> (SEQUEST summary)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001275">MS:1001275</a> (ProteinScape SearchEvent)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001276">MS:1001276</a> (ProteinScape Gel)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001399">MS:1001399</a> (OMSSA csv format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001400">MS:1001400</a> (OMSSA xml format)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001040">et al.</a></p>
</div>
<div class="paragraph">
<p>Path /MzIdentML/DataCollection/Inputs/SpectraData/FileFormat</p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000560">MS:1000560</a> (mass spectrometer file format) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000526">MS:1000526</a> (Waters raw format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000562">MS:1000562</a> (ABI WIFF format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000563">MS:1000563</a> (Thermo RAW format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000564">MS:1000564</a> (PSI mzData format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000565">MS:1000565</a> (Micromass PKL format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000566">MS:1000566</a> (ISB mzXML format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000567">MS:1000567</a> (Bruker/Agilent YEP format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000584">MS:1000584</a> (mzML format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000613">MS:1000613</a> (DTA format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000614">MS:1000614</a> (ProteinLynx Global Server mass spectrum XML format)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000560">et al.</a></p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001400" name="OMSSA xml file"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001348" name="FASTA format"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001062" name="Mascot MGF file"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001401" cvRef="PSI-MS" name="X\!Tandem xml file"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001199" cvRef="PSI-MS" name="Mascot DAT format"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1000584" cvRef="PSI-MS" name="mzML format"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000563" name="Thermo Raw file"&gt;&lt;/cvParam&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_filter">Element &lt;Filter&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Filters applied to the search database. The filter MUST include at least one of Include and Exclude. If both are used, it is assumed that inclusion is performed first.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">FilterType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-filtertype">FilterType</a> |1 |1 |The type of filter e.g. database taxonomy filter, pi filter, mw filter |<a href="#element-include">Include</a> |0 |1 |All sequences fulfilling the specifed criteria are included.
|<a href="#element-exclude">Exclude</a> |0 |1 |All sequences fulfilling the specifed criteria are excluded.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Filter&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FilterType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001020" cvRef="PSI-MS" name="DB filter taxonomy"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FilterType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Filter&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_filtertype">Element &lt;FilterType&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The type of filter e.g. database taxonomy filter, pi filter, mw filter</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ParamType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |1 |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |1 |1 |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;FilterType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001020" cvRef="PSI-MS" name="DB filter taxonomy"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FilterType&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/DatabaseFilters/Filter/FilterType</p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001511">MS:1001511</a> (Sequence database filter types) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001020">MS:1001020</a> (DB filter taxonomy)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001021">MS:1001021</a> (DB filter on accession numbers)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001022">MS:1001022</a> (DB MW filter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001023">MS:1001023</a> (DB PI filter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001027">MS:1001027</a> (DB filter on sequence pattern)</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;cvParam accession="MS:1001020" cvRef="PSI-MS" name="DB filter taxonomy"/&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_fragmentarray">Element &lt;FragmentArray&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">An array of values for a given type of measure and for a particular ion type, in parallel to the index of ions identified.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">FragmentArrayType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|measure_ref |xsd:string |required |A reference to the Measure defined in the FragmentationTable |values |listOfFloats |required |The values of this particular measure, corresponding to the index defined in ion type</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;FragmentArray measure_ref="Measure_Error" values="4.173258879802688E-4 -1.9794682032170385E-5 1.618474794895519E-5 0.001690052197886871 -0.0037214683721344954 0.0020060110579152024 -2.3719321211501665E-4 2.7168621795681247E-4 -0.0019049343519554895 0.0019553613780090018 2.6704080801209784E-4 0.007734020238103767 0.0013568713879976713 1.571508180404635E-4 -0.0017703817320580129 0.013774177127970688 0.0056154565579618065 0.004415735988004599 0.006145015418042021 -0.005059517131940083 0.01419863401793009 0.007626913448120831 0.007892192877989146"/&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_fragmentation">Element &lt;Fragmentation&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The product ions identified in this result.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">FragmentationType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-iontype">IonType</a> |1 |unbounded |IonType defines the index of fragmentation ions being reported, importing a CV term for the type of ion e.g. b ion.
Example: if b3 b7 b8 and b10 have been identified, the index attribute will contain 3 7 8 10, and the corresponding values will be reported in parallel arrays below</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Fragmentation&gt;</p>
</div>
<div class="paragraph">
<p>&lt;IonType charge="1" index="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_MZ" values="175.1193695 232.1403961 289.1618958 452.2268982 509.2429504 566.2701416 653.2999268 710.3218994 767.3411865 854.3770752 911.3968506 968.4257813 1065.472168 1122.492432 1285.553833 1399.612305 1456.62561 1513.645874 1570.669067 1733.721191 1830.793213 1887.808105 1944.829834"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_Int" values="5939.5844726563 4933.5014648438 13310.7265625 5077.6694335938 5685.9287109375 13253.552734375 7620.0947265625 7724.3696289063 16868.541015625 10552.126953125 11589.0576171875 7839.9741210938 47821.64453125 60335.71484375 21759.3984375 8742.5595703125 11512.0908203125 18130.890625 30577.375 3801.3923339844 8051.07421875 1954.5501708984 4844.9125976563"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_Error" values="4.173258879802688E-4 -1.9794682032170385E-5 1.618474794895519E-5 0.001690052197886871 -0.0037214683721344954 0.0020060110579152024 -2.3719321211501665E-4 2.7168621795681247E-4 -0.0019049343519554895 0.0019553613780090018 2.6704080801209784E-4 0.007734020238103767 0.0013568713879976713 1.571508180404635E-4 -0.0017703817320580129 0.013774177127970688 0.0056154565579618065 0.004415735988004599 0.006145015418042021 -0.005059517131940083 0.01419863401793009 0.007626913448120831 0.007892192877989146"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001220" name="frag: y ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/IonType&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/Fragmentation&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_fragmentationtable">Element &lt;FragmentationTable&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Contains the types of measures that will be reported in generic arrays for each SpectrumIdentificationItem e.g. product ion m/z, product ion intensity, product ion m/z error</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">FragmentationTableType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-measure">Measure</a> |1 |unbounded |References to CV terms defining the measures about product ions to be reported in SpectrumIdentificationItem</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;FragmentationTable&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Measure id="Measure_MZ"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001225" name="product ion m/z" unitCvRef="PSI-MS" unitAccession="MS:1000040" unitName="m/z" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Measure&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Measure id="Measure_Int"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001226" name="product ion intensity" unitCvRef="PSI-MS" unitAccession="MS:1000131" unitName="number of detector counts"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Measure&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/FragmentationTable&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_fragmenttolerance">Element &lt;FragmentTolerance&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The tolerance of the search given as a plus and minus value with units.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ToleranceType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;FragmentTolerance&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001412" name="search tolerance plus value" value="20.0 ppm" unitAccession="UO:0000169" unitName="parts per million" unitCvRef="UO"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001413" name="search tolerance minus value" value="20.0 ppm" unitAccession="UO:0000169" unitName="parts per million" unitCvRef="UO"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FragmentTolerance&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/FragmentTolerance</p>
</div>
<div class="paragraph">
<p>MUST supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001412">MS:1001412</a> (search tolerance plus value) only once</p>
</div>
<div class="paragraph">
<p>MUST supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001413">MS:1001413</a> (search tolerance minus value) only once</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001412" name="search tolerance plus value" value="0.7" unitAccession="UO:0000221" unitName="dalton" unitCvRef="UO"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001413" name="search tolerance minus value" value="0.7" unitAccession="UO:0000221" unitName="dalton" unitCvRef="UO"&gt;&lt;/cvParam&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_include">Element &lt;Include&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">All sequences fulfilling the specifed criteria are included.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ParamListType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |1 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/DatabaseFilters/Filter/Include</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001512">MS:1001512</a> (Sequence database filters) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001090">MS:1001090</a> (taxonomy nomenclature)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001201">MS:1001201</a> (DB MW filter maximum)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001202">MS:1001202</a> (DB MW filter minimum)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001203">MS:1001203</a> (DB PI filter maximum)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001204">MS:1001204</a> (DB PI filter minimum)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001467">MS:1001467</a> (taxonomy: NCBI TaxID)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001468">MS:1001468</a> (taxonomy: common name)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001469">MS:1001469</a> (taxonomy: scientific name)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001470">MS:1001470</a> (taxonomy: Swiss-Prot ID)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001513">MS:1001513</a> (DB sequence filter pattern)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001512">et al.</a></p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_inputs">Element &lt;Inputs&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The inputs to the analyses including the databases searched, the spectral data and the source

file converted to mzIdentML.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">InputsType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-sourcefile">SourceFile</a> |0 |unbounded |A file from which this mzIdentML instance was created.
|<a href="#element-searchdatabase">SearchDatabase</a> |0 |unbounded |A database for searching mass spectra.
Examples include a set of amino acid sequence entries, nucleotide databases (e.g. 6 frame translated) or annotated spectra libraries.
|<a href="#element-spectradata">SpectraData</a> |1 |unbounded |A data set containing spectra data (consisting of one or more spectra).</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Inputs xmlns="http://psidev.info/psi/pi/mzIdentML/1.2"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SourceFile location="C:\Work\PSI\mzIdentML\ProteinInference\Rosetta2\tandem\peaklist2a_plus_ecoli_versus_unimod_</p>
</div>
<div class="paragraph">
<p>full.xml" id="SourceFile_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001401" cvRef="PSI-MS" name="X\!Tandem xml file"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SourceFile&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SearchDatabase numDatabaseSequences="163648" location="C:/Work/PSI/mzIdentML/ProteinInference/Rosetta2/FASTAs, neat/Rosetta_uniprot_20130402_mouse_FULL_UNIPROT_can+iso.fasta" id="SearchDB_1"&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/Inputs&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_inputspectra">Element &lt;InputSpectra&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">One of the spectra data sets used.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">InputSpectraType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|spectraData_ref |xsd:string |required |A reference to the SpectraData element which locates the input spectra to an external file.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;InputSpectra spectraData_ref="SD_4299_120114_20_Orbi2_ZC_QC_220_HSAd0-d4-1to1-3_Din.raw"&gt;&lt;/InputSpectra&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_inputspectrumidentifications">Element &lt;InputSpectrumIdentifications&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The lists of spectrum identifications that are input to the protein detection process.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">InputSpectrumIdentificationsType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|spectrumIdentificationList_ref |xsd:string |required |A reference to the list of spectrum identifications that were input to the process.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;InputSpectrumIdentifications spectrumIdentificationList_ref="SII_LIST_1"/&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_iontype">Element &lt;IonType&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">IonType defines the index of fragmentation ions being reported, importing a CV term for the

Type of ion e.g. b ion. Example: if b3 b7 b8 and b10 have been identified, the index attribute

will contain 3 7 8 10, and the corresponding values will be reported in parallel arrays below</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">IonTypeType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|charge |xsd:int |required |The charge of the identified fragmentation ions.
|index |listOfIntegers |optional |The index of ions identified as integers, following standard notation for a-c, x-z e.g. if b3 b5 and b6 have been identified, the index would store "3 5 6".
For internal ions, the index contains pairs defining the start and end point - see specification document for examples.
For immonium ions, the index is the position of the identified ion within the peptide sequence - if the peptide contains the same amino acid in multiple positions that cannot be distinguished, all positions should be given.
For precursor ions, including neutral losses, the index value MUST be 0. For any other ions not related to the position within the peptide sequence e.g. quantification reporter ions, the index value MUST be 0.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-fragmentarray">FragmentArray</a> |0 |unbounded |An array of values for a given type of measure and for a particular ion type, in parallel to the index of ions identified.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;IonType charge="1" index="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_MZ" values="175.1193695 232.1403961 289.1618958</p>
</div>
<div class="paragraph">
<p>452.2268982 509.2429504 566.2701416 653.2999268 710.3218994 767.3411865 854.3770752 911.3968506 968</p>
</div>
<div class="paragraph">
<p>4257813 1065.472168 1122.492432 1285.553833 1399.612305 1456.62561 1513.645874 1570.669067 1733.721191</p>
</div>
<div class="paragraph">
<p>1830.793213 1887.808105 1944.829834"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_Int" values="5939.5844726563 4933.5014648438</p>
</div>
<div class="paragraph">
<p>13310.7265625 5077.6694335938 5685.9287109375 13253.552734375 7620.0947265625 7724.3696289063</p>
</div>
<div class="paragraph">
<p>16868.541015625 10552.126953125 11589.0576171875 7839.9741210938 47821.64453125 60335.71484375</p>
</div>
<div class="paragraph">
<p>21759.3984375 8742.5595703125 11512.0908203125 18130.890625 30577.375 3801.3923339844 8051.07421875</p>
</div>
<div class="paragraph">
<p>1954.5501708984 4844.9125976563"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_Error" values="4.173258879802688E-4</p>
</div>
<div class="paragraph">
<p>-1.9794682032170385E-5 1.618474794895519E-5 0.001690052197886871 -0.0037214683721344954</p>
</div>
<div class="paragraph">
<p>0.0020060110579152024 -2.3719321211501665E-4 2.7168621795681247E-4 -0.0019049343519554895</p>
</div>
<div class="paragraph">
<p>0.0019553613780090018 2.6704080801209784E-4 0.007734020238103767 0.0013568713879976713</p>
</div>
<div class="paragraph">
<p>1.571508180404635E-4 -0.0017703817320580129 0.013774177127970688 0.0056154565579618065</p>
</div>
<div class="paragraph">
<p>0.004415735988004599 0.006145015418042021 -0.005059517131940083 0.01419863401793009</p>
</div>
<div class="paragraph">
<p>0.007626913448120831 0.007892192877989146"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001220" name="frag: y ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/IonType&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/DataCollection/AnalysisData/SpectrumIdentificationList/SpectrumIdentificationResult/</p>
</div>
<div class="paragraph">
<p>SpectrumIdentificationItem/Fragmentation/IonType</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001221">MS:1001221</a> (fragmentation information) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000903">MS:1000903</a> (product ion series ordinal)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000904">MS:1000904</a> (product ion m/z delta)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000926">MS:1000926</a> (product interpretation rank)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001220">MS:1001220</a> (frag: y ion)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001222">MS:1001222</a> (frag: b ion - H2O)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001223">MS:1001223</a> (frag: y ion - H2O)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001224">MS:1001224</a> (frag: b ion)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001225">MS:1001225</a> (product ion m/z)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001227">MS:1001227</a> (product ion m/z error)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001228">MS:1001228</a> (frag: x ion)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001221">et al.</a></p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam accession="MS:1001224" cvRef="PSI-MS" name="frag: b ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001220" cvRef="PSI-MS" name="frag: y ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002681" cvRef="PSI-MS" name="OpenXQuest:combined score" value="21.9678562261903"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002511" cvRef="PSI-MS" name="crosslink spectrum identification item" value="3050674907789158263"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1000894" cvRef="PSI-MS" name="retention time" value="5468.0193" unitAccession="second" unitName="" unitCvRef="se"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001523" name="frag: precursor ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002466" name="PeptideShaker PSM score" value="0.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002467" name="PeptideShaker PSM confidence" value="0.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002469" name="PeptideShaker peptide confidence" value="4.0000000000000036"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002468" name="PeptideShaker peptide score" value="-0.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002500" name="peptide passes threshold" value="false"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002520" name="peptide group ID" value="QKAQAAATVVK"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001328" name="OMSSA:evalue" value="68.145917448381"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001117" name="theoretical mass" value="1113.6506071554904" unitCvRef="UO" unitAccession="UO:0000221" unitName="dalton"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002540" name="PeptideShaker PSM confidence type" value="Not Validated"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000796" name="spectrum title" value="qExactive01819.13825.13825. File:"qExactive01819.raw", NativeID:"controllerType=0 controllerNumber=1 scan=13825""/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001222" name="frag: b ion - H2O"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001239" name="frag: immonium ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001223" name="frag: y ion - H2O"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001233" name="frag: y ion - NH3"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001521" name="frag: precursor ion - H2O"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002536" name="D-Score" value="2:99.6124031007752:1:true"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001330" name="X!Tandem:expect" value="0.0067"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001232" name="frag: b ion - NH3"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001522" name="frag: precursor ion - NH3"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002674" name="frag: b ion - CH4OS"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001969" name="phosphoRS score" value="1:50.0:4:false"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002550" name="peptide:phosphoRS score" value="1:50.0:4:false"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002553" name="peptide:D-Score" value="1:1.4263074484944571:4:false"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002694" name="frag: precursor ion - CH4OS"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002686" name="frag: y ion - CH4OS"/&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example userParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;userParam name="crosslink_chain" unitName="xsd:string" values="alpha"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="crosslink_ioncategory" unitName="xsd:string" values="ci"/&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_masstable">Element &lt;MassTable&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The masses of residues used in the search.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MassTableType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|msLevel |listOfIntegers |required |The MS spectrum that the MassTable refers to e.g. "1" for MS1 "2" for MS2 or "1 2" for MS1 or MS2.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-residue">Residue</a> |0 |unbounded |The specification of a single residue within the mass table.
|<a href="#element-ambiguousresidue">AmbiguousResidue</a> |0 |unbounded |Ambiguous residues e.g. X can be specified by the Code attribute and a set of parameters for example giving the different masses that will be used in the search.
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;MassTable msLevel="1 2" id="MT"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Residue mass="71.03712" code="A"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Residue mass="103.009186" code="C"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Residue mass="115.02694" code="D"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Residue mass="129.04259" code="E"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Residue mass="147.06842" code="F"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Residue mass="57.021465" code="G"/&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/MassTable&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/MassTable</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001354">MS:1001354</a> (mass table options) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001346">MS:1001346</a> (AAIndex mass table)</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_measure">Element &lt;Measure&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">References to CV terms defining the measures about product ions to be reported in SpectrumIdentificationItem</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">MeasureType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Measure id="Measure_Int"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001226" name="product ion intensity" unitCvRef="PSI-MS" unitAccession="MS:1000131" unitName="number of detector counts"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Measure&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/DataCollection/AnalysisData/SpectrumIdentificationList/FragmentationTable/Measure</p>
</div>
<div class="paragraph">
<p>MUST supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001226">MS:1001226</a> (product ion intensity) only once</p>
</div>
<div class="paragraph">
<p>MUST supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001225">MS:1001225</a> (product ion m/z) only once</p>
</div>
<div class="paragraph">
<p>MUST supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001227">MS:1001227</a> (product ion m/z error) only once</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam accession="MS:1001225" cvRef="PSI-MS" unitCvRef="PSI-MS" unitName="m/z" unitAccession="MS:1000040" name="product ion m/z"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001226" cvRef="PSI-MS" unitCvRef="PSI-MS" unitName="number of detector counts" unitAccession="MS:1000131" name="product ion intensity"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001227" cvRef="PSI-MS" unitCvRef="PSI-MS" unitName="m/z" unitAccession="MS:1000040" name="product ion m/z error"/&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_modification">Element &lt;Modification&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A molecule modification specification. If <em>n</em> modifications have been found on a peptide, there should be <em>n</em> instances of Modification. If multiple modifications are provided as cvParams, it is assumed that the modification is ambiguous i.e. one modification or another. A cvParam MUST be provided with the identification of the modification sourced from a suitable CV e.g. UNIMOD. If the modification is not present in the CV (and this will be checked by the semantic validator within a given tolerance window), there is a “unknown modification” CV term that MUST be used instead. A neutral loss should be defined as an additional CVParam within Modification. If more complex information should be given about neutral losses (such as presence/absence on particular product ions), this can additionally be encoded within the FragmentationArray.

MAY also contain the CV term "search modification id ref" (MS:1003393) once to link the Modification to a SearchModification defined in the ModificationParams of the related SpectrumIdentificationProtocol (Section 7.12). The value of this term is the id of the SearchModification as defined by its “search modification id” (MS:1003392) CV term.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ModificationType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|avgMassDelta |xsd:double |optional |Atomic mass delta considering the natural distribution of isotopes in Daltons.
|location |xsd:int |optional |Location of the modification within the peptide - position in peptide sequence, counted from the N-terminus residue, starting at position 1. Specific modifications to the N-terminus should be given the location 0. Modification to the C-terminus should be given as peptide length + 1. If the modification location is unknown e.g. for PMF data, this attribute should be omitted.
|monoisotopicMassDelta |xsd:double |optional |Atomic mass delta when assuming only the most common isotope of elements in Daltons.
|residues |listOfChars |optional |Specification of the residue (amino acid) on which the modification occurs.
If multiple values are given, it is assumed that the exact residue modified is unknown i.e. the modification is to ONE of the residues listed.
Multiple residues would usually only be specified for PMF data.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Modification location="10" residues="K" monoisotopicMassDelta="138.0680796"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="XLMOD:02001" cvRef="XLMOD" name="DSS"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002509" cvRef="PSI-MS" name="crosslink donor" value="11309529182388590588"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Modification&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1003393" name="search modification id ref" value="DSSO_donor"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:35" name="Oxidation"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:37" name="Trimethyl"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:4" name="Carbamidomethyl"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:27" name="Glu&#8594;pyro-Glu"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:1" name="Acetyl"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:385" name="Ammonia-loss"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:28" name="Gln&#8594;pyro-Glu"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:575" name="Gly&#8594;Val"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="UNIMOD:214" cvRef="UNIMOD" name="iTRAQ4plex"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="UNIMOD:7" cvRef="UNIMOD" name="Deamidated"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="UNIMOD:39" cvRef="UNIMOD" name="Methylthio"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="XLMOD:02001" cvRef="XLMOD" name="DSS"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002509" cvRef="PSI-MS" name="crosslink donor" value="11309529182388590588"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002510" cvRef="PSI-MS" name="crosslink acceptor" value="2399294065069360606"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="UNIMOD:1020" name="xlink:DSS" cvRef="UNIMOD"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:01000" name="hydrolyzed BS3"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:01001" name="amidated BS3"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:02000" name="BS3"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:01008" name="hydrolyzed BS3-d4"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:01009" name="amidated BS3-d4"&gt;&lt;/cvParam&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Example for crosslinking:</strong></p>
</div>
<div class="paragraph">
<p>&lt;Modification location="2" residues="Y" monoisotopicMassDelta="156.07864430999996"&gt;</p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:01000" name="hydrolysed BS3"&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p>&lt;/Modification&gt;</p>
</div>
</div>
<div class="sect2">
<h3 id="_element_modificationparams">Element &lt;ModificationParams&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The specification of static/variable modifications (e.g. Oxidation of Methionine) that are to be considered in the spectra search.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ModificationParamsType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-searchmodification">SearchModification</a> |1 |unbounded |Specification of a search modification as parameter for a spectra search.
Contains the name of the modification, the mass, the specificity and whether it is a static modification.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;ModificationParams&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SearchModification residues="M" massDelta="15.9949" fixedMod="false"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="UNIMOD:35" cvRef="UNIMOD" name="Oxidation"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SearchModification&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SearchModification residues="." massDelta="144.102" fixedMod="false"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="UNIMOD:214" cvRef="UNIMOD" name="iTRAQ4plex"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001189" cvRef="PSI-MS" name="modification specificity peptide N-term"/&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/ModificationParams&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_organization">Element &lt;Organization&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Organizations are entities like companies, universities, government agencies. Any additional information such as the address, email etc. should be supplied either as CV parameters or as user parameters.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">OrganizationType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.
|<a href="#element-parent">Parent</a> |0 |1 |The containing organization (the university or business which a lab belongs to, etc.)</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Organization name="PeptideShaker developers" id="PS_DEV"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000586" name="contact name" value="PeptideShaker developers"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000587" name="contact address" value="Proteomics Unit, Building for Basic Biology, University of Bergen, Jonas Liesvei 91, N-5009 Bergen, Norway"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000588" name="contact URL" value="http://compomics.github.io/projects/peptide-shaker.html"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000589" name="contact email" value="<a href="mailto:peptide-shaker@googlegroups.com">peptide-shaker@googlegroups.com</a>"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Organization&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AuditCollection/Organization</p>
</div>
<div class="paragraph">
<p>SHOULD supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000588">MS:1000588</a> (contact URL) one or more times</p>
</div>
<div class="paragraph">
<p>SHOULD supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000587">MS:1000587</a> (contact address) one or more times</p>
</div>
<div class="paragraph">
<p>SHOULD supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000589">MS:1000589</a> (contact email) one or more times</p>
</div>
<div class="paragraph">
<p>SHOULD supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000586">MS:1000586</a> (contact name) one or more times</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000586" name="contact name" value="address"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1000588" cvRef="PSI-MS" value="http://www.matrixscience.com" name="contact URL"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000587" name="contact address" value="test"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000589" name="contact email" value="test"/&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_parent">Element &lt;Parent&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The containing organization (the university or business which a lab belongs to, etc.)</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ParentOrganizationType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|organization_ref |xsd:string |required |A reference to the organization this contact belongs to.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_parenttolerance">Element &lt;ParentTolerance&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The tolerance of the search given as a plus and minus value with units.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ToleranceType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;ParentTolerance&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001412" name="search tolerance plus value" value="6.0 ppm" unitAccession="UO:0000169" unitName="parts per million" unitCvRef="UO"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001413" name="search tolerance minus value" value="6.0 ppm" unitAccession="UO:0000169" unitName="parts per million" unitCvRef="UO"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/ParentTolerance&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/ParentTolerance</p>
</div>
<div class="paragraph">
<p>MUST supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001412">MS:1001412</a> (search tolerance plus value) only once</p>
</div>
<div class="paragraph">
<p>MUST supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001413">MS:1001413</a> (search tolerance minus value) only once</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001412" name="search tolerance plus value" value="10.0" unitAccession="UO:0000221" unitName="dalton" unitCvRef="UO"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001413" name="search tolerance minus value" value="10.0" unitAccession="UO:0000221" unitName="dalton" unitCvRef="UO"&gt;&lt;/cvParam&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_peptide">Element &lt;Peptide&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">One (poly)peptide (a sequence with modifications). The combination of Peptide sequence and

modifications MUST be unique in the file.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PeptideType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-peptidesequence">PeptideSequence</a> |1 |1 |The amino acid sequence of the (poly)peptide.
If a substitution modification has been found, the original sequence should be reported.
|<a href="#element-modification">Modification</a> |0 |unbounded |A molecule modification specification.
If n modifications have been found on a peptide, there should be n instances of Modification.
If multiple modifications are provided as cvParams, it is assumed that the modification is ambiguous i.e. one modification or another.
A cvParam MUST be provided with the identification of the modification sourced from a suitable CV e.g. UNIMOD.
If the modification is not present in the CV (and this will be checked by the semantic validator within a given tolerance window), there is a â€œunknown modificationâ€ CV term that MUST be used instead.
A neutral loss should be defined as an additional CVParam within Modification.
If more complex information should be given about neutral losses (such as presence/absence on particular product ions), this can additionally be encoded within the FragmentationArray.
|<a href="#element-substitutionmodification">SubstitutionModification</a> |0 |unbounded |A modification where one residue is substituted by another (amino acid change).
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Peptide id="CCPQCCSSGCSQNLCGPLCVTTPYYCTR_<mark>Carbamidomethyl&#169;:1</mark>Carbamidomethyl&#169;:2##Carbamidomethyl&#169;:</p>
</div>
<div class="paragraph">
<p>5<mark>Carbamidomethyl&#169;:6</mark>Carbamidomethyl&#169;:10<mark>Carbamidomethyl&#169;:15</mark>Carbamidomethyl&#169;:</p>
</div>
<div class="paragraph">
<p>19##Carbamidomethyl&#169;:26"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideSequence&gt;CCPQCCSSGCSQNLCGPLCVTTPYYCTR&lt;/PeptideSequence&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Modification location="1" residues="C" monoisotopicMassDelta="57.02147"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:4" name="Carbamidomethyl"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Modification&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Modification location="2" residues="C" monoisotopicMassDelta="57.02147"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:4" name="Carbamidomethyl"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/Peptide&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/SequenceCollection/Peptide</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001355">MS:1001355</a> (peptide descriptions) one or more times</p>
</div></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Example for crosslinking:</strong></p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;Peptide id="54603257_54604608_2_1_p1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideSequence&gt;KYLYEIAR&lt;/PeptideSequence&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Modification location="2" residues="Y" monoisotopicMassDelta="156.07864430999996"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:01000" name="hydrolysed BS3"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Modification&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Modification location="4" residues="Y" monoisotopicMassDelta="155.094628715"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:01001" name="amidated BS3"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Modification&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Modification location="2" monoisotopicMassDelta="0.0"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor" value="1"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Modification&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Peptide&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Peptide id="54603257_54604608_2_1_p0"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideSequence&gt;LSVEAFEK&lt;/PeptideSequence&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Modification location="3" monoisotopicMassDelta="138.06807961"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:02000" name="BS3"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002509" name="crosslink donor" value="1"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Modification&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Peptide&gt;</p>
</div>
</blockquote>
</div>
</div>
<div class="sect2">
<h3 id="_element_peptideevidence">Element &lt;PeptideEvidence&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">PeptideEvidence links a specific Peptide element to a specific position in a DBSequence.

There MUST only be one PeptideEvidence item per Peptide-to-DBSequence-position.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PeptideEvidenceType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|dBSequence_ref |xsd:string |required |A reference to the protein sequence in which the specified peptide has been linked.
|end |xsd:int |optional |The index position of the last amino acid of the peptide inside the protein sequence, where the first amino acid of the protein sequence is position 1. Must be provided unless this is a de novo search.
|frame |allowed_frames |optional |The translation frame of this sequence if this is PeptideEvidence derived from nucleic acid sequence |id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|isDecoy |xsd:boolean |optional |Set to true if the peptide is matched to a decoy sequence.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|peptide_ref |xsd:string |required |A reference to the identified (poly)peptide sequence in the Peptide element.
|post a| xsd:string with restriction</p>
</div>
<div class="paragraph">
<p>[ABCDEFGHIJKLMNOPQRSTUVWXYZ?\-]{1}</p>
</div>
<div class="paragraph">
<p>|optional |Post flanking residue.
If the peptide is C-terminal, post="-" and not post="".
If for any reason it is unknown (e.g. denovo), post="?" should be used.
|pre a| xsd:string with restriction</p>
</div>
<div class="paragraph">
<p>[ABCDEFGHIJKLMNOPQRSTUVWXYZ?\-]{1}</p>
</div>
<div class="paragraph">
<p>|optional |Previous flanking residue.
If the peptide is N-terminal, pre="-" and not pre="".
If for any reason it is unknown (e.g. denovo), pre="?" should be used.
|start |xsd:int |optional |Start position of the peptide inside the protein sequence, where the first amino acid of the protein sequence is position 1. Must be provided unless this is a de novo search.
|translationTable_ref |xsd:string |optional |A reference to the translation table used if this is PeptideEvidence derived from nucleic acid sequence</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 16.6666%;">
<col style="width: 16.6666%;">
<col style="width: 16.6666%;">
<col style="width: 16.6666%;">
<col style="width: 16.6666%;">
<col style="width: 16.667%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;PeptideEvidence dBSequence_ref="dbseq_generic</p>
</div></div></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">B_GENSCAN00000036974_REVERSED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:genscan" peptide_ref="CRCQYSGVNNLCHTSSHCPNQGSTCENVDTCLKPDEPK_<mark>Carbamidomethyl&#169;:1</mark>Carbamidomethyl&#169;:</p>
<p class="tableblock">3<mark>Carbamidomethyl&#169;:12</mark>Carbamidomethyl&#169;:18<mark>Carbamidomethyl&#169;:25</mark>Carbamidomethyl&#169;:</p>
<p class="tableblock">31##Ammonia-loss&#169;:1" start="494" end="531" pre="R" post="L" isDecoy="true" id="CRCQYSGVNNLCHTSSHCPNQGSTCENVDTCLKPDEPK_generic</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">B_GENSCAN00000036974_REVERSED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:genscan_494_531"&gt;</p>
<p class="tableblock">&lt;/PeptideEvidence&gt;</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Example for proteogenomics search:</strong></p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidence dBSequence_ref="dbseq_generic|A_ENSP00000287611|" peptide_ref="YNSQNQSNNQFVLYR_" start="44" end="58" pre="K" post="I" isDecoy="false" id="YNSQNQSNNQFVLYR_generic|A_ENSP00000287611|_44_58"&gt;</p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002640" name="peptide end on chromosome" value="186717716"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002641" name="peptide exon count" value="1"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002642" name="peptide exon nucleotide sizes" value="45"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002643" name="peptide start positions on chromosome" value="186717673"&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p>&lt;/PeptideEvidence&gt;</p>
</div>
</div>
<div class="sect2">
<h3 id="_element_peptideevidenceref">Element &lt;PeptideEvidenceRef&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Reference to the PeptideEvidence element identified. If a specific sequence can be assigned to

multiple proteins and or positions in a protein all possible PeptideEvidence elements should be

referenced here.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PeptideEvidenceRefType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|peptideEvidence_ref |xsd:string |required |A reference to the PeptideEvidenceItem element(s).</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="GEGGAQDGSGTEGVGATGAAGGRGAQGAPGGTGGAGSGSGLHHQQDSGYQGASGSGGAQSGGR_generic</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP</p>
<p class="tableblock">00000352272_REVERSED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:known_125_187"&gt;&lt;/PeptideEvidenceRef&gt;</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002356" name="PSM-level combined FDRScore" value="3.9523759266648643E-7"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002359" name="distinct peptide-level local FDR" value="0.0"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001868" name="distinct peptide-level q-value" value="0.0"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002360" name="distinct peptide-level FDRScore" value="3.0117913560694526E-7"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002500" name="peptide passes threshold" value="true"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002520" name="peptide group ID" value="AVMDDFAAFVEK_##Oxidation(M):3"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002439" name="final PSM list UNDER DISCUSSION"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002511" name="Crosslinked spectrum identification item." value="21"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002545" name="The xi result 'Score'." value="2.769918944845425"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000797" name="peak list scans" value="6655"&gt;&lt;/cvParam&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example userParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;userParam name="search engines identifying PSM" value="12"&gt;&lt;/userParam&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_peptidehypothesis">Element &lt;PeptideHypothesis&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Peptide evidence on which this ProteinHypothesis is based by reference to a PeptideEvidence element.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PeptideHypothesisType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|peptideEvidence_ref |xsd:string |required |A reference to the PeptideEvidence element on which this hypothesis is based.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-spectrumidentificationitemref">SpectrumIdentificationItemRef</a> |1 |unbounded |Reference(s) to the SpectrumIdentificationItem element(s) that support the given PeptideEvidence element.
Using these references it is possible to indicate which spectra were actually accepted as evidence for this peptide identification in the given protein.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;PeptideHypothesis peptideEvidence_ref="PepEv_1864"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_1780_1"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_2217_1"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_3245_1"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_4362_1"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_5349_1"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_5621_1"/&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/PeptideHypothesis&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam accession="MS:1001097" cvRef="PSI-MS" value="1" name="distinct peptide sequences"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002235" cvRef="PSI-MS" value="81.01860914459425" name="ProteoGrouper:PDH score"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002401" cvRef="PSI-MS" name="leading protein"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002403" cvRef="PSI-MS" name="group representative"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001594" cvRef="PSI-MS" value="PDH_15" name="sequence same-set protein"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002415" cvRef="PSI-MS" value="true" name="protein group passes threshold"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002236" cvRef="PSI-MS" value="43.73236628236426" name="ProteoGrouper:PAG score"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002407" cvRef="PSI-MS" value="2" name="cluster identifier"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002404" cvRef="PSI-MS" value="4" name="count of identified proteins"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001596" cvRef="PSI-MS" value="PDH_239" name="sequence sub-set protein"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002402" cvRef="PSI-MS" name="non-leading protein"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001598" cvRef="PSI-MS" value="PDH_167" name="sequence subsumable protein"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:score" value="1416.6296969697" cvRef="PSI-MS" accession="MS:1001171" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="PAnalyzer:conclusive protein" cvRef="PSI-MS" accession="MS:1002213" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="PAnalyzer:non-conclusive protein" cvRef="PSI-MS" accession="MS:1002215" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="PAnalyzer:indistinguishable protein" cvRef="PSI-MS" accession="MS:1002214" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001093" name="sequence coverage" value="0.19"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002470" name="PeptideShaker protein group score" value="100.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002471" name="PeptideShaker protein group confidence" value="100.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002542" name="PeptideShaker protein confidence type" value="Confident"/&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example userParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;userParam value="IINEPTAAAIAYGLDK" name="razor peptides"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="SLSDTLEEVLSSSGEK" name="unique peptides"/&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_peptidesequence">Element &lt;PeptideSequence&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The amino acid sequence of the (poly)peptide. If a substitution modification has been found, the original sequence should be reported.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">sequence</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;PeptideSequence&gt;GEGGAQDGSGTEGVGATGAAGGRGAQGAPGGTGGAGSGSGLHHQQDSGYQGASGSGGAQSGGR&lt;/PeptideSequence&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_person">Element &lt;Person&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A person&#8217;s name and contact details. Any additional information such as the address, contact email etc. should be supplied using CV parameters or user parameters.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PersonType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|firstName |xsd:string |optional |The Person&#8217;s first name.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|lastName |xsd:string |optional |The Person&#8217;s last/family name.
|midInitials |xsd:string |optional |The Person&#8217;s middle initial.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.
|<a href="#element-affiliation">Affiliation</a> |0 |unbounded |The organization a person belongs to.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Person firstName="test" lastName="test" id="PROVIDER"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000587" name="contact address" value="test"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000588" name="contact URL" value="test"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000589" name="contact email" value="testtest"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Affiliation organization_ref="ORG_DOC_OWNER"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Person&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AuditCollection/Person</p>
</div>
<div class="paragraph">
<p>SHOULD supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000588">MS:1000588</a> (contact URL) one or more times</p>
</div>
<div class="paragraph">
<p>SHOULD supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000587">MS:1000587</a> (contact address) one or more times</p>
</div>
<div class="paragraph">
<p>SHOULD supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000589">MS:1000589</a> (contact email) one or more times</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000587" name="contact address" value="address"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1000589" cvRef="PSI-MS" value="<a href="mailto:smartinez@proteored.org">smartinez@proteored.org</a>" name="contact email"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000588" name="contact URL" value="test"/&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_proteinambiguitygroup">Element &lt;ProteinAmbiguityGroup&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A set of logically related results from a protein detection, for example to represent conflicting assignments of peptides to proteins.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ProteinAmbiguityGroupType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-proteindetectionhypothesis">ProteinDetectionHypothesis</a> |1 |unbounded |A single result of the ProteinDetection analysis (i.e. a protein).
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;ProteinAmbiguityGroup id="PAG_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;ProteinDetectionHypothesis passThreshold="true" dBSequence_ref="dbseq_sp</p>
</div></div></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q64467</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">G3PT_MOUSE Glyceraldehyde-3-phosphate dehydrogenase, testis-specific OS=Mus&#8230;&#8203;" id="PDH_11"&gt;</p>
<p class="tableblock">&lt;PeptideHypothesis peptideEvidence_ref="PE13_2_63"&gt;</p>
<p class="tableblock">&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_13_1"/&gt;</p>
<p class="tableblock">&lt;/PeptideHypothesis&gt;</p>
<p class="tableblock">&lt;cvParam accession="MS:1001097" cvRef="PSI-MS" value="1" name="distinct peptide sequences"/&gt;</p>
<p class="tableblock">&lt;cvParam accession="MS:1002235" cvRef="PSI-MS" value="34.57557513936462" name="ProteoGrouper:PDH score"/&gt;</p>
<p class="tableblock">&#8230;&#8203;
&lt;/ProteinAmbiguityGroup&gt;</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Example for protein grouping:</strong></p>
</div>
<div class="paragraph">
<p>&lt;ProteinAmbiguityGroup id="PAG_4266"&gt;</p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;ProteinDetectionHypothesis dBSequence_ref="DBSeq_RRRRRQ7TMJ9|Q7TMJ9_MOUSE" passThreshold="true" id="PDH_RRRRRQ7TMJ9|Q7TMJ9_MOUSE_PAG_4266"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideHypothesis peptideEvidence_ref="PE_APVPPSQAR(0;144.1021)-332-340-RRRRRQ7TMJ9|Q7TMJ9_MOUSE"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="2:[0,144.1021]:APVPPSQAR:index=26699"&gt;&lt;/SpectrumIdentificationItemRef&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/PeptideHypothesis&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002394" name="PIA:protein score" value="2.0881360887005513"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002401" name="leading protein"&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p>&lt;/ProteinDetectionHypothesis&gt;</p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>cvParam cvRef="PSI-MS" accession="MS:1002415" name="protein group passes threshold" value="true"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>cvParam cvRef="PSI-MS" accession="MS:1002407" name="cluster identifier" value="2814"&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p>&lt;/ProteinAmbiguityGroup&gt;</p>
</div>
</div>
<div class="sect2">
<h3 id="_element_proteindetection">Element &lt;ProteinDetection&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">An Analysis which assembles a set of peptides (e.g. from a spectra search analysis) to proteins.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ProteinDetectionType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|activityDate |xsd:dateTime |optional |When the protocol was applied.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|proteinDetectionList_ref |xsd:string |required |A reference to the ProteinDetectionList in the DataCollection section.
|proteinDetectionProtocol_ref |xsd:string |required |A reference to the detection protocol used for this ProteinDetection.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-inputspectrumidentifications">InputSpectrumIdentifications</a> |1 |unbounded |The lists of spectrum identifications that are input to the protein detection process.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;ProteinDetection id="PD_1" activityDate="2014-01-11T19:42:49" proteinDetectionList_ref="PDL_PAnalyzer" proteinDetectionProtocol_ref="PDP_PAnalyzer"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;InputSpectrumIdentifications spectrumIdentificationList_ref="SIL_1" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/ProteinDetection&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_proteindetectionhypothesis">Element &lt;ProteinDetectionHypothesis&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A single result of the ProteinDetection analysis (i.e. a protein).</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ProteinDetectionHypothesisType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|dBSequence_ref |xsd:string |required |A reference to the corresponding DBSequence entry.
Note - this attribute was optional in mzIdentML 1.1 but is now mandatory in mzIdentML 1.2. Consuming software should assume that the DBSequence entry referenced here is the definitive identifier for the protein.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|passThreshold |xsd:boolean |required |Set to true if the producers of the file has deemed that the ProteinDetectionHypothesis has passed a given threshold or been validated as correct.
If no such threshold has been set, value of true should be given for all results.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-peptidehypothesis">PeptideHypothesis</a> |1 |unbounded |Peptide evidence on which this ProteinHypothesis is based by reference to a PeptideEvidence element.
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;ProteinDetectionHypothesis passThreshold="true" dBSequence_ref="dbseq_tr</p>
</div></div></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q3V2I5</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q3V2I5_MOUSE Glyceraldehyde-3-phosphate dehydrogenase (Fragment) OS=Mus&#8230;&#8203;" id="PDH_10"&gt;</p>
<p class="tableblock">&lt;PeptideHypothesis peptideEvidence_ref="PE13_2_62"&gt;</p>
<p class="tableblock">&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_13_1"/&gt;</p>
<p class="tableblock">&lt;/PeptideHypothesis&gt;</p>
<p class="tableblock">&lt;cvParam accession="MS:1001097" cvRef="PSI-MS" value="1" name="distinct peptide sequences"/&gt;</p>
<p class="tableblock">&lt;cvParam accession="MS:1002235" cvRef="PSI-MS" value="34.57557513936462" name="ProteoGrouper:PDH score"/&gt;</p>
<p class="tableblock">&lt;cvParam accession="MS:1001594" cvRef="PSI-MS" value="PDH_11" name="sequence same-set protein"/&gt;</p>
<p class="tableblock">&#8230;&#8203;
&lt;/ProteinDetectionHypothesis&gt;</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Example for protein grouping:</strong></p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002394" name="PIA:protein score" value="107.73038501509386"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002401" name="leading protein"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001594" name="sequence same-set protein" value="PDH_Q3TWF2_PAG_2106 PDH_Q3U7T8_PAG_2106 PDH_Q3U9G2_PAG_2106"&gt;&lt;/cvParam&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p>&lt;/ProteinDetectionHypothesis&gt;</p>
</div>
</div>
<div class="sect2">
<h3 id="_element_proteindetectionlist">Element &lt;ProteinDetectionList&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The protein list resulting from a protein detection process.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ProteinDetectionListType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-proteinambiguitygroup">ProteinAmbiguityGroup</a> |0 |unbounded |A set of logically related results from a protein detection, for example to represent conflicting assignments of peptides to proteins.
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
<col style="width: 25%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;ProteinDetectionList id="PDL_1" xmlns="http://psidev.info/psi/pi/mzIdentML/1.2"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;ProteinAmbiguityGroup id="PAG_0"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;ProteinDetectionHypothesis passThreshold="true" dBSequence_ref="dbseq_sp</p>
</div></div></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">P16627</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">HS71L_MOUSE Heat shock 70 kDa protein 1-like OS=Mus musculus GN=Hspa1l&#8230;&#8203;" id="PDH_15"&gt;</p>
<p class="tableblock">&lt;PeptideHypothesis peptideEvidence_ref="PE16_2_69"&gt;</p>
<p class="tableblock">&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_16_1"/&gt;</p>
<p class="tableblock">&lt;/PeptideHypothesis&gt;</p>
<p class="tableblock">&lt;PeptideHypothesis peptideEvidence_ref="PE17_2_107"&gt;</p>
<p class="tableblock">&#8230;&#8203;
&lt;/ProteinDetectionList&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_proteindetectionprotocol">Element &lt;ProteinDetectionProtocol&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The parameters and settings of a ProteinDetection process.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ProteinDetectionProtocolType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|analysisSoftware_ref |xsd:string |required |The protein detection software used, given as a reference to the SoftwareCollection section.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-analysisparams">AnalysisParams</a> |0 |1 |The parameters and settings for the protein detection given as CV terms.
|<a href="#element-threshold">Threshold</a> |1 |1 |The threshold(s) applied to determine that a result is significant.
If multiple terms are used it is assumed that all conditions are satisfied by the passing results.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Graphical Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="image"><img src="img//media/image7.png" alt="http://www.peptideatlas.org/PSI/schemas/mzIdentML/1.2/figures/ProteinDetectionProtocol.png" width="566" height="336"></span></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;ProteinDetectionProtocol id="PDP_PAnalyzer" analysisSoftware_ref="PAnalyzer"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;AnalysisParams&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:SigThreshold" value="0.05" cvRef="PSI-MS" accession="MS:1001316" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:MaxProteinHits" value="Auto" cvRef="PSI-MS" accession="MS:1001317" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:ProteinScoringMethod" value="MudPIT" cvRef="PSI-MS" accession="MS:1001318" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:MinMSMSThreshold" value="0" cvRef="PSI-MS" accession="MS:1001319" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam name="mascot:ShowHomologousProteinsWithSamePeptides" value="1" cvRef="PSI-MS" accession="MS:1001320" /&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/ProteinDetectionProtocol&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_provider">Element &lt;Provider&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The Provider of the mzIdentML record in terms of the contact and software.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ProviderType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|analysisSoftware_ref |xsd:string |optional |The Software that produced the document instance.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-contactrole">ContactRole</a> |0 |1 |The Contact that provided the document instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Provider id="PROVIDER" xmlns="http://psidev.info/psi/pi/mzIdentML/1.2"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;ContactRole contact_ref="PERSON_DOC_OWNER"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Role&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001271" cvRef="PSI-MS" name="researcher"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Role&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/ContactRole&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Provider&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_residue">Element &lt;Residue&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The specification of a single residue within the mass table.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ResidueType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|code |chars |required |The single letter code for the residue.
|mass |xsd:float |required |The residue mass in Daltons (not including any fixed modifications).</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;Residue code="C" mass="103.009186" /&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_role">Element &lt;Role&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The roles (lab equipment sales, contractor, etc.) the Contact fills.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">RoleType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |1 |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Role&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001267" name="software vendor"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Role&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/Provider/ContactRole/Role</p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001266">MS:1001266</a> (role type) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001267">MS:1001267</a> (software vendor)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001268">MS:1001268</a> (programmer)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001269">MS:1001269</a> (instrument vendor)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001270">MS:1001270</a> (lab personnel)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001271">MS:1001271</a> (researcher)</p>
</div>
<div class="paragraph">
<p>Path /MzIdentML/AnalysisSoftwareList/AnalysisSoftware/ContactRole/Role</p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001266">MS:1001266</a> (role type) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001267">MS:1001267</a> (software vendor)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001268">MS:1001268</a> (programmer)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001269">MS:1001269</a> (instrument vendor)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001270">MS:1001270</a> (lab personnel)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001271">MS:1001271</a> (researcher)</p>
</div>
<div class="paragraph">
<p>Path /MzIdentML/AnalysisSampleCollection/Sample/ContactRole/Role</p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001266">MS:1001266</a> (role type) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001267">MS:1001267</a> (software vendor)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001268">MS:1001268</a> (programmer)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001269">MS:1001269</a> (instrument vendor)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001270">MS:1001270</a> (lab personnel)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001271">MS:1001271</a> (researcher)</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001271" name="researcher"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001267" cvRef="PSI-MS" name="software vendor"/&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_sample">Element &lt;Sample&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A description of the sample analysed by mass spectrometry using CVParams or UserParams. If a composite sample has been analysed, a parent sample should be defined, which references subsamples. This represents any kind of substance used in an experimental workflow, such as whole organisms, cells, DNA, solutions, compounds and experimental substances (gels, arrays etc.).</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SampleType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-contactrole">ContactRole</a> |0 |unbounded |The Contact that provided the document instance.
|<a href="#element-subsample">SubSample</a> |0 |unbounded |References to the individual component samples within a mixed parent sample.
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_searchdatabase">Element &lt;SearchDatabase&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A database for searching mass spectra. Examples include a set of amino acid sequence entries, nucleotide databases (e.g. 6 frame translated) or annotated spectra libraries.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SearchDatabaseType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|location |xsd:anyURI |required |The location of the data file.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|numDatabaseSequences |xsd:long |optional |The total number of sequences in the database.
|numResidues |xsd:long |optional |The number of residues in the database.
|releaseDate |xsd:dateTime |optional |The date and time the database was released to the public; omit this attribute when the date and time are unknown or not applicable (e.g. custom databases).
|version |xsd:string |optional |The version of the database.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-externalformatdocumentation">ExternalFormatDocumentation</a> |0 |1 |A URI to access documentation and tools to interpret the external format of the ExternalData instance.
For example, XML Schema or static libraries (APIs) to access binary formats.
|<a href="#element-fileformat">FileFormat</a> |1 |1 |The format of the ExternalData file, for example "tiff" for image files.
|<a href="#element-databasename">DatabaseName</a> |1 |1 |The database name may be given as a cvParam if it maps exactly to one of the release databases listed in the CV, otherwise a userParam should be used.
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SearchDatabase numDatabaseSequences="40400" location="C:\Users\hba041\My_Git_Applications\peptide-shaker.wiki\data\2016_04_05\uniprot-human-reviewed-trypsin-april-2016_concatenated_target_decoy.fasta" id="SearchDB_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001348" name="FASTA format"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;DatabaseName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="uniprot-human-reviewed-trypsin-april-2016_concatenated_target_decoy.fasta"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/DatabaseName&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/SearchDatabase&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/DataCollection/Inputs/SearchDatabase</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000561">MS:1000561</a> (data file checksum type) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000568">MS:1000568</a> (MD5)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000569">MS:1000569</a> (SHA-1)</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001011">MS:1001011</a> (search database details) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001014">MS:1001014</a> (database local file path)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001015">MS:1001015</a> (database original uri)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001016">MS:1001016</a> (database version)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001017">MS:1001017</a> (database release date)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001020">MS:1001020</a> (DB filter taxonomy)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001021">MS:1001021</a> (DB filter on accession numbers)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001022">MS:1001022</a> (DB MW filter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001023">MS:1001023</a> (DB PI filter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001024">MS:1001024</a> (translation frame)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001025">MS:1001025</a> (translation table)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001011">et al.</a></p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_searchdatabaseref">Element &lt;SearchDatabaseRef&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">One of the search databases used.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SearchDatabaseRefType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|searchDatabase_ref |xsd:string |required |A reference to the database searched.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;SearchDatabaseRef searchDatabase_ref="SDB_4299_203"&gt;&lt;/SearchDatabaseRef&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_searchmodification">Element &lt;SearchModification&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Specification of a search modification as parameter for a spectra search. Contains the name of the modification, the mass, the specificity and whether it is a static modification.

It MAY provide the CV term “search modification id” (MS:1003392) once, to provide an identifier for this SearchModification (Section 7.12). This identifier MUST be unique within the &lt;SpectrumIdentificationProtocol&gt; element. If there are multiple &lt;SpectrumIdentificationProtocol&gt; elements with in the file, then the values of “search modification id” SHOULD be identical for identical modifications.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SearchModificationType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|fixedMod |xsd:boolean |required |True, if the modification is static (i.e. occurs always).
|massDelta |xsd:float |required |The mass delta of the searched modification in Daltons.
|residues |listOfCharsOrAny |required |The residue(s) searched with the specified modification.
For N or C terminal modifications that can occur on any residue, the . character should be used to specify any, otherwise the list of amino acids should be provided.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-specificityrules">SpecificityRules</a> |0 |unbounded |The specificity rules of the searched modification including for example the probability of a modification&#8217;s presence or peptide or protein termini.
Standard fixed or variable status should be provided by the attribute fixedMod.
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SearchModification residues="E" massDelta="-18.010565" fixedMod= "false" &gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpecificityRules&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001189" name="modification specificity peptide N-term"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpecificityRules&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:27" name="Glu&#8594;pyro-Glu"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002504" name="modification index" value="3"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SearchModification&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/ModificationParams/</p>
</div>
<div class="paragraph">
<p>SearchModification</p>
</div>
<div class="paragraph">
<p>MAY supply term <span class="underline">MS:1003392</span> (search modification id) only once</p>
</div>
<div class="paragraph">
<p>MAY supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002509"><span class="underline">MS:1002509</span></a> (crosslink donor) only once</p>
</div>
<div class="paragraph">
<p>MAY supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002510"><span class="underline">MS:1002510</span></a> (crosslink acceptor) only once</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/UNIMOD:0"><span class="underline">UNIMOD:0</span></a> (unimod root node) only once</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001471"><span class="underline">MS:1001471</span></a> (peptide modification details) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001460"><span class="underline">MS:1001460</span></a> (unknown modification)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001524"><span class="underline">MS:1001524</span></a> (fragment neutral loss)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001525"><span class="underline">MS:1001525</span></a> (precursor neutral loss)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001972"><span class="underline">MS:1001972</span></a> (PTM scoring algorithm version)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002028"><span class="underline">MS:1002028</span></a> (nucleic acid base modification)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002029"><span class="underline">MS:1002029</span></a> (original nucleic acid sequence)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002030"><span class="underline">MS:1002030</span></a> (modified nucleic acid sequence)</p>
</div>
<div class="paragraph">
<p>MUST supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001460"><span class="underline">MS:1001460</span></a> (unknown modification) only once</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/XLMOD:00002"><span class="underline">XLMOD:00002</span></a> (cross-linker related PTM) only once</p>
</div>
<div class="paragraph">
<p>MAY supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002504"><span class="underline">MS:1002504</span></a> (modification index) only once</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/XLMOD:00004"><span class="underline">XLMOD:00004</span></a> (cross-linker) only once</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MOD:00000"><span class="underline">MOD:00000</span></a> (protein modification) only once</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:35" name="Oxidation"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:4" name="Carbamidomethyl"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="UNIMOD:214" cvRef="UNIMOD" name="iTRAQ4plex"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001189" cvRef="PSI-MS" name="modification specificity peptide N-term"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="UNIMOD:39" cvRef="UNIMOD" name="Methylthio"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="UNIMOD:7" cvRef="UNIMOD" name="Deamidated"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002504" name="modification index" value="0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:02000" name="BS3"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002509" name="crosslink donor" value="0"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002510" name="crosslink acceptor" value="0"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:02004" name="BS3-d4"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:01001" name="amidated BS3"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:01000" name="hydrolyzed BS3"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="UNIMOD" accession="UNIMOD:1020" name="Xlink:DSS"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:01009" name="amidated BS3-d4"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="XLMOD" accession="XLMOD:01008" name="hydrolyzed BS3-d4"&gt;&lt;/cvParam&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_searchtype">Element &lt;SearchType&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The type of search performed e.g. PMF, Tag searches, MS-MS</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ParamType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |1 |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |1 |1 |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SearchType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001083" cvRef="PSI-MS" value="" name="ms-ms search"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SearchType&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/SearchType</p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001080">MS:1001080</a> (search type) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001010">MS:1001010</a> (de novo search)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001031">MS:1001031</a> (spectral library search)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001081">MS:1001081</a> (pmf search)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001082">MS:1001082</a> (tag search)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001083">MS:1001083</a> (ms-ms search)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001584">MS:1001584</a> (combined pmf + ms-ms search)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002490">MS:1002490</a> (peptide-level scoring)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002491">MS:1002491</a> (modification localization scoring)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002492">MS:1002492</a> (consensus scoring)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002493">MS:1002493</a> (sample pre-fractionation)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001080">et al.</a></p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;cvParam cvRef="PSI-MS" accession="MS:1001083" name="ms-ms search"&gt;&lt;/cvParam&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_seq">Element &lt;Seq&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The actual sequence of amino acids or nucleic acid.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">sequence</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Seq&gt;MMKFTVVAAALLLLGAVRAEEEDKKEDVGTVVGIDLGTTYSCVGVFKNGRVEIIANDQGNRITPSYVAFTPEGERLIGDAAKNQLTSNPENTVFDA</p>
</div>
<div class="paragraph">
<p>KRLIGRTWNDPSVQQDIKFLPFKVVEKKTKPYIQVDIGGGQTKTFAPEEISAMVLTKMKETAEAYLGKKVTHAVVTVPAYFNDAQRQATKDAGTIAGLNVM</p>
</div>
<div class="paragraph">
<p>RIINEPTAAAIAYGLDKREGEKNILVFDLGGGTFDVSLLTIDNGVFEVVATNGDTHLGGEDFDQRVMEHFIKLYKKKTGKDVRKDNRAVQKLRREVEKAKR</p>
</div>
<div class="paragraph">
<p>ALSSQHQARIEIESFFEREDFSETLTRAKFEELNMDLFRSTMKPVQKVLEDSDLKKSDIDEIVLVGGSTRIPKIQQLVKEFFNGKEPSRGINPDEAVAYGA</p>
</div>
<div class="paragraph">
<p>AVQAGVLSGGQDTGDLVLLDVCPLTLGIETVGGVMTKLIPRNTVVPTKKSQIFSTASDNQPTVTIKVYEGERPLTKDNHLLGTFDLTGIPPAPRGVPQIEV</p>
</div>
<div class="paragraph">
<p>TFEIDVNGILRVTAEDKGTGNKNKITITNDQNRLTPEEIERMVNDAEKFAEEDKKLKERIDTRNELESYAYSLKNQIGDKEKLGGKLSSEDKETMEKAVEE</p>
</div>
<div class="paragraph">
<p>KIEWLESHQDADIEDFKAKKKELEEIVQPIISKLYGSGGPPPTGEEDTSEKDEL&lt;/Seq&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_sequencecollection">Element &lt;SequenceCollection&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The collection of sequences (DBSequence or Peptide) identified and their relationship between each other (PeptideEvidence) to be referenced elsewhere in the results.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SequenceCollectionType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-dbsequence">DBSequence</a> |0 |unbounded |A database sequence from the specified SearchDatabase (nucleic acid or amino acid).
If the sequence is nucleic acid, the source nucleic acid sequence should be given in the seq attribute rather than a translated sequence.
|<a href="#element-peptide">Peptide</a> |0 |unbounded |One (poly)peptide (a sequence with modifications).
The combination of Peptide sequence and modifications MUST be unique in the file.
|<a href="#element-peptideevidence">PeptideEvidence</a> |0 |unbounded |PeptideEvidence links a specific Peptide element to a specific position in a DBSequence.
There MUST only be one PeptideEvidence item per Peptide-to-DBSequence-position.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Graphical Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="image"><img src="img//media/image8.png" alt="http://www.peptideatlas.org/PSI/schemas/mzIdentML/1.2/figures/SequenceCollection.png" width="576" height="652"></span></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SequenceCollection xmlns="http://psidev.info/psi/pi/mzIdentML/1.2"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;DBSequence accession="sp</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q64467</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">G3PT_MOUSE Glyceraldehyde-3-phosphate dehydrogenase, testis-speci</p>
<p class="tableblock">fic OS=Mus&#8230;&#8203;" searchDatabase_ref="SearchDB_1" length="440" name="sp</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q64467</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">G3PT_MOUSE Glyceraldehyde-3</p>
<p class="tableblock">-phosphate dehydrogenase, testis-specific OS=Mus musculus GN=Gapdhs PE=1 SV=1" id="dbseq_sp</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q64467</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">G3PT</p>
<p class="tableblock">_MOUSE Glyceraldehyde-3-phosphate dehydrogenase, testis-specific OS=Mus&#8230;&#8203;"&gt;</p>
<p class="tableblock">&lt;Seq&gt;MSRRDVVLTNVTVVQLRRDRCPCPCPCPCPCPCPVIRPPPPKLEDPPPTVEEQPPPPPPPPPPPPPPPPPPPPQIEPDKFEEAPPPPP</p>
<p class="tableblock">PPPPPPPPPP</p>
<p class="tableblock">PPLQKPARELTVGINGFGRIGRLVLRVCMEKGIRVVAVNDPFIDPEYMVYMFKYDSTHGRYKGNVEHKNGQLVVDNLEINTYQCKDPKEIPWSSIGNPYVVEC</p>
<p class="tableblock">TGVYLSIEAASAHISSGARRVVVTAPSPDAPMFVMGVNEKDYNPGSMTIVSNASCTTNCLAPLAKVIHENFGIVEGLMTTVHSYTATQKTVDGPSKKDWRGGR</p>
<p class="tableblock">GAHQNIIPSSTGAAKAVGKVIPELKGKLTGMAFRVPTPNVSVVDLTCRLAKPASYSAITEAVKAAAKGPLAGILAYTEDQVVSTDFNGNPHSSIFDAKAGIAL</p>
<p class="tableblock">NDNFVKLVAWYDNEYGYSNRVVDLLRYMFSREK&lt;/Seq&gt;</p>
<p class="tableblock">&lt;/DBSequence&gt;</p>
<p class="tableblock">&lt;DBSequence accession="tr</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q3UEM8</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q3UEM8_MOUSE Putative uncharacterized protein (Fragment) OS=Mus</p>
<p class="tableblock">musculus&#8230;&#8203;" searchDatabase_ref="SearchDB_1" length="520" name="tr</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q3UEM8</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q3UEM8_MOUSE Putative unchar</p>
<p class="tableblock">acterized protein (Fragment) OS=Mus musculus GN=Hspa5 PE=2 SV=1" id="dbseq_tr</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q3UEM8</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Q3UEM8_MOUSE Puta</p>
<p class="tableblock">tive uncharacterized protein (Fragment) OS=Mus musculus&#8230;&#8203;"&gt;</p>
<p class="tableblock">&lt;Seq&gt;MMKFTVVAAALLLLGAVRAEEEDKKEDVGTVVGIDLGTTYSCVGVFKNGRVEIIANDQGNRITPSYVAFTPEGERLIGDAAKNQLTSNP</p>
<p class="tableblock">ENTVFDAK</p>
<p class="tableblock">RLIGRTWNDPSVQQDIKFLPFKVVEKKTKPYIQVDIGGGQTKTFAPEEISAMVLTKMKETAEAYLGKKVTHAVVTVPAYFNDAQRQATKDAGTIAGLNVMRI</p>
<p class="tableblock">INEPTAAAIAYGLDKREGEKNILVFDLGGGTFDVSLLTIDNGVFEVVATNGDTHLGGEDFDQRVMEHFIKLYKKKTGKDVRKDNRAVQKLRREVEKAKRALS</p>
<p class="tableblock">SQHQARIEIESFFEGEDFSETLTRAKFEELNMDLFRSTIKPVQKVLEDSDLKKSDIDEIVLVGGSTRIPKIQQLVKEFFNGKEPSRGINPDEAVAYGAAVQA</p>
<p class="tableblock">GVLSGDQDTGDLVLLDVCPLTLGIETVGGVMTKLIPRNTVVPTKKSQIFSTASDNQPTVTIKVYEGERPLTKDNHLLGTFDLTGIPPAPRGVPQIEVTFEID</p>
<p class="tableblock">VNGILRVTAEDKGTG&lt;/Seq&gt;</p>
<p class="tableblock">&lt;/DBSequence&gt;</p>
<p class="tableblock">&#8230;&#8203;
&lt;/SequenceCollection&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_siteregexp">Element &lt;SiteRegexp&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Regular expression for specifying the enzyme cleavage site.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">xsd:string</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;SiteRegexp&gt;(?&#8656;[KR])&lt;/SiteRegexp&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_softwarename">Element &lt;SoftwareName&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The name of the analysis software package, sourced from a CV if available.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ParamType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |1 |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |1 |1 |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SoftwareName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002244" cvRef="PSI-MS" name="mzidLib:FalseDiscoveryRate"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SoftwareName&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisSoftwareList/AnalysisSoftware/SoftwareName</p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001456">MS:1001456</a> (analysis software) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000532">MS:1000532</a> (Xcalibur)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000533">MS:1000533</a> (Bioworks)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000534">MS:1000534</a> (MassLynx)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000535">MS:1000535</a> (FlexAnalysis)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000536">MS:1000536</a> (Data Explorer)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000537">MS:1000537</a> (4700 Explorer)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000539">MS:1000539</a> (Voyager Biospectrometry Workstation System)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000551">MS:1000551</a> (Analyst)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000600">MS:1000600</a> (Proteios)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000601">MS:1000601</a> (ProteinLynx Global Server)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001456">et al.</a></p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001475" name="OMSSA"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002237" name="mzidLib"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001476" cvRef="PSI-MS" name="X\!Tandem"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002239" cvRef="PSI-MS" name="mzidLib:Tandem2Mzid"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002244" cvRef="PSI-MS" name="mzidLib:FalseDiscoveryRate"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002242" cvRef="PSI-MS" name="mzidLib:Thresholder"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002241" cvRef="PSI-MS" name="mzidLib:ProteoGrouper"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002048" cvRef="PSI-MS" name="MS-GF+"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001207" cvRef="PSI-MS" name="Mascot"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001478" cvRef="PSI-MS" name="Mascot Parser"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002076" cvRef="PSI-MS" name="PAnalyzer"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001456" cvRef="PSI-MS" name="analysis software"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1000752" cvRef="PSI-MS" name="TOPP software"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002458" name="PeptideShaker"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002544" name="xiFDR"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002543" name="xiFDR"&gt;&lt;/cvParam&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_sourcefile">Element &lt;SourceFile&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A file from which this mzIdentML instance was created.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SourceFileType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|location |xsd:anyURI |required |The location of the data file.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-externalformatdocumentation">ExternalFormatDocumentation</a> |0 |1 |A URI to access documentation and tools to interpret the external format of the ExternalData instance.
For example, XML Schema or static libraries (APIs) to access binary formats.
|<a href="#element-fileformat">FileFormat</a> |1 |1 |The format of the ExternalData file, for example "tiff" for image files.
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SourceFile location="C:\Users\hba041\My_Git_Applications\peptide-shaker.wiki\data\2016_04_05\.PeptideShaker_unzip_temp\searchgui_out_PeptideShaker_temp\qExactive01819</p>
</div>
<div class="paragraph">
<div class="title">t.xml" id="SourceFile_2"&gt;</div>
<p>&lt;FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001401" name="X!Tandem xml format"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SourceFile&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/DataCollection/Inputs/SourceFile</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000561">MS:1000561</a> (data file checksum type) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000568">MS:1000568</a> (MD5)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000569">MS:1000569</a> (SHA-1)</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_specificityrules">Element &lt;SpecificityRules&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The specificity rules of the searched modification including for example the probability of a

modification&#8217;s presence or peptide or protein termini. Standard fixed or variable status should

be provided by the attribute fixedMod.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SpecificityRulesType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SpecificityRules&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002057" name="modification specificity protein N-</p>
</div>
<div class="paragraph">
<p>term"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpecificityRules&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/ModificationParams/SearchModifi</p>
</div>
<div class="paragraph">
<p>cation/SpecificityRules</p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001056">MS:1001056</a> (modification specificity rule) only once</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001189">MS:1001189</a> (modification specificity peptide N-term)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001190">MS:1001190</a> (modification specificity peptide C-term)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001875">MS:1001875</a> (modification motif)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001876">MS:1001876</a> (modification probability)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002057">MS:1002057</a> (modification specificity protein N-term)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1002058">MS:1002058</a> (modification specificity protein C-term)</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam accession="MS:1001189" cvRef="PSI-MS" name="modification specificity peptide N-term"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002057" cvRef="PSI-MS" name="modification specificity protein N-term"/&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_spectradata">Element &lt;SpectraData&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A data set containing spectra data (consisting of one or more spectra).</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SpectraDataType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|location |xsd:anyURI |required |The location of the data file.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-externalformatdocumentation">ExternalFormatDocumentation</a> |0 |1 |A URI to access documentation and tools to interpret the external format of the ExternalData instance.
For example, XML Schema or static libraries (APIs) to access binary formats.
|<a href="#element-fileformat">FileFormat</a> |1 |1 |The format of the ExternalData file, for example "tiff" for image files.
|<a href="#element-spectrumidformat">SpectrumIDFormat</a> |1 |1 |The format of the spectrum identifier within the source file</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SpectraData location="E:\Work\PSI\mzIdentML\ProteinInference\Rosetta2\Peaklistswithecoli\Rosetta2a_Ecoli_spectra</p>
</div>
<div class="paragraph">
<div class="title">mgf" name="Rosetta2a_Ecoli_spectra.mgf" id="SID_1"&gt;</div>
<p>&lt;FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001062" cvRef="PSI-MS" name="Mascot MGF format"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIDFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1000774" cvRef="PSI-MS" name="multiple peak list nativeID format"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIDFormat&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/SpectraData&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_spectrumidentification">Element &lt;SpectrumIdentification&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">An Analysis which tries to identify peptides in input spectra, referencing the database searched, the input spectra, the output results and the protocol that is run.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SpectrumIdentificationType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|activityDate |xsd:dateTime |optional |When the protocol was applied.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|spectrumIdentificationList_ref |xsd:string |required |A reference to the SpectrumIdentificationList produced by this analysis in the DataCollection section.
|spectrumIdentificationProtocol_ref |xsd:string |required |A reference to the search protocol used for this SpectrumIdentification.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-inputspectra">InputSpectra</a> |1 |unbounded |One of the spectra data sets used.
|<a href="#element-searchdatabaseref">SearchDatabaseRef</a> |1 |unbounded |One of the search databases used.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SpectrumIdentification spectrumIdentificationProtocol_ref="SearchProtocol_1_4299" spectrumIdentificationList_ref="SII_LIST_1_1_4299_120114_20_Orbi2_ZC_QC_220_HSAd0-d4-1to1-3_Din.raw" id="SpecIdent__4299_120114_20_Orbi2_ZC_QC_220_HSAd0-d4-1to1-3_Din.raw"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;InputSpectra spectraData_ref="SD_4299_120114_20_Orbi2_ZC_QC_220_HSAd0-d4-1to1-3_Din.raw"&gt;&lt;/InputSpectra&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SearchDatabaseRef searchDatabase_ref="SDB_4299_203"&gt;&lt;/SearchDatabaseRef&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentification&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_spectrumidentificationitem">Element &lt;SpectrumIdentificationItem&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">An identification of a single (poly)peptide, resulting from querying an input spectra, along with

the set of confidence values for that identification. PeptideEvidence elements should be given

for all mappings of the corresponding Peptide sequence within protein sequences.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SpectrumIdentificationItemType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|calculatedMassToCharge |xsd:double |optional |The theoretical mass-to-charge value calculated for the peptide in Daltons / charge.
|calculatedPI |xsd:float |optional |The calculated isoelectric point of the (poly)peptide, with relevant modifications included.
Do not supply this value if the PI cannot be calcuated properly.
|chargeState |xsd:int |required |The charge state of the identified peptide.
|experimentalMassToCharge |xsd:double |required |The mass-to-charge value measured in the experiment in Daltons / charge.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|massTable_ref |xsd:string |optional |A reference SHOULD be given to the MassTable used to calculate the sequenceMass only if more than one MassTable has been given.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|passThreshold |xsd:boolean |required |Set to true if the producers of the file has deemed that the identification has passed a given threshold or been validated as correct.
If no such threshold has been set, value of true should be given for all results.
|peptide_ref |xsd:string |required |A reference to the identified (poly)peptide sequence in the Peptide element.
|rank |xsd:int |required |For an MS/MS result set, this is the rank of the identification quality as scored by the search engine. 1 is the top rank.
If multiple identifications have the same top score, they should all be assigned rank =1. For PMF data, the rank attribute may be meaningless and values of rank = 0 should be given.
|sample_ref |xsd:string |optional |A reference should be provided to link the SpectrumIdentificationItem to a Sample if more than one sample has been described in the AnalysisSampleCollection.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-peptideevidenceref">PeptideEvidenceRef</a> |0 |unbounded |Reference to the PeptideEvidence element identified.
If a specific sequence can be assigned to multiple proteins and or positions in a protein all possible PeptideEvidence elements should be referenced here.
|<a href="#element-fragmentation">Fragmentation</a> |0 |1 |The product ions identified in this result.
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Graphical Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="image"><img src="img//media/image9.png" alt="http://www.peptideatlas.org/PSI/schemas/mzIdentML/1.2/figures/SpectrumIdentificationItem.png" width="576" height="326"></span></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SpectrumIdentificationItem passThreshold="true" rank="1" peptide_ref="NMGGPYGGGNYGPGGSGGSGGYGGR_15.99491461956-ATAA-2" calculatedMassToCharge="1103.45377262</p>
</div>
<div class="paragraph">
<p>38118" experimentalMassToCharge="1103.454029541017" chargeState="2" id="SII_1920_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="PepEv_5164"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Fragmentation&gt;</p>
</div>
<div class="paragraph">
<p>&lt;IonType charge="1" index="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_MZ" values="175.1193695 232.1403961 289.1618958</p>
</div>
<div class="paragraph">
<p>452.2268982 509.2429504 566.2701416 653.2999268 710.3218994 767.3411865 854.3770752 911.3968506</p>
</div>
<div class="paragraph">
<p>968.4257813 1065.472168 1122.492432 1285.553833 1399.612305 1456.62561 1513.645874 1570.669067</p>
</div>
<div class="paragraph">
<p>1733.721191 1830.793213 1887.808105 1944.829834"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_Int" values="5939.5844726563 4933.5014648438</p>
</div>
<div class="paragraph">
<p>13310.7265625 5077.6694335938 5685.9287109375 13253.552734375 7620.0947265625 7724.3696289063</p>
</div>
<div class="paragraph">
<p>16868.541015625 10552.126953125 11589.0576171875 7839.9741210938 47821.64453125 60335.71484375</p>
</div>
<div class="paragraph">
<p>21759.3984375 8742.5595703125 11512.0908203125 18130.890625 30577.375 3801.3923339844 8051.07421875</p>
</div>
<div class="paragraph">
<p>1954.5501708984 4844.9125976563"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_Error" values="4.173258879802688E-4</p>
</div>
<div class="paragraph">
<p>-1.9794682032170385E-5 1.618474794895519E-5 0.001690052197886871 -0.0037214683721344954</p>
</div>
<div class="paragraph">
<p>0.0020060110579152024 -2.3719321211501665E-4 2.7168621795681247E-4 -0.0019049343519554895</p>
</div>
<div class="paragraph">
<p>0.0019553613780090018 2.6704080801209784E-4 0.007734020238103767 0.0013568713879976713</p>
</div>
<div class="paragraph">
<p>1.571508180404635E-4 -0.0017703817320580129 0.013774177127970688 0.0056154565579618065</p>
</div>
<div class="paragraph">
<p>0.004415735988004599 0.006145015418042021 -0.005059517131940083 0.01419863401793009</p>
</div>
<div class="paragraph">
<p>0.007626913448120831 0.007892192877989146"/&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/SpectrumIdentificationItem&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/DataCollection/AnalysisData/SpectrumIdentificationList/SpectrumIdentificationResult/Spect</p>
</div>
<div class="paragraph">
<p>rumIdentificationItem</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001405">MS:1001405</a> (spectrum identification result details) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000016">MS:1000016</a> (scan start time)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000796">MS:1000796</a> (spectrum title)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000797">MS:1000797</a> (peak list scans)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000798">MS:1000798</a> (peak list raw scans)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000903">MS:1000903</a> (product ion series ordinal)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000904">MS:1000904</a> (product ion m/z delta)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000926">MS:1000926</a> (product interpretation rank)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001030">MS:1001030</a> (number of peptide seqs compared to each spectrum)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001035">MS:1001035</a> (date / time search performed)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001036">MS:1001036</a> (search time taken)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001405">et al.</a></p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam accession="MS:1001330" cvRef="PSI-MS" value="2.5E-6" name="X\!Tandem:expect"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001331" cvRef="PSI-MS" value="43.2" name="X\!Tandem:hyperscore"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001250" cvRef="PSI-MS" value="0.0" name="local FDR"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001868" cvRef="PSI-MS" value="0.0" name="distinct peptide-level q-value"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001874" cvRef="PSI-MS" value="3.113325031133251E-4" name="FDRScore"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1000796" cvRef="PSI-MS" value="Locus:11.1.1.4652.4 File:"R1 p450 iTRAQ QS CEX11.wiff"" name="spectrum title"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002049" cvRef="PSI-MS" value="129" name="MS-GF:RawScore"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002050" cvRef="PSI-MS" value="143" name="MS-GF:DeNovoScore"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002052" cvRef="PSI-MS" value="6.3746987E-25" name="MS-GF:SpecEValue"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002053" cvRef="PSI-MS" value="1.3781529E-17" name="MS-GF:EValue"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002054" cvRef="PSI-MS" value="0.0" name="MS-GF:QValue"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002055" cvRef="PSI-MS" value="0.0" name="MS-GF:PepQValue"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002351" cvRef="PSI-MS" value="0.0" name="PSM-level local FDR"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002354" cvRef="PSI-MS" value="0.0" name="PSM-level q-value"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002355" cvRef="PSI-MS" value="1.5603866050496166E-18" name="PSM-level FDRScore"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001171" cvRef="PSI-MS" value="25.37" name="Mascot:score"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001172" cvRef="PSI-MS" value="0.0813522191664226" name="Mascot:expectation value"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001175" cvRef="PSI-MS" name="peptide shared in multiple proteins"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001363" cvRef="PSI-MS" name="peptide unique to one protein"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001371" cvRef="PSI-MS" value="40" name="Mascot:identity threshold"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001370" cvRef="PSI-MS" value="27" name="Mascot:homology threshold"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001030" cvRef="PSI-MS" value="10148" name="number of peptide seqs compared to each spectrum"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001114" cvRef="PSI-MS" unitCvRef="UO" unitName="second" unitAccession="UO:0000010" value="1741" name="retention time(s)"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002681" cvRef="PSI-MS" name="OpenXQuest:combined score" value="0.552164719139592"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002511" cvRef="PSI-MS" name="crosslink spectrum identification item" value="11309529182388590588"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1000894" cvRef="PSI-MS" name="retention time" value="5468.0193" unitAccession="second" unitName="" unitCvRef="se"/&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example userParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;userParam value="0" name="IsotopeError"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam value="HCD" name="AssumedDissociationMethod"/&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p><strong>Example for peptide-level statistics:</strong></p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="3" experimentalMassToCharge="710.352539" calculatedMassToCharge="710.352984" peptide_ref="KMDLSDEGGGGVRYPGLHPK_##Oxidation(M):2" rank="1" passThreshold="false" id="SIR_3397_SII_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="KMDLSDEGGGGVRYPGLHPK_generic|B_GENSCAN00000016205_REVERSED|p:genscan_42_61"&gt;&lt;/PeptideEvidenceRef&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002356" name="PSM-level combined FDRScore" value="0.38641138028680944"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="search engines identifying PSM" value="2"&gt;&lt;/userParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002359" name="distinct peptide-level local FDR" value="0.419570671870644"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001868" name="distinct peptide-level q-value" value="0.4192650334075724"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002360" name="distinct peptide-level FDRScore" value="0.41934590570107133"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002500" name="peptide passes threshold" value="true"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002520" name="peptide group ID" value="KMDLSDEGGGGVRYPGLHPK_##Oxidation(M):2"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationItem&gt;</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p><strong>Example for crosslinking:</strong></p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="4" experimentalMassToCharge="0.0" peptide_ref="54600873_54605193_9_1_p1" rank="1" passThreshold="false" id="SII_21_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_psm252621611_pep54605193_protP02768-A_target_137"&gt;&lt;/PeptideEvidenceRef&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002511" name="crosslink spectrum identification item" value="21"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002545" name="The xi result 'Score'." value="2.769918944845425"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationItem&gt;</p>
</div>
</blockquote>
</div>
</div>
<div class="sect2">
<h3 id="_element_spectrumidentificationitemref">Element &lt;SpectrumIdentificationItemRef&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Reference(s) to the SpectrumIdentificationItem element(s) that support the given PeptideEvidence element. Using these references, it is possible to indicate which spectra were actually accepted as evidence for this peptide identification in the given protein.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SpectrumIdentificationItemRefType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|spectrumIdentificationItem_ref |xsd:string |required |A reference to the SpectrumIdentificationItem element(s).</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;SpectrumIdentificationItemRef spectrumIdentificationItem_ref="SII_1000_1"&gt;&lt;/SpectrumIdentificationItemRef&gt;</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001591" name="anchor protein"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001593" name="group member with undefined relationship OR ortholog protein"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002676" name="protein-pair-level global FDR" value="0.a:null:1.0:true"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002677" name="residue-pair-level global FDR" value="0.a:58:0.04716981132075472:true"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002415" name="protein group passes threshold" value="true"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002404" name="count of identified protein" value="2"&gt;&lt;/cvParam&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_spectrumidentificationlist">Element &lt;SpectrumIdentificationList&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">Represents the set of all search results from SpectrumIdentification.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SpectrumIdentificationListType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|numSequencesSearched |xsd:long |optional |The number of database sequences searched against.
This value should be provided unless a de novo search has been performed.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-fragmentationtable">FragmentationTable</a> |0 |1 |Contains the types of measures that will be reported in generic arrays for each SpectrumIdentificationItem e.g. product ion m/z, product ion intensity, product ion m/z error |<a href="#element-spectrumidentificationresult">SpectrumIdentificationResult</a> |1 |unbounded |All identifications made from searching one spectrum.
For PMF data, all peptide identifications will be listed underneath as SpectrumIdentificationItems.
For MS/MS data, there will be ranked SpectrumIdentificationItems corresponding to possible different peptide IDs.
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Graphical Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="image"><img src="img//media/image10.png" alt="http://www.peptideatlas.org/PSI/schemas/mzIdentML/1.2/figures/SpectrumIdentificationList.png" width="575" height="432"></span></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SpectrumIdentificationList xmlns="http://psidev.info/psi/pi/mzIdentML/1.2" id="SII_LIST_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult spectrumID="index=6451" spectraData_ref="SD_COMBINED_SE_0" id="SIR_8947"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="2" experimentalMassToCharge="679.817322" calculatedMassToCharge="679.818488" peptide_ref="AVMDDFAAFVEK_##Oxidation(M):3" rank="1" passThreshold="false" id="SIR_8947_SII_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="AVMDDFAAFVEK_generic</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP00000401820</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:known_378_389"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&lt;PeptideEvidenceRef peptideEvidence_ref="AVMDDFAAFVEK_generic</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP00000421027</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:putative_420_431"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&lt;PeptideEvidenceRef peptideEvidence_ref="AVMDDFAAFVEK_generic</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP00000483421</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:known_357_368"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&lt;PeptideEvidenceRef peptideEvidence_ref="AVMDDFAAFVEK_generic</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP00000480485</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:known_357_368"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&#8230;&#8203;
&lt;/SpectrumIdentificationList&gt;</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_spectrumidentificationprotocol">Element &lt;SpectrumIdentificationProtocol&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The parameters and settings of a SpectrumIdentification analysis.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SpectrumIdentificationProtocolType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|analysisSoftware_ref |xsd:string |required |The search algorithm used, given as a reference to the SoftwareCollection section.
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-searchtype">SearchType</a> |1 |1 |The type of search performed e.g. PMF, Tag searches, MS-MS |<a href="#element-additionalsearchparams">AdditionalSearchParams</a> |0 |1 |The search parameters other than the modifications searched.
|<a href="#element-modificationparams">ModificationParams</a> |0 |1 |The specification of static/variable modifications (e.g. Oxidation of Methionine) that are to be considered in the spectra search.
|<a href="#element-enzymes">Enzymes</a> |0 |1 |The list of enzymes used in experiment |<a href="#element-masstable">MassTable</a> |0 |unbounded |The masses of residues used in the search.
|<a href="#element-fragmenttolerance">FragmentTolerance</a> |0 |1 |The tolerance of the search given as a plus and minus value with units.
|<a href="#element-parenttolerance">ParentTolerance</a> |0 |1 |The tolerance of the search given as a plus and minus value with units.
|<a href="#element-threshold">Threshold</a> |1 |1 |The threshold(s) applied to determine that a result is significant.
If multiple terms are used it is assumed that all conditions are satisfied by the passing results.
|<a href="#element-databasefilters">DatabaseFilters</a> |0 |1 |The specification of filters applied to the database searched.
|<a href="#element-databasetranslation">DatabaseTranslation</a> |0 |1 |A specification of how a nucleic acid sequence database was translated for searching.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Graphical Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock"><span class="image"><img src="img//media/image11.png" alt="http://www.peptideatlas.org/PSI/schemas/mzIdentML/1.2/figures/SpectrumIdentificationProtocol.png" width="557" height="813"></span></p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SpectrumIdentificationProtocol id="SIP_10589554385233790425" analysisSoftware_ref="SOF_10581839310406754333"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SearchType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001083" cvRef="PSI-MS" name="ms-ms search"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SearchType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;AdditionalSearchParams&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002494" cvRef="PSI-MS" name="crosslinking search"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="input_consensusXML" unitName="xsd:string" value="leitner004.consensusXML"/&gt;</p>
</div>
<div class="paragraph">
<p>&#8230;&#8203;
&lt;/SpectrumIdentificationProtocol&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_spectrumidentificationresult">Element &lt;SpectrumIdentificationResult&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">All identifications made from searching one spectrum. For PMF data, all peptide identifications

will be listed underneath as SpectrumIdentificationItems. For MS/MS data, there will be ranked

SpectrumIdentificationItems corresponding to possible different peptide IDs.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SpectrumIdentificationResultType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.
|spectraData_ref |xsd:string |required |A reference to a spectra data set (e.g. a spectra file).
|spectrumID |xsd:string |required |The locally unique id for the spectrum in the spectra data set specified by SpectraData_ref.
External guidelines are provided on the use of consistent identifiers for spectra in different external formats.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-spectrumidentificationitem">SpectrumIdentificationItem</a> |1 |unbounded |An identification of a single (poly)peptide, resulting from querying an input spectra, along with the set of confidence values for that identification.
PeptideEvidence elements should be given for all mappings of the corresponding Peptide sequence within protein sequences.
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |0 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 8.3333%;">
<col style="width: 8.3333%;">
<col style="width: 8.3333%;">
<col style="width: 8.3333%;">
<col style="width: 8.3333%;">
<col style="width: 8.3333%;">
<col style="width: 8.3333%;">
<col style="width: 8.3333%;">
<col style="width: 8.3333%;">
<col style="width: 8.3333%;">
<col style="width: 8.3333%;">
<col style="width: 8.3337%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SpectrumIdentificationResult spectrumID="index=7665" spectraData_ref="SD_COMBINED_SE_0" id="SIR_7191"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="4" experimentalMassToCharge="1123.974121" calculatedMassTo</p>
</div>
<div class="paragraph">
<p>Charge="1123.968707" peptide_ref="CRCQYSGVNNLCHTSSHCPNQGSTCENVDTCLKPDEPK_<mark>Carbamidomethyl&#169;:1</mark>Carba</p>
</div>
<div class="paragraph">
<p>midomethyl&#169;:3##Carba</p>
</div>
<div class="paragraph">
<p>midomethyl&#169;:12<mark>Carbamidomethyl&#169;:18</mark>Carbamidomethyl&#169;:25<mark>Carbamidomethyl&#169;:31</mark>Ammonia-loss&#169;:</p>
</div>
<div class="paragraph">
<p>1" rank="1" passThreshold="false" id="SIR_7191_SII_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="CRCQYSGVNNLCHTSSHCPNQGSTCENVDTCLKPDEPK_generic</p>
</div></div></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP00000376692_REVERSED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:</p>
<p class="tableblock">novel_575_612"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&lt;PeptideEvidenceRef peptideEvidence_ref="CRCQYSGVNNLCHTSSHCPNQGSTCENVDTCLKPDEPK_generic</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">B_GENSCAN00000036974_REVERSED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:</p>
<p class="tableblock">genscan_494_531"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&lt;PeptideEvidenceRef peptideEvidence_ref="CRCQYSGVNNLCHTSSHCPNQGSTCENVDTCLKPDEPK_generic</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP00000471297_REVERSED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:</p>
<p class="tableblock">putative_641_678"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&lt;PeptideEvidenceRef peptideEvidence_ref="CRCQYSGVNNLCHTSSHCPNQGSTCENVDTCLKPDEPK_generic</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP00000319883_REVERSED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:</p>
<p class="tableblock">known_633_670"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&lt;PeptideEvidenceRef peptideEvidence_ref="CRCQYSGVNNLCHTSSHCPNQGSTCENVDTCLKPDEPK_generic</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">A_ENSP00000472280_REVERSED</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">p:</p>
<p class="tableblock">putative_622_659"&gt;&lt;/PeptideEvidenceRef&gt;</p>
<p class="tableblock">&#8230;&#8203;
&lt;/SpectrumIdentificationResult&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_spectrumidformat">Element &lt;SpectrumIDFormat&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The format of the spectrum identifier within the source file</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SpectrumIDFormatType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |1 |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;SpectrumIDFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1000774" cvRef="PSI-MS" name="multiple peak list nativeID format"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIDFormat&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/DataCollection/Inputs/SpectraData/SpectrumIDFormat</p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000767">MS:1000767</a> (native spectrum identifier format) only once</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000768">MS:1000768</a> (Thermo nativeID format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000769">MS:1000769</a> (Waters nativeID format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000770">MS:1000770</a> (WIFF nativeID format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000771">MS:1000771</a> (Bruker/Agilent YEP nativeID format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000772">MS:1000772</a> (Bruker BAF nativeID format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000773">MS:1000773</a> (Bruker FID nativeID format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000774">MS:1000774</a> (multiple peak list nativeID format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000775">MS:1000775</a> (single peak list nativeID format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000776">MS:1000776</a> (scan number only nativeID format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000777">MS:1000777</a> (spectrum identifier nativeID format)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1000767">et al.</a></p>
</div>
<div class="paragraph">
<p>MUST supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001529">MS:1001529</a> (spectra data details) only once</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001530">MS:1001530</a> (mzML unique identifier)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001531">MS:1001531</a> (spectrum from ProteinScape database nativeID format)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001532">MS:1001532</a> (spectrum from database string nativeID format)</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;cvParam cvRef="PSI-MS" accession="MS:1000774" name="multiple peak list nativeID format"&gt;&lt;/cvParam&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_subsample">Element &lt;SubSample&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">References to the individual component samples within a mixed parent sample.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SubSampleType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|sample_ref |xsd:string |required |A reference to the child sample.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_substitutionmodification">Element &lt;SubstitutionModification&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">A modification where one residue is substituted by another (amino acid change).</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">SubstitutionModificationType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|avgMassDelta |xsd:double |optional |Atomic mass delta considering the natural distribution of isotopes in Daltons.
This should only be reported if the original amino acid is known i.e. it is not "X" |location |xsd:int |optional |Location of the modification within the peptide - position in peptide sequence, counted from the N-terminus residue, starting at position 1. Specific modifications to the N-terminus should be given the location 0. Modification to the C-terminus should be given as peptide length + 1.
|monoisotopicMassDelta |xsd:double |optional |Atomic mass delta when assuming only the most common isotope of elements in Daltons.
This should only be reported if the original amino acid is known i.e. it is not "X" |originalResidue a| xsd:string with restriction</p>
</div>
<div class="paragraph">
<p>[ABCDEFGHIJKLMNOPQRSTUVWXYZ?\-]{1}</p>
</div>
<div class="paragraph">
<p>|required |The original residue before replacement.
|replacementResidue a| xsd:string with restriction</p>
</div>
<div class="paragraph">
<p>[ABCDEFGHIJKLMNOPQRSTUVWXYZ?\-]{1}</p>
</div>
<div class="paragraph">
<p>|required |The residue that replaced the originalResidue.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;SubstitutionModification originalResidue="X" replacementResidue="I" location="10" /&gt;</p></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_threshold">Element &lt;Threshold&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top"><strong>Depending on context (SpectrumIdentificationProtocol or ProteinDetectionProtocol):</strong><br>
<strong>1</strong>: The threshold(s) applied to determine that a spectrum identification is significant. If multiple terms are used it is assumed that all conditions are satisfied by the passing results.<br>
<strong>2</strong>: The threshold(s) applied to determine that a protein detection is significant. If multiple terms are used it is assumed that all conditions are satisfied by the passing results.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">ParamListType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |1 |unbounded |A single entry from an ontology or a controlled vocabulary.
|<a href="#element-userparam">userParam</a> |1 |unbounded |A single user-defined parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;Threshold&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001364" name="distinct peptide-level global FDR" value="1.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002350" name="PSM-level global FDR" value="1.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002567" name="phosphoRS score threshold" value="95.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002557" name="D-Score threshold" value="95.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Threshold&gt;</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/Threshold</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001302">MS:1001302</a> (search engine specific input parameter) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001005">MS:1001005</a> (SEQUEST:CleavesAt)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001007">MS:1001007</a> (SEQUEST:OutputLines)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001009">MS:1001009</a> (SEQUEST:DescriptionLines)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001026">MS:1001026</a> (SEQUEST:NormalizeXCorrValues)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001028">MS:1001028</a> (SEQUEST:SequenceHeaderFilter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001032">MS:1001032</a> (SEQUEST:SequencePartialFilter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001037">MS:1001037</a> (SEQUEST:ShowFragmentIons)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001038">MS:1001038</a> (SEQUEST:Consensus)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001042">MS:1001042</a> (SEQUEST:LimitTo)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001046">MS:1001046</a> (SEQUEST:sort by dCn)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001302">et al.</a></p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001153">MS:1001153</a> (search engine specific score) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001154">MS:1001154</a> (SEQUEST:probability)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001155">MS:1001155</a> (SEQUEST:xcorr)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001156">MS:1001156</a> (SEQUEST:deltacn)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001157">MS:1001157</a> (SEQUEST:sp)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001158">MS:1001158</a> (SEQUEST:Uniq)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001159">MS:1001159</a> (SEQUEST:expectation value)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001160">MS:1001160</a> (SEQUEST:sf)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001161">MS:1001161</a> (SEQUEST:matched ions)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001162">MS:1001162</a> (SEQUEST:total ions)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001163">MS:1001163</a> (SEQUEST:consensus score)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001153">et al.</a></p>
</div>
<div class="paragraph">
<p>MAY supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001494">MS:1001494</a> (no threshold) only once</p>
</div>
<div class="paragraph">
<p>MAY supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001448">MS:1001448</a> (pep:FDR threshold) only once</p>
</div>
<div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/ProteinDetectionProtocol/Threshold</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001302">MS:1001302</a> (search engine specific input parameter) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001005">MS:1001005</a> (SEQUEST:CleavesAt)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001007">MS:1001007</a> (SEQUEST:OutputLines)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001009">MS:1001009</a> (SEQUEST:DescriptionLines)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001026">MS:1001026</a> (SEQUEST:NormalizeXCorrValues)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001028">MS:1001028</a> (SEQUEST:SequenceHeaderFilter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001032">MS:1001032</a> (SEQUEST:SequencePartialFilter)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001037">MS:1001037</a> (SEQUEST:ShowFragmentIons)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001038">MS:1001038</a> (SEQUEST:Consensus)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001042">MS:1001042</a> (SEQUEST:LimitTo)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001046">MS:1001046</a> (SEQUEST:sort by dCn)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001302">et al.</a></p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001153">MS:1001153</a> (search engine specific score) one or more times</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001154">MS:1001154</a> (SEQUEST:probability)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001155">MS:1001155</a> (SEQUEST:xcorr)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001156">MS:1001156</a> (SEQUEST:deltacn)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001157">MS:1001157</a> (SEQUEST:sp)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001158">MS:1001158</a> (SEQUEST:Uniq)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001159">MS:1001159</a> (SEQUEST:expectation value)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001160">MS:1001160</a> (SEQUEST:sf)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001161">MS:1001161</a> (SEQUEST:matched ions)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001162">MS:1001162</a> (SEQUEST:total ions)</p>
</div>
<div class="paragraph">
<p>e.g.: <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001163">MS:1001163</a> (SEQUEST:consensus score)</p>
</div>
<div class="paragraph">
<p><a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001153">et al.</a></p>
</div>
<div class="paragraph">
<p>MAY supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001447">MS:1001447</a> (prot:FDR threshold) only once</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1002664&amp;lang=en&amp;viewMode=All&amp;siblings=false">MS:1002664</a> (interaction score derived from crosslinking) one or more times</p>
</div>
<div class="paragraph">
<p>MAY supply a <strong>child</strong> term of <a href="https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1002482&amp;lang=en&amp;viewMode=All&amp;siblings=false">MS:1002482</a> (statistical threshold) one or more times</p>
</div></div></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example cvParams:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001494" name="no threshold"&gt;&lt;/cvParam&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1002351" cvRef="PSI-MS" value="0.01" name="PSM-level local FDR"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001316" cvRef="PSI-MS" value="0.05" name="Mascot:SigThreshold"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001364" name="distinct peptide-level global FDR" value="1.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002350" name="PSM-level global FDR" value="1.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002567" name="phosphoRS score threshold" value="95.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002557" name="D-Score threshold" value="95.0"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002369" name="protein group-level global FDR" value="0.01"/&gt;</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_translationtable">Element &lt;TranslationTable&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">The table used to translate codons into nucleic acids e.g. by reference to the NCBI translation

table.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">TranslationTableType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|id |xsd:string |required |An identifier is an unambiguous string that is unique within the scope (i.e. a document, a set of related documents, or a repository) of its use.
|name |xsd:string |optional |The potentially ambiguous common identifier, such as a human-readable name for the instance.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Subelement Name</strong> |<strong>minOccurs</strong> |<strong>maxOccurs</strong> |<strong>Definition</strong>
|<a href="#element-cvparam">cvParam</a> |0 |unbounded |A single entry from an ontology or a controlled vocabulary.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>cvParam Mapping Rules:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"><div class="paragraph">
<p>Path /MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/DatabaseTranslation/Translation</p>
</div>
<div class="paragraph">
<p>Table</p>
</div>
<div class="paragraph">
<p>MUST supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001410">MS:1001410</a> (translation start codons) only once</p>
</div>
<div class="paragraph">
<p>MUST supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001025">MS:1001025</a> (translation table) only once</p>
</div>
<div class="paragraph">
<p>MUST supply term <a href="http://www.ebi.ac.uk/ols/beta/ontologies/ms/terms?iri=http://purl.obolibrary.org/obo/MS_1001423">MS:1001423</a> (translation table description) only once</p>
</div></div></td>
</tr>
</tbody>
</table>
</div>
<div class="sect2">
<h3 id="_element_userparam">Element &lt;userParam&gt;</h3>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Definition:</strong></th>
<th class="tableblock halign-left valign-top">In case more information about the ions annotation has to be conveyed, that has no fit in FragmentArray. Note: It is suggested that the value attribute takes the form of a list of the same size as FragmentArray values. However, there is no formal encoding and it cannot be expeceted that other software will process or impart that information properly.</th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Type:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">UserParamType</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Attributes:</strong></p></td>
<td class="tableblock halign-left valign-top"><div class="content"></div></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>|<strong>Attribute Name</strong> |<strong>Data Type</strong> |<strong>Use</strong> |<strong>Definition</strong>
|name |xsd:string |required |The name of the parameter.
|type |xsd:string |optional |The datatype of the parameter, where appropriate (e.g.: xsd:float).
|unitAccession |xsd:string |optional |An accession number identifying the unit within the OBO foundry Unit CV.
|unitCvRef |xsd:string |optional |If a unit term is referenced, this attribute MUST refer to the CV 'id' attribute defined in the cvList in this file.
|unitName |xsd:string |optional |The name of the unit.
|value |xsd:string |optional |The user-entered value of the parameter.</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Subelements:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">none</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock"><strong>Example Context:</strong></p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">&lt;userParam value="VLENAEGDR; ASSGLNEDEIQK; MQELAQVSQK; KTAEDYLGEPVTEAVITVPAYFNDAQR; SLGQFNLDGINPAPR; MPMVQK; IIAADNGDAWVEVK; DVSIMPFK; KDVNPDEAVAIGAAVQGGVLTGDVK; KFEELVQTR; NDPLAMQR; VAEFFGK; QVEEAGDKLPADDK; MAPPQISAEVLKK; KQVEEAGDKLPADDK; LINYLVEEFK; MAPPQISAEVLK; QAVTNPQNTLFAIK; TFEVLATNGDTHLGGEDFDSR; VALQDAGLSVSDIDDVILVGGQTR; FQDEEVQR" name="unique peptides"/&gt;</p></td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_specific_comments_on_schema">Specific Comments on schema</h2>
<div class="sectionbody">
<div class="paragraph">
<p>In this section, several points of documentation are elaborated beyond the core specification in Section 6..</p>
</div>
<div class="sect2">
<h3 id="_file_extension_and_compression">File extension and compression</h3>
<div class="paragraph">
<p>It is noted that standard file compression algorithms greatly reduce the mzIdentML file sizes, speeding up file transfers and uploads / downloads.
It is also noted that software implementing mzIdentML import or export will be expected to benefit in performance from working with compressed mzIdentML, since the compression and decompression algorithms are expected to give significant performance gains over disk access times for non-compressed files.
As such, it is RECOMMENDED that mzIdentML files are compressed using gzip from all software that exports mzIdentML and software that imports SHOULD be expected to read gzipped files, as well as native (non-compressed) mzIdentML files.
The file extension for native mzIdentML files SHOULD be “.mzid” and for compressed files SHOULD be “mzid.gz”.</p>
</div>
</div>
<div class="sect2">
<h3 id="_referencing_elements_within_the_document">Referencing elements within the document</h3>
<div class="paragraph">
<p>A number of elements within the schema have an attribute which is used to reference an element elsewhere in the file using the unique identifier of the referenced element.
These attributes are named following the convention: “[elementName]_ref”.
The uniqueness of the value in the “id” attribute of elements is validated using xsd:key, and the integrity of the reference is validated using xsd:keyref, defined within the schema.</p>
</div>
</div>
<div class="sect2">
<h3 id="_searches_against_nucleotide_sequences">Searches against nucleotide sequences</h3>
<div class="paragraph">
<p>The “seq” attribute on &lt;DBSequence&gt; SHOULD contain the nucleic acid sequence if a nucleic acid database was searched (rather than up to six translated sequences).
&lt;Peptide&gt; represents the identified amino acid sequence (including modifications) and, as such, the &lt;peptideSequence&gt; elements SHOULD store the translated amino acid sequences.
&lt;PeptideEvidence&gt; contains the <em>DBSequence_Ref</em> together with the translation frame and a <em>TranslationTable_Ref</em> attribute (see below).
The <em>Peptide_Ref</em> is done in &lt;SpectrumIdentificationItem&gt; as in the case for an amino acid database.
If protein detection is performed, there are &lt;PeptideHypothesis&gt; elements referencing &lt;PeptideEvidence&gt; elements from &lt;SpectrumIdentificationItem&gt; sections.
For clarification, see the example instance document for a nucleic acid search (Section 5.3).</p>
</div>
<div class="paragraph">
<p>In the &lt;SpectrumIdentificationProtocol&gt;, &lt;TranslationTable&gt; is used to specify how nucleic acid sequences are translated into amino acid sequences as follows:</p>
</div>
<div class="paragraph">
<p>&lt;DatabaseTranslation frames="1 2 3 -1 -2 -3"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;TranslationTable id="TT_1" name="Standard"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001025" name="translation table" cvRef="PSI-MS" value="FFLLSSSSYY**CC*WLLLLPPPPHHQQRRRRIIIMTTTTNNKKSSRRVVVVAAAADDEEGGGG" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001410" name="translation start codons" cvRef="PSI-MS" value="---M---------------M---------------M----------------------------" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001423" name="translation table description" cvRef="PSI-MS" value="http://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/index.cgi?chapter=cgencodes#SG1" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/TranslationTable&gt;</p>
</div>
<div class="paragraph">
<p>&lt;TranslationTable id="TT_2" name="Vertebrate Mitochondrial"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001025" name="translation table" cvRef="PSI-MS" value="FFLLSSSSYY<strong>CCWWLLLLPPPPHHQQRRRRIIMMTTTTNNKKSS</strong>VVVVAAAADDEEGGGG" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001410" name="translation start codons" cvRef="PSI-MS" value="--------------------------------MMMM---------------M------------" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001423" name="translation table description" cvRef="PSI-MS" value="http://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/index.cgi?chapter=cgencodes#SG2" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/TranslationTable&gt;</p>
</div>
<div class="paragraph">
<p>The attribute “frames” specifies which frames are considered and one or more translation tables can be specified using CV parameters.
The translation table is defined here: <a href="http://www.ncbi.nlm.nih.gov/IEB/ToolBox/SDKDOCS/SEQFEAT.HTML#_Genetic_Codes" class="bare">http://www.ncbi.nlm.nih.gov/IEB/ToolBox/SDKDOCS/SEQFEAT.HTML#_Genetic_Codes</a>:</p>
</div>
<div class="paragraph">
<p>"The genetic codes themselves are arrays of 64 amino acid codes.
The index to the position in the array of the amino acid is derived from the codon by the following method:</p>
</div>
<div class="paragraph">
<p>index = (base1 16) + (base2 4) + (base3 1)</p>
</div>
<div class="paragraph">
<p>where T=0, C=1, A=2, G=3”</p>
</div>
<div class="paragraph">
<p>The same encoding technique is used to specify start codons.
Alphabet names are prefixed with "s" (e.g. sncbieaa) to indicate start codon arrays.
Each cell of a start codon array contains either the gap code ("-" for ncbieaa) or an amino acid code if it is valid to use the codon as a start codon.
Currently all starts are set to code for methionine, since it has never been convincingly demonstrated that a protein can start with any other amino acid.
However, if other amino acids are shown to be used as starts, this structure can easily accommodate that information.</p>
</div>
<div class="paragraph">
<p>For each peptide, the frame and translation table should be specified in the &lt;PeptideEvidence&gt; element:</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidence id=”1” TranslationTable_ref="TT_1" frame="1" /&gt;</p>
</div>
</div>
<div class="sect2">
<h3 id="_reporting_peptide_and_protein_identifications_passing_a_significance_threshold">Reporting peptide and protein identifications passing a significance threshold</h3>
<div class="paragraph">
<p>The elements &lt;SpectrumIdentificationItem&gt; and &lt;ProteinDetectionHypothesis&gt; have a mandatory Boolean attribute <em>passThreshold</em> that allows a file producer to indicate that an identification has passed a given threshold or that it has been manually validated.
Depending on the intended purpose of the file, the file producer MAY wish to report a number of identifications that fall below the given significance threshold, for example to allow global statistical analyses to be performed which are not possible if only identifications passing the threshold are reported.
Thresholds for peptide-spectrum matches or for protein identification should be encoded as instances of &lt;cvParam&gt; within &lt;SpectrumDetectionProtocol&gt; or &lt;ProteinDetectionProtocol&gt;, respectively, as follows.
If the file producer does not want to indicate that a threshold has been set, all identifications MUST have passThreshold = “true” and the “no threshold” CV term should be given within the protocols.</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationProtocol id="SIP" AnalysisSoftware_ref="AS_mascot_server"&gt;</p>
</div>
<div class="paragraph">
<p>…</p>
</div>
<div class="paragraph">
<p>&lt;Threshold&gt;<br>
&lt;cvParam accession="MS:1001316" name="mascot:SigThreshold" cvRef="PSI-MS" value="0.05"/&gt;<br>
&lt;/Threshold&gt;</p>
</div>
<div class="paragraph">
<p>&lt;ProteinDetectionProtocol id="PDP_MascotParser_1" AnalysisSoftware_ref="AS_mascot_parser"&gt;</p>
</div>
<div class="paragraph">
<p>…</p>
</div>
<div class="paragraph">
<p>&lt;Threshold&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001316" name="mascot:SigThreshold" cvRef="PSI-MS" value="0.05"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Threshold&gt;</p>
</div>
<div class="paragraph">
<p>The reporting of significance thresholds at the PSM and peptide level (mzIdentML 1.2) is explained in section 5.2.7. Reporting of threshold for modification position (also mzIdentML 1.2) is explained in section 5.2.8.</p>
</div>
</div>
<div class="sect2">
<h3 id="_using_decoy_databases_to_set_different_thresholds_of_false_discovery_rate">Using decoy databases to set different thresholds of false discovery rate</h3>
<div class="paragraph">
<p>mzIdentML supports the reporting of searches against decoy databases, constructed and searched using many of the currently known methods.
A &lt;SpectrumIdentificationItem&gt; can be marked as matching a decoy peptide using the <em>isDecoy</em> attribute of the referenced &lt;PeptideEvidence&gt; element, thus allowing the false discovery rate to be calculated across an entire file.
The <em>DBSequence_Ref</em> references the decoy protein record.</p>
</div>
<div class="paragraph">
<p>Implementers of the format SHOULD report the peptide identifications that pass the threshold they wish to communicate to a consumer of the data.
For example, a threshold could be set by p-value, false discovery rate, by a native search engine score (or a more complex system documented with CV terms in &lt;Threshold&gt;), and those peptides reported (passing the threshold) are used to determine which proteins have been detected.
It is not guaranteed that a consumer of an mzIdentML file will be able to calculate other results, or global false discovery rates, using different thresholds from the reported information, although in some circumstances they may be able to, for example, if a user reports the complete output of a search against a target and decoy search.</p>
</div>
<div class="paragraph">
<p>&lt;SequenceCollection&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidence isDecoy="true" post="D" pre="K" end="404" start="392"</p>
</div>
<div class="paragraph">
<p>peptide_ref="HAVGGRYSSLLCK__57.0215@C$403;_"</p>
</div>
<div class="paragraph">
<p>dBSequence_ref="dbseq_REV_psu|NC_LIV_113200" id="PE6_2_4"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SequenceCollection&gt;</p>
</div>
<div class="paragraph">
<p>&lt;DataCollection&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Inputs&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SearchDatabase location="/localdirectory/18.E_coli_K12_edit.fasta" id="K12_nosignal" name="K12" numDatabaseSequences="9376" releaseDate="01-2008-08-2008" version="1.0" &gt;</p>
</div>
<div class="paragraph">
<p>&lt;FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001348" name="FASTA format" cvRef="PSI-MS"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FileFormat&gt;</p>
</div>
<div class="paragraph">
<p>&lt;DatabaseName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;userParam name="18.E_coli_K12_edit.fasta" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/DatabaseName&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001197" name="<strong>DB composition target+decoy</strong>" cvRef="PSI-MS"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001283" name="<strong>decoy DB accession regexp</strong>" value="<strong>Rnd</strong>" cvRef="PSI-MS"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001195" name="<strong>decoy DB type reverse</strong>" cvRef="PSI-MS"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SearchDatabase&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Inputs&gt;</p>
</div>
<div class="paragraph">
<p>&lt;AnalysisData&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationList&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem passThreshold="false" rank="1"<br>
peptide_ref="HAVGGRYSSLLCK__57.0215@C$403;_"<br>
experimentalMassToCharge="1448.756" chargeState="2" id="SII_6_1"&gt;<br>
&lt;PeptideEvidenceRef peptideEvidence_ref="PE6_2_4"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001329" name="OMSSA:pvalue" cvRef="PSI-MS"</p>
</div>
<div class="paragraph">
<p>value="0.00073351" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationItem&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationList&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/AnalysisData&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/DataCollection&gt;</p>
</div>
</div>
<div class="sect2">
<h3 id="_database_filter">Database Filter</h3>
<div class="paragraph">
<p>The format can specify that a sequence database has been filtered, for example based on pI, protein mass, taxonomy or even a set of accession numbers for a second pass search.
For example, all animals except mice would be encoded as (NCBI:33208 is metazoa, NCBI:10090 is <em>Mus musculus</em>):</p>
</div>
<div class="paragraph">
<p>&lt;DatabaseFilters&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Filter&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FilterType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001020" name="DB filter taxonomy" cvRef="PSI-MS" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FilterType&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Include&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001467" name="taxonomy: NCBI TaxID" cvRef="PSI-MS" value="33208"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Include&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Exclude&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1001467" name="taxonomy: NCBI TaxID" cvRef="PSI-MS" value="10090"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Exclude&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Filter&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/DatabaseFilters&gt;</p>
</div>
</div>
<div class="sect2">
<h3 id="_types_of_parameters_and_values">Types of parameters and values</h3>
<div class="paragraph">
<p>There are several types for parameters that are used in the schema:</p>
</div>
<div class="paragraph">
<p>&lt;ParamListType&gt;: A list (i.e. unbounded number) of &lt;ParamGroup&gt; elements.</p>
</div>
<div class="paragraph">
<p>&lt;ParamGroup&gt;: A choice between &lt;cvParam&gt; or &lt;userParam&gt; elements.</p>
</div>
<div class="paragraph">
<p>&lt;ParamType&gt;: A single reference to &lt;ParamGroup&gt;, which allows a choice between either &lt;cvParam&gt; or &lt;userParam&gt; elements at the specified point in the schema.</p>
</div>
<div class="paragraph">
<p>&lt;cvParamType&gt;: A single entry from an ontology or a CV.
Attributes: <em>accession</em>, <em>cvRef</em>, <em>name</em>, <em>value</em>, <em>unitAccession</em>, <em>unitName</em>, <em>unitCvRef</em>.</p>
</div>
<div class="paragraph">
<p>&lt;userParamType&gt;: A single user-defined parameter.
Attributes: <em>name</em>, <em>value</em>, <em>unitAccession</em>, <em>unitName</em>, <em>unitCvRef</em>.</p>
</div>
</div>
<div class="sect2">
<h3 id="_reporting_fragmentation_ions">Reporting fragmentation ions</h3>
<div class="paragraph">
<p>mzIdentML employs an array type structure to support the reporting of ion types identified in an MS/MS analysis, coupled with CV parameters to retain flexibility in the types of ions that can be reported.</p>
</div>
<div class="paragraph">
<p>A brief example is given here to explain how these structures should be used where <em>y11</em>, <em>y8</em> and <em>y7</em> have been identified with charge = 2+.
First, the types of measures to be reported are given in the &lt;FragmentationTable&gt; using &lt;cvParam&gt; instances.
Second, each &lt;SpectrumIdentificationItem&gt; contains an index of values (11, 8 and 7 for each y ion) and parallel arrays that reference back to each &lt;Measure&gt; defined in the &lt;FragmentationTable&gt;.
In the example, the <em>y8</em> ion has a product ion <em>m/z</em> = 436.4, product ion intensity = 11 and product ion <em>m/z</em> error = 0.1284 (the second position in the index of each array).</p>
</div>
<div class="paragraph">
<p>&lt;FragmentationTable&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Measure id="m_mz"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001225" name="product ion m/z"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Measure&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Measure id="m_intensity"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001226" name="product ion intensity"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Measure&gt;</p>
</div>
<div class="paragraph">
<p>&lt;Measure id="m_error"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001227" name="product ion m/z error" unitAccession="MS:1000040" unitName="m/z" unitCvRef="PSI-MS"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/Measure&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/FragmentationTable&gt;</p>
</div>
<div class="paragraph">
<p>…</p>
</div>
<div class="paragraph">
<p>&lt;IonType index="11 8 7" charge="2"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001220" name="frag: y ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray values="551.3 436.4 380.1 " measure_ref="m_mz"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray values="800 11 46" measure_ref="m_intensity"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray values="0.4752 0.1284 0.3704" measure_ref="m_error"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/IonType&gt;</p>
</div>
<div class="sect3">
<h4 id="_internal_fragments_and_immonium_ions">Internal fragments and immonium ions</h4>
<div class="paragraph">
<p>mzIdentML supports the reporting of internal fragment ions, of which an immonium ion is a special case comprising a single side chain (<a href="http://www.matrixscience.com/help/fragmentation_help.html" class="bare">http://www.matrixscience.com/help/fragmentation_help.html</a>).
For internal and immonium ions, the index is used in two different ways.
Internal fragments are reported using the index structure to identify the start and end of the ion within the sequence.
The example shows how the index performs this different role, as it identifies pairs of internal ions: <em>ya2-5</em>, <em>ya3-7</em>, <em>ya3-8</em>, <em>ya4-8</em>, <em>ya5-8</em>, <em>ya5-11</em>, <em>ya8-11</em>.</p>
</div>
<div class="paragraph">
<p>&lt;IonType index="2 5 3 7 3 8 4 8 5 8 5 11 8 11" charge="1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray values="315.2 388.1 501.4 444.1 342.8 669.901495 412.4 " measure_ref="m_mz"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray values="44 63 10430 75 48 6420 31" measure_ref="m_intensity"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray values="-0.0027 -0.1191 0.0969 -0.1817 -0.4340 0.4721 0.1082" measure_ref="m_error"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001366" name="frag: internal ya ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/IonType&gt;</p>
</div>
<div class="paragraph">
<p>For immonium ions, the index is the position of the identified ion within the peptide sequence.
If the peptide contains the same amino acid in multiple positions that cannot be distinguished, all positions should be given.
Example, where immonium ions have been found matching T and G in the following peptide sequence FGGEENTY (positions 2 or 3, and position 7):</p>
</div>
<div class="paragraph">
<p>&lt;IonType charge="1" index="2 3 7"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray values="288.2 286.1 387.2 371.127841 " measure_ref="m_mz"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray values="2137 83 656 1663" measure_ref="m_intensity"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray values="0.0260 -0.1125 -0.0602 -0.1011" measure_ref="m_error"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001239" name="frag: immonium ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/IonType&gt;</p>
</div>
</div>
<div class="sect3">
<h4 id="_encoding_neutral_loss_fragment_ions">Encoding Neutral loss fragment ions</h4>
<div class="paragraph">
<p>The encoding of the identification of neutral loss fragment ions has changed from version 1.1 to version 1.2.0. The CV previously contained an attempt to enumerate all possible neutral losses from all types of fragment ions, leading to a long and incomplete list of possible terms.
As such, pairs of CV terms are now allowed in version 1.2.0 to describe both the ion type and the type of neutral loss, as follows:</p>
</div>
<div class="paragraph">
<p>&lt;IonType charge="2" index="3"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_MZ" values="165.0899353"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_Int" values="2058.2841796875"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;FragmentArray measure_ref="Measure_Error" values="0.007899949187986977"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1001220" name="frag: y ion"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1000336" name="neutral loss" value="H3N"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/IonType&gt;</p>
</div>
</div>
</div>
<div class="sect2">
<h3 id="_enzyme_definition">Enzyme definition</h3>
<div class="paragraph">
<p>The &lt;SpectrumIdentificationProtocol&gt; SHOULD contain a specification of which enzyme (if any) was applied in the search.
The element &lt;Enzyme&gt; has optional sub-elements for specifying the &lt;EnzymeName&gt; using a CV term and the cleavage site, using a regular expression.
Regular expressions should be encoded following the notation of Perl Compatible Regular Expressions (PCRE regex, <a href="http://www.pcre.org/">http://www.pcre.org</a>, matching the syntax and semantics of Perl version 5).
The PSI-MS CV contains terms for the most common enzymes with pre-defined regular expressions (Table 6).
If the enzyme used is present in the PSI-MS CV, the term MUST be encoded under &lt;EnzymeName&gt; unless the rule given in the CV does not match that used by the software or if the enzyme used is not present in the CV, in which case the regular expression used MUST be given in the element &lt;SiteRegexp&gt;.
If the &lt;EnzymeName&gt; element is used, the regular expression MAY also be provided additionally.
For a no enzyme search, (i.e. one where there may be a cleavage at any residue), the CV term MS:1001091 ‘NoEnzyme’ MUST be specified, and the missedCleavages and semiSpecific attributes SHOULD NOT be specified.
If two or more enzymes are used, multiple &lt;Enzyme&gt; elements SHOULD be provided rather than trying to build a regular expression covering all cleavage sites.
If the software uses a name for an enzyme other than the one specified in the CV, a user param term MAY also be given.</p>
</div>
<div class="paragraph">
<p>The following guidelines SHOULD be followed when generating regular expressions in an instance document for enzymes not present in the CV: 1) use the PCRE supplied negation syntax for look-ahead and look-behind assertions and 2) use the most compact representation possible for a regex.
The start of a match specifies the cleavage point.
For example the enzyme trypsin, which cleaves following a K or R residue unless the next residue is P, has the regular expression:</p>
</div>
<div class="paragraph">
<p>(?&#8656;[KR])(?!P)</p>
</div>
<div class="paragraph">
<p>The ?&#8656; is a "zero-width positive look-behind assertion", and [] means one of this character set.
So, this rule is to look behind for a K or R. ?! is a zero-width positive look-ahead assertion, and ?!P means any character that is not P. An example of an “N-term” enzyme is Asp-N which cleaves before D or B. This can be described using the PCRE:</p>
</div>
<div class="paragraph">
<p>(?=[BD])</p>
</div>
<div class="paragraph">
<p>The ?= is a “zero-width positive look-ahead assertion.”</p>
</div>
<div class="paragraph">
<p>A simple 3 line perl program can be written to test a regular expression:</p>
</div>
<div class="paragraph">
<p>$protein = "ABCDKPEFGHIJKLMNOPQRSTUVWXYZ";</p>
</div>
<div class="paragraph">
<p>@peptides = split(/(?&#8656;[KR])( ?!P)/, $protein);</p>
</div>
<div class="paragraph">
<p>print join "\n", @peptides;</p>
</div>
<div class="paragraph">
<p>The program returns:</p>
</div>
<div class="paragraph">
<p>ABCDKPEFGHIJK</p>
</div>
<div class="paragraph">
<p>LMNOPQR</p>
</div>
<div class="paragraph">
<p>STUVWXYZ</p>
</div>
<table class="tableblock frame-all grid-all stretch">
<colgroup>
<col style="width: 50%;">
<col style="width: 50%;">
</colgroup>
<thead>
<tr>
<th class="tableblock halign-left valign-top"><strong>Enzyme Name</strong></th>
<th class="tableblock halign-left valign-top"><strong>Regular expression</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Trypsin</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;[KR])(?!P)</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Arg-C</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;R)(?!P)</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Asp-N</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?=[BD])</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Asp-N_ambic</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?=[DE])</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Chymotrypsin</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;[FYWL])(?!P)</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">CNBr</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;M)</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">Formic_acid</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">?&#8656;D</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">?=D</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Lys-C</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;K)(?!P)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Lys-C/P</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;K)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">PepsinA</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;[FL])</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">TrypChymo</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;[FYWLKR])(?!P)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Trypsin/P</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;[KR])</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">V8-DE</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;[BDEZ])(?!P)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">V8-E</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;[EZ])(?!P)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Leukocyte elastase</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;[ALIV])(?!P)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Proline endopeptidase</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;[HKR]P)(?!P)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">Glutamyl endopeptidase</p></td>
</tr>
<tr>
<td class="tableblock halign-left valign-top"><p class="tableblock">(?&#8656;[^E]E)</p></td>
<td class="tableblock halign-left valign-top"><p class="tableblock">2-iodobenzoate</p></td>
</tr>
</tbody>
</table>
<div class="paragraph">
<p>Table 6. Common enzymes and the cleavage site specified as regular expressions as represented in the PSI-MS CV.</p>
</div>
</div>
<div class="sect2">
<h3 id="_unknown_modifications">Unknown modifications</h3>
<div class="paragraph">
<p>In version 1.1.0 onwards of mzIdentML there has been a change with respect to how “unknown modifications” (i.e. those not present in an allowed CV) are reported on peptides.
In version 1.0, &lt;userParam&gt; elements were allowed on &lt;Peptide&gt; to capture these modifications.
In version 1.1.0 onwards, only &lt;cvParam&gt; elements can be given on &lt;Peptide&gt; and a term “unknown modification” has been added to the PSI-MS CV.
This term MUST only be used if the identified modification is not present in Unimod (or other allowed CV such as PSI-MOD, although PSI-MOD is now deprecated), according to the identity of the residue modified and the delta mass, within the parent tolerance specified in the search.
The semantic validator will check any uses of the “unknown modification” term (MS:1001460) and reject files if the modification is present in Unimod.</p>
</div>
</div>
<div class="sect2">
<h3 id="_identifications_based_on_multiple_mass_spectra">Identifications based on multiple mass spectra</h3>
<div class="paragraph">
<p>This Section has been added to version 1.3.0 of the specification.
Some analysis workflows utilise multiple spectra to arrive at a given identification, for instance, the following crosslinking search strategies:</p>
</div>
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>(i) “light” and “heavy” isotopes of the crosslinker are used as a pair, combined together and searched once;</p>
</div>
<div class="paragraph">
<p>(ii) multiple spectra of the same precursor are acquired, <em>e.g.</em> using different fragmentation techniques like EThcD;</p>
</div>
<div class="paragraph">
<p>(iii) when using a cleavable crosslinker and both MS3 spectra of the cleaved peptides and the MS2 spectrum of the crosslinked peptide pair are considered in the identification process.</p>
</div>
</blockquote>
</div>
<div class="paragraph">
<p>mzIdentML 1.2.0 included a method for encoding such cases using the “combined spectra” type of input file format.
This essentially associates a single &lt;SpectrumIdentificationResult&gt; element with a comma separated list of spectrum identifiers.
This would work for crosslinking search strategy (i) if all the spectra contributing to an identification share the same acquisition settings, but it presents a problem for cases (ii) and (iii).
The &lt;SpectrumIdentificationResult&gt; element with the comma separated list of identifiers is inside a single &lt;SpectrumIdentificationList&gt; element and this can only be associated with a single &lt;SpectrumAnalysisProtocol&gt; element.
But in cases (ii) and (iii) the spectra referenced have different acquisition settings, which would require the use of distinct &lt;SpectrumAnalysisProtocol&gt; elements to be encoded correctly.
Many of the workflows using cleavable crosslinkers fall into categories (ii) and (iii) above.</p>
</div>
<div class="paragraph">
<p>The consequence of the use of the “combined spectra” type of input file format is that the different spectra cannot be associated with different &lt;SpectrumAnalysisProtocol&gt; elements.
But doing so is necessary to correctly encode cases (ii) and (iii) above (to capture, for example, different acquisition settings for different fragmentation modes).</p>
</div>
<div class="paragraph">
<p>mzIdentML 1.3.0 now advises the use of the new CV term “identification based on multiple spectra” (MS:1003332) to encode identifications based on multiple spectra.
This CV term goes inside &lt;SpectrumIdentificationItem&gt; elements.
The “combined spectra” type of input file format from version 1.2.0 has been retired and is not part of the 1.3.0 specification.</p>
</div>
<div class="paragraph">
<p>The values of the “identification based on multiple spectra” CV term (MS:1003332) take the form: [identifier string]:[P or C].
For example:</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1003332" cvRef="PSI-MS" value="1234:P" name="identification based on multiple spectra"/&gt;</p>
</div>
<div class="paragraph">
<p>The letters ‘P’ and ‘C’ refer to ‘parent’ and ‘child’.
&lt;SpectrumIdentificationItem&gt; elements marked ‘parent’ cover the entire identification (in the case of crosslinking, the crosslinked peptide pair) and those marked ‘child’ only identify a constituent part of the whole identification (typically a single peptide in an MS3 scan).
Identifications based on multiple spectra MUST have both ‘P’ and ‘C’ marked constituent spectra identifications for cases that include at least one child element, or neither ‘P’ nor ‘C’ in the case that there is no parent/child relationship.
There is no limit on the number of constituent spectra that are marked either ‘P’ or ‘C’.</p>
</div>
<div class="paragraph">
<p>The unique identifier string (“1234” in the above example) can associate &lt;SpectrumIdentificationItem&gt; elements across different &lt;SpectrumIdentificationList&gt; elements.
These &lt;SpectrumIdentificationList&gt; elements can then be associated with different &lt;SpectrumIdentificationProtocol&gt; elements.</p>
</div>
<div class="paragraph">
<p>There may not be a parent-child relationship between the scan events supporting an identification since one scan may not represent a constituent part of the other.
To describe such a sibling relationship between scans, the ‘P’ and ‘C’ can be omitted, as is the case in Figure 6.</p>
</div>
<div class="paragraph">
<p>The associated example file <a href="https://github.com/colin-combe/mzIdentML/blob/master/examples/1_3_0examples/crosslinking/multiple_spectra_per_id_1.3.0_draft.mzid">multiple_spectra_per_id_1.3.0</a>_draft.mzid shows a common workflow for cleavable crosslinkers which uses two levels of MS2 (one using HCD as the fragmentation mode, one using ETD as the fragmentation mode) followed by MS3 scans.
To encode this, three distinct &lt;SpectrumIdentificationProtocol&gt; elements are needed: one for HCD MS2 scans, one for ETD MS2 scans, and one for MS3 scans.
Three distinct &lt;SpectrumIdentification&gt; elements associate each of the &lt;SpectrumIdentificationProtocol&gt; elements with different &lt;SpectrumIdentificationList&gt; elements.
Figure 5 shows an excerpt from this example file to illustrate the use of the new CV terms.</p>
</div>
<div class="paragraph">
<p>Figure 6 shows this approach used in the context of encoding identifications of glycopeptides.</p>
</div>
<div class="paragraph">
<p>&lt;AnalysisData&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- for the ETD to be correctly encoded it needs to go into a separate list -&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationList id="sil_ETD"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult spectrumID="index=2" spectraData_ref="pk_id" id="SIR_2"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="3" experimentalMassToCharge="100" peptide_ref="p1"</p>
</div>
<div class="paragraph">
<p>rank="1" passThreshold="false" id="ETD_SII_0"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_p1"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002511"</p>
</div>
<div class="paragraph">
<p>name="crosslink spectrum identification item" value="1"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- this flags it as part of the crosslinked identification '1234' -&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1003332" cvRef="PSI-MS" value="<strong>1234:P</strong>"</p>
</div>
<div class="paragraph">
<p>name="identification based on multiple spectra"/&gt;</p>
</div>
<div class="paragraph">
<p>                &lt;cvParam cvRef="PSI-MS" accession="MS:1003336"</p>
</div>
<div class="paragraph">
<p>name="Posterior Error Probability from multiple spectra identification"</p>
</div>
<div class="paragraph">
<p>value="<strong>1234:1E-08</strong>"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationItem&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="3" experimentalMassToCharge="100" peptide_ref="p2"</p>
</div>
<div class="paragraph">
<p>rank="1" passThreshold="false" id="ETD_SII_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_p2"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002511"</p>
</div>
<div class="paragraph">
<p>name="crosslink spectrum identification item" value="1"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- this flags it as part of the crosslinked identification '1234' -&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1003332" cvRef="PSI-MS" value="<strong>1234:P</strong>"</p>
</div>
<div class="paragraph">
<p>name="identification based on multiple spectra"/&gt;</p>
</div>
<div class="paragraph">
<p>            &lt;cvParam cvRef="PSI-MS" accession="MS:1003336"</p>
</div>
<div class="paragraph">
<p>name="Posterior Error Probability from multiple spectra identification"</p>
</div>
<div class="paragraph">
<p>value="<strong>1234:1E-08</strong>"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationItem&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationResult&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationList&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- as the MS3 may have different search params, e.g. mass tolerance, they need to go into a separate list -&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationList id="sil_MS3"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- HCD MS3 match peptide 1 A-&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult spectrumID="index=3" spectraData_ref="pk_id" id="SIR_3"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="3" experimentalMassToCharge="50"</p>
</div>
<div class="paragraph">
<p>peptide_ref="p1_a" rank="1" passThreshold="false" id="MS3_SII_0"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_p1_a"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- this flags it as part of the crosslinked identification '1234' -&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1003332" cvRef="PSI-MS" value="<strong>1234:C</strong>"</p>
</div>
<div class="paragraph">
<p>name="identification based on multiple spectra"/&gt;</p>
</div>
<div class="paragraph">
<p>                &lt;cvParam cvRef="PSI-MS" accession="MS:1003336"</p>
</div>
<div class="paragraph">
<p>name="Posterior Error Probability from multiple spectra identification"</p>
</div>
<div class="paragraph">
<p>value="<strong>1234:1E-0</strong>8"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationItem&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationResult&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- HCD MS3 match peptide 2 T-&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult spectrumID="index=4" spectraData_ref="pk_id" id="SIR_4"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="3" experimentalMassToCharge="50"</p>
</div>
<div class="paragraph">
<p>peptide_ref="p2_t" rank="1" passThreshold="false" id="MS3_SII_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_p2_t"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- this flags it as part of the crosslinked identification '1234' -&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1003332" cvRef="PSI-MS" value="<strong>1234:C</strong>"</p>
</div>
<div class="paragraph">
<p>name="identification based on multiple spectra"/&gt;</p>
</div>
<div class="paragraph">
<p>                &lt;cvParam cvRef="PSI-MS" accession="MS:1003336"</p>
</div>
<div class="paragraph">
<p>name="Posterior Error Probability from multiple spectra identification"</p>
</div>
<div class="paragraph">
<p>value="<strong>1234:1E-08</strong>"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationItem&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationResult&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- HCD MS3 match peptide 1 T-&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult spectrumID="index=3" spectraData_ref="pk_id" id="SIR_5"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="3" experimentalMassToCharge="51"</p>
</div>
<div class="paragraph">
<p>peptide_ref="p1_t" rank="1" passThreshold="false" id="MS3_SII_2"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_p1_t"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- this flags it as part of the crosslinked identification '1234' →</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1003332" cvRef="PSI-MS" value="<strong>1234:C</strong>"</p>
</div>
<div class="paragraph">
<p>name="identification based on multiple spectra"/&gt;</p>
</div>
<div class="paragraph">
<p>                &lt;cvParam cvRef="PSI-MS" accession="MS:1003336"</p>
</div>
<div class="paragraph">
<p>name="Posterior Error Probability from multiple spectra identification"</p>
</div>
<div class="paragraph">
<p>value="<strong>1234:1E-08</strong>"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationItem&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationResult&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- HCD MS3 match peptide 2 A-&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult spectrumID="index=4" spectraData_ref="pk_id" id="SIR_6"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="3" experimentalMassToCharge="49"</p>
</div>
<div class="paragraph">
<p>peptide_ref="p2_a" rank="1" passThreshold="false" id="MS3_SII_3"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="pepevid_p2_a"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- this flags it as part of the crosslinked identification '1234' -&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1003332" cvRef="PSI-MS" value="<strong>1234:C</strong>"</p>
</div>
<div class="paragraph">
<p>name="identification based on multiple spectra"/&gt;</p>
</div>
<div class="paragraph">
<p>                &lt;cvParam cvRef="PSI-MS" accession="MS:1003336"</p>
</div>
<div class="paragraph">
<p>name="Posterior Error Probability from multiple spectra identification"</p>
</div>
<div class="paragraph">
<p>value="<strong>1234:1E-08</strong>"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationItem&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationResult&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationList&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/AnalysisData&gt;</p>
</div>
<div class="paragraph">
<p><strong>Figure 5.</strong> XML snippet showing the new CV terms "identification based on multiple spectra" (MS:1003332) and "Posterior Error Probability from multiple spectra identification" (MS:1003336) used in the context of crosslinking.</p>
</div>
<div class="paragraph">
<p>To encode peptide level scores for multiple spectra identifications different CV terms MUST be used.
The values of these new terms take the form: [multiple spectra identification identifier]: [score].
The identifier before the colon MUST be an identifier used for an identification based on multiple spectra.
For example:</p>
</div>
<div class="paragraph">
<p>&lt;cvParam accession="MS:1003332" cvRef="PSI-MS" value="<strong>1234:P</strong>" name="identification based on multiple spectra"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1003336"</p>
</div>
<div class="paragraph">
<p>name="Posterior Error Probability for an identification based on multiple spectra" value="<strong>1234:1E-08</strong>"/&gt;</p>
</div>
<div class="paragraph">
<p>A new CV term for "Posterior Error Probability from multiple spectra identification" (MS:1003336) has also been introduced in this context.
For search specific, match level scores for identifications based on multiple spectra, new CV terms must be created, these should be children of MS:1003334 (“Parent term for PSM-level scores for identifications based on multiple spectra”).</p>
</div>
<div class="paragraph">
<p>&lt;AnalysisData&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- HCD Spectrum List -&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationList id="SII_LIST_4_1" &gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult spectrumID="controllerType=0 controllerNumber=1 scan=3832"</p>
</div>
<div class="paragraph">
<p>spectraData_ref="SD_17022_recal_B210619_02_Lumos_ZC_CO_190_D2I_SDA-</p>
</div>
<div class="paragraph">
<p>WT12019_09_19_OPRmix_35trig_EThcD35.raw"</p>
</div>
<div class="paragraph">
<p>id="SIR_4"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="3" experimentalMassToCharge="821.6863"</p>
</div>
<div class="paragraph">
<p>calculatedMassToCharge="821.68451"</p>
</div>
<div class="paragraph">
<p>peptide_ref="T_O-Glycosylation_EAQT_O-Glycosylation_T_O-</p>
</div>
<div class="paragraph">
<p>Glycosylation_PLAA_Hex_4_HexNAc_4" rank="1"</p>
</div>
<div class="paragraph">
<p>passThreshold="true" id="SII_4_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef peptideEvidence_ref="T_O-Glycosylation_EAQT_O-Glycosylation_T_O-</p>
</div>
<div class="paragraph">
<p>Glycosylation_PLAA_Hex_4_HexNAc_4_pe1"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002520" value="T_O-Glycosylation_EAQT_O-</p>
</div>
<div class="paragraph">
<p>Glycosylation_T_O-Glycosylation_PLAA_Hex_4_HexNAc_4:256"</p>
</div>
<div class="paragraph">
<p>name="peptide group ID"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002354" value="0.005" name="PSM-level q-value"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef=”PSI-MS” accession="MS:1003332" value="<strong>256</strong>"</p>
</div>
<div class="paragraph">
<p>name="identification based on multiple spectra" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1003336"</p>
</div>
<div class="paragraph">
<p>name="Posterior Error Probability from multiple spectra identification"</p>
</div>
<div class="paragraph">
<p>value="256:1E-08"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationItem&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationResult&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationList&gt;</p>
</div>
<div class="paragraph">
<p>&lt;!-- EThcD Spectrum List -&#8594;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationList id="SII_LIST_5_1" &gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationResult spectrumID="controllerType=0 controllerNumber=1 scan=3836"</p>
</div>
<div class="paragraph">
<p>spectraData_ref="SD_17022_recal_B210619_02_Lumos_ZC_CO_190_D2I_SDA-</p>
</div>
<div class="paragraph">
<p>WT12019_09_19_OPRmix_35trig_EThcD35.raw"</p>
</div>
<div class="paragraph">
<p>id="SIR_5"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;SpectrumIdentificationItem chargeState="3" experimentalMassToCharge="821.6863"</p>
</div>
<div class="paragraph">
<p>calculatedMassToCharge="821.68451"</p>
</div>
<div class="paragraph">
<p>peptide_ref="T_O-Glycosylation_EAQT_O-Glycosylation_T_O-</p>
</div>
<div class="paragraph">
<p>Glycosylation_PLAA_Hex_4_HexNAc_4" rank="1"</p>
</div>
<div class="paragraph">
<p>passThreshold="true" id="SII_5_1"&gt;</p>
</div>
<div class="paragraph">
<p>&lt;PeptideEvidenceRef</p>
</div>
<div class="paragraph">
<p>peptideEvidence_ref="T_Hex_1_HexNAc_1_EAQT_Hex_2_HexNAc_2_T_Hex_1_HexNAc_1_PLAA_pe1"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002520"</p>
</div>
<div class="paragraph">
<p>value="T_O-Glycosylation_EAQT_O-Glycosylation_T_O-</p>
</div>
<div class="paragraph">
<p>Glycosylation_PLAA_Hex_4_HexNAc_4:256"</p>
</div>
<div class="paragraph">
<p>name="peptide group ID"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1003147" value="1:0.97:1:true"</p>
</div>
<div class="paragraph">
<p>name="PTMProphet probability"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1003147" value="2:0.83:5:true"</p>
</div>
<div class="paragraph">
<p>name="PTMProphet probability"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1003147" value="3:0.89:6:true"</p>
</div>
<div class="paragraph">
<p>name="PTMProphet probability"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1002354" value="0.001" name="PSM-level q-value"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef=”PSI-MS” accession="MS:1003332"</p>
</div>
<div class="paragraph">
<p>name="identification based on multiple spectra" value="<strong>256</strong>" /&gt;</p>
</div>
<div class="paragraph">
<p>&lt;cvParam cvRef="PSI-MS" accession="MS:1003336"</p>
</div>
<div class="paragraph">
<p>name="Posterior Error Probability from multiple spectra identification"</p>
</div>
<div class="paragraph">
<p>value="256:1E-08"/&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationItem&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationResult&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/SpectrumIdentificationList&gt;</p>
</div>
<div class="paragraph">
<p>&lt;/AnalysisData&gt;</p>
</div>
<div class="paragraph">
<p><strong>Figure 6.</strong> XML snippet showing the new CV terms "identification based on multiple spectra" (MS:1003332) and "Posterior Error Probability from multiple spectra identification" (MS:1003336) used in the context of encoding glycopeptides, using HCD-EThCD spectrum pairs for glycan localization (taken from dataset PXD020077).</p>
</div>
</div>
<div class="sect2">
<h3 id="_linking_searchmodification_elements_to_modification_elements">Linking SearchModification elements to Modification elements</h3>
<div class="paragraph">
<p>mzIdentML version 1.3.0 introduces two new CV terms to link &lt;SearchModification&gt; elements and &lt;Modification&gt; elements - “search modification id” (MS:1003392) which goes inside &lt;SearchModification&gt; elements, and “search modification id ref” (MS:1003393) which goes inside &lt;Modification&gt; elements.
This allows for more detailed information on modifications to be provided without redundant repetition of this information throughout the file.
Making this link is optional but recommended where possible.
In the case of open modification searches, such a link cannot be made.</p>
</div>
<div class="paragraph">
<p>The values of “search modification id” (MS:1003392) MUST be unique within the &lt;SpectrumIdentificationProtocol&gt; element.</p>
</div>
<div class="paragraph">
<p>However, if there are multiple &lt;SpectrumIdentificationProtocol&gt; elements within the file, then the values of “search modification id” SHOULD be identical for identical modifications across different &lt;SpectrumIdentificationProtocol&gt; elements.
This is to avoid the duplication of &lt;Peptide&gt; elements with the same &lt;Modification&gt; but identified with a different &lt;SpectrumIdentificationProtocol&gt;).</p>
</div>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_conclusions">Conclusions</h2>
<div class="sectionbody">
<div class="paragraph">
<p>This document contains the specifications for using the mzIdentML format to represent results from peptide and protein identification pipelines, in the context of a proteomics investigation.
This specification, in conjunction with the XML Schema, mapping file and CV constitute a proposal for a standard from the Proteomics Standards Initiative.</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_authors_and_contributors">Authors and Contributors</h2>
<div class="sectionbody">
<div class="paragraph">
<p>Colin W. Combe, University of Edinburgh</p>
</div>
<div class="paragraph">
<p>Lars Kolbowski, Technical University Berlin</p>
</div>
<div class="paragraph">
<p>Lutz Fischer, University of Edinburgh</p>
</div>
<div class="paragraph">
<p>Ville Koskinen, Matrix Science, London</p>
</div>
<div class="paragraph">
<p>Joshua Klein, Boston University</p>
</div>
<div class="paragraph">
<p>Alexander Leitner, ETH Zurich</p>
</div>
<div class="paragraph">
<p>Juan Antonio Vizcaíno, European Informatics Institute</p>
</div>
<div class="paragraph">
<p>Andy Jones, University of Liverpool</p>
</div>
<div class="paragraph">
<p>Juri Rappsilber, University of Edinburg &amp; Technical University Berlin</p>
</div>
<div class="paragraph">
<p>The mzIdentML version 1.2.0 authors were as follows:</p>
</div>
<div class="paragraph">
<p>Juan Antonio Vizcaíno, European Bioinformatics Institute</p>
</div>
<div class="paragraph">
<p>Gerhard Mayer, Medizinisches Proteom-Center, Ruhr-Universität Bochum</p>
</div>
<div class="paragraph">
<p>Martin Eisenacher, Medizinisches Proteom-Center, Ruhr-Universität Bochum</p>
</div>
<div class="paragraph">
<p>Sean Seymour, SCIEX / Applied Biosystems</p>
</div>
<div class="paragraph">
<p>Terry Farrah, Institute for Systems Biology</p>
</div>
<div class="paragraph">
<p>Eric Deutsch, Institute for Systems Biology</p>
</div>
<div class="paragraph">
<p>Andrew R Jones, University of Liverpool</p>
</div>
<div class="paragraph">
<p>The mzIdentML version 1.1.0 authors were as follows:</p>
</div>
<div class="paragraph">
<p>David Creasy, Matrix Science</p>
</div>
<div class="paragraph">
<p>Florian Reisinger, European Bioinformatics Institute</p>
</div>
<div class="paragraph">
<p>Johannes Griss, European Bioinformatics Institute</p>
</div>
<div class="paragraph">
<p>Juan Antonio Vizcaíno, European Bioinformatics Institute</p>
</div>
<div class="paragraph">
<p>Matthew Chambers, Vanderbilt University Medical Center</p>
</div>
<div class="paragraph">
<p>Gerhard Mayer, Medizinisches Proteom-Center, Ruhr-Universität Bochum</p>
</div>
<div class="paragraph">
<p>Martin Eisenacher, Medizinisches Proteom-Center, Ruhr-Universität Bochum</p>
</div>
<div class="paragraph">
<p>Andrew Jones, University of Liverpool</p>
</div>
<div class="paragraph">
<p>The mzIdentML version 1.0 authors were as follows:</p>
</div>
<div class="paragraph">
<p>Angel Pizarro, Center for Bioinformatics, University of Pennsylvania</p>
</div>
<div class="paragraph">
<p>David Creasy, MatrixScience</p>
</div>
<div class="paragraph">
<p>Phil Jones, European Bioinformatics Institute</p>
</div>
<div class="paragraph">
<p>Andreas Bertsch, Eberhard Karls University Tübingen</p>
</div>
<div class="paragraph">
<p>Jenny Siepen, University of Manchester</p>
</div>
<div class="paragraph">
<p>Martin Eisenacher, Medizinisches Proteom-Center, Ruhr-Universität Bochum</p>
</div>
<div class="paragraph">
<p>Andrew Jones, University of Liverpool</p>
</div>
<div class="paragraph">
<p>In addition to the authors indicated above, the following people contributed to the model development, gave feedback or tested mzIdentML:</p>
</div>
<div class="ulist">
<ul>
<li>
<p>Simon Hubbard, University of Manchester</p>
</li>
<li>
<p>Julian Selley, University of Manchester</p>
</li>
<li>
<p>Zsuzsanna Bencsath-Makkai, Biomedical Engineering, McGill University</p>
</li>
<li>
<p>Randy Julian, IndigoBio</p>
</li>
<li>
<p>Pierre-Alain Binz, GeneBio Geneva</p>
</li>
<li>
<p>Alex Masselot, GeneBio Geneva</p>
</li>
<li>
<p>Lennart Martens, European Bioinformatics Institute</p>
</li>
<li>
<p>Henning Hermjakob, European Bioinformatics Institute</p>
</li>
<li>
<p>Luisa Montecchi, European Bioinformatics Institute</p>
</li>
<li>
<p>Richard Côté, European Bioinformatics Institute</p>
</li>
<li>
<p>Marc Sturm, Eberhard Karls University, Tübingen</p>
</li>
<li>
<p>Jim Shofstahl, Thermo Fisher</p>
</li>
<li>
<p>David Horn, Agilent</p>
</li>
<li>
<p>Jimmy Eng, Fred Hutchinson Cancer Research</p>
</li>
<li>
<p>Brian Searle, Proteome Software</p>
</li>
<li>
<p>Phillip Young, Waters</p>
</li>
<li>
<p>Michael Kohl, Medizinisches Proteom-Center, Ruhr-Universität Bochum, Germany</p>
</li>
<li>
<p>Christian Stephan, Medizinisches Proteom-Center, Ruhr-Universität Bochum, Germany</p>
</li>
<li>
<p>Eugene Kapp, Ludwig Institute for Cancer Research</p>
</li>
<li>
<p>Michael Coleman, Stowers Institute</p>
</li>
<li>
<p>Julian Uszkoreit, Medizinisches Proteom-Center, Ruhr-Universität Bochum, Germany</p>
</li>
<li>
<p>Oliver Kohlbacher, Eberhard Karls University, Tübingen</p>
</li>
<li>
<p>Mathias Walzer, Eberhard Karls University, Tübingen</p>
</li>
<li>
<p>David Ovelleiro, European Bioinformatics Institute</p>
</li>
<li>
<p>Alberto Medina, ProteoRed Consortium, Spain</p>
</li>
<li>
<p>Salvador Martínez-Bartolomé, ProteoRed Consortium, Spain</p>
</li>
<li>
<p>Laurent Gatto, University of Cambridge</p>
</li>
<li>
<p>Simon Perkins, University of Liverpool</p>
</li>
<li>
<p>Harald Barsnes, University of Bergen</p>
</li>
<li>
<p>Marc Vaudel, University of Bergen</p>
</li>
<li>
<p>Yasset Perez-Riverol, European Bioinformatics Institute</p>
</li>
<li>
<p>Tobias Ternent, European Bioinformatics Institute</p>
</li>
<li>
<p>Eugen Netz, Max Planck Institute for Developmental Biology, Tübingen</p>
</li>
<li>
<p>Robert J. Chalkley, University of California San Franscisco</p>
</li>
<li>
<p>Fawaz Ghali, University of Liverpool</p>
</li>
<li>
<p>Timo Sachsenberg, Eberhard Karls University, Tübingen</p>
</li>
<li>
<p>Andrea Sinz, Martin Luther University Halle-Wittenberg</p>
</li>
</ul>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_references">References</h2>
<div class="sectionbody">
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p>1. Martens, L., Chambers, M., Sturm, M., Kessner, D., Levander, F., Shofstahl, J., Tang, W. H., Römpp, A., Neumann, S., Pizarro, A. D., Montecchi-Palazzi, L., Tasman, N., Coleman, M., Reisinger, F., Souda, P., Hermjakob, H., Binz, P.-A., and Deutsch, E. W. (2011) mzML—a Community Standard for Mass Spectrometry Data. <em>Molecular &amp; Cellular Proteomics</em> 10, R110.0001332. Walzer, M., Qi, D., Mayer, G., Uszkoreit, J., Eisenacher, M., Sachsenberg, T., Gonzalez-Galarza, F. F., Fan, J., Bessant, C., Deutsch, E. W., Reisinger, F., Vizcaino, J. A., Medina-Aunon, J. A., Albar, J. P., Kohlbacher, O., and Jones, A. R. (2013) The mzQuantML data standard for mass spectrometry-based quantitative studies in proteomics. <em>Molecular &amp; cellular proteomics : MCP</em> 12, 2332-23403. Griss, J., Jones, A. R., Sachsenberg, T., Walzer, M., Gatto, L., Hartler, J., Thallinger, G. G., Salek, R. M., Steinbeck, C., Neuhauser, N., Cox, J., Neumann, S., Fan, J., Reisinger, F., Xu, Q. W., Del Toro, N., Perez-Riverol, Y., Ghali, F., Bandeira, N., Xenarios, I., Kohlbacher, O., Vizcaino, J. A., and Hermjakob, H. (2014) The mzTab data exchange format: communicating mass-spectrometry-based proteomics and metabolomics experimental results to a wider audience. <em>Molecular &amp; cellular proteomics : MCP</em> 13, 2765-27754. Kerrien, S., Orchard, S., Montecchi-Palazzi, L., Aranda, B., Quinn, A., Vinod, N., Bader, G., Xenarios, I., Wojcik, J., Sherman, D., Tyers, M., Salama, J., Moore, S., Ceol, A., Chatr-aryamontri, A., Oesterheld, M., Stumpflen, V., Salwinski, L., Nerothin, J., Cerami, E., Cusick, M., Vidal, M., Gilson, M., Armstrong, J., Woollard, P., Hogue, C., Eisenberg, D., Cesareni, G., Apweiler, R., and Hermjakob, H. (2007) Broadening the horizon - level 2.5 of the HUPO-PSI format for molecular interactions. <em>BMC Biology</em> 5, 445. Mayer, G., Montecchi-Palazzi, L., Ovelleiro, D., Jones, A. R., Binz, P.-A., Deutsch, E. W., Chambers, M., Kallhardt, M., Levander, F., Shofstahl, J., Orchard, S., Antonio Vizcaíno, J., Hermjakob, H., Stephan, C., Meyer, H. E., and Eisenacher, M. (2013) The HUPO proteomics standards initiative- mass spectrometry controlled vocabulary. <em>Database</em> 20136. Seymour, S. L., Farrah, T., Binz, P. A., Chalkley, R. J., Cottrell, J. S., Searle, B. C., Tabb, D. L., Vizcaino, J. A., Prieto, G., Uszkoreit, J., Eisenacher, M., Martinez-Bartolome, S., Ghali, F., and Jones, A. R. (2014) A standardized framing for reporting protein identifications in mzIdentML 1.2. <em>Proteomics</em> 14, 2389-23997. Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., and Durbin, R. (2009) The Sequence Alignment/Map format and SAMtools. <em>Bioinformatics (Oxford, England)</em> 25, 2078-2079</p>
</div>
</blockquote>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_intellectual_property_statement">Intellectual Property Statement</h2>
<div class="sectionbody">
<div class="paragraph">
<p>The PSI takes no position regarding the validity or scope of any intellectual property or other rights that might be claimed to pertain to the implementation or use of the technology described in this document or the extent to which any license under such rights might or might not be available; neither does it represent that it has made any effort to identify any such rights.
Copies of claims of rights made available for publication and any assurances of licenses to be made available, or the result of an attempt made to obtain a general license or permission for the use of such proprietary rights by implementers or users of this specification can be obtained from the PSI Chair.</p>
</div>
<div class="paragraph">
<p>The PSI invites any interested party to bring to its attention any copyrights, patents or patent applications, or other proprietary rights that may cover technology that may be required to practice this recommendation.
Please address the information to the PSI Chair (see contacts information at PSI website).</p>
</div>
</div>
</div>
<div class="sect1">
<h2 id="_copyright_notice">Copyright Notice</h2>
<div class="sectionbody">
<div class="paragraph">
<p>Copyright &#169; Proteomics Standards Initiative (2024).
All Rights Reserved.</p>
</div>
<div class="paragraph">
<p>This document and translations of it may be copied and furnished to others, and derivative works that comment on or otherwise explain it or assist in its implementation may be prepared, copied, published and distributed, in whole or in part, without restriction of any kind, provided that the above copyright notice and this paragraph are included on all such copies and derivative works.
However, this document itself may not be modified in any way, such as by removing the copyright notice or references to the PSI or other organizations, except as needed for the purpose of developing Proteomics Recommendations in which case the procedures for copyrights defined in the PSI Document process must be followed, or as required to translate it into languages other than English.</p>
</div>
<div class="paragraph">
<p>The limited permissions granted above are perpetual and will not be revoked by the PSI or its successors or assigns.</p>
</div>
<div class="paragraph">
<p>This document and the information contained herein is provided on an "AS IS" basis and THE PROTEOMICS STANDARDS INITIATIVE DISCLAIMS ALL WARRANTIES, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTY THAT THE USE OF THE INFORMATION HEREIN WILL NOT INFRINGE ANY RIGHTS OR ANY IMPLIED WARRANTIES OF MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE."</p>
</div>
</div>
</div>
</div>
<div id="footer">
<div id="footer-text">
Last updated 2025-02-20 17:25:40 +0100
</div>
</div>
</body>
</html>