= Technical Specification

== XML Schema Elements

=== Element `<MzIdentML>`

Definition:: The upper-most hierarchy level of mzIdentML with sub-containers for example describing software, protocols and search results (spectrum identifications or protein detection results).

Type:: `MzIdentMLType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cv>`

Definition:: A source controlled vocabulary from which cvParams will be obtained.

Type:: `cvType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<AnalysisSoftware>`

Definition:: The software used for performing the analyses.

Type:: `AnalysisSoftwareType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Sample>`

Definition:: A description of the sample analysed by mass spectrometry using CVParams or UserParams. If a composite sample has been analysed, a parent sample should be defined, which references subsamples. This represents any kind of substance used in an experimental workflow, such as whole organisms, cells, DNA, solutions, compounds and experimental substances (gels, arrays etc.).

Type:: `SampleType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<DBSequence>`

Definition:: A database sequence from the specified SearchDatabase (nucleic acid or amino acid). If the sequence is nucleic acid, the source nucleic acid sequence
should be given in the seq attribute rather than a translated sequence.

Type:: `DBSequenceType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Peptide>`

Definition:: One (poly)peptide (a sequence with modifications). The combination of Peptide sequence and modifications must be unique in the file.

Type:: `PeptideType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<PeptideEvidence>`

Definition:: PeptideEvidence links a specific Peptide element to a specific position in a DBSequence. There must only be one PeptideEvidence item per Peptide-to-DBSequence-position.

Type:: `PeptideEvidenceType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SpectrumIdentification>`

Definition:: An Analysis which tries to identify peptides in input spectra, referencing the database searched, the input spectra, the output results and the protocol that is run.

Type:: `SpectrumIdentificationType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<ProteinDetection>`

Definition:: An Analysis which assembles a set of peptides (e.g. from a spectra search analysis) to proteins.

Type:: `ProteinDetectionType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SpectrumIdentificationProtocol>`

Definition:: The parameters and settings of a SpectrumIdentification analysis.

Type:: `SpectrumIdentificationProtocolType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<ProteinDetectionProtocol>`

Definition:: The parameters and settings of a ProteinDetection process.

Type:: `ProteinDetectionProtocolType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SourceFile>`

Definition:: A file from which this mzIdentML instance was created.

Type:: `SourceFileType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SearchDatabase>`

Definition:: A database for searching mass spectra. Examples include a set of amino acid sequence entries, nucleotide databases (e.g. 6 frame translated) or annotated spectra libraries.

Type:: `SearchDatabaseType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SpectraData>`

Definition:: A data set containing spectra data (consisting of one or more spectra).

Type:: `SpectraDataType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SpectrumIdentificationList>`

Definition:: Represents the set of all search results from SpectrumIdentification.

Type:: `SpectrumIdentificationListType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<ProteinDetectionList>`

Definition:: The protein list resulting from a protein detection process.

Type:: `ProteinDetectionListType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Inputs>`

Definition:: The inputs to the analyses including the databases searched, the spectral data and the source file converted to mzIdentML.

Type:: `InputsType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<AnalysisData>`

Definition:: No documentation available

Type:: `psi-pi:AnalysisDataType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvList>`

Definition:: The list of controlled vocabularies used in the file.

Type:: `CVListType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: A single entry from an ontology or a controlled
vocabulary.

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<AnalysisSoftwareList>`

Definition:: The software packages used to perform the analyses.

Type:: `AnalysisSoftwareListType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Provider>`

Definition:: The Provider of the mzIdentML record in terms of the contact and software.

Type:: `ProviderType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<AuditCollection>`

Definition:: The complete set of Contacts (people and organisations) for this file.

Type:: `AuditCollectionType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<AnalysisSampleCollection>`

Definition:: The samples analysed can optionally be recorded using CV terms for descriptions. If a composite sample has been analysed, the subsample association can be used to build a hierarchical description.

Type:: `AnalysisSampleCollectionType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SequenceCollection>`

Definition:: The collection of sequences (DBSequence or Peptide) identified and their relationship between each other (PeptideEvidence) to be referenced elsewhere in the results.

Type:: `SequenceCollectionType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<AnalysisCollection>`

Definition:: The analyses performed to get the results, which map the input and output data sets. Analyses are for example: SpectrumIdentification (resulting in peptides) or ProteinDetection (assemble proteins from peptides).

Type:: `AnalysisCollectionType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<AnalysisProtocolCollection>`

Definition:: The collection of protocols which include the parameters and settings of the performed analyses.

Type:: `AnalysisProtocolCollectionType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<DataCollection>`

Definition:: The collection of input and output data sets of the analyses.

Type:: `DataCollectionType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<BibliographicReference>`

Definition:: Any bibliographic references associated with the file

Type:: `BibliographicReferenceType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<DatabaseName>`

Definition:: The database name may be given as a cvParam if it maps exactly to one of the release databases listed in the CV, otherwise a userParam should be used.

Type:: `ParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: A single entry from an ontology or a controlled
vocabulary.

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SearchModification>`

Definition:: Specification of a search modification as parameter for a spectra search. Contains the name of the modification, the mass, the specificity and whether it is a static modification.

Type:: `SearchModificationType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<FilterType>`

Definition:: The type of filter e.g. database taxonomy filter, pi filter, mw filter

Type:: `ParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Include>`

Definition:: All sequences fulfilling the specifed criteria are included.

Type:: `ParamListType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Exclude>`

Definition:: All sequences fulfilling the specifed criteria are excluded.

Type:: `ParamListType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Filter>`

Definition:: Filters applied to the search database. The filter must include at least one of Include and Exclude. If both are used, it is assumed that inclusion is performed first.

Type:: `FilterType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: The details specifying this translation table are captured as cvParams, e.g. translation table, translation start codons and translation table description (see specification document and mapping file)

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<TranslationTable>`

Definition:: The table used to translate codons into nucleic acids e.g. by reference to the NCBI translation table.

Type:: `TranslationTableType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SearchType>`

Definition:: The type of search performed e.g. PMF, Tag searches, MS-MS

Type:: `ParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<AdditionalSearchParams>`

Definition:: The search parameters other than the modifications searched.

Type:: `ParamListType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<ModificationParams>`

Definition:: The specification of static/variable modifications (e.g. Oxidation of Methionine) that are to be considered in the spectra search.

Type:: `ModificationParamsType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Enzymes>`

Definition:: The list of enzymes used in experiment

Type:: `EnzymesType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<MassTable>`

Definition:: The masses of residues used in the search.

Type:: `MassTableType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<FragmentTolerance>`

Definition:: The tolerance of the search given as a plus and minus value with units.

Type:: `ToleranceType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<ParentTolerance>`

Definition:: The tolerance of the search given as a plus and minus value with units.

Type:: `ToleranceType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Threshold>`

Definition:: The threshold(s) applied to determine that a result is significant. If multiple terms are used it is assumed that all conditions are satisfied by the passing results.

Type:: `ParamListType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<DatabaseFilters>`

Definition:: The specification of filters applied to the database searched.

Type:: `DatabaseFiltersType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<DatabaseTranslation>`

Definition:: A specification of how a nucleic acid sequence database was translated for searching.

Type:: `DatabaseTranslationType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<InputSpectra>`

Definition:: One of the spectra data sets used.

Type:: `InputSpectraType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SearchDatabaseRef>`

Definition:: One of the search databases used.

Type:: `SearchDatabaseRefType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: A single entry from an ontology or a controlled
vocabulary.

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Measure>`

Definition:: References to CV terms defining the measures about product ions to be reported in SpectrumIdentificationItem

Type:: `MeasureType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<FragmentationTable>`

Definition:: Contains the types of measures that will be reported in generic arrays for each SpectrumIdentificationItem e.g. product ion m/z, product ion intensity, product ion m/z error

Type:: `FragmentationTableType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SpectrumIdentificationResult>`

Definition:: All identifications made from searching one spectrum. For PMF data, all peptide identifications will be listed underneath as SpectrumIdentificationItems. For MS/MS data, there will be ranked SpectrumIdentificationItems corresponding to possible different peptide IDs.

Type:: `SpectrumIdentificationResultType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: A single entry from an ontology or a controlled
vocabulary.

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SpecificityRules>`

Definition:: The specificity rules of the searched modification including for example the probability of a modification's presence or peptide or protein termini. Standard fixed or variable status should be provided by the attribute fixedMod.

Type:: `SpecificityRulesType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: The modification is uniquely identified by references to external CVs such as UNIMOD, see specification document and mapping file for more details.

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<FragmentArray>`

Definition:: An array of values for a given type of measure and for a particular ion type, in parallel to the index of ions identified.

Type:: `FragmentArrayType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<userParam>`

Definition:: In case more information about the ions annotation has to be conveyed, that has no fit in FragmentArray. Note: It is suggested that the value attribute takes the form of a list of the same size as FragmentArray values. However, there is no formal encoding and it cannot be expeceted that other software will process or impart that information properly.

Type:: `UserParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: The type of ion identified. In the case of neutral losses, one term should report the ion type, a second term should report the neutral loss - note: this is a change in practice from mzIdentML 1.1.

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<IonType>`

Definition:: IonType defines the index of fragmentation ions being reported, importing a CV term for the type of ion e.g. b ion. Example: if b3 b7 b8 and b10 have been identified, the index attribute will contain 3 7 8 10, and the corresponding values will be reported in parallel arrays below

Type:: `IonTypeType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<PeptideEvidenceRef>`

Definition:: Reference to the PeptideEvidence element identified. If a specific sequence can be assigned to multiple proteins and or positions in a protein all possible PeptideEvidence elements should be referenced here.

Type:: `PeptideEvidenceRefType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Fragmentation>`

Definition:: The product ions identified in this result.

Type:: `FragmentationType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SpectrumIdentificationItem>`

Definition:: An identification of a single (poly)peptide, resulting from querying an input spectra, along with the set of confidence values for that identification.
PeptideEvidence elements should be given for all mappings of the corresponding Peptide sequence within protein sequences.

Type:: `SpectrumIdentificationItemType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<InputSpectrumIdentifications>`

Definition:: The lists of spectrum identifications that are input to the protein detection process.

Type:: `InputSpectrumIdentificationsType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<AnalysisParams>`

Definition:: The parameters and settings for the protein detection given as CV terms.

Type:: `ParamListType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Threshold>`

Definition:: The threshold(s) applied to determine that a result is significant. If multiple terms are used it is assumed that all conditions are satisfied by the passing results.

Type:: `ParamListType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<ProteinAmbiguityGroup>`

Definition:: A set of logically related results from a protein detection, for example to represent conflicting assignments of peptides to proteins.

Type:: `ProteinAmbiguityGroupType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SpectrumIdentificationItemRef>`

Definition:: Reference(s) to the SpectrumIdentificationItem element(s) that support the given PeptideEvidence element. Using these references it is possible to indicate which spectra were actually accepted as evidence for this peptide identification in the given protein.

Type:: `SpectrumIdentificationItemRefType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<PeptideHypothesis>`

Definition:: Peptide evidence on which this ProteinHypothesis is based by reference to a PeptideEvidence element.

Type:: `PeptideHypothesisType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<ProteinDetectionHypothesis>`

Definition:: A single result of the ProteinDetection analysis (i.e. a protein).

Type:: `ProteinDetectionHypothesisType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: CV terms capturing the modification, sourced from an appropriate controlled vocabulary.

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<PeptideSequence>`

Definition:: The amino acid sequence of the (poly)peptide. If a substitution modification has been found, the original sequence
should be reported.

Type:: `sequence`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Modification>`

Definition:: A molecule modification specification. If n modifications have been found on a peptide, there should be n instances of Modification. If multiple modifications are provided as cvParams, it is assumed that the modification is ambiguous i.e. one modification or another. A cvParam must be provided with the identification of the modification sourced from a suitable CV e.g. UNIMOD. If the modification is not present in the CV (and this will be checked by the semantic validator within a given tolerance window), there is a â€œunknown modificationâ€ CV term that must be used instead. A neutral loss should be defined as an additional CVParam within Modification. If more complex information should be given about neutral losses (such as presence/absence on particular product ions), this can additionally be encoded within the FragmentationArray.

Type:: `ModificationType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SubstitutionModification>`

Definition:: A modification where one residue is substituted by another (amino acid change).

Type:: `SubstitutionModificationType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SpectrumIDFormat>`

Definition:: The format of the spectrum identifier within the source file

Type:: `SpectrumIDFormatType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<ContactRole>`

Definition:: The contact details of the organisation or person that produced the software

Type:: `ContactRoleType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SoftwareName>`

Definition:: The name of the analysis software package, sourced from a CV if available.

Type:: `ParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Customizations>`

Definition:: Any customizations to the software, such as alternative scoring mechanisms implemented, should be documented here as free text.

Type:: `xsd:string`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SiteRegexp>`

Definition:: Regular expression for specifying the enzyme cleavage site.

Type:: `xsd:string`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<EnzymeName>`

Definition:: The name of the enzyme from a CV.

Type:: `ParamListType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Enzyme>`

Definition:: The details of an individual cleavage enzyme should be provided by giving a regular expression or a CV term if a "standard" enzyme cleavage has been performed.

Type:: `EnzymeType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Residue>`

Definition:: The specification of a single residue within the mass table.

Type:: `ResidueType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<AmbiguousResidue>`

Definition:: Ambiguous residues e.g. X can be specified by the Code attribute and a set of parameters for example giving the different masses that will be used in the search.

Type:: `AmbiguousResidueType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: CV terms capturing the tolerance plus and minus values.

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: CV term capturing the type of identifier used.

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Seq>`

Definition:: The actual sequence of amino acids or nucleic acid.

Type:: `sequence`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<ContactRole>`

Definition:: Contact details for the Material. The association to ContactRole could specify, for example, the creator or provider of the Material.

Type:: `ContactRoleType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<SubSample>`

Definition:: References to the individual component samples within a mixed parent sample.

Type:: `SubSampleType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<ExternalFormatDocumentation>`

Definition:: A URI to access documentation and tools to interpret the external format of the ExternalData instance. For example, XML Schema or static libraries (APIs) to access binary formats.

Type:: `xsd:anyURI`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<FileFormat>`

Definition:: The format of the ExternalData file, for example "tiff" for image files.

Type:: `FileFormatType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: cvParam capturing file formats

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Affiliation>`

Definition:: The organization a person belongs to.

Type:: `AffiliationType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Parent>`

Definition:: The containing organization (the university or business which a lab belongs to, etc.)

Type:: `ParentOrganizationType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Role>`

Definition:: The roles (lab equipment sales, contractor, etc.) the Contact fills.

Type:: `RoleType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: CV term for contact roles, such as software provider.

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Person>`

Definition:: A person's name and contact details. Any additional information such as the address, contact email etc. should be supplied using CV parameters or user parameters.

Type:: `PersonType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<Organization>`

Definition:: Organizations are entities like companies, universities, government agencies. Any additional information such as the address, email etc. should be supplied either as CV parameters or as user parameters.

Type:: `OrganizationType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<ContactRole>`

Definition:: The Contact that provided the document instance.

Type:: `ContactRoleType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<cvParam>`

Definition:: A single entry from an ontology or a controlled
vocabulary.

Type:: `CVParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

=== Element `<userParam>`

Definition:: A single user-defined parameter.

Type:: `UserParamType`

[cols=1,2, options=header]
|===
| Attribute | Description

| `id` | Unique identifier for the element.
| `name` | Optional human-readable name.
|===

